{
 "cells": [
  {
   "cell_type": "code",
   "execution_count": 35,
   "metadata": {},
   "outputs": [],
   "source": [
    "import polars as pl\n",
    "import os\n",
    "import cv2\n",
    "from figure import detect_subfigure_labels, dummy_detect_label_letter, classify_subfigures, save_subfigures, show_image_with_bboxes"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 30,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "9\n"
     ]
    }
   ],
   "source": [
    "import re\n",
    "\n",
    "def count_subfigures(caption):\n",
    "    patterns = [\n",
    "        r\"(?:^|\\.\\s)([a-z](?=,))\", # a, \n",
    "        r\"(?:^|\\.\\s|,\\s)([a-z])\\.\", # a.\n",
    "        r\"(?:^|\\.\\s|,\\s)\\(([a-z])\\)\", # (a)\n",
    "        r\"\\(([a-z])-([a-z])\\)\", # (a-b)\n",
    "        r\"\\(([a-z])\\sand\\s([a-z])\\)\", # (a and b)\n",
    "        r\"\\,([a-z])\\,\", # ,a,\n",
    "        r\"(?:^|\\.\\s)[a-z]–([a-z])\" # a–b -- captures the second one\n",
    "    ]\n",
    "    all_patterns = r\"|\".join(patterns)\n",
    "    matches = re.finditer(all_patterns, caption, re.IGNORECASE)\n",
    "    letters = []\n",
    "    for match in matches:\n",
    "        groups = [group.lower() for group in match.groups() if group is not None]\n",
    "        letters.append(max(groups))\n",
    "\n",
    "    if len(letters) == 0:\n",
    "        return 1\n",
    "    return ord(max(letters)) - ord('a') + 1\n",
    "\n",
    "sample_caption = \"a, UMAP shows CTB and STB nuclei profiled with integration of snRNA-seq and snATAC-seq in early pregnancy. b, Heatmap shows pseudotime ordering of the 23,746 DARs in early pregnancy (left). Zoomed-in genomic tracks show the cis-element accessibility of representative genes. The normalized accessibility of DARs is presented with color intensities. c, TF-mining heatmap shows candidate master TFs of each nucleus type. The NES and expression z score are presented with dot color and dot size, respectively. Dots with bold edges show selected TFs used for network construction. d, Regulatory networks with three layers (TF, cis-element and target gene) represent the cis-regulatory architecture covering the complete STB differentiation process. The gene percentage, peak accessibility and gene expression score are presented with circle sizes, edge width and color intensities, respectively. e, smFISH staining (left) and fluorescence intensities (right) show the colocalization of TFs (STAT5A and FOSL2) and representative genes (PAPPA and FLT1) of eSTB mature 1 and eSTB mature 2. hCG, STB marker. The white dotted lines show the outline of CTB. The white lines represent the plotted tracks of fluorescence intensity. f, UMAP shows CTB and STB nuclei profiled with integration of snRNA-seq of STB-CT30, STB-BL and placenta villi in early pregnancy. g, Heatmaps show marker gene expression in STB-BL, STB-CT30 and placental villi in early pregnancy. The expression levels are presented with color intensities. Binarized gene expression levels (positive or negative) are calculated and visualized beside heatmaps for the two marker genes, PAPPA and FLT1. h, Pseudotime ordering shows three differential trajectories of CTB and STB nuclei (upper left). The differentiation time is presented with color intensities. Quantitative classifications for each cluster (upper right) and specific pseudotime of cluster 9 and cluster 7 on the integrated UMAP (bottom) are shown. i, Heatmap shows the different expression patterns of marker genes, master TFs and representative hormones among in vitro trophoblast models and in vivo placental villi in early pregnancy. The z scores are presented with color intensities. Examples of genes with similar expression patterns are highlighted in bold black, and those with different patterns are colored in red.\"\n",
    "print(count_subfigures(sample_caption))"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 3,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "\n",
      "image 1/1 /Users/etowah/projects/forks/exsclaim/imgs/10335933_Fig5.jpg: 640x544 13 labels, 146.3ms\n",
      "Speed: 5.6ms preprocess, 146.3ms inference, 7.6ms postprocess per image at shape (1, 3, 640, 544)\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "/Users/etowah/projects/forks/exsclaim/figures/models/yolo_layer.py:374: UserWarning: The given NumPy array is not writable, and PyTorch does not support non-writable tensors. This means writing to this tensor will result in undefined behavior. You may want to copy the array to protect its data or make it writable before converting it to a tensor. This type of warning will be suppressed for the rest of this program. (Triggered internally at /Users/runner/work/pytorch/pytorch/pytorch/torch/csrc/utils/tensor_numpy.cpp:212.)\n",
      "  x_shift = dtype(np.broadcast_to(\n"
     ]
    }
   ],
   "source": [
    "image_folder = \"imgs\"\n",
    "image_files = [os.path.join(image_folder, file) for file in os.listdir(image_folder) if file.endswith((\".png\", \".jpg\", \".gif\"))]\n",
    "\n",
    "image_files = [\"imgs/10335933_Fig5.jpg\"]\n",
    "for image_file in image_files:\n",
    "    bounding_boxes = detect_subfigure_labels(image_file)\n",
    "    # show_image_with_bboxes(image_file, bounding_boxes)\n",
    "    subfigure_info, concate_img = dummy_detect_label_letter(image_file, bounding_boxes)\n",
    "    # show_labels_on_image(image_file, subfigure_info, save=False, show=True)\n",
    "    figure_json = classify_subfigures(image_file, subfigure_info, concate_img)\n",
    "    save_subfigures(image_file, figure_json, out_dir=\"test\")"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 42,
   "metadata": {},
   "outputs": [],
   "source": [
    "def non_max_suppression(boxes, scores, threshold):\n",
    "    \"\"\"\n",
    "    Perform non-max suppression on a set of bounding boxes and corresponding scores.\n",
    "\n",
    "    :param boxes: a list of bounding boxes in the format [xmin, ymin, xmax, ymax]\n",
    "    :param scores: a list of corresponding scores\n",
    "    :param threshold: the IoU (intersection-over-union) threshold for merging bounding boxes\n",
    "    :return: a list of indices of the boxes to keep after non-max suppression\n",
    "    \"\"\"\n",
    "    # Sort the boxes by score in descending order\n",
    "    order = sorted(range(len(scores)), key=lambda i: scores[i], reverse=True)\n",
    "    keep = []\n",
    "    while order:\n",
    "        i = order.pop(0)\n",
    "        keep.append(i)\n",
    "        for j in order:\n",
    "            # Calculate the IoU between the two boxes\n",
    "            intersection = max(0, min(boxes[i][2], boxes[j][2]) - max(boxes[i][0], boxes[j][0])) * \\\n",
    "                           max(0, min(boxes[i][3], boxes[j][3]) - max(boxes[i][1], boxes[j][1]))\n",
    "            union = (boxes[i][2] - boxes[i][0]) * (boxes[i][3] - boxes[i][1]) + \\\n",
    "                    (boxes[j][2] - boxes[j][0]) * (boxes[j][3] - boxes[j][1]) - intersection\n",
    "            iou = intersection / union\n",
    "\n",
    "            # Remove boxes with IoU greater than the threshold\n",
    "            if iou > threshold:\n",
    "                order.remove(j)\n",
    "    return keep"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 46,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "4\n",
      "\n",
      "image 1/1 /Users/etowah/projects/PMC-figure-downloader/img/10937393_Fig1.jpg: 640x544 5 labels, 77.1ms\n",
      "Speed: 2.2ms preprocess, 77.1ms inference, 0.7ms postprocess per image at shape (1, 3, 640, 544)\n",
      "5\n",
      "\n",
      "image 1/1 /Users/etowah/projects/PMC-figure-downloader/img/10937393_Fig2.jpg: 640x608 5 labels, 81.2ms\n",
      "Speed: 2.3ms preprocess, 81.2ms inference, 0.6ms postprocess per image at shape (1, 3, 640, 608)\n",
      "6\n",
      "\n",
      "image 1/1 /Users/etowah/projects/PMC-figure-downloader/img/10937393_Fig3.jpg: 576x640 7 labels, 82.5ms\n",
      "Speed: 2.3ms preprocess, 82.5ms inference, 0.7ms postprocess per image at shape (1, 3, 576, 640)\n",
      "Subfigure count mismatch: 7 detected, 6 expected\n",
      "3\n",
      "\n",
      "image 1/1 /Users/etowah/projects/PMC-figure-downloader/img/10937393_Fig4.jpg: 640x640 4 labels, 104.3ms\n",
      "Speed: 2.3ms preprocess, 104.3ms inference, 1.4ms postprocess per image at shape (1, 3, 640, 640)\n",
      "4\n",
      "\n",
      "image 1/1 /Users/etowah/projects/PMC-figure-downloader/img/10937393_Fig5.jpg: 640x544 5 labels, 78.5ms\n",
      "Speed: 2.0ms preprocess, 78.5ms inference, 0.5ms postprocess per image at shape (1, 3, 640, 544)\n",
      "4\n",
      "\n",
      "image 1/1 /Users/etowah/projects/PMC-figure-downloader/img/10937393_Fig6.jpg: 640x320 4 labels, 62.6ms\n",
      "Speed: 1.4ms preprocess, 62.6ms inference, 0.5ms postprocess per image at shape (1, 3, 640, 320)\n",
      "6\n",
      "\n",
      "image 1/1 /Users/etowah/projects/PMC-figure-downloader/img/10937393_Fig7.jpg: 640x640 6 labels, 106.8ms\n",
      "Speed: 2.6ms preprocess, 106.8ms inference, 1.5ms postprocess per image at shape (1, 3, 640, 640)\n",
      "1\n",
      "Relative expression in each cell type was calculated based on DESeq2-normalized counts from 6-8 control donors, and was log2-transformed for visualization (see Methods). The marker genes specific for each interneuron subtype were selected based on single-nucleus RNA sequencing (snRNA-seq) data19. The FANS-isolated ETV1 + TAC3- population of Parvalbumin-expressing interneuron nuclei most likely captured both the major PVALB+ interneuron population and the related smaller PVALB + TH+ interneuron population defined as a separate subtype in snRNA-seq analysis19.\n",
      "3\n",
      "\n",
      "image 1/1 /Users/etowah/projects/PMC-figure-downloader/img/10937393_Fig9.jpg: 640x640 3 labels, 92.4ms\n",
      "Speed: 2.3ms preprocess, 92.4ms inference, 1.2ms postprocess per image at shape (1, 3, 640, 640)\n",
      "4\n",
      "\n",
      "image 1/1 /Users/etowah/projects/PMC-figure-downloader/img/10937393_Fig10.jpg: 640x512 4 labels, 102.2ms\n",
      "Speed: 2.2ms preprocess, 102.2ms inference, 0.4ms postprocess per image at shape (1, 3, 640, 512)\n",
      "1\n",
      "a, Heatmaps depict log2-transformed relative expression in each cell type, calculated based on the mean of DESeq2-normalized counts from 7-8 control donors. Genes identified as HD age at onset-modifying candidates are marked with an asterisk. Genes known to influence CAG tract instability in HD mouse models are marked with an arrowhead. b-d, Relative levels of MSH2 (b), MSH3 (c) and FAN1 (d) transcripts in MSNs compared to interneurons, calculated with DESeq2 using cell type-specific FANS-seq data from control donors. The largest P adj. value of all MSN vs. interneuron comparisons (by DESeq2, P adjusted for multiple comparisons) is indicated for each gene. For caudate nucleus samples: n = 8 individuals for dMSNs, iMSNs, TAC3 + IN, PVALB + IN and SST + IN. For putamen samples: n = 8 individuals for dMSNs, iMSNs, TAC3 + IN, n = 7 individuals for PVALB + IN and n = 6 individuals for SST + IN.\n",
      "9\n",
      "\n",
      "image 1/1 /Users/etowah/projects/PMC-figure-downloader/img/10937393_Fig12.jpg: 640x512 9 labels, 75.6ms\n",
      "Speed: 2.0ms preprocess, 75.6ms inference, 0.9ms postprocess per image at shape (1, 3, 640, 512)\n",
      "Subfigure count mismatch: 8 detected, 9 expected\n",
      "4\n",
      "\n",
      "image 1/1 /Users/etowah/projects/PMC-figure-downloader/img/10937393_Fig13.jpg: 544x640 4 labels, 81.7ms\n",
      "Speed: 1.9ms preprocess, 81.7ms inference, 0.8ms postprocess per image at shape (1, 3, 544, 640)\n",
      "7\n",
      "\n",
      "image 1/1 /Users/etowah/projects/PMC-figure-downloader/img/10937382_Fig1.jpg: 640x512 8 labels, 72.4ms\n",
      "Speed: 1.9ms preprocess, 72.4ms inference, 0.5ms postprocess per image at shape (1, 3, 640, 512)\n",
      "9\n",
      "\n",
      "image 1/1 /Users/etowah/projects/PMC-figure-downloader/img/10937382_Fig2.jpg: 640x512 11 labels, 85.0ms\n",
      "Speed: 2.0ms preprocess, 85.0ms inference, 0.5ms postprocess per image at shape (1, 3, 640, 512)\n",
      "5\n",
      "\n",
      "image 1/1 /Users/etowah/projects/PMC-figure-downloader/img/10937382_Fig3.jpg: 640x608 5 labels, 65.0ms\n",
      "Speed: 1.9ms preprocess, 65.0ms inference, 0.5ms postprocess per image at shape (1, 3, 640, 608)\n",
      "4\n",
      "\n",
      "image 1/1 /Users/etowah/projects/PMC-figure-downloader/img/10937382_Fig4.jpg: 640x416 4 labels, 72.7ms\n",
      "Speed: 2.0ms preprocess, 72.7ms inference, 1.1ms postprocess per image at shape (1, 3, 640, 416)\n",
      "4\n",
      "\n",
      "image 1/1 /Users/etowah/projects/PMC-figure-downloader/img/10937382_Fig5.jpg: 576x640 4 labels, 86.7ms\n",
      "Speed: 2.4ms preprocess, 86.7ms inference, 0.5ms postprocess per image at shape (1, 3, 576, 640)\n",
      "2\n",
      "\n",
      "image 1/1 /Users/etowah/projects/PMC-figure-downloader/img/10937382_Fig6.jpg: 352x640 2 labels, 84.1ms\n",
      "Speed: 1.4ms preprocess, 84.1ms inference, 0.8ms postprocess per image at shape (1, 3, 352, 640)\n",
      "1\n",
      "Oncoprint from cBioPortal analysis of human muscle-invasive bladder cancers from The Cancer Genome Atlas bladder cancer (TCGA-BLCA) cohort showing select genes for which gain- or loss-of-function events were incorporated into the bladder urothelial LV pool.\n",
      "1\n",
      "(a–e) Low- and high-magnification images of H&E-stained sections and high-magnification images of IHC-stained sections for GFP and pan-cytokeratin (panCK) expression depicting characteristic histologies. Scale bars = 50 µm.\n",
      "2\n",
      "\n",
      "image 1/1 /Users/etowah/projects/PMC-figure-downloader/img/10937382_Fig9.jpg: 640x384 2 labels, 76.5ms\n",
      "Speed: 1.8ms preprocess, 76.5ms inference, 0.8ms postprocess per image at shape (1, 3, 640, 384)\n",
      "4\n",
      "\n",
      "image 1/1 /Users/etowah/projects/PMC-figure-downloader/img/10937382_Fig10.jpg: 640x480 5 labels, 68.9ms\n",
      "Speed: 1.9ms preprocess, 68.9ms inference, 0.5ms postprocess per image at shape (1, 3, 640, 480)\n",
      "Subfigure count mismatch: 5 detected, 4 expected\n",
      "1\n",
      "MuSiCal decomposes a mutation count matrix X into a signature matrix W and an exposure matrix H through four main modules: preprocessing, de novo discovery, matching and refitting, and in silico validation/optimization. The new methods implemented in each module are highlighted. Solid purple arrows represent a typical workflow of mutational signature analysis. Dashed orange arrows denote that some steps in the workflow potentially require repeating an earlier step (‘Results’).\n",
      "3\n",
      "\n",
      "image 1/1 /Users/etowah/projects/PMC-figure-downloader/img/10937379_Fig2.jpg: 640x512 3 labels, 86.9ms\n",
      "Speed: 2.2ms preprocess, 86.9ms inference, 0.5ms postprocess per image at shape (1, 3, 640, 512)\n",
      "6\n",
      "\n",
      "image 1/1 /Users/etowah/projects/PMC-figure-downloader/img/10937379_Fig3.jpg: 576x640 6 labels, 81.4ms\n",
      "Speed: 2.3ms preprocess, 81.4ms inference, 0.8ms postprocess per image at shape (1, 3, 576, 640)\n",
      "3\n",
      "\n",
      "image 1/1 /Users/etowah/projects/PMC-figure-downloader/img/10937379_Fig4.jpg: 544x640 3 labels, 72.7ms\n",
      "Speed: 2.2ms preprocess, 72.7ms inference, 0.5ms postprocess per image at shape (1, 3, 544, 640)\n",
      "6\n",
      "\n",
      "image 1/1 /Users/etowah/projects/PMC-figure-downloader/img/10937379_Fig5.jpg: 320x640 8 labels, 56.5ms\n",
      "Speed: 1.2ms preprocess, 56.5ms inference, 0.5ms postprocess per image at shape (1, 3, 320, 640)\n",
      "3\n",
      "\n",
      "image 1/1 /Users/etowah/projects/PMC-figure-downloader/img/10937379_Fig6.jpg: 640x512 3 labels, 73.9ms\n",
      "Speed: 1.9ms preprocess, 73.9ms inference, 0.4ms postprocess per image at shape (1, 3, 640, 512)\n",
      "4\n",
      "\n",
      "image 1/1 /Users/etowah/projects/PMC-figure-downloader/img/10937379_Fig7.jpg: 384x640 4 labels, 61.9ms\n",
      "Speed: 1.5ms preprocess, 61.9ms inference, 0.4ms postprocess per image at shape (1, 3, 384, 640)\n",
      "4\n",
      "\n",
      "image 1/1 /Users/etowah/projects/PMC-figure-downloader/img/10937379_Fig8.jpg: 640x640 4 labels, 109.8ms\n",
      "Speed: 2.6ms preprocess, 109.8ms inference, 0.5ms postprocess per image at shape (1, 3, 640, 640)\n",
      "2\n",
      "\n",
      "image 1/1 /Users/etowah/projects/PMC-figure-downloader/img/10937379_Fig9.jpg: 640x576 6 labels, 89.8ms\n",
      "Speed: 2.2ms preprocess, 89.8ms inference, 3.2ms postprocess per image at shape (1, 3, 640, 576)\n",
      "Subfigure count mismatch: 6 detected, 2 expected\n",
      "4\n",
      "\n",
      "image 1/1 /Users/etowah/projects/PMC-figure-downloader/img/10937379_Fig10.jpg: 352x640 4 labels, 60.4ms\n",
      "Speed: 1.3ms preprocess, 60.4ms inference, 0.4ms postprocess per image at shape (1, 3, 352, 640)\n",
      "4\n",
      "\n",
      "image 1/1 /Users/etowah/projects/PMC-figure-downloader/img/10937379_Fig11.jpg: 352x640 4 labels, 54.1ms\n",
      "Speed: 1.4ms preprocess, 54.1ms inference, 0.5ms postprocess per image at shape (1, 3, 352, 640)\n",
      "4\n",
      "\n",
      "image 1/1 /Users/etowah/projects/PMC-figure-downloader/img/10937379_Fig12.jpg: 576x640 4 labels, 83.5ms\n",
      "Speed: 2.2ms preprocess, 83.5ms inference, 0.6ms postprocess per image at shape (1, 3, 576, 640)\n",
      "1\n",
      "a-c. An example with the PCAWG ColoRect.AdenoCA dataset to demonstrate that cohort stratification can benefit de novo signature discovery. After an initial signature discovery with the entire dataset of 60 samples, hierarchical clustering is performed using the de novo exposure matrix H, and 3 clusters are selected by the gap statistic, as shown in panel (a). The 3 clusters correspond to samples with MMRP, MMRD, and POLE mutations, respectively, as shown in panel (b). In panel (c), de novo signatures discovered from the entire dataset of 60 samples and the MMRP cluster (43 samples) are plotted, demonstrating improved sensitivity of signature discovery after cohort stratification. Specifically, 12 signatures are discovered when de novo discovery is performed with the entire dataset of 60 samples, while 11 are discovered with the MMRP cluster alone. Out of the 11 MMRP-specific signatures, only 7 are discovered before cohort stratification. Therefore, a separate run of de novo discovery with the MMRP cluster allows 4 more MMRP-specific signatures to be discovered. In panel (a), reasonable k means any k satisfying Gap(k)≥Gap(k + 1) − sk+1, where Gap(k) denotes the gap statistic (indicated by dots in the plot) and sk the standard deviation of 50 independent simulations after accounting for simulation errors (indicated by error bars in the plot). The smallest reasonable k is chosen as the optimal k. See60,61 for more details. d-f. An example with the PCAWG CNS.GBM dataset to demonstrate that outlier removal can benefit de novo signature discovery. When signature discovery is performed with the entire dataset, 4 de novo signatures are discovered, as shown in panel (d), where Signature A corresponds to SBS11 associated with temozolomide treatment. When the de novo exposures are inspected with the Gini coefficient, a single outlier sample with an exceptionally strong exposure of Signature A is detected, as shown in panel (e). After removing this outlier and rerunning signature discovery with the remaining samples, an additional signature is discovered, as shown in panel (f), demonstrating improved sensitivity.\n",
      "6\n",
      "\n",
      "image 1/1 /Users/etowah/projects/PMC-figure-downloader/img/10937379_Fig14.jpg: 640x608 6 labels, 85.5ms\n",
      "Speed: 2.5ms preprocess, 85.5ms inference, 0.6ms postprocess per image at shape (1, 3, 640, 608)\n",
      "2\n",
      "\n",
      "image 1/1 /Users/etowah/projects/PMC-figure-downloader/img/10937379_Fig15.jpg: 608x640 2 labels, 98.1ms\n",
      "Speed: 2.4ms preprocess, 98.1ms inference, 0.4ms postprocess per image at shape (1, 3, 608, 640)\n",
      "4\n",
      "\n",
      "image 1/1 /Users/etowah/projects/PMC-figure-downloader/img/10937379_Fig16.jpg: 640x640 4 labels, 89.7ms\n",
      "Speed: 2.3ms preprocess, 89.7ms inference, 0.4ms postprocess per image at shape (1, 3, 640, 640)\n",
      "1\n",
      "Exposures of SBS40 and ID5 are correlated for both MuSiCal- (top) and PCAWG-derived (middle) signature assignments in kidney and bladder cancers, which are the only tumor types where SBS40 is assigned by MuSiCal. PCAWG also assigns SBS40 in many other tumor types (bottom), where a strong correlation is not observed. Pearson correlation coefficients are shown for both absolute (left) and relative (right) exposures (normalized by mutation burden). Error bands (shaded areas) indicate 95% confidence intervals of the linear regressions.\n",
      "10\n",
      "\n",
      "image 1/1 /Users/etowah/projects/PMC-figure-downloader/img/10937375_Fig1.jpg: 640x576 11 labels, 77.6ms\n",
      "Speed: 2.3ms preprocess, 77.6ms inference, 0.4ms postprocess per image at shape (1, 3, 640, 576)\n",
      "10\n",
      "\n",
      "image 1/1 /Users/etowah/projects/PMC-figure-downloader/img/10937375_Fig2.jpg: 640x512 10 labels, 69.8ms\n",
      "Speed: 2.1ms preprocess, 69.8ms inference, 0.5ms postprocess per image at shape (1, 3, 640, 512)\n",
      "11\n",
      "\n",
      "image 1/1 /Users/etowah/projects/PMC-figure-downloader/img/10937375_Fig3.jpg: 640x480 12 labels, 77.0ms\n",
      "Speed: 2.1ms preprocess, 77.0ms inference, 0.4ms postprocess per image at shape (1, 3, 640, 480)\n",
      "Subfigure count mismatch: 12 detected, 11 expected\n",
      "7\n",
      "\n",
      "image 1/1 /Users/etowah/projects/PMC-figure-downloader/img/10937375_Fig4.jpg: 640x512 7 labels, 67.3ms\n",
      "Speed: 2.1ms preprocess, 67.3ms inference, 0.6ms postprocess per image at shape (1, 3, 640, 512)\n",
      "14\n",
      "\n",
      "image 1/1 /Users/etowah/projects/PMC-figure-downloader/img/10937375_Fig5.jpg: 640x608 15 labels, 82.4ms\n",
      "Speed: 2.3ms preprocess, 82.4ms inference, 1.3ms postprocess per image at shape (1, 3, 640, 608)\n",
      "11\n",
      "\n",
      "image 1/1 /Users/etowah/projects/PMC-figure-downloader/img/10937375_Fig6.jpg: 640x544 11 labels, 78.4ms\n",
      "Speed: 3.2ms preprocess, 78.4ms inference, 0.4ms postprocess per image at shape (1, 3, 640, 544)\n",
      "9\n",
      "\n",
      "image 1/1 /Users/etowah/projects/PMC-figure-downloader/img/10937375_Fig7.jpg: 608x640 9 labels, 114.7ms\n",
      "Speed: 2.4ms preprocess, 114.7ms inference, 0.4ms postprocess per image at shape (1, 3, 608, 640)\n",
      "10\n",
      "\n",
      "image 1/1 /Users/etowah/projects/PMC-figure-downloader/img/10937375_Fig8.jpg: 640x512 10 labels, 90.0ms\n",
      "Speed: 2.1ms preprocess, 90.0ms inference, 0.5ms postprocess per image at shape (1, 3, 640, 512)\n",
      "9\n",
      "\n",
      "image 1/1 /Users/etowah/projects/PMC-figure-downloader/img/10937375_Fig9.jpg: 512x640 9 labels, 71.2ms\n",
      "Speed: 2.0ms preprocess, 71.2ms inference, 0.4ms postprocess per image at shape (1, 3, 512, 640)\n",
      "8\n",
      "\n",
      "image 1/1 /Users/etowah/projects/PMC-figure-downloader/img/10937375_Fig10.jpg: 640x512 7 labels, 78.6ms\n",
      "Speed: 1.9ms preprocess, 78.6ms inference, 0.4ms postprocess per image at shape (1, 3, 640, 512)\n",
      "Subfigure count mismatch: 7 detected, 8 expected\n",
      "3\n",
      "\n",
      "image 1/1 /Users/etowah/projects/PMC-figure-downloader/img/10937375_Fig11.jpg: 608x640 3 labels, 95.3ms\n",
      "Speed: 2.3ms preprocess, 95.3ms inference, 0.4ms postprocess per image at shape (1, 3, 608, 640)\n",
      "3\n",
      "\n",
      "image 1/1 /Users/etowah/projects/PMC-figure-downloader/img/10937375_Fig12.jpg: 416x640 4 labels, 74.7ms\n",
      "Speed: 1.6ms preprocess, 74.7ms inference, 1.8ms postprocess per image at shape (1, 3, 416, 640)\n",
      "Subfigure count mismatch: 4 detected, 3 expected\n",
      "12\n",
      "\n",
      "image 1/1 /Users/etowah/projects/PMC-figure-downloader/img/10937375_Fig13.jpg: 640x576 12 labels, 73.4ms\n",
      "Speed: 2.4ms preprocess, 73.4ms inference, 0.6ms postprocess per image at shape (1, 3, 640, 576)\n",
      "8\n",
      "\n",
      "image 1/1 /Users/etowah/projects/PMC-figure-downloader/img/10937375_Fig14.jpg: 640x512 9 labels, 87.0ms\n",
      "Speed: 2.0ms preprocess, 87.0ms inference, 1.4ms postprocess per image at shape (1, 3, 640, 512)\n",
      "Subfigure count mismatch: 9 detected, 8 expected\n",
      "6\n",
      "\n",
      "image 1/1 /Users/etowah/projects/PMC-figure-downloader/img/10937375_Fig15.jpg: 608x640 7 labels, 92.1ms\n",
      "Speed: 2.5ms preprocess, 92.1ms inference, 0.4ms postprocess per image at shape (1, 3, 608, 640)\n",
      "5\n",
      "\n",
      "image 1/1 /Users/etowah/projects/PMC-figure-downloader/img/10937375_Fig16.jpg: 640x480 5 labels, 70.0ms\n",
      "Speed: 1.8ms preprocess, 70.0ms inference, 0.4ms postprocess per image at shape (1, 3, 640, 480)\n",
      "6\n",
      "\n",
      "image 1/1 /Users/etowah/projects/PMC-figure-downloader/img/10864188_Fig1.jpg: 640x448 7 labels, 73.4ms\n",
      "Speed: 1.8ms preprocess, 73.4ms inference, 0.4ms postprocess per image at shape (1, 3, 640, 448)\n",
      "3\n",
      "\n",
      "image 1/1 /Users/etowah/projects/PMC-figure-downloader/img/10864188_Fig2.jpg: 512x640 3 labels, 89.9ms\n",
      "Speed: 2.0ms preprocess, 89.9ms inference, 1.6ms postprocess per image at shape (1, 3, 512, 640)\n",
      "1\n",
      "The colors of the ribbon plot represent molecular categories of samples in the pAML cohort (n = 887).\n",
      "4\n",
      "\n",
      "image 1/1 /Users/etowah/projects/PMC-figure-downloader/img/10864188_Fig4.jpg: 640x512 5 labels, 79.8ms\n",
      "Speed: 2.1ms preprocess, 79.8ms inference, 0.5ms postprocess per image at shape (1, 3, 640, 512)\n",
      "6\n",
      "\n",
      "image 1/1 /Users/etowah/projects/PMC-figure-downloader/img/10864188_Fig5.jpg: 640x544 6 labels, 82.3ms\n",
      "Speed: 2.2ms preprocess, 82.3ms inference, 0.4ms postprocess per image at shape (1, 3, 640, 544)\n",
      "5\n",
      "\n",
      "image 1/1 /Users/etowah/projects/PMC-figure-downloader/img/10864188_Fig6.jpg: 640x576 5 labels, 79.6ms\n",
      "Speed: 2.8ms preprocess, 79.6ms inference, 0.4ms postprocess per image at shape (1, 3, 640, 576)\n",
      "6\n",
      "\n",
      "image 1/1 /Users/etowah/projects/PMC-figure-downloader/img/10864188_Fig7.jpg: 512x640 6 labels, 72.4ms\n",
      "Speed: 2.0ms preprocess, 72.4ms inference, 0.5ms postprocess per image at shape (1, 3, 512, 640)\n",
      "7\n",
      "\n",
      "image 1/1 /Users/etowah/projects/PMC-figure-downloader/img/10864188_Fig8.jpg: 640x480 7 labels, 72.9ms\n",
      "Speed: 2.0ms preprocess, 72.9ms inference, 1.0ms postprocess per image at shape (1, 3, 640, 480)\n",
      "1\n",
      "Pair-wise correlation among most frequent 97 genetic alterations (n ≥ 5 in the entire cohort) from GRIN analysis and chromosomal changes (complex karyotype and monosomy 7) (a) and category-defining gene alterations (b). KMT2A-PTD (partial tandem duplication) is independently included from other KMT2A alterations, and FLT3 alterations are classified into ITD (internal tandem duplication), TKD (tyrosine kinase domain) mutations, and non-TKD mutations due to the known functional difference. Colors correspond to Pearson correlation. Statistical significance was assessed by two-sided Fisher’s exact test to calculate P values followed by the Benjamini-Hochberg adjustment for multiple testing to calculate q values (*P < 0.05, **q < 0.05).\n",
      "5\n",
      "\n",
      "image 1/1 /Users/etowah/projects/PMC-figure-downloader/img/10864188_Fig10.jpg: 640x512 5 labels, 72.0ms\n",
      "Speed: 2.0ms preprocess, 72.0ms inference, 0.6ms postprocess per image at shape (1, 3, 640, 512)\n",
      "2\n",
      "\n",
      "image 1/1 /Users/etowah/projects/PMC-figure-downloader/img/10864188_Fig11.jpg: 640x512 2 labels, 79.7ms\n",
      "Speed: 2.1ms preprocess, 79.7ms inference, 0.6ms postprocess per image at shape (1, 3, 640, 512)\n",
      "4\n",
      "\n",
      "image 1/1 /Users/etowah/projects/PMC-figure-downloader/img/10864188_Fig12.jpg: 640x512 5 labels, 104.1ms\n",
      "Speed: 1.9ms preprocess, 104.1ms inference, 0.6ms postprocess per image at shape (1, 3, 640, 512)\n",
      "Subfigure count mismatch: 5 detected, 4 expected\n",
      "2\n",
      "\n",
      "image 1/1 /Users/etowah/projects/PMC-figure-downloader/img/10864188_Fig13.jpg: 640x480 1 label, 78.4ms\n",
      "Speed: 2.0ms preprocess, 78.4ms inference, 0.5ms postprocess per image at shape (1, 3, 640, 480)\n",
      "Subfigure count mismatch: 1 detected, 2 expected\n",
      "8\n",
      "\n",
      "image 1/1 /Users/etowah/projects/PMC-figure-downloader/img/10864188_Fig14.jpg: 640x544 6 labels, 82.0ms\n",
      "Speed: 2.1ms preprocess, 82.0ms inference, 0.4ms postprocess per image at shape (1, 3, 640, 544)\n",
      "Subfigure count mismatch: 6 detected, 8 expected\n",
      "4\n",
      "\n",
      "image 1/1 /Users/etowah/projects/PMC-figure-downloader/img/10864188_Fig15.jpg: 640x640 4 labels, 107.1ms\n",
      "Speed: 2.4ms preprocess, 107.1ms inference, 0.4ms postprocess per image at shape (1, 3, 640, 640)\n",
      "5\n",
      "\n",
      "image 1/1 /Users/etowah/projects/PMC-figure-downloader/img/10864188_Fig16.jpg: 640x448 5 labels, 64.5ms\n",
      "Speed: 1.9ms preprocess, 64.5ms inference, 0.4ms postprocess per image at shape (1, 3, 640, 448)\n",
      "4\n",
      "\n",
      "image 1/1 /Users/etowah/projects/PMC-figure-downloader/img/10864188_Fig17.jpg: 512x640 4 labels, 66.3ms\n",
      "Speed: 1.8ms preprocess, 66.3ms inference, 0.5ms postprocess per image at shape (1, 3, 512, 640)\n",
      "15\n",
      "\n",
      "image 1/1 /Users/etowah/projects/PMC-figure-downloader/img/10864183_Fig1.jpg: 640x576 15 labels, 70.5ms\n",
      "Speed: 2.4ms preprocess, 70.5ms inference, 0.5ms postprocess per image at shape (1, 3, 640, 576)\n",
      "8\n",
      "\n",
      "image 1/1 /Users/etowah/projects/PMC-figure-downloader/img/10864183_Fig2.jpg: 640x544 8 labels, 101.5ms\n",
      "Speed: 2.1ms preprocess, 101.5ms inference, 0.4ms postprocess per image at shape (1, 3, 640, 544)\n",
      "14\n",
      "\n",
      "image 1/1 /Users/etowah/projects/PMC-figure-downloader/img/10864183_Fig3.jpg: 544x640 14 labels, 79.0ms\n",
      "Speed: 1.9ms preprocess, 79.0ms inference, 0.5ms postprocess per image at shape (1, 3, 544, 640)\n",
      "10\n",
      "\n",
      "image 1/1 /Users/etowah/projects/PMC-figure-downloader/img/10864183_Fig4.jpg: 640x640 10 labels, 103.8ms\n",
      "Speed: 2.5ms preprocess, 103.8ms inference, 0.4ms postprocess per image at shape (1, 3, 640, 640)\n",
      "9\n",
      "\n",
      "image 1/1 /Users/etowah/projects/PMC-figure-downloader/img/10864183_Fig5.jpg: 448x640 9 labels, 71.2ms\n",
      "Speed: 1.7ms preprocess, 71.2ms inference, 0.7ms postprocess per image at shape (1, 3, 448, 640)\n",
      "7\n",
      "\n",
      "image 1/1 /Users/etowah/projects/PMC-figure-downloader/img/10864183_Fig6.jpg: 480x640 7 labels, 61.7ms\n",
      "Speed: 1.9ms preprocess, 61.7ms inference, 0.4ms postprocess per image at shape (1, 3, 480, 640)\n",
      "1\n",
      "(a) Previous mapping efforts located qRgls1 to a 1.4-Mb region on the short arm of maize chromosome 8. (b–d) Sequential fine-mapping of qRgls1 (n = 3290). The schematic diagrams to the right represent the genomic architecture of each recombinant type. The markers used in the fine-mapping are indicated, with red markers showing the limits of the qRgls1 region as defined at each fine-mapping step. The tables to the right summarize the DSI from homozygous and heterozygous backcross progeny of recombinants. Significant difference (P < 0.05) in DSI between heterozygous and homozygous offspring indicates the presence of the resistance allele at qRgls1 in the heterozygous region of their parental recombinant; otherwise, there is no qRgls1 (P > 0.05). Data are shown as means ± s.d. in (b)–(d) and the P value was determined based on two-sided Student’s t-test.\n",
      "4\n",
      "\n",
      "image 1/1 /Users/etowah/projects/PMC-figure-downloader/img/10864183_Fig8.jpg: 352x640 6 labels, 64.5ms\n",
      "Speed: 1.6ms preprocess, 64.5ms inference, 0.4ms postprocess per image at shape (1, 3, 352, 640)\n",
      "Subfigure count mismatch: 6 detected, 4 expected\n",
      "1\n",
      "Schematic diagrams of ZmWAKL (a) and ZmPR5L (b) loci, as determined by aligning the full-size complementary DNA (cDNA) with the corresponding genomic sequence. Rectangles, exons; lines, introns. The colors indicate different domains: yellow, SP (signal peptide); light orange, GUB (galacturonan-binding domain of wall-associated receptor kinase); pink, TM (transmembrane domain); blue, Ser/Thr kinase. Gray shading between the Y32 and Q11 alleles represents the syntenic region; red lines denote SNPs; black spaces or lines represent InDels.\n",
      "8\n",
      "\n",
      "image 1/1 /Users/etowah/projects/PMC-figure-downloader/img/10864183_Fig10.jpg: 480x640 7 labels, 69.9ms\n",
      "Speed: 2.3ms preprocess, 69.9ms inference, 0.4ms postprocess per image at shape (1, 3, 480, 640)\n",
      "Subfigure count mismatch: 7 detected, 8 expected\n",
      "14\n",
      "\n",
      "image 1/1 /Users/etowah/projects/PMC-figure-downloader/img/10864183_Fig11.jpg: 480x640 6 labels, 70.1ms\n",
      "Speed: 1.8ms preprocess, 70.1ms inference, 0.4ms postprocess per image at shape (1, 3, 480, 640)\n",
      "Subfigure count mismatch: 6 detected, 14 expected\n",
      "1\n",
      "(a) Relative ZmWAKL expression in tissues collected from NIL-Q11 and NIL-Y32 plants (n = 3). (b,c) Relative expression levels of ZmWAKL in NIL-Q11 and NIL-Y32 after inoculation with C. zeina in the greenhouse (b) or field (c). (d,e) Subcellular localization of ZmWAKL in onion epidermal cells (d) and ZmWAKL-GFP overexpression transgenic plants (e). Plasmolysis was achieved by incubating cells in 0.3 g/mL sucrose for 5 min. Scale bars, 50 μm. DIC, differential interference contrast. The membrane was stained with the membrane-impermeable dye FM4-64. This experiment was repeated two times with the same results. In (a–c), values are means ± s.d., statistical significance was determined by a two-sided Student’s t-test.\n",
      "8\n",
      "\n",
      "image 1/1 /Users/etowah/projects/PMC-figure-downloader/img/10864183_Fig13.jpg: 608x640 10 labels, 82.7ms\n",
      "Speed: 2.2ms preprocess, 82.7ms inference, 6.2ms postprocess per image at shape (1, 3, 608, 640)\n",
      "Subfigure count mismatch: 10 detected, 8 expected\n",
      "4\n",
      "\n",
      "image 1/1 /Users/etowah/projects/PMC-figure-downloader/img/10864183_Fig14.jpg: 576x640 4 labels, 76.9ms\n",
      "Speed: 2.3ms preprocess, 76.9ms inference, 0.5ms postprocess per image at shape (1, 3, 576, 640)\n",
      "6\n",
      "\n",
      "image 1/1 /Users/etowah/projects/PMC-figure-downloader/img/10864183_Fig15.jpg: 544x640 7 labels, 74.8ms\n",
      "Speed: 5.2ms preprocess, 74.8ms inference, 0.4ms postprocess per image at shape (1, 3, 544, 640)\n",
      "1\n",
      "(a) Schematic diagram of ZmRBOH4 and the sgRNA target site in the first exon for CRISPR/Cas9-mediated editing. The sequences of two homozygous knockout lines with insertion (ko #1) and deletion (ko #2) are shown. The wild-type B73 sequence is shown on top. The target sites and PAM are highlighted in red and gray, respectively, with deletions indicated by dashes. (b,c) Plant morphology (b) and height (c) of B73 and two homozygous knockout lines (n = 183). The knockout lines exhibit significantly reduced height. Scale bar, 30 cm. Data in (c) are displayed as box and whisker plots with individual data points, the asterisk denotes the mean, and the box limits indicate the interquartile range. The center line indicates the median, and different lowercase letters indicate a significant difference (P < 0.05) based on one-way ANOVA with LSD test.\n",
      "4\n",
      "\n",
      "image 1/1 /Users/etowah/projects/PMC-figure-downloader/img/10864181_Fig1.jpg: 512x640 4 labels, 73.6ms\n",
      "Speed: 1.8ms preprocess, 73.6ms inference, 0.4ms postprocess per image at shape (1, 3, 512, 640)\n",
      "2\n",
      "\n",
      "image 1/1 /Users/etowah/projects/PMC-figure-downloader/img/10864181_Fig2.jpg: 480x640 3 labels, 69.5ms\n",
      "Speed: 1.7ms preprocess, 69.5ms inference, 0.5ms postprocess per image at shape (1, 3, 480, 640)\n",
      "4\n",
      "\n",
      "image 1/1 /Users/etowah/projects/PMC-figure-downloader/img/10864181_Fig3.jpg: 480x640 4 labels, 85.5ms\n",
      "Speed: 1.9ms preprocess, 85.5ms inference, 0.5ms postprocess per image at shape (1, 3, 480, 640)\n",
      "5\n",
      "\n",
      "image 1/1 /Users/etowah/projects/PMC-figure-downloader/img/10864181_Fig4.jpg: 640x416 5 labels, 91.5ms\n",
      "Speed: 2.9ms preprocess, 91.5ms inference, 0.5ms postprocess per image at shape (1, 3, 640, 416)\n",
      "4\n",
      "\n",
      "image 1/1 /Users/etowah/projects/PMC-figure-downloader/img/10864181_Fig5.jpg: 544x640 4 labels, 83.1ms\n",
      "Speed: 2.2ms preprocess, 83.1ms inference, 2.6ms postprocess per image at shape (1, 3, 544, 640)\n",
      "6\n",
      "\n",
      "image 1/1 /Users/etowah/projects/PMC-figure-downloader/img/10864181_Fig6.jpg: 640x512 6 labels, 85.4ms\n",
      "Speed: 2.0ms preprocess, 85.4ms inference, 0.6ms postprocess per image at shape (1, 3, 640, 512)\n",
      "9\n",
      "\n",
      "image 1/1 /Users/etowah/projects/PMC-figure-downloader/img/10864176_Fig1.jpg: 640x512 10 labels, 69.1ms\n",
      "Speed: 2.0ms preprocess, 69.1ms inference, 3.1ms postprocess per image at shape (1, 3, 640, 512)\n",
      "7\n",
      "\n",
      "image 1/1 /Users/etowah/projects/PMC-figure-downloader/img/10864176_Fig2.jpg: 640x480 8 labels, 65.6ms\n",
      "Speed: 1.9ms preprocess, 65.6ms inference, 0.4ms postprocess per image at shape (1, 3, 640, 480)\n",
      "9\n",
      "\n",
      "image 1/1 /Users/etowah/projects/PMC-figure-downloader/img/10864176_Fig3.jpg: 640x512 10 labels, 72.5ms\n",
      "Speed: 2.3ms preprocess, 72.5ms inference, 0.4ms postprocess per image at shape (1, 3, 640, 512)\n",
      "9\n",
      "\n",
      "image 1/1 /Users/etowah/projects/PMC-figure-downloader/img/10864176_Fig4.jpg: 640x512 9 labels, 64.7ms\n",
      "Speed: 2.0ms preprocess, 64.7ms inference, 0.4ms postprocess per image at shape (1, 3, 640, 512)\n",
      "9\n",
      "\n",
      "image 1/1 /Users/etowah/projects/PMC-figure-downloader/img/10864176_Fig5.jpg: 640x448 9 labels, 74.5ms\n",
      "Speed: 2.1ms preprocess, 74.5ms inference, 0.7ms postprocess per image at shape (1, 3, 640, 448)\n",
      "2\n",
      "\n",
      "image 1/1 /Users/etowah/projects/PMC-figure-downloader/img/10864174_Fig1.jpg: 256x640 2 labels, 51.2ms\n",
      "Speed: 1.0ms preprocess, 51.2ms inference, 0.6ms postprocess per image at shape (1, 3, 256, 640)\n",
      "6\n",
      "\n",
      "image 1/1 /Users/etowah/projects/PMC-figure-downloader/img/10864174_Fig2.jpg: 448x640 6 labels, 67.4ms\n",
      "Speed: 1.6ms preprocess, 67.4ms inference, 0.5ms postprocess per image at shape (1, 3, 448, 640)\n",
      "3\n",
      "\n",
      "image 1/1 /Users/etowah/projects/PMC-figure-downloader/img/10864174_Fig3.jpg: 640x640 4 labels, 106.1ms\n",
      "Speed: 2.3ms preprocess, 106.1ms inference, 0.6ms postprocess per image at shape (1, 3, 640, 640)\n",
      "Subfigure count mismatch: 4 detected, 3 expected\n",
      "2\n",
      "\n",
      "image 1/1 /Users/etowah/projects/PMC-figure-downloader/img/10864174_Fig4.jpg: 320x640 2 labels, 57.9ms\n",
      "Speed: 1.4ms preprocess, 57.9ms inference, 0.5ms postprocess per image at shape (1, 3, 320, 640)\n",
      "1\n",
      "Individual-level genotype data are subject to data sharing restrictions. SVD-based processing creates anonymous data describing variation in case genotypes without storing individual data that could be shared with no restrictions. Remote server with a pool of controls selects a set of control genotype variation matching cases, estimates allele frequency for sites to be used for association study and delivers results to the user.\n",
      "7\n",
      "\n",
      "image 1/1 /Users/etowah/projects/PMC-figure-downloader/img/10864173_Fig2.jpg: 640x544 8 labels, 79.5ms\n",
      "Speed: 2.0ms preprocess, 79.5ms inference, 0.5ms postprocess per image at shape (1, 3, 640, 544)\n",
      "8\n",
      "\n",
      "image 1/1 /Users/etowah/projects/PMC-figure-downloader/img/10864173_Fig3.jpg: 640x448 8 labels, 81.3ms\n",
      "Speed: 1.7ms preprocess, 81.3ms inference, 0.4ms postprocess per image at shape (1, 3, 640, 448)\n",
      "5\n",
      "\n",
      "image 1/1 /Users/etowah/projects/PMC-figure-downloader/img/10864173_Fig4.jpg: 640x512 5 labels, 74.2ms\n",
      "Speed: 2.0ms preprocess, 74.2ms inference, 0.5ms postprocess per image at shape (1, 3, 640, 512)\n",
      "6\n",
      "\n",
      "image 1/1 /Users/etowah/projects/PMC-figure-downloader/img/10864173_Fig5.jpg: 576x640 6 labels, 77.2ms\n",
      "Speed: 2.3ms preprocess, 77.2ms inference, 0.9ms postprocess per image at shape (1, 3, 576, 640)\n",
      "5\n",
      "\n",
      "image 1/1 /Users/etowah/projects/PMC-figure-downloader/img/10937405_Fig1.jpg: 640x512 5 labels, 110.7ms\n",
      "Speed: 2.2ms preprocess, 110.7ms inference, 0.6ms postprocess per image at shape (1, 3, 640, 512)\n",
      "8\n",
      "\n",
      "image 1/1 /Users/etowah/projects/PMC-figure-downloader/img/10937405_Fig2.jpg: 640x608 8 labels, 104.1ms\n",
      "Speed: 2.8ms preprocess, 104.1ms inference, 0.8ms postprocess per image at shape (1, 3, 640, 608)\n",
      "6\n",
      "\n",
      "image 1/1 /Users/etowah/projects/PMC-figure-downloader/img/10937405_Fig3.jpg: 576x640 6 labels, 185.5ms\n",
      "Speed: 3.0ms preprocess, 185.5ms inference, 1.8ms postprocess per image at shape (1, 3, 576, 640)\n",
      "4\n",
      "\n",
      "image 1/1 /Users/etowah/projects/PMC-figure-downloader/img/10937405_Fig4.jpg: 544x640 4 labels, 89.9ms\n",
      "Speed: 2.1ms preprocess, 89.9ms inference, 0.5ms postprocess per image at shape (1, 3, 544, 640)\n",
      "10\n",
      "\n",
      "image 1/1 /Users/etowah/projects/PMC-figure-downloader/img/10937405_Fig5.jpg: 640x512 10 labels, 78.7ms\n",
      "Speed: 2.1ms preprocess, 78.7ms inference, 0.5ms postprocess per image at shape (1, 3, 640, 512)\n",
      "7\n",
      "\n",
      "image 1/1 /Users/etowah/projects/PMC-figure-downloader/img/10937405_Fig6.jpg: 640x320 8 labels, 80.1ms\n",
      "Speed: 1.4ms preprocess, 80.1ms inference, 0.6ms postprocess per image at shape (1, 3, 640, 320)\n",
      "3\n",
      "\n",
      "image 1/1 /Users/etowah/projects/PMC-figure-downloader/img/10937399_Fig1.jpg: 608x640 3 labels, 92.0ms\n",
      "Speed: 2.3ms preprocess, 92.0ms inference, 0.5ms postprocess per image at shape (1, 3, 608, 640)\n",
      "3\n",
      "\n",
      "image 1/1 /Users/etowah/projects/PMC-figure-downloader/img/10937399_Fig2.jpg: 640x512 3 labels, 80.4ms\n",
      "Speed: 1.9ms preprocess, 80.4ms inference, 0.4ms postprocess per image at shape (1, 3, 640, 512)\n",
      "10\n",
      "\n",
      "image 1/1 /Users/etowah/projects/PMC-figure-downloader/img/10937399_Fig3.jpg: 640x608 11 labels, 91.8ms\n",
      "Speed: 2.9ms preprocess, 91.8ms inference, 0.6ms postprocess per image at shape (1, 3, 640, 608)\n",
      "23\n",
      "\n",
      "image 1/1 /Users/etowah/projects/PMC-figure-downloader/img/10937399_Fig4.jpg: 640x512 22 labels, 101.4ms\n",
      "Speed: 2.3ms preprocess, 101.4ms inference, 0.5ms postprocess per image at shape (1, 3, 640, 512)\n",
      "Subfigure count mismatch: 22 detected, 23 expected\n",
      "4\n",
      "\n",
      "image 1/1 /Users/etowah/projects/PMC-figure-downloader/img/10937399_Fig5.jpg: 640x640 4 labels, 90.5ms\n",
      "Speed: 2.4ms preprocess, 90.5ms inference, 1.0ms postprocess per image at shape (1, 3, 640, 640)\n",
      "1\n",
      "Runtimes of BayesSpace, FICT, Giotto’s HMRF module, GraphST, MERINGUE’s spatial clustering module, SpaGCN, SpiceMix, STAGATE, nonspatial clustering (Seurat) and BANKSY for increasing cell numbers, up to 2 million cells. All methods were benchmarked on a 16-CPU 128-GB machine. Runtimes are shown up to the maximum cell number accommodated by each method, with a cutoff of 16 h.\n",
      "2\n",
      "\n",
      "image 1/1 /Users/etowah/projects/PMC-figure-downloader/img/10937399_Fig7.jpg: 640x512 2 labels, 81.1ms\n",
      "Speed: 2.2ms preprocess, 81.1ms inference, 0.4ms postprocess per image at shape (1, 3, 640, 512)\n",
      "1\n",
      "a) Non-spatial and BANKSY UMAPs colored by their respective clusters, with cell type labels for BANKSY clusters. b) Spatial plots for the full dataset and a single ROI colored by BANKSY clusters. c) Spatial distribution of macrophages, with the Mac.2 cluster enriched in the submucosa. Violin plots show DE genes between the two macrophage subpopulations. Mac.1 n = 1177, Mac.2 n = 214.\n",
      "6\n",
      "\n",
      "image 1/1 /Users/etowah/projects/PMC-figure-downloader/img/10937399_Fig9.jpg: 640x448 7 labels, 78.1ms\n",
      "Speed: 2.0ms preprocess, 78.1ms inference, 0.7ms postprocess per image at shape (1, 3, 640, 448)\n",
      "1\n",
      "The number of clusters was varied by adjusting parameters related to clustering resolution (number of nearest neighbors, k, in the graph for MERINGUE, and the number of clusters, N, for SpaGCN and FICT). Each column represents one clustering run for the indicated method at a given effective resolution. Blue arrows indicate the runs compared in Fig. 4. Green and red boxes indicate success and failure to identify the known neuronal or oligodendrocyte subclusters (Fig. 4; Supplementary Fig. 18). Conventional non-spatial clustering and MERINGUE (at any number of clusters) failed to separate both the oligodendrocyte subclusters and the neuronal subclusters. While Giotto separated the CA3 neurons (orange) from the somatosensory cortex neurons (brown), and at some resolutions was able to identify the fornix oligodendrocytes (dark green), it failed to identify the thalamic oligodendrocytes (purple) at all resolutions. Importantly, Giotto tended to cause clusters to form spatially disconnected groups of cells (‘patches’) that do not correspond to any known anatomical structures in the thalamus or hippocampal formation (dashed red boxes). BayesSpace merged several neuronal clusters: at N = 16 - 18, it merged the somatosensory cortex neurons with the CA3, CA1 and dentate gyrus neurons (blue clusters; all anatomical regions labeled in Fig. 4 in the main text), and at N = 19 - 20, it separated out the CA3 neurons, from the somatosensory cortex neurons, but merged the CA1 and dentate gyrus neurons. It also failed to separate out the thalamic oligodendrocytes from the fornix oligodendrocytes (red boxes around the dark green cluster).\n",
      "1\n",
      "Similar to Extended Data Fig. 4. As before, the number of clusters was varied by adjusting parameters related to clustering resolution. Each column represents one clustering run for the indicated method at a given resolution (controlled by the number of clusters input parameter, N). Blue arrows indicate the runs compared in Fig. 4. Green and red boxes indicate success and failure (respectively) to identify the known neuronal or oligodendrocyte subclusters (Fig. 4; Supplementary Fig. 18). SpiceMix correctly separated the thalamic and fornix oligodendrocytes, merged the somatosensory cortex neurons with either the CA3 neuron (orange clusters) or the dentate gyrus neurons (brown and light green clusters). SpaGCN was able to identify the CA3 neurons, but merged the somatosensory cortex neurons with cells in the dentate gyrus (brown cluster), and was unable to separate the thalamic and fornix oligodendrocytes (red boxed green clusters). FICT, like Giotto in Extended Data Fig. 4, tended to group cells into disconnected spatial patches that do not conform to known anatomical structures (dashed red boxes), and tended to merge the dentate gyrus, CA1, CA3 and somatosensory cortex neurons into one or two clusters (light and dark blue clusters).\n",
      "3\n",
      "\n",
      "image 1/1 /Users/etowah/projects/PMC-figure-downloader/img/10937399_Fig12.jpg: 640x640 3 labels, 112.9ms\n",
      "Speed: 2.4ms preprocess, 112.9ms inference, 0.8ms postprocess per image at shape (1, 3, 640, 640)\n",
      "3\n",
      "\n",
      "image 1/1 /Users/etowah/projects/PMC-figure-downloader/img/10937399_Fig13.jpg: 576x640 3 labels, 101.8ms\n",
      "Speed: 2.1ms preprocess, 101.8ms inference, 0.4ms postprocess per image at shape (1, 3, 576, 640)\n",
      "1\n",
      "a). BANKSY UMAP and spatial plots colored by tissue segment annotations. b). BANKSY UMAP and spatial plots colored by BANKSY clusters. c). Non-spatial UMAP and spatial plots colored by non-spatial clusters. d). Comparison of BANKSY and non-spatial clusters with community domain annotations. e). Marker genes for smooth muscle and mature epithelial communities identified by BANKSY. Boxplots are colored as in (d). Center line: median, dotted line: mean, height of box: interquartile range (IQR), whiskers: 1.5 x IQR. CD66 Annotation: n = 3179 (mean = 1.47, median = 0.636) vs 6189 (mean = 0.107, median = -0.167) cells, BANKSY: n = 3435 (mean = 1.70, median = 0.935) vs 5232 (mean = -0.184, median = -0.249), aSMA BANKSY: n = 3222 (mean = 1.26, median = 1.04) vs 4826 (mean = 0.245, median = 0.138). All p-values < 2.2e-16.\n",
      "1\n",
      "Performance of non-spatial clustering, BayesSpace and BANKSY with and without batch correction on joint clustering of spots from n = 12 DLPFC Visium datasets. ARI: adjusted Rand index. NMI: normalized mutual information. MCC: Matthew’s correlation coefficient. Center line: median, red dotted line: mean, height of box: interquartile range (IQR), whiskers: 1.5 x IQR.\n",
      "1\n",
      "All methods were benchmarked using a NVIDIA T4 GPU with 16GB of memory. Runtimes are shown up to the maximum cell number accommodated by each method.\n",
      "7\n",
      "\n",
      "image 1/1 /Users/etowah/projects/PMC-figure-downloader/img/10937392_Fig1.jpg: 640x512 7 labels, 71.2ms\n",
      "Speed: 1.9ms preprocess, 71.2ms inference, 0.4ms postprocess per image at shape (1, 3, 640, 512)\n",
      "8\n",
      "\n",
      "image 1/1 /Users/etowah/projects/PMC-figure-downloader/img/10937392_Fig2.jpg: 640x544 8 labels, 97.2ms\n",
      "Speed: 2.1ms preprocess, 97.2ms inference, 0.5ms postprocess per image at shape (1, 3, 640, 544)\n",
      "12\n",
      "\n",
      "image 1/1 /Users/etowah/projects/PMC-figure-downloader/img/10937392_Fig3.jpg: 640x544 12 labels, 89.8ms\n",
      "Speed: 2.3ms preprocess, 89.8ms inference, 0.5ms postprocess per image at shape (1, 3, 640, 544)\n",
      "11\n",
      "\n",
      "image 1/1 /Users/etowah/projects/PMC-figure-downloader/img/10937392_Fig4.jpg: 640x512 12 labels, 76.2ms\n",
      "Speed: 1.9ms preprocess, 76.2ms inference, 1.1ms postprocess per image at shape (1, 3, 640, 512)\n",
      "Subfigure count mismatch: 12 detected, 11 expected\n",
      "9\n",
      "\n",
      "image 1/1 /Users/etowah/projects/PMC-figure-downloader/img/10937392_Fig5.jpg: 640x480 10 labels, 101.2ms\n",
      "Speed: 2.0ms preprocess, 101.2ms inference, 0.6ms postprocess per image at shape (1, 3, 640, 480)\n",
      "12\n",
      "\n",
      "image 1/1 /Users/etowah/projects/PMC-figure-downloader/img/10937392_Fig6.jpg: 640x480 13 labels, 81.8ms\n",
      "Speed: 2.1ms preprocess, 81.8ms inference, 0.5ms postprocess per image at shape (1, 3, 640, 480)\n",
      "9\n",
      "\n",
      "image 1/1 /Users/etowah/projects/PMC-figure-downloader/img/10937392_Fig7.jpg: 640x608 9 labels, 102.3ms\n",
      "Speed: 2.8ms preprocess, 102.3ms inference, 0.7ms postprocess per image at shape (1, 3, 640, 608)\n",
      "8\n",
      "\n",
      "image 1/1 /Users/etowah/projects/PMC-figure-downloader/img/10937392_Fig8.jpg: 640x608 8 labels, 92.6ms\n",
      "Speed: 1.9ms preprocess, 92.6ms inference, 0.7ms postprocess per image at shape (1, 3, 640, 608)\n",
      "10\n",
      "\n",
      "image 1/1 /Users/etowah/projects/PMC-figure-downloader/img/10937392_Fig9.jpg: 640x576 11 labels, 76.1ms\n",
      "Speed: 2.0ms preprocess, 76.1ms inference, 0.4ms postprocess per image at shape (1, 3, 640, 576)\n",
      "8\n",
      "\n",
      "image 1/1 /Users/etowah/projects/PMC-figure-downloader/img/10937392_Fig10.jpg: 640x544 9 labels, 80.3ms\n",
      "Speed: 2.2ms preprocess, 80.3ms inference, 0.4ms postprocess per image at shape (1, 3, 640, 544)\n",
      "8\n",
      "\n",
      "image 1/1 /Users/etowah/projects/PMC-figure-downloader/img/10937392_Fig11.jpg: 640x480 8 labels, 75.5ms\n",
      "Speed: 1.9ms preprocess, 75.5ms inference, 0.4ms postprocess per image at shape (1, 3, 640, 480)\n",
      "11\n",
      "\n",
      "image 1/1 /Users/etowah/projects/PMC-figure-downloader/img/10937392_Fig12.jpg: 640x480 11 labels, 70.2ms\n",
      "Speed: 1.8ms preprocess, 70.2ms inference, 0.6ms postprocess per image at shape (1, 3, 640, 480)\n",
      "3\n",
      "\n",
      "image 1/1 /Users/etowah/projects/PMC-figure-downloader/img/10937392_Fig13.jpg: 640x480 5 labels, 92.9ms\n",
      "Speed: 2.2ms preprocess, 92.9ms inference, 1.1ms postprocess per image at shape (1, 3, 640, 480)\n",
      "Subfigure count mismatch: 5 detected, 3 expected\n",
      "5\n",
      "\n",
      "image 1/1 /Users/etowah/projects/PMC-figure-downloader/img/10937392_Fig14.jpg: 640x480 6 labels, 61.1ms\n",
      "Speed: 1.9ms preprocess, 61.1ms inference, 0.4ms postprocess per image at shape (1, 3, 640, 480)\n",
      "5\n",
      "\n",
      "image 1/1 /Users/etowah/projects/PMC-figure-downloader/img/10937392_Fig15.jpg: 448x640 5 labels, 84.4ms\n",
      "Speed: 1.8ms preprocess, 84.4ms inference, 0.6ms postprocess per image at shape (1, 3, 448, 640)\n",
      "5\n",
      "\n",
      "image 1/1 /Users/etowah/projects/PMC-figure-downloader/img/10937392_Fig16.jpg: 640x608 7 labels, 81.2ms\n",
      "Speed: 2.7ms preprocess, 81.2ms inference, 0.8ms postprocess per image at shape (1, 3, 640, 608)\n",
      "Subfigure count mismatch: 7 detected, 5 expected\n",
      "5\n",
      "\n",
      "image 1/1 /Users/etowah/projects/PMC-figure-downloader/img/10937392_Fig17.jpg: 640x576 5 labels, 80.9ms\n",
      "Speed: 2.2ms preprocess, 80.9ms inference, 0.4ms postprocess per image at shape (1, 3, 640, 576)\n",
      "10\n",
      "\n",
      "image 1/1 /Users/etowah/projects/PMC-figure-downloader/img/10937392_Fig18.jpg: 640x480 12 labels, 68.6ms\n",
      "Speed: 2.0ms preprocess, 68.6ms inference, 0.5ms postprocess per image at shape (1, 3, 640, 480)\n",
      "Subfigure count mismatch: 12 detected, 10 expected\n",
      "3\n",
      "\n",
      "image 1/1 /Users/etowah/projects/PMC-figure-downloader/img/10937387_Fig1.jpg: 640x480 3 labels, 71.9ms\n",
      "Speed: 1.9ms preprocess, 71.9ms inference, 1.5ms postprocess per image at shape (1, 3, 640, 480)\n",
      "9\n",
      "\n",
      "image 1/1 /Users/etowah/projects/PMC-figure-downloader/img/10937387_Fig2.jpg: 576x640 8 labels, 89.0ms\n",
      "Speed: 2.1ms preprocess, 89.0ms inference, 1.1ms postprocess per image at shape (1, 3, 576, 640)\n",
      "Subfigure count mismatch: 7 detected, 9 expected\n",
      "4\n",
      "\n",
      "image 1/1 /Users/etowah/projects/PMC-figure-downloader/img/10937387_Fig3.jpg: 224x640 5 labels, 46.6ms\n",
      "Speed: 0.9ms preprocess, 46.6ms inference, 0.4ms postprocess per image at shape (1, 3, 224, 640)\n",
      "20\n",
      "\n",
      "image 1/1 /Users/etowah/projects/PMC-figure-downloader/img/10937387_Fig4.jpg: 512x640 21 labels, 84.0ms\n",
      "Speed: 2.0ms preprocess, 84.0ms inference, 1.4ms postprocess per image at shape (1, 3, 512, 640)\n",
      "3\n",
      "\n",
      "image 1/1 /Users/etowah/projects/PMC-figure-downloader/img/10937387_Fig5.jpg: 640x640 3 labels, 103.7ms\n",
      "Speed: 2.5ms preprocess, 103.7ms inference, 0.6ms postprocess per image at shape (1, 3, 640, 640)\n",
      "2\n",
      "\n",
      "image 1/1 /Users/etowah/projects/PMC-figure-downloader/img/10937387_Fig6.jpg: 224x640 2 labels, 63.5ms\n",
      "Speed: 1.0ms preprocess, 63.5ms inference, 0.4ms postprocess per image at shape (1, 3, 224, 640)\n",
      "16\n",
      "\n",
      "image 1/1 /Users/etowah/projects/PMC-figure-downloader/img/10937387_Fig7.jpg: 608x640 3 labels, 79.3ms\n",
      "Speed: 2.3ms preprocess, 79.3ms inference, 1.0ms postprocess per image at shape (1, 3, 608, 640)\n",
      "Subfigure count mismatch: 3 detected, 16 expected\n",
      "1\n",
      "RNA sequencing data from patients carrying SRPK3 truncating variants (LII:1, DII:1 and YII:3) were analyzed for SRPK3 expression between patient (n = 3) and control (n = 6) samples. Counts per million (CPM) values were normalized by gene length. The boxes represent the first and third quartiles (25% and 75% percentile) with the center line at the median value. The whiskers extend from the hinge to the furthest value not beyond 1.5 times the interquartile range from the hinge. Differential expression was performed in edgeR using a two-sided exact test, with no adjustment for multiple comparisons. Uncorrected P-value = 7.106 × 10−11.\n",
      "16\n",
      "\n",
      "image 1/1 /Users/etowah/projects/PMC-figure-downloader/img/10937387_Fig9.jpg: 576x640 2 labels, 86.7ms\n",
      "Speed: 2.1ms preprocess, 86.7ms inference, 0.6ms postprocess per image at shape (1, 3, 576, 640)\n",
      "Subfigure count mismatch: 2 detected, 16 expected\n",
      "1\n",
      "TTN truncating variants (nonsense, splice sites and frameshift) are shown in black and missense variants in green. No missense variants in TTN exons 344 or 364, associated with HMERF and tibial muscular dystrophy, respectively, were found. TTN variants were located mainly in the A-band and I-band; however, no clustering was observed. Two frameshift variants occurred in meta transcript only exons (in pink).\n",
      "1\n",
      "RNA sequencing data from SRPK3/TTN myopathy patients (LII:1, DII:1 and YII:3) were analyzed for TTN expression between patient (n = 3) and control (n = 6) samples. Counts per million (CPM) values were normalized by gene length. The boxes represent the first and third quartiles (25% and 75% percentile) with the center line at the median value. The whiskers extend from the hinge to the furthest value not beyond 1.5 times the interquartile range from the hinge. Differential expression was performed in edgeR using a two-sided exact test, with no adjustment for multiple comparisons. Uncorrected P-value = 0.0008015. This was likely not due to nonsense-mediated mRNA decay, as there was no evidence of allele-specific expression at any of the heterozygous TTN loci examined.\n",
      "1\n",
      "Comparison between the number of TTNtv (stop gain, splice sites and frameshift variants) in the SRPK3/TTN myopathy families (n = 25) and three cohorts of patients with genetically confirmed forms of limb girdle muscular dystrophy: LGMD-R1 (n = 170), LGMD-R2 (n = 94) and LGMD-R12 (n = 56). A Fisher’s test (two-sided, with no adjustment for multiple comparisons), following the proposal of Agresti and Coull to add two successes and two failures to each data set was calculated. P-values: (*) = 6.13 × 10−19, (**) = 9.18 × 10−17 and (***) = 4.89 × 10−12.\n",
      "2\n",
      "\n",
      "image 1/1 /Users/etowah/projects/PMC-figure-downloader/img/10937387_Fig13.jpg: 640x640 2 labels, 98.5ms\n",
      "Speed: 2.6ms preprocess, 98.5ms inference, 0.5ms postprocess per image at shape (1, 3, 640, 640)\n",
      "6\n",
      "\n",
      "image 1/1 /Users/etowah/projects/PMC-figure-downloader/img/10937387_Fig14.jpg: 448x640 6 labels, 69.2ms\n",
      "Speed: 1.5ms preprocess, 69.2ms inference, 0.6ms postprocess per image at shape (1, 3, 448, 640)\n",
      "2\n",
      "\n",
      "image 1/1 /Users/etowah/projects/PMC-figure-downloader/img/10937387_Fig15.jpg: 256x640 2 labels, 60.7ms\n",
      "Speed: 1.1ms preprocess, 60.7ms inference, 0.6ms postprocess per image at shape (1, 3, 256, 640)\n",
      "1\n",
      "The table shows the top enriched GO terms from the comparisons of srpk3−/−; ttn.1+/− and srpk3−/−; ttn.1+/+ embryos to wild-type (P < 0.001 in one of the comparisons). The topGO P-value columns show the P-value from the enrichment test for each comparison. The dots in the number of DE genes columns represent the number of DE genes annotated to the term that appear either in srpk3−/−; ttn.1+/− alone (blue), srpk3−/−; ttn.1+/+ alone (red) or in both (purple). For most GO terms, most of the DE genes causing the enrichment are shared between both lists. GO term enrichment was done using the topGO package using a one-sided Fisher’s exact test without adjustment for multiple testing.\n",
      "4\n",
      "\n",
      "image 1/1 /Users/etowah/projects/PMC-figure-downloader/img/10937381_Fig1.jpg: 608x640 4 labels, 86.8ms\n",
      "Speed: 2.3ms preprocess, 86.8ms inference, 0.6ms postprocess per image at shape (1, 3, 608, 640)\n",
      "1\n",
      "Contact matrices displaying in vivo13 and in vitro Micro-C data with corresponding insulation scores and nucleosome occupancy profiles (MNase-seq) for three genomic regions. Gene (arrows) and sequence composition annotation are shown at the top (A, adenine; T, thymine). In vivo19 and in vitro ChIP–seq data for Abf1 (blue) and Reb1 (red) are shown below the in vivo data. Chromatin used for in vitro Micro-C was incubated with the indicated remodeler and the TFs Abf1 and Reb1 or with the TFs only. Strongly insulating boundaries are labeled with gray dots. Micro-C data are plotted as log10 interaction counts at 40 bp resolution. Insulation scores are calculated at 80 bp resolution with three different sliding windows.\n",
      "6\n",
      "\n",
      "image 1/1 /Users/etowah/projects/PMC-figure-downloader/img/10937381_Fig3.jpg: 608x640 6 labels, 78.1ms\n",
      "Speed: 2.3ms preprocess, 78.1ms inference, 0.4ms postprocess per image at shape (1, 3, 608, 640)\n",
      "4\n",
      "\n",
      "image 1/1 /Users/etowah/projects/PMC-figure-downloader/img/10937381_Fig4.jpg: 448x640 4 labels, 75.7ms\n",
      "Speed: 1.9ms preprocess, 75.7ms inference, 0.6ms postprocess per image at shape (1, 3, 448, 640)\n",
      "2\n",
      "\n",
      "image 1/1 /Users/etowah/projects/PMC-figure-downloader/img/10937381_Fig5.jpg: 608x640 2 labels, 91.6ms\n",
      "Speed: 2.2ms preprocess, 91.6ms inference, 0.4ms postprocess per image at shape (1, 3, 608, 640)\n",
      "3\n",
      "\n",
      "image 1/1 /Users/etowah/projects/PMC-figure-downloader/img/10937381_Fig6.jpg: 544x640 3 labels, 76.9ms\n",
      "Speed: 2.4ms preprocess, 76.9ms inference, 0.5ms postprocess per image at shape (1, 3, 544, 640)\n",
      "2\n",
      "\n",
      "image 1/1 /Users/etowah/projects/PMC-figure-downloader/img/10937381_Fig7.jpg: 640x448 2 labels, 66.0ms\n",
      "Speed: 1.8ms preprocess, 66.0ms inference, 0.4ms postprocess per image at shape (1, 3, 640, 448)\n",
      "5\n",
      "\n",
      "image 1/1 /Users/etowah/projects/PMC-figure-downloader/img/10937381_Fig8.jpg: 288x640 5 labels, 59.1ms\n",
      "Speed: 1.1ms preprocess, 59.1ms inference, 0.4ms postprocess per image at shape (1, 3, 288, 640)\n",
      "5\n",
      "\n",
      "image 1/1 /Users/etowah/projects/PMC-figure-downloader/img/10937381_Fig9.jpg: 448x640 6 labels, 72.6ms\n",
      "Speed: 1.7ms preprocess, 72.6ms inference, 0.4ms postprocess per image at shape (1, 3, 448, 640)\n",
      "1\n",
      "Contact matrices displaying in vivo13 and in vitro Micro-C data and corresponding nucleosome occupancy profiles (MNase-seq) for three genomic regions with low Abf1 and Reb1 occupancy. Gene annotation is shown at the top19. Chromatin used for in vitro Micro-C was incubated with the indicated remodeler and the TFs Abf1 and Reb1. Micro-C data are plotted as log10 interaction counts at 40 bp resolution.\n",
      "1\n",
      "Contact matrices displaying in vivo13 and in vitro Micro-C data for three genomic regions. Gene and sequence composition annotation are shown at the top (A = adenine; T = thymine). In vivo19 and in vitro ChIP-seq data for Abf1 (blue) and Reb1 (red) are shown below the in vivo data. Chromatin used for in vitro Micro-C was incubated with or without chromatin remodelers and the TFs Abf1 and Reb1 as indicated. Micro-C data are plotted as log10 interaction counts at 40 bp resolution.\n",
      "3\n",
      "\n",
      "image 1/1 /Users/etowah/projects/PMC-figure-downloader/img/10937381_Fig12.jpg: 640x640 3 labels, 79.7ms\n",
      "Speed: 2.3ms preprocess, 79.7ms inference, 0.4ms postprocess per image at shape (1, 3, 640, 640)\n",
      "4\n",
      "\n",
      "image 1/1 /Users/etowah/projects/PMC-figure-downloader/img/10937381_Fig13.jpg: 544x640 4 labels, 80.2ms\n",
      "Speed: 2.0ms preprocess, 80.2ms inference, 0.4ms postprocess per image at shape (1, 3, 544, 640)\n",
      "3\n",
      "\n",
      "image 1/1 /Users/etowah/projects/PMC-figure-downloader/img/10937381_Fig14.jpg: 480x640 4 labels, 72.1ms\n",
      "Speed: 2.2ms preprocess, 72.1ms inference, 0.5ms postprocess per image at shape (1, 3, 480, 640)\n",
      "Subfigure count mismatch: 4 detected, 3 expected\n",
      "2\n",
      "\n",
      "image 1/1 /Users/etowah/projects/PMC-figure-downloader/img/10937370_Fig1.jpg: 608x640 2 labels, 95.2ms\n",
      "Speed: 2.5ms preprocess, 95.2ms inference, 0.9ms postprocess per image at shape (1, 3, 608, 640)\n",
      "1\n",
      "Comparison of log odds ratios in GWAS of GDM (x axis) and T2D in males (y axis) for top-associated SNPs from GDM (13 SNPs) and T2D (15 SNPs). The following two distinct classes of SNP effects were identified by a Bayesian classifier in shared variants analysis: class T (blue) containing SNPs with T2D-predominant genetic effects and class G (red) with GDM-predominant effects (Supplementary Table 27). Gray SNPs were not confidently assigned to either class (posterior probability >95%). Dotted ellipses indicate the 95% probability regions of the fitted bivariate effect size distributions with each class.\n",
      "1\n",
      "Cell-type specificity analysis was performed for GDM (a) and for prior meta-analysis of T2D (b) from ref. 18 using high-quality mouse single-cell RNA-seq datasets with FUMA v1.3.4 (Supplementary Tables 32–35). Unadjusted P values are reported for the two-sided association test between relative gene expression in the given cell type and multimarker analysis of genomic annotation (MAGMA) gene-level associations in the GWAS. Results are shown for cell types that both are significantly associated with at least one GWAS after correction for multiple testing of all cell types in all datasets (a) and have putatively independent association conditional on other cell types in the same RNA-seq dataset (b). Colors indicate the RNA-seq dataset source and significance.\n",
      "1\n",
      "We included data from 21 cohorts with diverse ancestry. We assigned individuals into ancestry/ethnic groups and carried out association analyses with MD for each. Subsequently, we meta-analyzed the results by ancestry/ethnic group. We tested whether previously reported MD loci from European ancestry studies are transferable to these groups. We also used the results for discovery of novel depression associations and MR to assess the causal effects of cardiometabolic traits by ancestry. We subsequently merged all ancestry/ethnicity-specific results in a multi-ancestry meta-analysis that also included samples with European ancestry. The multi-ancestry meta-analysis results formed the basis for locus discovery, fine mapping to identify causal variants and several gene prioritization approaches to identify genes linked to MD risk. ST.(n) refers to the corresponding Supplementary Table. ST.2* (in green) refers to Supplementary Table 2, showing genomic inflation estimates of multiple analyses.\n",
      "2\n",
      "\n",
      "image 1/1 /Users/etowah/projects/PMC-figure-downloader/img/10864182_Fig2.jpg: 320x640 2 labels, 71.6ms\n",
      "Speed: 1.5ms preprocess, 71.6ms inference, 0.7ms postprocess per image at shape (1, 3, 320, 640)\n",
      "1\n",
      "A plot showing the genome-wide genetic correlations between the African, European, East Asian and Hispanic/Latin American groups. The intensity of the coloring reflects the strength of the correlation. The estimated coefficients and standard errors are also shown in each cell. We only present estimates where the s.e.m. was smaller than 0.3; otherwise, the field is colored in gray.\n",
      "2\n",
      "\n",
      "image 1/1 /Users/etowah/projects/PMC-figure-downloader/img/10864182_Fig4.jpg: 288x640 2 labels, 55.9ms\n",
      "Speed: 1.1ms preprocess, 55.9ms inference, 0.4ms postprocess per image at shape (1, 3, 288, 640)\n",
      "1\n",
      "Significant gene associations are shown as red dots (354 significant genes, 205 of them novel), and the 50 most significant gene names are highlighted on both sides of the plot. Novel associations are shown in black, while genes previously associated with MD are shown in gray. The red lines indicate the significance threshold (P < 1.37 × 10−6). For genes on the top part of the graph, increased expression was associated with increased depression risk, while expression of the genes on the bottom part of the plot showed an inverse association. NT, novel transcript.\n",
      "1\n",
      "The data are presented with a β and a 95% CI. Nominally significant associations are marked with a red asterisk. Statistics have been derived using the β and standard errors for the number of variants used as IVs in each analysis, shown as N SNPs. Results are not shown for diastolic blood pressure for which there were no significant associations. *P < 0.05 (P values in order from top to bottom: 6.88 × 10−11, 8.22 × 10−3, 9.22 ×10−3, 7.93 × 10−7, 7.67 × 10−3, 3.43 ×’10−3, 0.03 and 0.01). More details can be found in Supplementary Table 16. AFR, African ancestry; EAS, East Asian ancestry; EUR, European ancestry; HIS, Hispanic/Latin American group; SAS, South Asian ancestry.\n",
      "4\n",
      "\n",
      "image 1/1 /Users/etowah/projects/PMC-figure-downloader/img/10864182_Fig7.jpg: 416x640 4 labels, 60.8ms\n",
      "Speed: 1.6ms preprocess, 60.8ms inference, 0.4ms postprocess per image at shape (1, 3, 416, 640)\n",
      "4\n",
      "\n",
      "image 1/1 /Users/etowah/projects/PMC-figure-downloader/img/10864182_Fig8.jpg: 416x640 4 labels, 59.5ms\n",
      "Speed: 1.5ms preprocess, 59.5ms inference, 0.6ms postprocess per image at shape (1, 3, 416, 640)\n",
      "1\n",
      "The y-axes show the −log10P values for the associations between each single-nucleotide polymorphism and major depression. The x-axes show the chromosomal position (GRCh37). The red line represents the genome-wide significance threshold of 5 × 10−8 and the blue line 10−5. Association P values have been adjusted by the LDSC intercept of 1.0185.\n",
      "1\n",
      "The y-axes show the −log10P values for the associations between each single-nucleotide polymorphism and major depression. The x-axes show the chromosomal position (GRCh37). The red line represents the genome-wide significance threshold of 5 × 10−8 and the blue line 10−5.\n",
      "13\n",
      "\n",
      "image 1/1 /Users/etowah/projects/PMC-figure-downloader/img/10786726_Fig1.jpg: 640x512 13 labels, 89.9ms\n",
      "Speed: 1.9ms preprocess, 89.9ms inference, 0.4ms postprocess per image at shape (1, 3, 640, 512)\n",
      "11\n",
      "\n",
      "image 1/1 /Users/etowah/projects/PMC-figure-downloader/img/10786726_Fig2.jpg: 640x608 11 labels, 81.2ms\n",
      "Speed: 3.7ms preprocess, 81.2ms inference, 0.6ms postprocess per image at shape (1, 3, 640, 608)\n",
      "14\n",
      "\n",
      "image 1/1 /Users/etowah/projects/PMC-figure-downloader/img/10786726_Fig3.jpg: 640x544 16 labels, 70.3ms\n",
      "Speed: 2.0ms preprocess, 70.3ms inference, 0.4ms postprocess per image at shape (1, 3, 640, 544)\n",
      "Subfigure count mismatch: 15 detected, 14 expected\n",
      "7\n",
      "\n",
      "image 1/1 /Users/etowah/projects/PMC-figure-downloader/img/10786726_Fig4.jpg: 544x640 7 labels, 74.3ms\n",
      "Speed: 2.0ms preprocess, 74.3ms inference, 0.4ms postprocess per image at shape (1, 3, 544, 640)\n",
      "6\n",
      "\n",
      "image 1/1 /Users/etowah/projects/PMC-figure-downloader/img/10786726_Fig5.jpg: 640x608 6 labels, 85.5ms\n",
      "Speed: 2.7ms preprocess, 85.5ms inference, 0.4ms postprocess per image at shape (1, 3, 640, 608)\n",
      "7\n",
      "\n",
      "image 1/1 /Users/etowah/projects/PMC-figure-downloader/img/10786726_Fig6.jpg: 640x640 7 labels, 87.0ms\n",
      "Speed: 2.6ms preprocess, 87.0ms inference, 0.5ms postprocess per image at shape (1, 3, 640, 640)\n",
      "7\n",
      "\n",
      "image 1/1 /Users/etowah/projects/PMC-figure-downloader/img/10786726_Fig7.jpg: 512x640 8 labels, 89.8ms\n",
      "Speed: 1.9ms preprocess, 89.8ms inference, 1.6ms postprocess per image at shape (1, 3, 512, 640)\n",
      "Subfigure count mismatch: 8 detected, 7 expected\n",
      "1\n",
      "At tumor initiation, continuous APOBEC3B expression and activity induces CIN and p53 pathway activation, resulting in cell death. With targeted therapy, NF-κB induction leads to increased A3B expression, fueling TKI resistance. Figure created in BioRender.com.\n",
      "13\n",
      "\n",
      "image 1/1 /Users/etowah/projects/PMC-figure-downloader/img/10786726_Fig9.jpg: 640x512 14 labels, 83.7ms\n",
      "Speed: 2.8ms preprocess, 83.7ms inference, 0.9ms postprocess per image at shape (1, 3, 640, 512)\n",
      "4\n",
      "\n",
      "image 1/1 /Users/etowah/projects/PMC-figure-downloader/img/10786726_Fig10.jpg: 384x640 4 labels, 69.3ms\n",
      "Speed: 1.4ms preprocess, 69.3ms inference, 0.4ms postprocess per image at shape (1, 3, 384, 640)\n",
      "4\n",
      "\n",
      "image 1/1 /Users/etowah/projects/PMC-figure-downloader/img/10786726_Fig11.jpg: 448x640 5 labels, 75.8ms\n",
      "Speed: 1.5ms preprocess, 75.8ms inference, 0.5ms postprocess per image at shape (1, 3, 448, 640)\n",
      "9\n",
      "\n",
      "image 1/1 /Users/etowah/projects/PMC-figure-downloader/img/10786726_Fig12.jpg: 640x480 9 labels, 73.9ms\n",
      "Speed: 2.6ms preprocess, 73.9ms inference, 0.5ms postprocess per image at shape (1, 3, 640, 480)\n",
      "9\n",
      "\n",
      "image 1/1 /Users/etowah/projects/PMC-figure-downloader/img/10786726_Fig13.jpg: 576x640 10 labels, 144.2ms\n",
      "Speed: 2.6ms preprocess, 144.2ms inference, 1.0ms postprocess per image at shape (1, 3, 576, 640)\n",
      "15\n",
      "\n",
      "image 1/1 /Users/etowah/projects/PMC-figure-downloader/img/10786726_Fig14.jpg: 640x608 17 labels, 90.0ms\n",
      "Speed: 2.4ms preprocess, 90.0ms inference, 0.6ms postprocess per image at shape (1, 3, 640, 608)\n",
      "8\n",
      "\n",
      "image 1/1 /Users/etowah/projects/PMC-figure-downloader/img/10786726_Fig15.jpg: 576x640 8 labels, 82.0ms\n",
      "Speed: 2.1ms preprocess, 82.0ms inference, 0.6ms postprocess per image at shape (1, 3, 576, 640)\n",
      "13\n",
      "\n",
      "image 1/1 /Users/etowah/projects/PMC-figure-downloader/img/10786726_Fig16.jpg: 640x608 13 labels, 95.8ms\n",
      "Speed: 2.2ms preprocess, 95.8ms inference, 0.6ms postprocess per image at shape (1, 3, 640, 608)\n",
      "4\n",
      "\n",
      "image 1/1 /Users/etowah/projects/PMC-figure-downloader/img/10786726_Fig17.jpg: 256x640 4 labels, 43.4ms\n",
      "Speed: 1.0ms preprocess, 43.4ms inference, 0.4ms postprocess per image at shape (1, 3, 256, 640)\n",
      "3\n",
      "\n",
      "image 1/1 /Users/etowah/projects/PMC-figure-downloader/img/10786726_Fig18.jpg: 384x640 3 labels, 51.7ms\n",
      "Speed: 1.8ms preprocess, 51.7ms inference, 0.4ms postprocess per image at shape (1, 3, 384, 640)\n",
      "5\n",
      "\n",
      "image 1/1 /Users/etowah/projects/PMC-figure-downloader/img/10786715_Fig1.jpg: 544x640 5 labels, 82.0ms\n",
      "Speed: 2.1ms preprocess, 82.0ms inference, 0.7ms postprocess per image at shape (1, 3, 544, 640)\n",
      "2\n",
      "\n",
      "image 1/1 /Users/etowah/projects/PMC-figure-downloader/img/10786715_Fig2.jpg: 384x640 2 labels, 66.4ms\n",
      "Speed: 1.6ms preprocess, 66.4ms inference, 0.5ms postprocess per image at shape (1, 3, 384, 640)\n",
      "3\n",
      "\n",
      "image 1/1 /Users/etowah/projects/PMC-figure-downloader/img/10786715_Fig3.jpg: 512x640 3 labels, 71.4ms\n",
      "Speed: 1.9ms preprocess, 71.4ms inference, 1.8ms postprocess per image at shape (1, 3, 512, 640)\n",
      "12\n",
      "\n",
      "image 1/1 /Users/etowah/projects/PMC-figure-downloader/img/10786715_Fig4.jpg: 640x448 12 labels, 68.0ms\n",
      "Speed: 1.9ms preprocess, 68.0ms inference, 0.5ms postprocess per image at shape (1, 3, 640, 448)\n",
      "2\n",
      "\n",
      "image 1/1 /Users/etowah/projects/PMC-figure-downloader/img/10786715_Fig5.jpg: 256x640 2 labels, 48.7ms\n",
      "Speed: 1.1ms preprocess, 48.7ms inference, 0.4ms postprocess per image at shape (1, 3, 256, 640)\n",
      "9\n",
      "\n",
      "image 1/1 /Users/etowah/projects/PMC-figure-downloader/img/10786715_Fig6.jpg: 640x544 9 labels, 99.2ms\n",
      "Speed: 2.1ms preprocess, 99.2ms inference, 0.7ms postprocess per image at shape (1, 3, 640, 544)\n",
      "4\n",
      "\n",
      "image 1/1 /Users/etowah/projects/PMC-figure-downloader/img/10786715_Fig7.jpg: 384x640 4 labels, 62.0ms\n",
      "Speed: 2.0ms preprocess, 62.0ms inference, 0.4ms postprocess per image at shape (1, 3, 384, 640)\n",
      "1\n",
      "Distribution of blood cytometry readouts under baseline and three perturbation conditions. Aggregate counts for each bin were calculated across all samples for each condition and normalized to the number of cells measured in each channel. Each channel records three dimensions (forward scatter, side scatter and side fluorescence). Plots show the two dimensions used for gating cell types in each channel.\n",
      "3\n",
      "\n",
      "image 1/1 /Users/etowah/projects/PMC-figure-downloader/img/10786715_Fig9.jpg: 640x384 3 labels, 61.6ms\n",
      "Speed: 1.7ms preprocess, 61.6ms inference, 0.4ms postprocess per image at shape (1, 3, 640, 384)\n",
      "1\n",
      "Distance correlations between all pairs of blood trait readouts with significant genetic associations were projected into a 2-dimensional embedding using UMAP. Each trait is assigned a color by the cell type, the type of readout, associated candidate genes at the GWAS locus and the perturbation condition.\n",
      "8\n",
      "\n",
      "image 1/1 /Users/etowah/projects/PMC-figure-downloader/img/10786715_Fig11.jpg: 640x640 6 labels, 83.8ms\n",
      "Speed: 2.5ms preprocess, 83.8ms inference, 1.0ms postprocess per image at shape (1, 3, 640, 640)\n",
      "Subfigure count mismatch: 6 detected, 8 expected\n",
      "1\n",
      "a-d, Quantification of GFP+ cells at the tail transection site at 4 h and 24 h under control, and hyperglycemia conditions for control Tracer RNA injected (a, control n = 12, hyperglycemia n = 15), hk1 (b, control n = 7, hyperglycemia n = 12), pfkpa/pfkpb (c, control n = 8, hyperglycemia n = 9), and acsl1a/acsl1b (d, control n = 9, hyperglycemia n = 8) gRNA injected zebrafish. Each data point indicates an individual zebrafish. Paired two-sided non-parametric test (Wilcoxon test) was used to determine P values. *P < 0.05, **P < 0.01, and ***P < 0.001.\n",
      "1\n",
      "Forest plots of ancestry-specific associations for selected lead SNPs. Each plot shows estimated log HR and 95% CI across distinct ancestry groups after harmonizing the effect allele. Genotype counts are shown in parentheses. Asterisks next to each ancestry group denote levels of significance reached: *P < 0.05, **P < 0.01, ***P < 0.001. The size of the squares represents the weight of each study in the meta-analysis. The diamond at the bottom of each forest plot represents the combined effect size and its confidence interval from the multi-ancestry meta-analysis. The SNPs shown are rs644592 (RHCE), rs67760360 (BCL2A1), rs6480404 (HK1) and rs5743618 (TLR1).\n",
      "2\n",
      "\n",
      "image 1/1 /Users/etowah/projects/PMC-figure-downloader/img/10786714_Fig1.jpg: 224x640 1 label, 49.6ms\n",
      "Speed: 1.0ms preprocess, 49.6ms inference, 0.4ms postprocess per image at shape (1, 3, 224, 640)\n",
      "Subfigure count mismatch: 1 detected, 2 expected\n",
      "2\n",
      "\n",
      "image 1/1 /Users/etowah/projects/PMC-figure-downloader/img/10786714_Fig2.jpg: 640x480 2 labels, 60.7ms\n",
      "Speed: 1.9ms preprocess, 60.7ms inference, 0.4ms postprocess per image at shape (1, 3, 640, 480)\n",
      "3\n",
      "\n",
      "image 1/1 /Users/etowah/projects/PMC-figure-downloader/img/10786714_Fig3.jpg: 544x640 4 labels, 72.5ms\n",
      "Speed: 1.9ms preprocess, 72.5ms inference, 0.4ms postprocess per image at shape (1, 3, 544, 640)\n",
      "Subfigure count mismatch: 4 detected, 3 expected\n",
      "1\n",
      "We visualize IBD segments 12–16 cM long (for IBD sharing in other length classes see Extended Data Fig. 3). We applied ancIBD to identify IBD segments between all pairs of 304 West Eurasian ancient individuals (all previously published data; Supplementary Table 3) organized into 24 archaeological groups. The number in the parenthesis indicates the sample size for each archaeological group. For each pair of groups, we plot the fraction of all possible pairs of individuals that share at least one IBD 12–16 cM long, which we obtained by dividing the total number of pairs that share such IBD segments by the total number of all possible pairs: between two different groups of n1 and n2 individuals, one has n1n2 pairs, while within a group (on the diagonal in the figure) of size n one has n(n − 1)/2 pairs. LN, Late Neolithic; BAC, Battle Axe Culture; C, Chalcolithic; TRB, Trichterbecherkultur (Funnelbeaker culture); GAC, Globular Amphora Culture.\n",
      "4\n",
      "\n",
      "image 1/1 /Users/etowah/projects/PMC-figure-downloader/img/10786714_Fig5.jpg: 512x640 4 labels, 79.0ms\n",
      "Speed: 1.9ms preprocess, 79.0ms inference, 0.4ms postprocess per image at shape (1, 3, 512, 640)\n",
      "4\n",
      "\n",
      "image 1/1 /Users/etowah/projects/PMC-figure-downloader/img/10786714_Fig6.jpg: 640x576 (no detections), 78.3ms\n",
      "Speed: 2.2ms preprocess, 78.3ms inference, 0.3ms postprocess per image at shape (1, 3, 640, 576)\n",
      "Subfigure count mismatch: 0 detected, 4 expected\n",
      "1\n",
      "We applied both methods with their default settings to genotype data imputed after downsampling to various coverages. For each coverage, we report the average precision and recall of each length bin across 50 independent replicates. The error bar represents ± SE of the estimated precision and recall. Each row represents a length bin and each column represents one input data type (either WGS data or 1240k data). Note that the y axis ranges are different for different rows.\n",
      "1\n",
      "As in Fig. 4, but for shared IBD [8 − 12 cM], [12 − 16 cM], [16 − 20 cM], > 20 cM long. We used ancIBD to infer IBD segments between all pairs of groups and visualize the fraction of pairs that share at least one IBD for each pair of populations and for the four different IBD length bins.\n",
      "1\n",
      "We downsampled all individuals from a previously published English Neolithic pedigree26 with coverage at least 1x both to 1x and 0.75x. For each coverage, we downsampled 10 times, each with different random seeds, to create 10 replicates. Therefore, not all dots are independent pairs of relatives; they may be the same pair downsampled with different random seeds. The relationship annotations are obtained from Supp. Table 5 of ref. 26. All relatives more distant than 3rd degree are depicted as hollow dots.\n",
      "1\n",
      "To benchmark runtimes, we applied ancIBD on empirical ancient DNA data in .hdf5 format imputed at 1240k sites. We used the imputed hdf5 file from the Eurasian application (Fig. 3), choosing samples and pairs at random. Left: For each sample pair, all autosomes are screened for IBD. In one experiment all pairs of samples were run independently, leading to a linear dependency on pair number, as expected. In a second experiment, all samples were loaded into memory and then each sample pair was screened for IBD. The apparent sub-linear behaviour is due to the fact that loading n samples scales slower than the actual runtime of n(n − 1)/2 sample pairs. Right: We depict the runtimes normalized per sample pair when screening all pairs of sample batches of various sizes for IBD. We visualize the loading time (the time it takes to load the hdf5 genotype data into memory), the preprocessing time (including preparing the transition and emission matrix), as well as the runtime of screening for IBD that includes the forward-backward algorithm as well as postprocessing. Due to the decrease in the impact of the time to load the data, which scales linearly with batch size while the number of sample pair scales quadratically, we observe substantially increased runtimes per pair.\n",
      "6\n",
      "\n",
      "image 1/1 /Users/etowah/projects/PMC-figure-downloader/img/10703699_Fig1.jpg: 576x640 6 labels, 98.4ms\n",
      "Speed: 2.6ms preprocess, 98.4ms inference, 0.6ms postprocess per image at shape (1, 3, 576, 640)\n",
      "9\n",
      "\n",
      "image 1/1 /Users/etowah/projects/PMC-figure-downloader/img/10703699_Fig2.jpg: 640x544 9 labels, 69.3ms\n",
      "Speed: 2.0ms preprocess, 69.3ms inference, 0.4ms postprocess per image at shape (1, 3, 640, 544)\n",
      "8\n",
      "\n",
      "image 1/1 /Users/etowah/projects/PMC-figure-downloader/img/10703699_Fig3.jpg: 640x544 8 labels, 91.3ms\n",
      "Speed: 2.2ms preprocess, 91.3ms inference, 0.5ms postprocess per image at shape (1, 3, 640, 544)\n",
      "8\n",
      "\n",
      "image 1/1 /Users/etowah/projects/PMC-figure-downloader/img/10703699_Fig4.jpg: 640x512 8 labels, 75.7ms\n",
      "Speed: 2.4ms preprocess, 75.7ms inference, 0.5ms postprocess per image at shape (1, 3, 640, 512)\n",
      "12\n",
      "\n",
      "image 1/1 /Users/etowah/projects/PMC-figure-downloader/img/10703699_Fig5.jpg: 640x512 12 labels, 80.2ms\n",
      "Speed: 2.1ms preprocess, 80.2ms inference, 0.4ms postprocess per image at shape (1, 3, 640, 512)\n",
      "7\n",
      "\n",
      "image 1/1 /Users/etowah/projects/PMC-figure-downloader/img/10703699_Fig6.jpg: 640x512 9 labels, 99.8ms\n",
      "Speed: 1.9ms preprocess, 99.8ms inference, 1.0ms postprocess per image at shape (1, 3, 640, 512)\n",
      "11\n",
      "\n",
      "image 1/1 /Users/etowah/projects/PMC-figure-downloader/img/10703699_Fig7.jpg: 640x544 11 labels, 83.0ms\n",
      "Speed: 2.2ms preprocess, 83.0ms inference, 1.0ms postprocess per image at shape (1, 3, 640, 544)\n",
      "5\n",
      "\n",
      "image 1/1 /Users/etowah/projects/PMC-figure-downloader/img/10703699_Fig8.jpg: 544x640 5 labels, 72.4ms\n",
      "Speed: 1.9ms preprocess, 72.4ms inference, 0.8ms postprocess per image at shape (1, 3, 544, 640)\n",
      "4\n",
      "\n",
      "image 1/1 /Users/etowah/projects/PMC-figure-downloader/img/10703699_Fig9.jpg: 640x544 4 labels, 75.0ms\n",
      "Speed: 2.2ms preprocess, 75.0ms inference, 0.4ms postprocess per image at shape (1, 3, 640, 544)\n",
      "5\n",
      "\n",
      "image 1/1 /Users/etowah/projects/PMC-figure-downloader/img/10703699_Fig10.jpg: 640x384 5 labels, 93.6ms\n",
      "Speed: 1.6ms preprocess, 93.6ms inference, 0.6ms postprocess per image at shape (1, 3, 640, 384)\n",
      "3\n",
      "\n",
      "image 1/1 /Users/etowah/projects/PMC-figure-downloader/img/10703699_Fig11.jpg: 640x512 4 labels, 84.1ms\n",
      "Speed: 2.1ms preprocess, 84.1ms inference, 0.7ms postprocess per image at shape (1, 3, 640, 512)\n",
      "3\n",
      "\n",
      "image 1/1 /Users/etowah/projects/PMC-figure-downloader/img/10703699_Fig12.jpg: 640x544 3 labels, 76.8ms\n",
      "Speed: 2.1ms preprocess, 76.8ms inference, 0.5ms postprocess per image at shape (1, 3, 640, 544)\n",
      "3\n",
      "\n",
      "image 1/1 /Users/etowah/projects/PMC-figure-downloader/img/10703699_Fig13.jpg: 608x640 4 labels, 87.5ms\n",
      "Speed: 2.5ms preprocess, 87.5ms inference, 0.7ms postprocess per image at shape (1, 3, 608, 640)\n",
      "Subfigure count mismatch: 4 detected, 3 expected\n",
      "6\n",
      "\n",
      "image 1/1 /Users/etowah/projects/PMC-figure-downloader/img/10703699_Fig14.jpg: 640x544 6 labels, 75.6ms\n",
      "Speed: 2.1ms preprocess, 75.6ms inference, 0.4ms postprocess per image at shape (1, 3, 640, 544)\n",
      "8\n",
      "\n",
      "image 1/1 /Users/etowah/projects/PMC-figure-downloader/img/10703699_Fig15.jpg: 640x480 10 labels, 120.9ms\n",
      "Speed: 1.8ms preprocess, 120.9ms inference, 1.8ms postprocess per image at shape (1, 3, 640, 480)\n",
      "Subfigure count mismatch: 10 detected, 8 expected\n",
      "7\n",
      "\n",
      "image 1/1 /Users/etowah/projects/PMC-figure-downloader/img/10703699_Fig16.jpg: 640x640 9 labels, 103.2ms\n",
      "Speed: 2.4ms preprocess, 103.2ms inference, 0.6ms postprocess per image at shape (1, 3, 640, 640)\n",
      "Subfigure count mismatch: 9 detected, 7 expected\n",
      "5\n",
      "\n",
      "image 1/1 /Users/etowah/projects/PMC-figure-downloader/img/10703693_Fig1.jpg: 640x608 5 labels, 116.3ms\n",
      "Speed: 2.6ms preprocess, 116.3ms inference, 0.7ms postprocess per image at shape (1, 3, 640, 608)\n",
      "2\n",
      "\n",
      "image 1/1 /Users/etowah/projects/PMC-figure-downloader/img/10703693_Fig2.jpg: 224x640 2 labels, 63.9ms\n",
      "Speed: 1.0ms preprocess, 63.9ms inference, 0.5ms postprocess per image at shape (1, 3, 224, 640)\n",
      "1\n",
      "Manhattan plot showing logistic regression GWAS results (−log10(P) versus chromosomal position) from 16,306 cases and 159,913 controls. The horizontal red line corresponds to a P value of 5 × 10−8. Named loci have unconditional P values of <5 × 10−8. Loci are named by the nearest gene or plausible candidate. The TERT and TCL1A loci are offscale, and their P values are indicated on the plot. Detailed data for named loci are in Supplementary Table 9. Several high-effect, rare variants were deemed to require further confirmation and were not considered further (indicated in Supplementary Table 9).\n",
      "3\n",
      "\n",
      "image 1/1 /Users/etowah/projects/PMC-figure-downloader/img/10703693_Fig4.jpg: 640x608 3 labels, 87.6ms\n",
      "Speed: 2.3ms preprocess, 87.6ms inference, 0.5ms postprocess per image at shape (1, 3, 640, 608)\n",
      "4\n",
      "\n",
      "image 1/1 /Users/etowah/projects/PMC-figure-downloader/img/10703693_Fig5.jpg: 480x640 5 labels, 72.3ms\n",
      "Speed: 1.8ms preprocess, 72.3ms inference, 0.4ms postprocess per image at shape (1, 3, 480, 640)\n",
      "7\n",
      "\n",
      "image 1/1 /Users/etowah/projects/PMC-figure-downloader/img/10703693_Fig6.jpg: 448x640 8 labels, 67.3ms\n",
      "Speed: 1.5ms preprocess, 67.3ms inference, 0.9ms postprocess per image at shape (1, 3, 448, 640)\n",
      "3\n",
      "\n",
      "image 1/1 /Users/etowah/projects/PMC-figure-downloader/img/10703693_Fig7.jpg: 384x640 2 labels, 61.5ms\n",
      "Speed: 1.5ms preprocess, 61.5ms inference, 0.4ms postprocess per image at shape (1, 3, 384, 640)\n",
      "Subfigure count mismatch: 2 detected, 3 expected\n",
      "2\n",
      "\n",
      "image 1/1 /Users/etowah/projects/PMC-figure-downloader/img/10703693_Fig8.jpg: 224x640 2 labels, 52.7ms\n",
      "Speed: 0.8ms preprocess, 52.7ms inference, 1.1ms postprocess per image at shape (1, 3, 224, 640)\n",
      "3\n",
      "\n",
      "image 1/1 /Users/etowah/projects/PMC-figure-downloader/img/10703693_Fig9.jpg: 640x544 4 labels, 103.2ms\n",
      "Speed: 2.3ms preprocess, 103.2ms inference, 0.8ms postprocess per image at shape (1, 3, 640, 544)\n",
      "Subfigure count mismatch: 4 detected, 3 expected\n",
      "1\n",
      "Data are a meta-analysis of ISL and UKB. GWAS variants were included if they were significantly associated with barcode-CH or CPLDneg-CH. The plotted points are association effect estimates (loge odds ratio) and 95%CI from logistic regression association testing for variants in barcode-CH (16,306 cases, 159,913 controls) and CPLDneg-CH (11,692 cases, 151,277 controls) respectively. The fitted inverse variance weighted linear regression, fixed through the origin, is shown as a red dotted line. Variants that were newly discovered in the CPLDneg-CH GWAS are colored green. Labeled loci are discussed in the text.\n",
      "1\n",
      "GWAS variants were included if they were significantly associated with barcode-CH or any of the CPLD-CH types. The x-axes show the effects (loge odds ratio) and 95%CI (horizontal lines) for each variant in barcode-CH, determined by logistic regression. The y-axes show the corresponding effects and 95%CI (vertical lines) for each variant in the different types of CPLD-CH, as indicated above each panel. The dotted line shows the position of the diagonal. Gray lines indicate the position of no effect. Detailed data including case and control numbers are in Supplementary Table 12. The chr14:TCL1A rs2887399_T allele was protective against barcode-CH, TET2-CH and ASXL1-CH whilst the same allele increased risk of DNMT3A-CH, in line with previous reports. The chr14:TCL1A variant is indicated in the DNMT3A-CH and ASXL1-CH panels to illustrate the reversal of effect. Similarly, the chr6:CD164 chr6:CD164 rs3056655_A allele increased risk of barcode-CH and DNMT3A-CH but decreased risk of TET2-CH13,14. The latter result was seen only in UKB, whereas ISL data could not confirm it. The chr3:SMC4 rs201009932 variant had no discernible effect on ASXL1-CH while it had a pronounced effect on JAK2-CH. chr3:THRB had no apparent effect on DNMT3A-CH and chr5:TERT\n",
      "rs7705526 had no effect on PPM1D-CH. Other variants showed prominent effects only in specific CPLD-CH types: chr12:SOX5 and chr14:DLK1 had no evident effects outside of barcode-CH, while chr13:KLF12 had no apparent effect outside of PPM1D-CH. The chr9:JAK2 rs16922785_G allele (indicated in the JAK2-CH panel) only conferred CH risk in the context of the JAK2 Val617Phe somatic mutation and was preferentially linked to it in cis, as has been noted previously for the 46/1 JAK2 haplotype and MPN risk104. rs16922785 is in moderate LD with the 46/1 haplotype (r2 = 0.68) and had a somewhat stronger association with JAK2-CH than the 46/1 haplotype tagger rs12343867_C (P = 1.60 × 10−9\n",
      "vs 1.04 × 10−7).\n",
      "2\n",
      "\n",
      "image 1/1 /Users/etowah/projects/PMC-figure-downloader/img/10703693_Fig12.jpg: 224x640 3 labels, 75.3ms\n",
      "Speed: 1.0ms preprocess, 75.3ms inference, 0.4ms postprocess per image at shape (1, 3, 224, 640)\n",
      "1\n",
      "A GWAS was conducted on a sub-sample of UKB from which proven CH cases had been removed (n = 111,523). The effects of LTL GWAS variants were compared between the two samples: LTL effect on the x-axis and the no CH LTL effect on the y-axis. The plotted points are association the effect estimates from linear regression and the bars indicate 95%CI. The red dotted line indicates the fitted inverse variance weighted (IVW) regression. Gray lines indicate the position of no effect.\n",
      "2\n",
      "\n",
      "image 1/1 /Users/etowah/projects/PMC-figure-downloader/img/10703693_Fig14.jpg: 352x640 2 labels, 78.3ms\n",
      "Speed: 1.6ms preprocess, 78.3ms inference, 1.0ms postprocess per image at shape (1, 3, 352, 640)\n",
      "2\n",
      "\n",
      "image 1/1 /Users/etowah/projects/PMC-figure-downloader/img/10703692_Fig1.jpg: 640x480 2 labels, 78.9ms\n",
      "Speed: 2.4ms preprocess, 78.9ms inference, 0.6ms postprocess per image at shape (1, 3, 640, 480)\n",
      "3\n",
      "\n",
      "image 1/1 /Users/etowah/projects/PMC-figure-downloader/img/10703692_Fig2.jpg: 480x640 3 labels, 97.9ms\n",
      "Speed: 2.2ms preprocess, 97.9ms inference, 0.5ms postprocess per image at shape (1, 3, 480, 640)\n",
      "5\n",
      "\n",
      "image 1/1 /Users/etowah/projects/PMC-figure-downloader/img/10703692_Fig3.jpg: 576x640 5 labels, 83.4ms\n",
      "Speed: 2.3ms preprocess, 83.4ms inference, 0.4ms postprocess per image at shape (1, 3, 576, 640)\n",
      "3\n",
      "\n",
      "image 1/1 /Users/etowah/projects/PMC-figure-downloader/img/10703692_Fig4.jpg: 544x640 3 labels, 108.6ms\n",
      "Speed: 2.2ms preprocess, 108.6ms inference, 0.8ms postprocess per image at shape (1, 3, 544, 640)\n",
      "6\n",
      "\n",
      "image 1/1 /Users/etowah/projects/PMC-figure-downloader/img/10703692_Fig5.jpg: 640x512 6 labels, 97.8ms\n",
      "Speed: 2.1ms preprocess, 97.8ms inference, 1.0ms postprocess per image at shape (1, 3, 640, 512)\n",
      "4\n",
      "\n",
      "image 1/1 /Users/etowah/projects/PMC-figure-downloader/img/10703692_Fig6.jpg: 640x640 5 labels, 80.2ms\n",
      "Speed: 2.5ms preprocess, 80.2ms inference, 0.6ms postprocess per image at shape (1, 3, 640, 640)\n",
      "2\n",
      "\n",
      "image 1/1 /Users/etowah/projects/PMC-figure-downloader/img/10703692_Fig7.jpg: 384x640 2 labels, 67.4ms\n",
      "Speed: 1.6ms preprocess, 67.4ms inference, 0.8ms postprocess per image at shape (1, 3, 384, 640)\n",
      "2\n",
      "\n",
      "image 1/1 /Users/etowah/projects/PMC-figure-downloader/img/10703692_Fig8.jpg: 640x288 3 labels, 85.9ms\n",
      "Speed: 3.5ms preprocess, 85.9ms inference, 0.4ms postprocess per image at shape (1, 3, 640, 288)\n",
      "Subfigure count mismatch: 3 detected, 2 expected\n",
      "4\n",
      "\n",
      "image 1/1 /Users/etowah/projects/PMC-figure-downloader/img/10703692_Fig9.jpg: 640x544 4 labels, 87.6ms\n",
      "Speed: 2.3ms preprocess, 87.6ms inference, 0.5ms postprocess per image at shape (1, 3, 640, 544)\n",
      "1\n",
      "(a) Number of genes that pass TWAS (blue) and isoTWAS (red) CV R2 cutoffs to be available for testing in the trait-mapping step (X axis) (b) Percent difference in CV R2 (X axis) of prediction of isoform expression models using multivariate models versus univariate models. The label shows the proportion of isoforms with improved performance using multivariate models. All boxplots represent the median, 25% and 75% quantiles, and whiskers correspond to the 10% and 90% quantiles.\n",
      "2\n",
      "\n",
      "image 1/1 /Users/etowah/projects/PMC-figure-downloader/img/10703692_Fig11.jpg: 544x640 2 labels, 87.5ms\n",
      "Speed: 1.8ms preprocess, 87.5ms inference, 1.3ms postprocess per image at shape (1, 3, 544, 640)\n",
      "8\n",
      "\n",
      "image 1/1 /Users/etowah/projects/PMC-figure-downloader/img/10703692_Fig12.jpg: 640x512 8 labels, 173.7ms\n",
      "Speed: 2.6ms preprocess, 173.7ms inference, 1.4ms postprocess per image at shape (1, 3, 640, 512)\n",
      "3\n",
      "\n",
      "image 1/1 /Users/etowah/projects/PMC-figure-downloader/img/10703692_Fig13.jpg: 640x544 4 labels, 89.0ms\n",
      "Speed: 2.1ms preprocess, 89.0ms inference, 0.5ms postprocess per image at shape (1, 3, 640, 544)\n",
      "Subfigure count mismatch: 4 detected, 3 expected\n",
      "4\n",
      "\n",
      "image 1/1 /Users/etowah/projects/PMC-figure-downloader/img/10703692_Fig14.jpg: 640x544 4 labels, 75.0ms\n",
      "Speed: 2.0ms preprocess, 75.0ms inference, 0.5ms postprocess per image at shape (1, 3, 640, 544)\n",
      "1\n",
      "QQ-plots of gene-level P values using TWAS (red) and isoTWAS (blue) across 15 traits.\n",
      "4\n",
      "\n",
      "image 1/1 /Users/etowah/projects/PMC-figure-downloader/img/10703692_Fig16.jpg: 640x416 7 labels, 75.6ms\n",
      "Speed: 1.5ms preprocess, 75.6ms inference, 0.5ms postprocess per image at shape (1, 3, 640, 416)\n",
      "Subfigure count mismatch: 6 detected, 4 expected\n",
      "1\n",
      "From top to bottom, 2 loci were identified in EAS ancestry, 1 for AMR, 2 for AFR, and 22 for EUR (red dots). Nearby genes are shown for orientation. P values were calculated with bidirectional Wald’s test. The field standard GWS threshold of P < 5 × 10−8 (horizontal red line) was used to determine significant associations. Other colors indicate different chromosomes. Chr, chromosome.\n",
      "1\n",
      "Comparison of genetic correlations between CanUD and cannabis use18. Left axis depicts the range of rG between 1 and −1. Red bars and blue bars depict the rG point estimate per trait for CanUD and cannabis use, respectively. Black error bars represent the standard error. Right axis displays the absolute magnitude of the Z statistic for the difference in rG between CanUD and cannabis use (gold line). SNP-based heritability for each comparison trait is included in the table. EA, educational attainment; ReExp, re-experience; AUDIT-C, Alcohol Use Disorders Identification Test-Consumption; TDI, Townsend deprivation index.\n",
      "4\n",
      "\n",
      "image 1/1 /Users/etowah/projects/PMC-figure-downloader/img/10703690_Fig3.jpg: 480x640 4 labels, 76.3ms\n",
      "Speed: 1.7ms preprocess, 76.3ms inference, 0.4ms postprocess per image at shape (1, 3, 480, 640)\n",
      "1\n",
      "gSEM was used to cluster 14 traits correlated with CanUD. Exploratory and confirmatory factor analysis indicated four factors fit the data best. Factors fit mostly into categories that we designated as functional impairment (factor 1), impulsivity and risk taking (factor 2), psychopathology (factor 3) and substance dependence (factor 4). CanUD fit best (and strongly) in the substance dependence cluster. FTND fit into factors for functional impairment and substance dependence. Suicide attempts fit into functional impairment, impulsivity and risk taking, and psychopathology. Numbers outside the parenthesis are correlation coefficients between factors. Numbers inside parenthesis are stadard errors of means. Smoke init, initiation of regular smoking; GAD, generalized anxiety disorder; OUD, opioid use disorder.\n",
      "5\n",
      "\n",
      "image 1/1 /Users/etowah/projects/PMC-figure-downloader/img/10703688_Fig1.jpg: 416x640 5 labels, 80.6ms\n",
      "Speed: 1.9ms preprocess, 80.6ms inference, 1.0ms postprocess per image at shape (1, 3, 416, 640)\n",
      "8\n",
      "\n",
      "image 1/1 /Users/etowah/projects/PMC-figure-downloader/img/10703688_Fig2.jpg: 608x640 8 labels, 76.9ms\n",
      "Speed: 2.3ms preprocess, 76.9ms inference, 1.3ms postprocess per image at shape (1, 3, 608, 640)\n",
      "11\n",
      "\n",
      "image 1/1 /Users/etowah/projects/PMC-figure-downloader/img/10703688_Fig3.jpg: 480x640 11 labels, 93.0ms\n",
      "Speed: 1.9ms preprocess, 93.0ms inference, 2.1ms postprocess per image at shape (1, 3, 480, 640)\n",
      "8\n",
      "\n",
      "image 1/1 /Users/etowah/projects/PMC-figure-downloader/img/10703688_Fig4.jpg: 480x640 8 labels, 71.7ms\n",
      "Speed: 2.0ms preprocess, 71.7ms inference, 0.5ms postprocess per image at shape (1, 3, 480, 640)\n",
      "4\n",
      "\n",
      "image 1/1 /Users/etowah/projects/PMC-figure-downloader/img/10703688_Fig5.jpg: 416x640 4 labels, 70.7ms\n",
      "Speed: 1.5ms preprocess, 70.7ms inference, 0.5ms postprocess per image at shape (1, 3, 416, 640)\n",
      "4\n",
      "\n",
      "image 1/1 /Users/etowah/projects/PMC-figure-downloader/img/10703688_Fig6.jpg: 640x640 6 labels, 85.4ms\n",
      "Speed: 2.5ms preprocess, 85.4ms inference, 1.3ms postprocess per image at shape (1, 3, 640, 640)\n",
      "Subfigure count mismatch: 6 detected, 4 expected\n",
      "4\n",
      "\n",
      "image 1/1 /Users/etowah/projects/PMC-figure-downloader/img/10703688_Fig7.jpg: 384x640 4 labels, 62.3ms\n",
      "Speed: 1.5ms preprocess, 62.3ms inference, 0.5ms postprocess per image at shape (1, 3, 384, 640)\n",
      "7\n",
      "\n",
      "image 1/1 /Users/etowah/projects/PMC-figure-downloader/img/10703688_Fig8.jpg: 640x480 8 labels, 72.1ms\n",
      "Speed: 1.9ms preprocess, 72.1ms inference, 0.4ms postprocess per image at shape (1, 3, 640, 480)\n",
      "6\n",
      "\n",
      "image 1/1 /Users/etowah/projects/PMC-figure-downloader/img/10703688_Fig9.jpg: 640x608 6 labels, 81.1ms\n",
      "Speed: 2.3ms preprocess, 81.1ms inference, 0.5ms postprocess per image at shape (1, 3, 640, 608)\n",
      "5\n",
      "\n",
      "image 1/1 /Users/etowah/projects/PMC-figure-downloader/img/10703688_Fig10.jpg: 416x640 5 labels, 61.5ms\n",
      "Speed: 1.7ms preprocess, 61.5ms inference, 0.4ms postprocess per image at shape (1, 3, 416, 640)\n",
      "5\n",
      "\n",
      "image 1/1 /Users/etowah/projects/PMC-figure-downloader/img/10703688_Fig11.jpg: 640x320 5 labels, 54.5ms\n",
      "Speed: 1.4ms preprocess, 54.5ms inference, 0.4ms postprocess per image at shape (1, 3, 640, 320)\n",
      "3\n",
      "\n",
      "image 1/1 /Users/etowah/projects/PMC-figure-downloader/img/10703688_Fig12.jpg: 640x448 3 labels, 73.0ms\n",
      "Speed: 1.8ms preprocess, 73.0ms inference, 0.5ms postprocess per image at shape (1, 3, 640, 448)\n",
      "6\n",
      "\n",
      "image 1/1 /Users/etowah/projects/PMC-figure-downloader/img/10703688_Fig13.jpg: 608x640 6 labels, 76.2ms\n",
      "Speed: 2.2ms preprocess, 76.2ms inference, 0.4ms postprocess per image at shape (1, 3, 608, 640)\n",
      "3\n",
      "\n",
      "image 1/1 /Users/etowah/projects/PMC-figure-downloader/img/10703688_Fig14.jpg: 480x640 3 labels, 71.4ms\n",
      "Speed: 1.7ms preprocess, 71.4ms inference, 0.4ms postprocess per image at shape (1, 3, 480, 640)\n",
      "7\n",
      "\n",
      "image 1/1 /Users/etowah/projects/PMC-figure-downloader/img/10703688_Fig15.jpg: 640x448 8 labels, 65.5ms\n",
      "Speed: 2.0ms preprocess, 65.5ms inference, 0.9ms postprocess per image at shape (1, 3, 640, 448)\n",
      "Subfigure count mismatch: 8 detected, 7 expected\n",
      "1\n",
      "AutoComplete defines a feed-forward encoder-decoder architecture h trained using copy-masking, a procedure that simulates realistic missingness patterns that the model uses to impute missing values. AutoComplete minimizes the loss function \\documentclass[12pt]{minimal}\n",
      "\t\t\t\t\\usepackage{amsmath}\n",
      "\t\t\t\t\\usepackage{wasysym} \n",
      "\t\t\t\t\\usepackage{amsfonts} \n",
      "\t\t\t\t\\usepackage{amssymb} \n",
      "\t\t\t\t\\usepackage{amsbsy}\n",
      "\t\t\t\t\\usepackage{mathrsfs}\n",
      "\t\t\t\t\\usepackage{upgreek}\n",
      "\t\t\t\t\\setlength{\\oddsidemargin}{-69pt}\n",
      "\t\t\t\t\\begin{document}$${\\mathscr{L}}$$\\end{document}L that is defined over the observed and masked values. AutoComplete supports the imputation of continuous and binary features.\n",
      "3\n",
      "\n",
      "image 1/1 /Users/etowah/projects/PMC-figure-downloader/img/10703681_Fig2.jpg: 640x576 3 labels, 78.0ms\n",
      "Speed: 2.2ms preprocess, 78.0ms inference, 0.4ms postprocess per image at shape (1, 3, 640, 576)\n",
      "1\n",
      "Results of GWAS of the observed portions of bilirubin, LifetimeMDD and cannabis ever taken in the UKBB (indicated as Obs), where each phenotype had 21%, 80% and 67% missingness respectively. GWAS was then performed for all individuals in the dataset after using AutoComplete to impute the missing entries for each phenotype (indicated as ImpAll). The significance threshold of \\documentclass[12pt]{minimal}\n",
      "\t\t\t\t\\usepackage{amsmath}\n",
      "\t\t\t\t\\usepackage{wasysym} \n",
      "\t\t\t\t\\usepackage{amsfonts} \n",
      "\t\t\t\t\\usepackage{amssymb} \n",
      "\t\t\t\t\\usepackage{amsbsy}\n",
      "\t\t\t\t\\usepackage{mathrsfs}\n",
      "\t\t\t\t\\usepackage{upgreek}\n",
      "\t\t\t\t\\setlength{\\oddsidemargin}{-69pt}\n",
      "\t\t\t\t\\begin{document}$$P < 5\\times 1{0}^{-8}$$\\end{document}P<5×10−8 is indicated by a red line, and SNPs passing the threshold are highlighted in red.\n",
      "2\n",
      "\n",
      "image 1/1 /Users/etowah/projects/PMC-figure-downloader/img/10703681_Fig4.jpg: 544x640 3 labels, 71.9ms\n",
      "Speed: 2.1ms preprocess, 71.9ms inference, 0.4ms postprocess per image at shape (1, 3, 544, 640)\n",
      "12\n",
      "\n",
      "image 1/1 /Users/etowah/projects/PMC-figure-downloader/img/10703679_Fig1.jpg: 640x480 12 labels, 70.0ms\n",
      "Speed: 1.9ms preprocess, 70.0ms inference, 0.4ms postprocess per image at shape (1, 3, 640, 480)\n",
      "11\n",
      "\n",
      "image 1/1 /Users/etowah/projects/PMC-figure-downloader/img/10703679_Fig2.jpg: 640x448 10 labels, 73.5ms\n",
      "Speed: 1.9ms preprocess, 73.5ms inference, 0.4ms postprocess per image at shape (1, 3, 640, 448)\n",
      "Subfigure count mismatch: 10 detected, 11 expected\n",
      "8\n",
      "\n",
      "image 1/1 /Users/etowah/projects/PMC-figure-downloader/img/10703679_Fig3.jpg: 640x480 8 labels, 74.6ms\n",
      "Speed: 1.9ms preprocess, 74.6ms inference, 0.5ms postprocess per image at shape (1, 3, 640, 480)\n",
      "10\n",
      "\n",
      "image 1/1 /Users/etowah/projects/PMC-figure-downloader/img/10703679_Fig4.jpg: 640x512 10 labels, 70.4ms\n",
      "Speed: 1.8ms preprocess, 70.4ms inference, 0.8ms postprocess per image at shape (1, 3, 640, 512)\n",
      "6\n",
      "\n",
      "image 1/1 /Users/etowah/projects/PMC-figure-downloader/img/10703679_Fig5.jpg: 640x640 6 labels, 92.9ms\n",
      "Speed: 2.4ms preprocess, 92.9ms inference, 0.6ms postprocess per image at shape (1, 3, 640, 640)\n",
      "15\n",
      "\n",
      "image 1/1 /Users/etowah/projects/PMC-figure-downloader/img/10703679_Fig6.jpg: 544x640 15 labels, 70.2ms\n",
      "Speed: 2.0ms preprocess, 70.2ms inference, 0.5ms postprocess per image at shape (1, 3, 544, 640)\n",
      "9\n",
      "\n",
      "image 1/1 /Users/etowah/projects/PMC-figure-downloader/img/10703679_Fig7.jpg: 544x640 10 labels, 77.4ms\n",
      "Speed: 2.0ms preprocess, 77.4ms inference, 0.9ms postprocess per image at shape (1, 3, 544, 640)\n",
      "Subfigure count mismatch: 10 detected, 9 expected\n",
      "8\n",
      "\n",
      "image 1/1 /Users/etowah/projects/PMC-figure-downloader/img/10703679_Fig8.jpg: 640x512 9 labels, 82.4ms\n",
      "Speed: 2.0ms preprocess, 82.4ms inference, 1.8ms postprocess per image at shape (1, 3, 640, 512)\n",
      "4\n",
      "\n",
      "image 1/1 /Users/etowah/projects/PMC-figure-downloader/img/10703679_Fig9.jpg: 640x640 6 labels, 77.3ms\n",
      "Speed: 2.2ms preprocess, 77.3ms inference, 0.5ms postprocess per image at shape (1, 3, 640, 640)\n",
      "Subfigure count mismatch: 5 detected, 4 expected\n",
      "7\n",
      "\n",
      "image 1/1 /Users/etowah/projects/PMC-figure-downloader/img/10703679_Fig10.jpg: 640x544 10 labels, 79.6ms\n",
      "Speed: 2.2ms preprocess, 79.6ms inference, 0.5ms postprocess per image at shape (1, 3, 640, 544)\n",
      "Subfigure count mismatch: 9 detected, 7 expected\n",
      "4\n",
      "\n",
      "image 1/1 /Users/etowah/projects/PMC-figure-downloader/img/10703679_Fig11.jpg: 512x640 4 labels, 71.8ms\n",
      "Speed: 2.0ms preprocess, 71.8ms inference, 0.4ms postprocess per image at shape (1, 3, 512, 640)\n",
      "4\n",
      "\n",
      "image 1/1 /Users/etowah/projects/PMC-figure-downloader/img/10703679_Fig12.jpg: 640x576 5 labels, 94.2ms\n",
      "Speed: 2.3ms preprocess, 94.2ms inference, 0.5ms postprocess per image at shape (1, 3, 640, 576)\n",
      "4\n",
      "\n",
      "image 1/1 /Users/etowah/projects/PMC-figure-downloader/img/10703679_Fig13.jpg: 640x480 6 labels, 77.6ms\n",
      "Speed: 2.0ms preprocess, 77.6ms inference, 0.5ms postprocess per image at shape (1, 3, 640, 480)\n",
      "Subfigure count mismatch: 6 detected, 4 expected\n",
      "10\n",
      "\n",
      "image 1/1 /Users/etowah/projects/PMC-figure-downloader/img/10703679_Fig14.jpg: 576x640 10 labels, 80.4ms\n",
      "Speed: 2.1ms preprocess, 80.4ms inference, 1.0ms postprocess per image at shape (1, 3, 576, 640)\n",
      "6\n",
      "\n",
      "image 1/1 /Users/etowah/projects/PMC-figure-downloader/img/10703679_Fig15.jpg: 640x512 6 labels, 75.5ms\n",
      "Speed: 2.4ms preprocess, 75.5ms inference, 0.4ms postprocess per image at shape (1, 3, 640, 512)\n",
      "1\n",
      "The 1000 Genomes Project reference panel (1KG Phase 3). BBJ1-180K consists of approximately 180,000 individuals from BBJ1. BBJ1-12K consists of approximately 12,000 individuals from BBJ1. BBJ2-42K consists of approximately 42,000 individuals from BBJ2. TMM-50K consists of approximately 50,000 individuals from Tohoku Medical Megabank.\n",
      "1\n",
      "Two-sided P values were derived from the cross-ancestry meta-analysis of 52,032 cases and 905,344 controls of EAS or EUR ancestry. Meta-analysis was performed using the IVW method under the fixed-effect model. For variants above the top light-gray dashed line (−log10(P) > 20), values are rescaled. Lead variants are annotated with the nearest gene name(s). New loci are highlighted in red. Variants are plotted against GRCh37 (hg19). The bottom dark-gray dashed line indicates the genome-wide significance threshold (P < 5.0 × 10−8). Variants with −log10(P) < 2 were omitted.\n",
      "2\n",
      "\n",
      "image 1/1 /Users/etowah/projects/PMC-figure-downloader/img/10703676_Fig3.jpg: 384x640 2 labels, 59.9ms\n",
      "Speed: 1.6ms preprocess, 59.9ms inference, 0.4ms postprocess per image at shape (1, 3, 384, 640)\n",
      "2\n",
      "\n",
      "image 1/1 /Users/etowah/projects/PMC-figure-downloader/img/10703676_Fig4.jpg: 288x640 3 labels, 60.1ms\n",
      "Speed: 1.4ms preprocess, 60.1ms inference, 0.4ms postprocess per image at shape (1, 3, 288, 640)\n",
      "Subfigure count mismatch: 3 detected, 2 expected\n",
      "4\n",
      "\n",
      "image 1/1 /Users/etowah/projects/PMC-figure-downloader/img/10703676_Fig5.jpg: 576x640 4 labels, 95.0ms\n",
      "Speed: 2.2ms preprocess, 95.0ms inference, 0.4ms postprocess per image at shape (1, 3, 576, 640)\n",
      "3\n",
      "\n",
      "image 1/1 /Users/etowah/projects/PMC-figure-downloader/img/10703676_Fig6.jpg: 640x640 3 labels, 85.9ms\n",
      "Speed: 2.5ms preprocess, 85.9ms inference, 1.0ms postprocess per image at shape (1, 3, 640, 640)\n",
      "4\n",
      "\n",
      "image 1/1 /Users/etowah/projects/PMC-figure-downloader/img/10786727_Fig1.jpg: 608x640 4 labels, 67.3ms\n",
      "Speed: 2.4ms preprocess, 67.3ms inference, 1.7ms postprocess per image at shape (1, 3, 608, 640)\n",
      "8\n",
      "\n",
      "image 1/1 /Users/etowah/projects/PMC-figure-downloader/img/10786727_Fig2.jpg: 640x576 8 labels, 65.8ms\n",
      "Speed: 2.4ms preprocess, 65.8ms inference, 0.4ms postprocess per image at shape (1, 3, 640, 576)\n",
      "5\n",
      "\n",
      "image 1/1 /Users/etowah/projects/PMC-figure-downloader/img/10786727_Fig3.jpg: 640x576 5 labels, 70.8ms\n",
      "Speed: 2.4ms preprocess, 70.8ms inference, 0.8ms postprocess per image at shape (1, 3, 640, 576)\n",
      "7\n",
      "\n",
      "image 1/1 /Users/etowah/projects/PMC-figure-downloader/img/10786727_Fig4.jpg: 640x576 7 labels, 93.8ms\n",
      "Speed: 2.3ms preprocess, 93.8ms inference, 0.6ms postprocess per image at shape (1, 3, 640, 576)\n",
      "8\n",
      "\n",
      "image 1/1 /Users/etowah/projects/PMC-figure-downloader/img/10786727_Fig5.jpg: 640x576 8 labels, 82.7ms\n",
      "Speed: 2.4ms preprocess, 82.7ms inference, 0.8ms postprocess per image at shape (1, 3, 640, 576)\n",
      "6\n",
      "\n",
      "image 1/1 /Users/etowah/projects/PMC-figure-downloader/img/10786727_Fig6.jpg: 640x384 8 labels, 68.1ms\n",
      "Speed: 1.6ms preprocess, 68.1ms inference, 0.8ms postprocess per image at shape (1, 3, 640, 384)\n",
      "Subfigure count mismatch: 8 detected, 6 expected\n",
      "5\n",
      "\n",
      "image 1/1 /Users/etowah/projects/PMC-figure-downloader/img/10786727_Fig7.jpg: 640x544 5 labels, 133.0ms\n",
      "Speed: 4.1ms preprocess, 133.0ms inference, 0.5ms postprocess per image at shape (1, 3, 640, 544)\n",
      "10\n",
      "\n",
      "image 1/1 /Users/etowah/projects/PMC-figure-downloader/img/10786727_Fig8.jpg: 640x448 10 labels, 64.0ms\n",
      "Speed: 1.8ms preprocess, 64.0ms inference, 0.4ms postprocess per image at shape (1, 3, 640, 448)\n",
      "11\n",
      "\n",
      "image 1/1 /Users/etowah/projects/PMC-figure-downloader/img/10786727_Fig9.jpg: 640x448 11 labels, 75.5ms\n",
      "Speed: 1.9ms preprocess, 75.5ms inference, 0.4ms postprocess per image at shape (1, 3, 640, 448)\n",
      "14\n",
      "\n",
      "image 1/1 /Users/etowah/projects/PMC-figure-downloader/img/10786727_Fig10.jpg: 640x512 5 labels, 70.4ms\n",
      "Speed: 2.0ms preprocess, 70.4ms inference, 0.5ms postprocess per image at shape (1, 3, 640, 512)\n",
      "Subfigure count mismatch: 5 detected, 14 expected\n",
      "18\n",
      "\n",
      "image 1/1 /Users/etowah/projects/PMC-figure-downloader/img/10786727_Fig11.jpg: 640x512 10 labels, 67.6ms\n",
      "Speed: 1.9ms preprocess, 67.6ms inference, 0.4ms postprocess per image at shape (1, 3, 640, 512)\n",
      "Subfigure count mismatch: 10 detected, 18 expected\n",
      "14\n",
      "\n",
      "image 1/1 /Users/etowah/projects/PMC-figure-downloader/img/10786727_Fig12.jpg: 640x512 13 labels, 121.4ms\n",
      "Speed: 2.0ms preprocess, 121.4ms inference, 0.5ms postprocess per image at shape (1, 3, 640, 512)\n",
      "Subfigure count mismatch: 13 detected, 14 expected\n",
      "14\n",
      "\n",
      "image 1/1 /Users/etowah/projects/PMC-figure-downloader/img/10786727_Fig13.jpg: 640x640 8 labels, 98.4ms\n",
      "Speed: 2.3ms preprocess, 98.4ms inference, 0.4ms postprocess per image at shape (1, 3, 640, 640)\n",
      "Subfigure count mismatch: 8 detected, 14 expected\n",
      "10\n",
      "\n",
      "image 1/1 /Users/etowah/projects/PMC-figure-downloader/img/10786727_Fig14.jpg: 640x448 10 labels, 64.7ms\n",
      "Speed: 1.8ms preprocess, 64.7ms inference, 0.4ms postprocess per image at shape (1, 3, 640, 448)\n",
      "18\n",
      "\n",
      "image 1/1 /Users/etowah/projects/PMC-figure-downloader/img/10786727_Fig15.jpg: 640x512 4 labels, 70.8ms\n",
      "Speed: 2.0ms preprocess, 70.8ms inference, 0.4ms postprocess per image at shape (1, 3, 640, 512)\n",
      "Subfigure count mismatch: 4 detected, 18 expected\n",
      "6\n",
      "\n",
      "image 1/1 /Users/etowah/projects/PMC-figure-downloader/img/10786724_Fig1.jpg: 480x640 8 labels, 105.0ms\n",
      "Speed: 1.8ms preprocess, 105.0ms inference, 0.4ms postprocess per image at shape (1, 3, 480, 640)\n",
      "Subfigure count mismatch: 8 detected, 6 expected\n",
      "5\n",
      "\n",
      "image 1/1 /Users/etowah/projects/PMC-figure-downloader/img/10786724_Fig2.jpg: 608x640 5 labels, 78.6ms\n",
      "Speed: 2.2ms preprocess, 78.6ms inference, 0.5ms postprocess per image at shape (1, 3, 608, 640)\n",
      "4\n",
      "\n",
      "image 1/1 /Users/etowah/projects/PMC-figure-downloader/img/10786724_Fig3.jpg: 576x640 4 labels, 66.6ms\n",
      "Speed: 2.1ms preprocess, 66.6ms inference, 0.7ms postprocess per image at shape (1, 3, 576, 640)\n",
      "6\n",
      "\n",
      "image 1/1 /Users/etowah/projects/PMC-figure-downloader/img/10786724_Fig4.jpg: 640x352 6 labels, 56.7ms\n",
      "Speed: 1.5ms preprocess, 56.7ms inference, 0.4ms postprocess per image at shape (1, 3, 640, 352)\n",
      "6\n",
      "\n",
      "image 1/1 /Users/etowah/projects/PMC-figure-downloader/img/10786724_Fig5.jpg: 640x480 6 labels, 76.5ms\n",
      "Speed: 2.0ms preprocess, 76.5ms inference, 0.4ms postprocess per image at shape (1, 3, 640, 480)\n",
      "2\n",
      "\n",
      "image 1/1 /Users/etowah/projects/PMC-figure-downloader/img/10786724_Fig6.jpg: 288x640 2 labels, 47.2ms\n",
      "Speed: 1.2ms preprocess, 47.2ms inference, 0.4ms postprocess per image at shape (1, 3, 288, 640)\n",
      "6\n",
      "\n",
      "image 1/1 /Users/etowah/projects/PMC-figure-downloader/img/10786724_Fig7.jpg: 576x640 7 labels, 82.8ms\n",
      "Speed: 2.2ms preprocess, 82.8ms inference, 0.4ms postprocess per image at shape (1, 3, 576, 640)\n",
      "9\n",
      "\n",
      "image 1/1 /Users/etowah/projects/PMC-figure-downloader/img/10786724_Fig8.jpg: 640x576 10 labels, 74.4ms\n",
      "Speed: 2.2ms preprocess, 74.4ms inference, 0.6ms postprocess per image at shape (1, 3, 640, 576)\n",
      "2\n",
      "\n",
      "image 1/1 /Users/etowah/projects/PMC-figure-downloader/img/10786724_Fig9.jpg: 640x288 6 labels, 46.5ms\n",
      "Speed: 1.4ms preprocess, 46.5ms inference, 0.4ms postprocess per image at shape (1, 3, 640, 288)\n",
      "Subfigure count mismatch: 6 detected, 2 expected\n",
      "5\n",
      "\n",
      "image 1/1 /Users/etowah/projects/PMC-figure-downloader/img/10786724_Fig10.jpg: 640x448 5 labels, 66.7ms\n",
      "Speed: 1.8ms preprocess, 66.7ms inference, 0.8ms postprocess per image at shape (1, 3, 640, 448)\n",
      "7\n",
      "\n",
      "image 1/1 /Users/etowah/projects/PMC-figure-downloader/img/10786724_Fig11.jpg: 640x640 7 labels, 81.7ms\n",
      "Speed: 2.5ms preprocess, 81.7ms inference, 0.4ms postprocess per image at shape (1, 3, 640, 640)\n",
      "1\n",
      "Western blots were dually stained with IRDye 800CW secondary antibody against primary antibodies targeting the TFs (panel a) and IRDye 680RD secondary antibody against primary antibodies targeting histone H3 (panel b) and imaged using respective filters on the Odyssey DLx Imaging System.\n",
      "5\n",
      "\n",
      "image 1/1 /Users/etowah/projects/PMC-figure-downloader/img/10786720_Fig1.jpg: 416x640 5 labels, 67.0ms\n",
      "Speed: 1.7ms preprocess, 67.0ms inference, 0.5ms postprocess per image at shape (1, 3, 416, 640)\n",
      "13\n",
      "\n",
      "image 1/1 /Users/etowah/projects/PMC-figure-downloader/img/10786720_Fig2.jpg: 608x640 13 labels, 83.6ms\n",
      "Speed: 2.1ms preprocess, 83.6ms inference, 0.7ms postprocess per image at shape (1, 3, 608, 640)\n",
      "8\n",
      "\n",
      "image 1/1 /Users/etowah/projects/PMC-figure-downloader/img/10786720_Fig3.jpg: 640x608 8 labels, 77.3ms\n",
      "Speed: 2.3ms preprocess, 77.3ms inference, 0.4ms postprocess per image at shape (1, 3, 640, 608)\n",
      "6\n",
      "\n",
      "image 1/1 /Users/etowah/projects/PMC-figure-downloader/img/10786720_Fig4.jpg: 640x640 6 labels, 89.6ms\n",
      "Speed: 2.4ms preprocess, 89.6ms inference, 0.8ms postprocess per image at shape (1, 3, 640, 640)\n",
      "8\n",
      "\n",
      "image 1/1 /Users/etowah/projects/PMC-figure-downloader/img/10786720_Fig5.jpg: 640x608 8 labels, 73.2ms\n",
      "Speed: 2.3ms preprocess, 73.2ms inference, 0.4ms postprocess per image at shape (1, 3, 640, 608)\n",
      "8\n",
      "\n",
      "image 1/1 /Users/etowah/projects/PMC-figure-downloader/img/10786720_Fig6.jpg: 640x512 8 labels, 76.0ms\n",
      "Speed: 2.0ms preprocess, 76.0ms inference, 0.4ms postprocess per image at shape (1, 3, 640, 512)\n",
      "14\n",
      "\n",
      "image 1/1 /Users/etowah/projects/PMC-figure-downloader/img/10786720_Fig7.jpg: 640x480 17 labels, 73.1ms\n",
      "Speed: 2.0ms preprocess, 73.1ms inference, 0.8ms postprocess per image at shape (1, 3, 640, 480)\n",
      "Subfigure count mismatch: 15 detected, 14 expected\n",
      "8\n",
      "\n",
      "image 1/1 /Users/etowah/projects/PMC-figure-downloader/img/10786720_Fig8.jpg: 480x640 8 labels, 91.2ms\n",
      "Speed: 1.9ms preprocess, 91.2ms inference, 0.7ms postprocess per image at shape (1, 3, 480, 640)\n",
      "8\n",
      "\n",
      "image 1/1 /Users/etowah/projects/PMC-figure-downloader/img/10786720_Fig9.jpg: 576x640 8 labels, 76.1ms\n",
      "Speed: 2.1ms preprocess, 76.1ms inference, 0.5ms postprocess per image at shape (1, 3, 576, 640)\n",
      "3\n",
      "\n",
      "image 1/1 /Users/etowah/projects/PMC-figure-downloader/img/10786720_Fig10.jpg: 448x640 3 labels, 60.3ms\n",
      "Speed: 1.8ms preprocess, 60.3ms inference, 0.4ms postprocess per image at shape (1, 3, 448, 640)\n",
      "4\n",
      "\n",
      "image 1/1 /Users/etowah/projects/PMC-figure-downloader/img/10786720_Fig11.jpg: 512x640 5 labels, 87.4ms\n",
      "Speed: 1.9ms preprocess, 87.4ms inference, 0.4ms postprocess per image at shape (1, 3, 512, 640)\n",
      "Subfigure count mismatch: 5 detected, 4 expected\n",
      "11\n",
      "\n",
      "image 1/1 /Users/etowah/projects/PMC-figure-downloader/img/10786720_Fig12.jpg: 640x512 12 labels, 72.0ms\n",
      "Speed: 2.0ms preprocess, 72.0ms inference, 0.4ms postprocess per image at shape (1, 3, 640, 512)\n",
      "9\n",
      "\n",
      "image 1/1 /Users/etowah/projects/PMC-figure-downloader/img/10786720_Fig13.jpg: 640x544 9 labels, 89.0ms\n",
      "Speed: 2.0ms preprocess, 89.0ms inference, 0.5ms postprocess per image at shape (1, 3, 640, 544)\n",
      "7\n",
      "\n",
      "image 1/1 /Users/etowah/projects/PMC-figure-downloader/img/10786720_Fig14.jpg: 576x640 7 labels, 66.5ms\n",
      "Speed: 2.1ms preprocess, 66.5ms inference, 0.4ms postprocess per image at shape (1, 3, 576, 640)\n",
      "8\n",
      "\n",
      "image 1/1 /Users/etowah/projects/PMC-figure-downloader/img/10786720_Fig15.jpg: 640x480 9 labels, 65.8ms\n",
      "Speed: 1.9ms preprocess, 65.8ms inference, 0.4ms postprocess per image at shape (1, 3, 640, 480)\n",
      "6\n",
      "\n",
      "image 1/1 /Users/etowah/projects/PMC-figure-downloader/img/10786720_Fig16.jpg: 640x512 7 labels, 73.2ms\n",
      "Speed: 2.1ms preprocess, 73.2ms inference, 0.7ms postprocess per image at shape (1, 3, 640, 512)\n",
      "4\n",
      "\n",
      "image 1/1 /Users/etowah/projects/PMC-figure-downloader/img/10786719_Fig1.jpg: 576x640 4 labels, 81.0ms\n",
      "Speed: 2.2ms preprocess, 81.0ms inference, 0.4ms postprocess per image at shape (1, 3, 576, 640)\n",
      "5\n",
      "\n",
      "image 1/1 /Users/etowah/projects/PMC-figure-downloader/img/10786719_Fig2.jpg: 352x640 5 labels, 55.0ms\n",
      "Speed: 1.3ms preprocess, 55.0ms inference, 0.4ms postprocess per image at shape (1, 3, 352, 640)\n",
      "7\n",
      "\n",
      "image 1/1 /Users/etowah/projects/PMC-figure-downloader/img/10786719_Fig3.jpg: 640x640 7 labels, 86.8ms\n",
      "Speed: 2.3ms preprocess, 86.8ms inference, 0.4ms postprocess per image at shape (1, 3, 640, 640)\n",
      "1\n",
      "Drug sensitivity profiling of isogenic RPE1-BRCA1−/−, BRCA2−/−, and control cells to CX-5461, topoisomerase II poison, etoposide (ETO), and G-quadruplex stabilising compound, pyridostatin (PDS) confirmed synthetic lethality of CX-5461 in RPE1-BRCA1−/−, BRCA2−/− cells. RPE1-LIG4−/− was used as a positive control. Cells were also profiled against two other therapeutic agents commonly used for the treatment of BRCA1/2-mutated cancers, cisplatin, and olaparib. Data are mean ± standard errors (error bars), n = 3 independent biological replicates. All comparisons were made against WT. Two-tailed Wilcoxon signed-rank test was used to calculate P values.\n",
      "4\n",
      "\n",
      "image 1/1 /Users/etowah/projects/PMC-figure-downloader/img/10786719_Fig5.jpg: 640x576 4 labels, 91.7ms\n",
      "Speed: 2.3ms preprocess, 91.7ms inference, 0.8ms postprocess per image at shape (1, 3, 640, 576)\n",
      "1\n",
      "Top panel: four ancestry groups used in the meta-analysis. Middle panel: MAMA and the two methods used. Random-effect (top) is better suited for risk variants with homogeneous effect direction across different ancestries, whereas MR-MEGA (bottom) can identify risk variants with heterogeneous effects due to population stratification introduced by ancestry differences. The densely dashed lines indicate Bonferroni adjusted suggestive threshold of two-sided P < 1 × 10−6, and the loosely dashed lines indicate Bonferroni adjusted significant threshold of two-sided P < 5 × 10−9. Bottom panel: downstream analyses and their examples. Created with Biorender.com.\n",
      "2\n",
      "\n",
      "image 1/1 /Users/etowah/projects/PMC-figure-downloader/img/10786718_Fig2.jpg: 576x640 2 labels, 110.8ms\n",
      "Speed: 2.0ms preprocess, 110.8ms inference, 1.0ms postprocess per image at shape (1, 3, 576, 640)\n",
      "2\n",
      "\n",
      "image 1/1 /Users/etowah/projects/PMC-figure-downloader/img/10786718_Fig3.jpg: 640x608 2 labels, 85.7ms\n",
      "Speed: 2.2ms preprocess, 85.7ms inference, 0.6ms postprocess per image at shape (1, 3, 640, 608)\n",
      "2\n",
      "\n",
      "image 1/1 /Users/etowah/projects/PMC-figure-downloader/img/10786716_Fig1.jpg: 352x640 2 labels, 54.7ms\n",
      "Speed: 1.5ms preprocess, 54.7ms inference, 0.4ms postprocess per image at shape (1, 3, 352, 640)\n",
      "2\n",
      "\n",
      "image 1/1 /Users/etowah/projects/PMC-figure-downloader/img/10786716_Fig2.jpg: 480x640 2 labels, 86.5ms\n",
      "Speed: 1.9ms preprocess, 86.5ms inference, 0.6ms postprocess per image at shape (1, 3, 480, 640)\n",
      "2\n",
      "\n",
      "image 1/1 /Users/etowah/projects/PMC-figure-downloader/img/10786716_Fig3.jpg: 448x640 2 labels, 67.3ms\n",
      "Speed: 1.9ms preprocess, 67.3ms inference, 0.4ms postprocess per image at shape (1, 3, 448, 640)\n",
      "14\n",
      "\n",
      "image 1/1 /Users/etowah/projects/PMC-figure-downloader/img/10786716_Fig4.jpg: 288x640 2 labels, 53.0ms\n",
      "Speed: 1.2ms preprocess, 53.0ms inference, 0.4ms postprocess per image at shape (1, 3, 288, 640)\n",
      "Subfigure count mismatch: 2 detected, 14 expected\n",
      "1\n",
      "Relative loss measured by ratio of models’ variance explained (R2) without and with the causal variants included. Results are shown for BridgePRS, PRS-CSx, PRS-CS-mult and PRSice-meta across six simulation scenarios for African and East Asian ancestry samples. a SNP heritability \\documentclass[12pt]{minimal}\n",
      "\t\t\t\t\\usepackage{amsmath}\n",
      "\t\t\t\t\\usepackage{wasysym} \n",
      "\t\t\t\t\\usepackage{amsfonts} \n",
      "\t\t\t\t\\usepackage{amssymb} \n",
      "\t\t\t\t\\usepackage{amsbsy}\n",
      "\t\t\t\t\\usepackage{mathrsfs}\n",
      "\t\t\t\t\\usepackage{upgreek}\n",
      "\t\t\t\t\\setlength{\\oddsidemargin}{-69pt}\n",
      "\t\t\t\t\\begin{document}$${{{{{\\rm{h}}}}}^{2}}_{{{{\\rm{SNP}}}}}=0.25$$\\end{document}h2SNP=0.25 and b SNP heritability \\documentclass[12pt]{minimal}\n",
      "\t\t\t\t\\usepackage{amsmath}\n",
      "\t\t\t\t\\usepackage{wasysym} \n",
      "\t\t\t\t\\usepackage{amsfonts} \n",
      "\t\t\t\t\\usepackage{amssymb} \n",
      "\t\t\t\t\\usepackage{amsbsy}\n",
      "\t\t\t\t\\usepackage{mathrsfs}\n",
      "\t\t\t\t\\usepackage{upgreek}\n",
      "\t\t\t\t\\setlength{\\oddsidemargin}{-69pt}\n",
      "\t\t\t\t\\begin{document}$${{{{{\\rm{h}}}}}^{2}}_{{{{\\rm{SNP}}}}}=0.5$$\\end{document}h2SNP=0.5, ten simulated phenotypes per scenario. Under each set of analyses the proportion of causal variants and the relative power of the data used is shown, measured by nh2/m up to proportionality, where n is the GWAS sample size, h2 heritability and m the number of causal variants. The central rectangular boxes show the interquartile range, horizontal lines inside the boxes show the median, whiskers extend to the most extreme results and points show results for each of the 10 simulated phenotypes. PRSice-meta results for East Asian analyses were unstable and removed for clarity.\n",
      "1\n",
      "Sample sizes of 40K European and 10K non-European were used. Results are shown for BridgePRS, PRS-CSx, PRS-CS-mult and PRSice-meta across six simulation scenarios, with and without the causal variants included in the model for African and East Asian ancestry samples. a SNP heritability \\documentclass[12pt]{minimal}\n",
      "\t\t\t\t\\usepackage{amsmath}\n",
      "\t\t\t\t\\usepackage{wasysym} \n",
      "\t\t\t\t\\usepackage{amsfonts} \n",
      "\t\t\t\t\\usepackage{amssymb} \n",
      "\t\t\t\t\\usepackage{amsbsy}\n",
      "\t\t\t\t\\usepackage{mathrsfs}\n",
      "\t\t\t\t\\usepackage{upgreek}\n",
      "\t\t\t\t\\setlength{\\oddsidemargin}{-69pt}\n",
      "\t\t\t\t\\begin{document}$${{{{{\\rm{h}}}}}^{2}}_{{{{\\rm{SNP}}}}}=0.25$$\\end{document}h2SNP=0.25 and b SNP heritability \\documentclass[12pt]{minimal}\n",
      "\t\t\t\t\\usepackage{amsmath}\n",
      "\t\t\t\t\\usepackage{wasysym} \n",
      "\t\t\t\t\\usepackage{amsfonts} \n",
      "\t\t\t\t\\usepackage{amssymb} \n",
      "\t\t\t\t\\usepackage{amsbsy}\n",
      "\t\t\t\t\\usepackage{mathrsfs}\n",
      "\t\t\t\t\\usepackage{upgreek}\n",
      "\t\t\t\t\\setlength{\\oddsidemargin}{-69pt}\n",
      "\t\t\t\t\\begin{document}$${{{{{\\rm{h}}}}}^{2}}_{{{{\\rm{SNP}}}}}=0.5$$\\end{document}h2SNP=0.5, ten simulated phenotypes per scenario. Under each set of analyses the proportion of causal variants and the relative power of the data used is shown, measured by nh2/m up to proportionality, where n is the GWAS sample size, h2 heritability and m the number of causal variants. The central rectangular boxes show the interquartile range, horizontal lines inside the boxes show the median, whiskers extend to the most extreme results and points show results for each of the 10 simulated phenotypes.\n",
      "1\n",
      "Results are shown for BridgePRS, PRS-CSx, PRS-CS-mult and PRSice-meta across six simulation scenarios, with and without the causal variants included in the model for African and East Asian ancestry samples, ten simulated phenotypes per scenario. Under each set of analyses the proportion of causal variants and the relative power of the data used is shown, measured by nh2/m up to proportionality, where n is the GWAS sample size, h2 heritability and m the number of causal variants. The central rectangular boxes show the interquartile range, horizontal lines inside the boxes show the median, whiskers extend to the most extreme results and points show results for each of the 10 simulated phenotypes.\n",
      "1\n",
      "Results are shown for BridgePRS and PRS-CSx across six simulation scenarios, 10 simulated phenotypes per scenario with h2=0.25 for African and East Asian ancestry samples. Data was simulated using 1000 Genomes as reference. Under each set of analyses the proportion of causal variants and the relative power of the data used is shown, measured by nh2/m up to proportionality, where n is the GWAS sample size, h2 heritability and m the number of causal variants. The central rectangular boxes show the interquartile range, horizontal lines inside the boxes show the median, whiskers extend to the most extreme results and points show results for each of the 10 simulated phenotypes.\n",
      "1\n",
      "For each trait variance explained (R2), point estimates and 95% confidence intervals, by BridgePRS, PRS-CSx, PRS-CS-mult and PRSice-meta are shown for African, South Asian and East Asian ancestry samples. n indicates sample size. Neutro count=Neutrophil count, MCV=Mean corpuscular volume, Platelets=Platelet count, Retic count=Reticulocyte per- centage, ALP=Alkaline phosphatase, Mono count=Monocyte count, apoA1=Apolipoprotein A, BMI=Body mass index, RDW=Red blood cell distribution width, Eos count=Eosinophil count, TG=Triglycerides, Baso %=Basophil percentage, CRP=C-reactive protein. Confidence intervals were calculated by bootstrapping using 10,000 replicates.\n",
      "6\n",
      "\n",
      "image 1/1 /Users/etowah/projects/PMC-figure-downloader/img/10703697_Fig1.jpg: 640x480 6 labels, 107.1ms\n",
      "Speed: 2.1ms preprocess, 107.1ms inference, 0.5ms postprocess per image at shape (1, 3, 640, 480)\n",
      "4\n",
      "\n",
      "image 1/1 /Users/etowah/projects/PMC-figure-downloader/img/10703697_Fig2.jpg: 640x480 4 labels, 88.3ms\n",
      "Speed: 2.1ms preprocess, 88.3ms inference, 0.4ms postprocess per image at shape (1, 3, 640, 480)\n",
      "3\n",
      "\n",
      "image 1/1 /Users/etowah/projects/PMC-figure-downloader/img/10703697_Fig3.jpg: 640x608 5 labels, 91.8ms\n",
      "Speed: 2.4ms preprocess, 91.8ms inference, 0.6ms postprocess per image at shape (1, 3, 640, 608)\n",
      "Subfigure count mismatch: 4 detected, 3 expected\n",
      "4\n",
      "\n",
      "image 1/1 /Users/etowah/projects/PMC-figure-downloader/img/10703697_Fig4.jpg: 640x448 4 labels, 68.2ms\n",
      "Speed: 1.8ms preprocess, 68.2ms inference, 0.8ms postprocess per image at shape (1, 3, 640, 448)\n",
      "7\n",
      "\n",
      "image 1/1 /Users/etowah/projects/PMC-figure-downloader/img/10703697_Fig5.jpg: 640x480 7 labels, 88.7ms\n",
      "Speed: 3.2ms preprocess, 88.7ms inference, 0.7ms postprocess per image at shape (1, 3, 640, 480)\n",
      "6\n",
      "\n",
      "image 1/1 /Users/etowah/projects/PMC-figure-downloader/img/10703697_Fig6.jpg: 640x480 6 labels, 79.7ms\n",
      "Speed: 2.3ms preprocess, 79.7ms inference, 0.5ms postprocess per image at shape (1, 3, 640, 480)\n",
      "7\n",
      "\n",
      "image 1/1 /Users/etowah/projects/PMC-figure-downloader/img/10703697_Fig7.jpg: 640x544 7 labels, 73.1ms\n",
      "Speed: 2.2ms preprocess, 73.1ms inference, 0.6ms postprocess per image at shape (1, 3, 640, 544)\n",
      "1\n",
      "A schematic representation of the contribution of aSHM to transformation from FL to DHL, along with various mechanisms of PAX5/ZCCHC7 locus alterations that lead to aberrant rRNA processing and rewiring of lymphomagenic protein synthesis (created with BioRender.com).\n",
      "6\n",
      "\n",
      "image 1/1 /Users/etowah/projects/PMC-figure-downloader/img/10703697_Fig9.jpg: 640x480 6 labels, 74.2ms\n",
      "Speed: 2.1ms preprocess, 74.2ms inference, 0.8ms postprocess per image at shape (1, 3, 640, 480)\n",
      "4\n",
      "\n",
      "image 1/1 /Users/etowah/projects/PMC-figure-downloader/img/10703697_Fig10.jpg: 640x480 4 labels, 71.6ms\n",
      "Speed: 2.0ms preprocess, 71.6ms inference, 0.7ms postprocess per image at shape (1, 3, 640, 480)\n",
      "5\n",
      "\n",
      "image 1/1 /Users/etowah/projects/PMC-figure-downloader/img/10703697_Fig11.jpg: 640x448 5 labels, 75.7ms\n",
      "Speed: 1.9ms preprocess, 75.7ms inference, 0.4ms postprocess per image at shape (1, 3, 640, 448)\n",
      "5\n",
      "\n",
      "image 1/1 /Users/etowah/projects/PMC-figure-downloader/img/10703697_Fig12.jpg: 640x544 5 labels, 83.2ms\n",
      "Speed: 2.1ms preprocess, 83.2ms inference, 0.4ms postprocess per image at shape (1, 3, 640, 544)\n",
      "6\n",
      "\n",
      "image 1/1 /Users/etowah/projects/PMC-figure-downloader/img/10703697_Fig13.jpg: 640x480 6 labels, 76.8ms\n",
      "Speed: 1.9ms preprocess, 76.8ms inference, 1.3ms postprocess per image at shape (1, 3, 640, 480)\n",
      "5\n",
      "\n",
      "image 1/1 /Users/etowah/projects/PMC-figure-downloader/img/10703697_Fig14.jpg: 640x608 5 labels, 95.6ms\n",
      "Speed: 5.9ms preprocess, 95.6ms inference, 0.6ms postprocess per image at shape (1, 3, 640, 608)\n",
      "5\n",
      "\n",
      "image 1/1 /Users/etowah/projects/PMC-figure-downloader/img/10703697_Fig15.jpg: 640x480 5 labels, 75.8ms\n",
      "Speed: 2.1ms preprocess, 75.8ms inference, 0.5ms postprocess per image at shape (1, 3, 640, 480)\n",
      "7\n",
      "\n",
      "image 1/1 /Users/etowah/projects/PMC-figure-downloader/img/10703697_Fig16.jpg: 640x512 8 labels, 74.6ms\n",
      "Speed: 2.0ms preprocess, 74.6ms inference, 0.6ms postprocess per image at shape (1, 3, 640, 512)\n",
      "4\n",
      "\n",
      "image 1/1 /Users/etowah/projects/PMC-figure-downloader/img/10703697_Fig17.jpg: 640x480 6 labels, 74.1ms\n",
      "Speed: 2.0ms preprocess, 74.1ms inference, 0.4ms postprocess per image at shape (1, 3, 640, 480)\n",
      "8\n",
      "\n",
      "image 1/1 /Users/etowah/projects/PMC-figure-downloader/img/10703697_Fig18.jpg: 640x480 10 labels, 90.2ms\n",
      "Speed: 2.0ms preprocess, 90.2ms inference, 0.4ms postprocess per image at shape (1, 3, 640, 480)\n",
      "7\n",
      "\n",
      "image 1/1 /Users/etowah/projects/PMC-figure-downloader/img/10703696_Fig1.jpg: 576x640 7 labels, 134.1ms\n",
      "Speed: 2.6ms preprocess, 134.1ms inference, 2.0ms postprocess per image at shape (1, 3, 576, 640)\n",
      "6\n",
      "\n",
      "image 1/1 /Users/etowah/projects/PMC-figure-downloader/img/10703696_Fig2.jpg: 640x640 6 labels, 104.8ms\n",
      "Speed: 2.9ms preprocess, 104.8ms inference, 0.5ms postprocess per image at shape (1, 3, 640, 640)\n",
      "7\n",
      "\n",
      "image 1/1 /Users/etowah/projects/PMC-figure-downloader/img/10703696_Fig3.jpg: 640x576 7 labels, 80.2ms\n",
      "Speed: 2.1ms preprocess, 80.2ms inference, 0.4ms postprocess per image at shape (1, 3, 640, 576)\n",
      "3\n",
      "\n",
      "image 1/1 /Users/etowah/projects/PMC-figure-downloader/img/10703696_Fig4.jpg: 256x640 3 labels, 43.9ms\n",
      "Speed: 1.1ms preprocess, 43.9ms inference, 0.4ms postprocess per image at shape (1, 3, 256, 640)\n",
      "5\n",
      "\n",
      "image 1/1 /Users/etowah/projects/PMC-figure-downloader/img/10703696_Fig5.jpg: 640x640 6 labels, 84.0ms\n",
      "Speed: 2.4ms preprocess, 84.0ms inference, 0.5ms postprocess per image at shape (1, 3, 640, 640)\n",
      "5\n",
      "\n",
      "image 1/1 /Users/etowah/projects/PMC-figure-downloader/img/10703696_Fig6.jpg: 416x640 5 labels, 61.2ms\n",
      "Speed: 1.6ms preprocess, 61.2ms inference, 0.4ms postprocess per image at shape (1, 3, 416, 640)\n",
      "1\n",
      "Kaplan-Meier curves indicating (a) overall survival and (b) progression-free survival for n = 338 medulloblastoma patients, stratified by the amplifications present in the patient tumors. Patients with ecDNA amplification had significantly worse overall and progression-free survival compared to those with linear amplifications and compared to those without focal somatic copy number amplification (fSCNA). * p < 0.05; ** p < 0.005.\n",
      "3\n",
      "\n",
      "image 1/1 /Users/etowah/projects/PMC-figure-downloader/img/10703696_Fig8.jpg: 320x640 3 labels, 64.4ms\n",
      "Speed: 1.4ms preprocess, 64.4ms inference, 0.8ms postprocess per image at shape (1, 3, 320, 640)\n",
      "1\n",
      "Forest plots of β coefficients (log hazard ratios) of Cox Proportional Hazards models fitted on n = 322 patients using L2 ridge regression to control instability due to collinearity. WNT subgroup patients were excluded due to perfect separation. Log hazard estimates are relative to Group 4 and female patients. Log hazard ratios for (a) OS and (b) PFS of Cox models including age, sex, subgroup, ecDNA and TP53 mutation as covariates. Data are presented as maximum a posteriori (MAP) estimate with a Gaussian prior (L2 regularization) +/− 95% confidence intervals.\n",
      "6\n",
      "\n",
      "image 1/1 /Users/etowah/projects/PMC-figure-downloader/img/10703696_Fig10.jpg: 640x640 9 labels, 85.5ms\n",
      "Speed: 2.4ms preprocess, 85.5ms inference, 0.4ms postprocess per image at shape (1, 3, 640, 640)\n",
      "Subfigure count mismatch: 9 detected, 6 expected\n",
      "5\n",
      "\n",
      "image 1/1 /Users/etowah/projects/PMC-figure-downloader/img/10703696_Fig11.jpg: 640x544 5 labels, 86.4ms\n",
      "Speed: 2.2ms preprocess, 86.4ms inference, 0.4ms postprocess per image at shape (1, 3, 640, 544)\n",
      "6\n",
      "\n",
      "image 1/1 /Users/etowah/projects/PMC-figure-downloader/img/10703696_Fig12.jpg: 640x512 7 labels, 80.6ms\n",
      "Speed: 2.0ms preprocess, 80.6ms inference, 0.9ms postprocess per image at shape (1, 3, 640, 512)\n",
      "2\n",
      "\n",
      "image 1/1 /Users/etowah/projects/PMC-figure-downloader/img/10703696_Fig13.jpg: 640x608 2 labels, 88.1ms\n",
      "Speed: 2.4ms preprocess, 88.1ms inference, 0.5ms postprocess per image at shape (1, 3, 640, 608)\n",
      "4\n",
      "\n",
      "image 1/1 /Users/etowah/projects/PMC-figure-downloader/img/10703696_Fig14.jpg: 192x640 4 labels, 41.5ms\n",
      "Speed: 0.8ms preprocess, 41.5ms inference, 0.3ms postprocess per image at shape (1, 3, 192, 640)\n",
      "2\n",
      "\n",
      "image 1/1 /Users/etowah/projects/PMC-figure-downloader/img/10703696_Fig15.jpg: 640x576 2 labels, 75.7ms\n",
      "Speed: 2.2ms preprocess, 75.7ms inference, 0.4ms postprocess per image at shape (1, 3, 640, 576)\n",
      "2\n",
      "\n",
      "image 1/1 /Users/etowah/projects/PMC-figure-downloader/img/10703691_Fig1.jpg: 480x640 2 labels, 73.6ms\n",
      "Speed: 1.9ms preprocess, 73.6ms inference, 0.4ms postprocess per image at shape (1, 3, 480, 640)\n",
      "3\n",
      "\n",
      "image 1/1 /Users/etowah/projects/PMC-figure-downloader/img/10703691_Fig2.jpg: 448x640 3 labels, 109.5ms\n",
      "Speed: 2.2ms preprocess, 109.5ms inference, 0.4ms postprocess per image at shape (1, 3, 448, 640)\n",
      "4\n",
      "\n",
      "image 1/1 /Users/etowah/projects/PMC-figure-downloader/img/10703691_Fig3.jpg: 320x640 4 labels, 56.6ms\n",
      "Speed: 1.3ms preprocess, 56.6ms inference, 0.4ms postprocess per image at shape (1, 3, 320, 640)\n",
      "11\n",
      "\n",
      "image 1/1 /Users/etowah/projects/PMC-figure-downloader/img/10703691_Fig4.jpg: 640x352 5 labels, 63.1ms\n",
      "Speed: 1.5ms preprocess, 63.1ms inference, 0.4ms postprocess per image at shape (1, 3, 640, 352)\n",
      "Subfigure count mismatch: 5 detected, 11 expected\n",
      "3\n",
      "\n",
      "image 1/1 /Users/etowah/projects/PMC-figure-downloader/img/10703691_Fig5.jpg: 640x512 3 labels, 82.7ms\n",
      "Speed: 2.1ms preprocess, 82.7ms inference, 0.5ms postprocess per image at shape (1, 3, 640, 512)\n",
      "2\n",
      "\n",
      "image 1/1 /Users/etowah/projects/PMC-figure-downloader/img/10703691_Fig6.jpg: 640x448 2 labels, 67.0ms\n",
      "Speed: 1.8ms preprocess, 67.0ms inference, 0.4ms postprocess per image at shape (1, 3, 640, 448)\n",
      "4\n",
      "\n",
      "image 1/1 /Users/etowah/projects/PMC-figure-downloader/img/10703691_Fig7.jpg: 640x608 6 labels, 86.6ms\n",
      "Speed: 2.3ms preprocess, 86.6ms inference, 0.5ms postprocess per image at shape (1, 3, 640, 608)\n",
      "Subfigure count mismatch: 6 detected, 4 expected\n",
      "3\n",
      "\n",
      "image 1/1 /Users/etowah/projects/PMC-figure-downloader/img/10703691_Fig8.jpg: 512x640 3 labels, 71.1ms\n",
      "Speed: 1.8ms preprocess, 71.1ms inference, 0.4ms postprocess per image at shape (1, 3, 512, 640)\n",
      "4\n",
      "\n",
      "image 1/1 /Users/etowah/projects/PMC-figure-downloader/img/10703691_Fig9.jpg: 640x608 4 labels, 83.7ms\n",
      "Speed: 2.4ms preprocess, 83.7ms inference, 0.5ms postprocess per image at shape (1, 3, 640, 608)\n",
      "4\n",
      "\n",
      "image 1/1 /Users/etowah/projects/PMC-figure-downloader/img/10703691_Fig10.jpg: 416x640 4 labels, 73.7ms\n",
      "Speed: 1.6ms preprocess, 73.7ms inference, 0.5ms postprocess per image at shape (1, 3, 416, 640)\n",
      "5\n",
      "\n",
      "image 1/1 /Users/etowah/projects/PMC-figure-downloader/img/10703691_Fig11.jpg: 640x448 5 labels, 72.4ms\n",
      "Speed: 1.8ms preprocess, 72.4ms inference, 0.4ms postprocess per image at shape (1, 3, 640, 448)\n",
      "3\n",
      "\n",
      "image 1/1 /Users/etowah/projects/PMC-figure-downloader/img/10703691_Fig12.jpg: 640x512 3 labels, 74.6ms\n",
      "Speed: 2.0ms preprocess, 74.6ms inference, 0.7ms postprocess per image at shape (1, 3, 640, 512)\n",
      "4\n",
      "\n",
      "image 1/1 /Users/etowah/projects/PMC-figure-downloader/img/10703691_Fig13.jpg: 640x448 4 labels, 86.0ms\n",
      "Speed: 2.4ms preprocess, 86.0ms inference, 0.5ms postprocess per image at shape (1, 3, 640, 448)\n",
      "2\n",
      "\n",
      "image 1/1 /Users/etowah/projects/PMC-figure-downloader/img/10703691_Fig14.jpg: 640x512 5 labels, 63.0ms\n",
      "Speed: 2.0ms preprocess, 63.0ms inference, 0.4ms postprocess per image at shape (1, 3, 640, 512)\n",
      "Subfigure count mismatch: 5 detected, 2 expected\n",
      "1\n",
      "(1), We imputed LifetimeMDD using a partially observed matrix of depression-relevant phenotypes in UK Biobank (UKB). We focused on using SoftImpute, which also produces latent phenome-wide factors. (2),(3), We then performed GWASs on observed and imputed values of LifetimeMDD (2), as well as downstream polygenic analyses, including in-sample and out-of-sample PRS predictions of MDD (3). (4),(5), We also studied the genetic basis of the latent factors of the depression phenome (4) and residualized latent factors from shallow MDD phenotypes to remove nonspecific pleiotropic effects (5).\n",
      "7\n",
      "\n",
      "image 1/1 /Users/etowah/projects/PMC-figure-downloader/img/10703686_Fig2.jpg: 640x608 8 labels, 82.7ms\n",
      "Speed: 2.5ms preprocess, 82.7ms inference, 1.1ms postprocess per image at shape (1, 3, 640, 608)\n",
      "Subfigure count mismatch: 8 detected, 7 expected\n",
      "7\n",
      "\n",
      "image 1/1 /Users/etowah/projects/PMC-figure-downloader/img/10703686_Fig3.jpg: 640x512 9 labels, 79.6ms\n",
      "Speed: 2.0ms preprocess, 79.6ms inference, 0.9ms postprocess per image at shape (1, 3, 640, 512)\n",
      "Subfigure count mismatch: 8 detected, 7 expected\n",
      "5\n",
      "\n",
      "image 1/1 /Users/etowah/projects/PMC-figure-downloader/img/10703686_Fig4.jpg: 640x640 5 labels, 99.2ms\n",
      "Speed: 2.5ms preprocess, 99.2ms inference, 0.5ms postprocess per image at shape (1, 3, 640, 640)\n",
      "3\n",
      "\n",
      "image 1/1 /Users/etowah/projects/PMC-figure-downloader/img/10703686_Fig5.jpg: 640x576 4 labels, 120.4ms\n",
      "Speed: 2.5ms preprocess, 120.4ms inference, 1.0ms postprocess per image at shape (1, 3, 640, 576)\n",
      "Subfigure count mismatch: 4 detected, 3 expected\n",
      "3\n",
      "\n",
      "image 1/1 /Users/etowah/projects/PMC-figure-downloader/img/10703686_Fig6.jpg: 224x640 3 labels, 71.2ms\n",
      "Speed: 1.5ms preprocess, 71.2ms inference, 0.4ms postprocess per image at shape (1, 3, 224, 640)\n",
      "4\n",
      "\n",
      "image 1/1 /Users/etowah/projects/PMC-figure-downloader/img/10703686_Fig7.jpg: 512x640 (no detections), 139.8ms\n",
      "Speed: 2.7ms preprocess, 139.8ms inference, 0.3ms postprocess per image at shape (1, 3, 512, 640)\n",
      "Subfigure count mismatch: 0 detected, 4 expected\n",
      "3\n",
      "\n",
      "image 1/1 /Users/etowah/projects/PMC-figure-downloader/img/10703686_Fig8.jpg: 352x640 3 labels, 78.5ms\n",
      "Speed: 1.6ms preprocess, 78.5ms inference, 0.6ms postprocess per image at shape (1, 3, 352, 640)\n",
      "5\n",
      "\n",
      "image 1/1 /Users/etowah/projects/PMC-figure-downloader/img/10703686_Fig9.jpg: 512x640 5 labels, 92.2ms\n",
      "Speed: 2.2ms preprocess, 92.2ms inference, 0.6ms postprocess per image at shape (1, 3, 512, 640)\n",
      "3\n",
      "\n",
      "image 1/1 /Users/etowah/projects/PMC-figure-downloader/img/10703686_Fig10.jpg: 192x640 4 labels, 34.7ms\n",
      "Speed: 0.8ms preprocess, 34.7ms inference, 0.5ms postprocess per image at shape (1, 3, 192, 640)\n",
      "1\n",
      "a-d, PRS Pleiotropy of GPpsy (n = 332,629) (a,b) and Soft-ImpAll (n = 337,126) (c,d) at full sample size and down-sampled to n = 50K and 100K for 62 phenotypes in UKB. Plotted values show the mean PRS Pleiotropy from 10-fold cross-validation in UKB.\n",
      "1\n",
      "a-d, PRS Pleiotropy of imputed and MTAG GWAS for 62 phenotypes in UKB (which are significantly predicted by at least one full-sample PRS, as shown in Fig. 6), where the PRS is constructed with 136,563 LD-pruned SNPs (r2 < 0.2) in UKB (a,b) and 91,315 SNPs from PGC29 GWAS clumped at Pthreshold = 1 (c,d). Plotted values show the mean PRS Pleiotropy from 10-fold cross-validation in UKB.\n",
      "4\n",
      "\n",
      "image 1/1 /Users/etowah/projects/PMC-figure-downloader/img/10703684_Fig1.jpg: 320x640 4 labels, 71.6ms\n",
      "Speed: 1.3ms preprocess, 71.6ms inference, 0.7ms postprocess per image at shape (1, 3, 320, 640)\n",
      "3\n",
      "\n",
      "image 1/1 /Users/etowah/projects/PMC-figure-downloader/img/10703684_Fig2.jpg: 480x640 3 labels, 110.3ms\n",
      "Speed: 2.2ms preprocess, 110.3ms inference, 0.9ms postprocess per image at shape (1, 3, 480, 640)\n",
      "3\n",
      "\n",
      "image 1/1 /Users/etowah/projects/PMC-figure-downloader/img/10703684_Fig3.jpg: 544x640 4 labels, 72.7ms\n",
      "Speed: 2.2ms preprocess, 72.7ms inference, 0.4ms postprocess per image at shape (1, 3, 544, 640)\n",
      "Subfigure count mismatch: 4 detected, 3 expected\n",
      "3\n",
      "\n",
      "image 1/1 /Users/etowah/projects/PMC-figure-downloader/img/10703684_Fig4.jpg: 512x640 4 labels, 74.7ms\n",
      "Speed: 2.0ms preprocess, 74.7ms inference, 0.4ms postprocess per image at shape (1, 3, 512, 640)\n",
      "Subfigure count mismatch: 4 detected, 3 expected\n",
      "4\n",
      "\n",
      "image 1/1 /Users/etowah/projects/PMC-figure-downloader/img/10703684_Fig5.jpg: 640x512 5 labels, 74.6ms\n",
      "Speed: 2.0ms preprocess, 74.6ms inference, 0.5ms postprocess per image at shape (1, 3, 640, 512)\n",
      "Subfigure count mismatch: 5 detected, 4 expected\n",
      "9\n",
      "\n",
      "image 1/1 /Users/etowah/projects/PMC-figure-downloader/img/10703684_Fig6.jpg: 640x448 11 labels, 62.0ms\n",
      "Speed: 2.0ms preprocess, 62.0ms inference, 0.5ms postprocess per image at shape (1, 3, 640, 448)\n",
      "Subfigure count mismatch: 11 detected, 9 expected\n",
      "3\n",
      "\n",
      "image 1/1 /Users/etowah/projects/PMC-figure-downloader/img/10703684_Fig7.jpg: 544x640 4 labels, 69.3ms\n",
      "Speed: 1.9ms preprocess, 69.3ms inference, 1.0ms postprocess per image at shape (1, 3, 544, 640)\n",
      "Subfigure count mismatch: 4 detected, 3 expected\n",
      "3\n",
      "\n",
      "image 1/1 /Users/etowah/projects/PMC-figure-downloader/img/10703684_Fig8.jpg: 544x640 4 labels, 119.5ms\n",
      "Speed: 1.9ms preprocess, 119.5ms inference, 1.3ms postprocess per image at shape (1, 3, 544, 640)\n",
      "Subfigure count mismatch: 4 detected, 3 expected\n",
      "3\n",
      "\n",
      "image 1/1 /Users/etowah/projects/PMC-figure-downloader/img/10703684_Fig9.jpg: 576x640 4 labels, 133.9ms\n",
      "Speed: 2.2ms preprocess, 133.9ms inference, 0.6ms postprocess per image at shape (1, 3, 576, 640)\n",
      "Subfigure count mismatch: 4 detected, 3 expected\n",
      "3\n",
      "\n",
      "image 1/1 /Users/etowah/projects/PMC-figure-downloader/img/10703684_Fig10.jpg: 576x640 4 labels, 100.8ms\n",
      "Speed: 2.4ms preprocess, 100.8ms inference, 0.6ms postprocess per image at shape (1, 3, 576, 640)\n",
      "Subfigure count mismatch: 4 detected, 3 expected\n",
      "3\n",
      "\n",
      "image 1/1 /Users/etowah/projects/PMC-figure-downloader/img/10703684_Fig11.jpg: 544x640 4 labels, 97.9ms\n",
      "Speed: 2.3ms preprocess, 97.9ms inference, 0.5ms postprocess per image at shape (1, 3, 544, 640)\n",
      "Subfigure count mismatch: 4 detected, 3 expected\n",
      "3\n",
      "\n",
      "image 1/1 /Users/etowah/projects/PMC-figure-downloader/img/10703684_Fig12.jpg: 544x640 4 labels, 153.4ms\n",
      "Speed: 2.2ms preprocess, 153.4ms inference, 1.6ms postprocess per image at shape (1, 3, 544, 640)\n",
      "Subfigure count mismatch: 4 detected, 3 expected\n",
      "6\n",
      "\n",
      "image 1/1 /Users/etowah/projects/PMC-figure-downloader/img/10703678_Fig1.jpg: 640x640 6 labels, 140.1ms\n",
      "Speed: 2.8ms preprocess, 140.1ms inference, 2.2ms postprocess per image at shape (1, 3, 640, 640)\n",
      "8\n",
      "\n",
      "image 1/1 /Users/etowah/projects/PMC-figure-downloader/img/10703678_Fig2.jpg: 640x544 8 labels, 106.1ms\n",
      "Speed: 9.3ms preprocess, 106.1ms inference, 0.9ms postprocess per image at shape (1, 3, 640, 544)\n",
      "12\n",
      "\n",
      "image 1/1 /Users/etowah/projects/PMC-figure-downloader/img/10703678_Fig3.jpg: 640x544 12 labels, 82.9ms\n",
      "Speed: 2.1ms preprocess, 82.9ms inference, 0.4ms postprocess per image at shape (1, 3, 640, 544)\n",
      "8\n",
      "\n",
      "image 1/1 /Users/etowah/projects/PMC-figure-downloader/img/10703678_Fig4.jpg: 544x640 8 labels, 94.5ms\n",
      "Speed: 2.3ms preprocess, 94.5ms inference, 0.5ms postprocess per image at shape (1, 3, 544, 640)\n",
      "15\n",
      "\n",
      "image 1/1 /Users/etowah/projects/PMC-figure-downloader/img/10703678_Fig5.jpg: 640x480 16 labels, 62.7ms\n",
      "Speed: 1.9ms preprocess, 62.7ms inference, 0.4ms postprocess per image at shape (1, 3, 640, 480)\n",
      "1\n",
      "HRC, Haplotype Reference Consortium; MAC, minor allele count; PC, principal component.\n",
      "1\n",
      "Manhattan plots of −log10(P values) across the chromosomes and corresponding quantile–quantile plot of observed versus expected −log10(P values) for meta-analyses of SNP associations with PW in the fetal GWAS (top, n = 65,405 children), the maternal GWAS (middle, n = 61,228 mothers) and the paternal GWAS (bottom, n = 52,392 fathers).\n",
      "4\n",
      "\n",
      "image 1/1 /Users/etowah/projects/PMC-figure-downloader/img/10632150_Fig3.jpg: 448x640 4 labels, 81.4ms\n",
      "Speed: 1.8ms preprocess, 81.4ms inference, 0.4ms postprocess per image at shape (1, 3, 448, 640)\n",
      "4\n",
      "\n",
      "image 1/1 /Users/etowah/projects/PMC-figure-downloader/img/10632150_Fig4.jpg: 640x640 4 labels, 95.8ms\n",
      "Speed: 2.4ms preprocess, 95.8ms inference, 0.6ms postprocess per image at shape (1, 3, 640, 640)\n",
      "Subfigure count mismatch: 3 detected, 4 expected\n",
      "2\n",
      "\n",
      "image 1/1 /Users/etowah/projects/PMC-figure-downloader/img/10632150_Fig5.jpg: 352x640 2 labels, 72.1ms\n",
      "Speed: 1.4ms preprocess, 72.1ms inference, 0.4ms postprocess per image at shape (1, 3, 352, 640)\n",
      "1\n",
      "Point (effect size estimate) and line (95% CI of effect size) of two-sample MR analyses estimating the causal effects of maternal height, glycemic traits and blood pressure on PW, as well as the effects of fetal genetic predisposition to the same traits on PW. Maternal WLMs are adjusted for the effects of the fetal genotypes, and fetal WLM analyses are adjusted for the effects of the maternal genotypes. WLM, weighted linear model;SBP, systolic blood pressure; DBP, diastolic blood pressure. Units for exposures are height (s.d.), fasting glucose (s.d.), disposition index (s.d.), fasting insulin (s.d.), SBP (10 mmHg) and DBP (10 mmHg). Causal estimates are grams of PW per unit exposure.\n",
      "1\n",
      "Summary of results and conclusions from fetal, maternal and paternal meta-analyses (left panel) and a schematic of the results of Mendelian randomization analyses (right panel).\n",
      "1\n",
      "Variants identified in the fetal, maternal and paternal analyses are shown in black, red, and blue, respectively. The lines indicate effect sizes needed to have 80% power to detect variants at genome-wide significance with the sample sizes of the fetal, maternal, and paternal analyses. Circles indicate main signals and triangles indicate secondary signals (fetal n = 65,405; maternal n = 61,228; paternal n = 52,392). Error bars represent 95% confidence intervals.\n",
      "1\n",
      "Scatter plot showing SNP heritability estimates (h2) for fetal, maternal and paternal genomes estimated using genomic SEM (fetal n = 65,405; maternal n = 61,228; paternal n = 52,392). Error bars represent 95% confidence intervals.\n",
      "1\n",
      "Shown are the variants which were classified as having a fetal effect. Estimates are provided for fetal, maternal, and paternal effects for the meta-analysis results and after weighted linear model adjustment (fetal n = 65,405; maternal n = 61,228; paternal n = 52,392). Circles and triangles are association estimates, and error bars represent 95% confidence intervals. Abbreviation: WLM, weighted linear model. *Note different scale on x-axis for rs140691414 - TSNAX-DISC1 & rs541541049 NUDT3.\n",
      "1\n",
      "Shown are the remaining variants. Estimates are provided for fetal, maternal, and paternal effects for the meta-analysis results and after weighted linear model adjustment (fetal n = 65,405; maternal n = 61,228; paternal n = 52,392). Circles and triangles are association estimates, and error bars represent 95% confidence intervals. Abbreviation: WLM, weighted linear model.\n",
      "5\n",
      "\n",
      "image 1/1 /Users/etowah/projects/PMC-figure-downloader/img/10632150_Fig12.jpg: 640x416 7 labels, 57.9ms\n",
      "Speed: 1.6ms preprocess, 57.9ms inference, 0.4ms postprocess per image at shape (1, 3, 640, 416)\n",
      "Subfigure count mismatch: 7 detected, 5 expected\n",
      "1\n",
      "Plot illustrating the enrichment or depletion of RNA expression of the 31 placental weight-associated protein-coding genes identified in the fetal GWAS, in 61 different tissues. Each dot represents a specific tissue and plots the difference in average rank of expression levels between the 31 placental weight-associated genes and all the other genes (x-axis) with associated -log(P value) based on the Wilcoxon rank-sum test (y-axis). The size of the points is inversely proportional to the log P value. The two dashed horizontal lines represent significance levels with (red) or without (orange) Bonferroni correction (n = 61). Tissues with nominally significant (P value < 0.05) higher or lower expression of the test genes are plotted and labeled as red or blue dots, respectively. Tissues with Bonferroni corrected significance are highlighted by labels with yellow background.\n",
      "1\n",
      "Plot illustrating the enrichment or depletion of RNA expression of the 31 nearest protein-coding genes at placental weight loci identified in the fetal GWAS in 32 different cell types at the early maternal-fetal interface. Each dot represents a specific cell-type and plots the difference in average rank of expression between the 31 placental genes and all the other genes (x-axis) with associated -log(P value) based on the Wilcoxon rank-sum test (y-axis). The size of the points is inversely proportional to the log P value. The two dashed horizontal lines represent significance levels with (red) or without (orange) after Bonferroni correction (n = 32). Cell types with nominally significant (P value < 0.05) higher or lower expression of the test genes are plotted and labeled as red or blue dots, respectively. Cell types with Bonferroni corrected significance are highlighted by labels with yellow background. Abbreviations of cell types with nominally significant difference in expression: Endo (f), endothelial cells (fetal); SCT, syncytiotrophoblast (fetal); ILC, innate lymphocyte cells (maternal); HB, Hofbauer cells (fetal); fFB1 and fFB2, fibroblasts (fetal); Epi1 and Epi2, epithelial glandular cells (unassigned or maternal); dM3, Maternal macrophages (maternal cell in placenta); EVT, extravillous trophoblast; DC1, dendritic cells; T cells, T cells (maternal or fetal).\n",
      "4\n",
      "\n",
      "image 1/1 /Users/etowah/projects/PMC-figure-downloader/img/10632150_Fig15.jpg: 640x640 4 labels, 89.4ms\n",
      "Speed: 2.2ms preprocess, 89.4ms inference, 0.4ms postprocess per image at shape (1, 3, 640, 640)\n",
      "1\n",
      "The panels show associations between quantiles of fetal polygenic scores for placental weight and standardized observed placental weight in the iPSYCH cohort (n = 33,035). Points represent association effect estimates and error bars show 95% confidence intervals. Black dots show associations for the population representative sample used as controls in iPSYCH, and blue dots show associations for cases of four different neuropsychiatric diseases. Abbreviations: CI, confidence interval; ADHD, attention deficit/hyperactivity disorder; ASD, autism spectrum disorder; PGS, polygenic score.\n",
      "1\n",
      "The figure shows odds ratios (ORs) from logistic regressions of four neuropsychiatric diseases on standardized observed placental weight (upper panel) and fetal polygenic score of placental weight (lower panel). The ORs correspond to a change of one standard deviation in standardized observed placental weight or PGS for placental weight. Abbreviations: CI, confidence interval; ADHD, attention deficit/hyperactivity disorder; ASD, autism spectrum disorder; OR, odds ratio; PGS, polygenic score.\n",
      "4\n",
      "\n",
      "image 1/1 /Users/etowah/projects/PMC-figure-downloader/img/10632148_Fig1.jpg: 608x640 4 labels, 112.4ms\n",
      "Speed: 2.3ms preprocess, 112.4ms inference, 0.4ms postprocess per image at shape (1, 3, 608, 640)\n",
      "3\n",
      "\n",
      "image 1/1 /Users/etowah/projects/PMC-figure-downloader/img/10632148_Fig2.jpg: 480x640 3 labels, 78.1ms\n",
      "Speed: 1.7ms preprocess, 78.1ms inference, 0.4ms postprocess per image at shape (1, 3, 480, 640)\n",
      "2\n",
      "\n",
      "image 1/1 /Users/etowah/projects/PMC-figure-downloader/img/10632148_Fig3.jpg: 640x544 2 labels, 91.3ms\n",
      "Speed: 2.2ms preprocess, 91.3ms inference, 0.7ms postprocess per image at shape (1, 3, 640, 544)\n",
      "2\n",
      "\n",
      "image 1/1 /Users/etowah/projects/PMC-figure-downloader/img/10632148_Fig4.jpg: 640x544 2 labels, 87.3ms\n",
      "Speed: 3.2ms preprocess, 87.3ms inference, 1.5ms postprocess per image at shape (1, 3, 640, 544)\n",
      "2\n",
      "\n",
      "image 1/1 /Users/etowah/projects/PMC-figure-downloader/img/10632148_Fig5.jpg: 352x640 2 labels, 71.9ms\n",
      "Speed: 1.4ms preprocess, 71.9ms inference, 0.4ms postprocess per image at shape (1, 3, 352, 640)\n",
      "3\n",
      "\n",
      "image 1/1 /Users/etowah/projects/PMC-figure-downloader/img/10632148_Fig6.jpg: 288x640 3 labels, 53.1ms\n",
      "Speed: 1.2ms preprocess, 53.1ms inference, 0.6ms postprocess per image at shape (1, 3, 288, 640)\n",
      "1\n",
      "PPE surgery was performed on day 0 and maximal aortic diameter was monitored by B-mode ultrasound. Knockout of Pcsk9 resulted in blunted AAA growth starting at experimental day 7 (PPE ctrl group n = 6; Pcsk9−/− group n = 6). Graphs show aortic aneurysm diameter percentage increase versus baseline (means ± s.e.m.). Two-sided P values from the Mann–Whitney test were reported.\n",
      "1\n",
      "The AAA GWAS loci for which we have identified a candidate causal gene are depicted along with the plausible relationship to its underlying biological mechanism. Loci names are based on the candidate causal gene identified in our analysis. However, the biological pathway(s) remain unclear for many associated loci and, as such, the resultant annotation may prove incorrect in some cases. Gene names are colored according to the gene prioritization scheme used to obtain the candidate (red, protein-altering variants; blue, consensus of indicators and green, nearest gene).\n",
      "1\n",
      "Top left, input consists of disease diagnoses as a function of age. Top right, ATM assigns a topic weight to each patient. Bottom, ATM infers age-dependent topic loadings.\n",
      "3\n",
      "\n",
      "image 1/1 /Users/etowah/projects/PMC-figure-downloader/img/10632146_Fig2.jpg: 640x448 3 labels, 60.6ms\n",
      "Speed: 1.9ms preprocess, 60.6ms inference, 0.8ms postprocess per image at shape (1, 3, 640, 448)\n",
      "1\n",
      "We report topic loadings averaged across younger ages (age at diagnosis <60 years) and older ages (age at diagnosis >60 years). Row labels denote disease categories ordered by Phecode systems, with alternating blue and red colors for visualization purposes; ‘Other’ is a merge of five Phecode systems: ‘congenital anomalies’, ‘symptoms’, ‘injuries and poisoning’, ‘other tests’ and ‘death’ (which is treated as an additional disease; Methods). Topics are ordered by the corresponding Phecode system. Further details on the ten topics are provided in Table 1. Further details on the diseases discussed in the text (type 2 diabetes and breast cancer) are provided in Extended Data Fig. 2. Numerical results are reported in Supplementary Table 4.\n",
      "3\n",
      "\n",
      "image 1/1 /Users/etowah/projects/PMC-figure-downloader/img/10632146_Fig4.jpg: 576x640 3 labels, 71.9ms\n",
      "Speed: 2.3ms preprocess, 71.9ms inference, 0.4ms postprocess per image at shape (1, 3, 576, 640)\n",
      "3\n",
      "\n",
      "image 1/1 /Users/etowah/projects/PMC-figure-downloader/img/10632146_Fig5.jpg: 640x640 3 labels, 99.4ms\n",
      "Speed: 2.6ms preprocess, 99.4ms inference, 0.6ms postprocess per image at shape (1, 3, 640, 640)\n",
      "3\n",
      "\n",
      "image 1/1 /Users/etowah/projects/PMC-figure-downloader/img/10632146_Fig6.jpg: 448x640 3 labels, 66.1ms\n",
      "Speed: 1.8ms preprocess, 66.1ms inference, 0.9ms postprocess per image at shape (1, 3, 448, 640)\n",
      "2\n",
      "\n",
      "image 1/1 /Users/etowah/projects/PMC-figure-downloader/img/10632146_Fig7.jpg: 576x640 2 labels, 83.7ms\n",
      "Speed: 2.0ms preprocess, 83.7ms inference, 0.5ms postprocess per image at shape (1, 3, 576, 640)\n",
      "1\n",
      "M is the number of subjects; Ns is the number of records within sth subject. All plates (circles) are variables in the generative process, where the plates with shade w is the observed variable and plates without shade are unobserved variables to be inferred; β is the topic weight; z is diagnosis-specific topic probability; t is the age at diagnosis for each diagnosis; β is the topic loadings, which are functions of age t; α is the (non-informative) hyperparameter of the prior distribution of θ. The generative process is described in the Methods and Supplementary Note.\n",
      "1\n",
      "The figure has the same legends as Fig. 3 but focusing on 52 diseases that have a subtype with at least 500 incidences. Distribution of average topic assignment for these 52 diseases is reported in Supplementary Fig. 15.\n",
      "1\n",
      "Each dot represents results from running either ATM or LDA on the same random training and testing split. The models were run with different topic numbers, and we chose a cubic spline with one knot for configuring ATM topic loadings. The prediction odds ratios are computed on the testing data using topic loadings inferred from the training data and topic weights inferred using previous diseases of testing individuals. The odds ratios are between the odds that target diseases are within model-predicted top percentile disease set versus the odds that target diseases are within the prevalence-ordered top percentile disease set. For the optimal model with 10 topics, ATM has an average prediction odds ratio 1.71 (across 10 random training-testing splits); LDA has an average prediction odds ratio 1.58 (across 10 random training-testing splits). Box plots show the distributions of the dots; center, box bounds, and whisker ends denote median, quartiles, and minima/maxima.\n",
      "1\n",
      "The figure is the same as Fig. 3 but comparing the topics that are inferred from female and male populations separately.\n",
      "1\n",
      "Each box of the heatmap shows the correlation of average diagnosis-specific topic probability between a disease from All of Us and the other disease from UK Biobank. The diagnosis-specific topic probabilities from All of Us were mapped to UK Biobank based on proportional variance between the two topic spaces (Methods).\n",
      "1\n",
      "The bar plot shows the estimated changes in s.d. of PRS per unit changes in the patient topic weight in disease cases. The PRS is estimated using all the cases in British Isle Ancestry. Error bars denote the 95% confidence interval. The stars show disease-topic pairs that are significant at FDR = 0.05. Numerical results are reported in Supplementary Table 13.\n",
      "1\n",
      "P-values are for testing case-FST significantly higher than control-FST of similar topic weight distribution. The permutation controls are sampled for 1,000 times with the same topic weights distribution and sample size to the disease subtypes. We focus on 49 of the 52 diseases that have more than one subgroup of at least 500 cases. Subtypes are defined based on the maximum value of the diagnosis-specific topic probability. Three diseases (‘hypertension’, ‘hypercholesterolemia’, and ‘arthropathy’) are excluded as there are not enough controls that match the topic weights of cases. The color shows the value of FST across subtypes. Exact P-values are reported in Supplementary Table 16.\n",
      "1\n",
      "For each example, we report main SNP effects (log odds ratios) specific to each quartile of topic weights across individuals, for both the focal SNP (blue dots) and background SNPs for that disease and topic (genome-wide significant main effect (P < 5 × 10−8) but non-significant SNP × topic interaction effect (P > 0.05); gray dots). Dashed red lines denote aggregate main SNP effects for each focal SNP. Error bars denote 95% confidence intervals. Gray lines denote linear regression of gray dots, with gray shading denoting corresponding 95% confidence intervals. P-values for interaction are for testing the interaction regression coefficients; P-values for top/bottom differences are for two-sided t-test. Panels A–D are examples for type 2 diabetes, asthma, hypertension and hypothyroidism, respectively. Numerical results are reported in Supplementary Table 18.\n",
      "3\n",
      "\n",
      "image 1/1 /Users/etowah/projects/PMC-figure-downloader/img/10632138_Fig1.jpg: 256x640 3 labels, 55.9ms\n",
      "Speed: 1.0ms preprocess, 55.9ms inference, 0.4ms postprocess per image at shape (1, 3, 256, 640)\n",
      "3\n",
      "\n",
      "image 1/1 /Users/etowah/projects/PMC-figure-downloader/img/10632138_Fig2.jpg: 512x640 3 labels, 143.0ms\n",
      "Speed: 2.4ms preprocess, 143.0ms inference, 0.7ms postprocess per image at shape (1, 3, 512, 640)\n",
      "6\n",
      "\n",
      "image 1/1 /Users/etowah/projects/PMC-figure-downloader/img/10632138_Fig3.jpg: 640x608 6 labels, 82.4ms\n",
      "Speed: 2.3ms preprocess, 82.4ms inference, 0.4ms postprocess per image at shape (1, 3, 640, 608)\n",
      "2\n",
      "\n",
      "image 1/1 /Users/etowah/projects/PMC-figure-downloader/img/10632138_Fig4.jpg: 256x640 2 labels, 54.8ms\n",
      "Speed: 1.1ms preprocess, 54.8ms inference, 0.5ms postprocess per image at shape (1, 3, 256, 640)\n",
      "7\n",
      "\n",
      "image 1/1 /Users/etowah/projects/PMC-figure-downloader/img/10632138_Fig5.jpg: 640x544 7 labels, 94.1ms\n",
      "Speed: 3.0ms preprocess, 94.1ms inference, 0.5ms postprocess per image at shape (1, 3, 640, 544)\n",
      "6\n",
      "\n",
      "image 1/1 /Users/etowah/projects/PMC-figure-downloader/img/10632138_Fig6.jpg: 640x512 6 labels, 71.4ms\n",
      "Speed: 2.1ms preprocess, 71.4ms inference, 0.6ms postprocess per image at shape (1, 3, 640, 512)\n",
      "1\n",
      "Scatterplot of differential abundance log-Fold Change (DA logFC) against fraction of out-of-reference (OOR) cells for each neighbourhood, in simulations with different removed OOR cell populations (indicated in y-axis). Colored points indicate neighbourhoods where the enrichment was significant (10% SpatialFDR, logFC > 0). The dotted red line indicates the threshold used to define true positives for precision-recall analysis (20% of the higher fraction in the simulation).\n",
      "1\n",
      "Quantification of overlap between cell type labels (as a measure of biological conservation, left) and sample IDs (as a measure of batch effect, right) and clusters of disease cells on latent dimensions after scArches mapping with different designs (color). The overlap between clusters and covariates is measured by the Normalised Mutual Information (NMI), using the implementation in scikit-learn v1.1.2 (ref. 74). Each box plot shows the median and interquartile range for simulations with different OOR cell populations (n = 15 simulations). NMI values for leiden clustering with increasing resolution (x-axis) are shown. In boxplots the center line denotes the median; box limits, first and third quartiles; whiskers, 1.5X interquartile range.\n",
      "1\n",
      "Boxplots of false discovery rate (FDR), false positive rate (FPR), true positive rate (TPR) and Area Under the Precision-Recall Curve (AUPRC) for detection of OOR cell states with different reference designs (boxplot colour) using 3 different methods for differential cell abundance analysis: co-varying neighbourhood analysis (CNA), MELD and Milo. Points represent simulations with different OOR populations (n = 8, selecting OOR states with at least 250 cells). Tests on the same simulated data are connected. In boxplots the center line denotes the median; box limits, first and third quartiles; whiskers, 1.5X interquartile range.\n",
      "1\n",
      "Scatterplot of number of cells in the simulated OOR state (x-axis) against the true positive rate (TPR, y-axis) of OOR state detection with alternative reference designs.\n",
      "4\n",
      "\n",
      "image 1/1 /Users/etowah/projects/PMC-figure-downloader/img/10632138_Fig11.jpg: 640x512 4 labels, 87.5ms\n",
      "Speed: 2.1ms preprocess, 87.5ms inference, 0.8ms postprocess per image at shape (1, 3, 640, 512)\n",
      "7\n",
      "\n",
      "image 1/1 /Users/etowah/projects/PMC-figure-downloader/img/10632138_Fig12.jpg: 640x608 7 labels, 88.3ms\n",
      "Speed: 2.6ms preprocess, 88.3ms inference, 0.9ms postprocess per image at shape (1, 3, 640, 608)\n",
      "5\n",
      "\n",
      "image 1/1 /Users/etowah/projects/PMC-figure-downloader/img/10632138_Fig13.jpg: 512x640 5 labels, 82.4ms\n",
      "Speed: 2.4ms preprocess, 82.4ms inference, 0.4ms postprocess per image at shape (1, 3, 512, 640)\n",
      "4\n",
      "\n",
      "image 1/1 /Users/etowah/projects/PMC-figure-downloader/img/10632138_Fig14.jpg: 640x480 4 labels, 60.9ms\n",
      "Speed: 1.7ms preprocess, 60.9ms inference, 0.7ms postprocess per image at shape (1, 3, 640, 480)\n",
      "2\n",
      "\n",
      "image 1/1 /Users/etowah/projects/PMC-figure-downloader/img/10632138_Fig15.jpg: 640x448 2 labels, 59.9ms\n",
      "Speed: 1.9ms preprocess, 59.9ms inference, 0.4ms postprocess per image at shape (1, 3, 640, 448)\n",
      "1\n",
      "The graph shows data for migraine (ncase/control = 74,495/1,259,808), MA (ncase/control = 16,603/1,336,517), MO (ncase/control = 11,718/1,330,747), VD (ncase/control = 30,297/86,134) and BRH (ncase/control = 51,803/123,732). See Supplementary Table 1 for ncase/control for each cohort. On the x axis, variants are plotted along the 22 autosomes and the X chromosome. On the y axis is the statistical significance of their association with the respective phenotypes from meta-analyses using a fixed-effects inverse-variance method based on effect estimates and s.e. under the additive model, in which each dataset was assumed to have a common OR but allowed to have different population frequencies for alleles and genotypes. Gray dots are not significant variants. Variant associations that reach the P threshold weighted by variant annotation21 are represented by color-coded dots. Adjacent chromosomes are presented in different shades of gray. Known migraine loci are represented by gene names in black text, and new loci are represented by gene names in blue text.\n",
      "1\n",
      "The x axis (VD, ncase/control = 30,297/86,134) and the y axis (MA, ncase/control = 16,603/1,336,517; migraine, ncase/control = 74,495/1,259,808 and MO, ncase/control = 11,718/1,330,747) show the logarithmic estimated odds ratios, log(OR), for the associations with the respective phenotypes from meta-analyses using a fixed-effects inverse-variance method based on effect estimates and s.e. under the additive model, in which each dataset was assumed to have a common OR but allowed to have different population frequencies for alleles and genotypes. All effects are shown for the VD risk allele, and black crosses indicate 95% CIs. The dashed red lines represent slope (s.d.) based on a simple linear regression through the origin using 1/s.e. as weights. Effect estimates are 73%, 29% and 0% of VD effect estimates for MA, migraine and MO, respectively.\n",
      "2\n",
      "\n",
      "image 1/1 /Users/etowah/projects/PMC-figure-downloader/img/10632135_Fig3.jpg: 320x640 2 labels, 79.5ms\n",
      "Speed: 1.9ms preprocess, 79.5ms inference, 0.4ms postprocess per image at shape (1, 3, 320, 640)\n",
      "1\n",
      "Effect–effect plot of clinically defined migraine (ncase/control = 74,495/1,259,808) vs. self-reported BRH (ncase/control = 51,803/123,732) effects for 42 lead variants identified in this study (excluding high-impact variants in PRRT2 and A3GALT2; see Supplementary Table 7 for their associations with the respective phenotypes). Effects are from meta-analyses using a fixed-effects inverse-variance method based on effect estimates and s.e. under the additive model, in which each dataset was assumed to have a common OR but allowed to have different population frequencies for alleles and genotypes. The x axis and the y axis show the logarithmic estimated ORs for the associations with the respective phenotypes. Error bars represent 95% CI. The dashed red lines represent slope (s.d.) based on a simple linear regression through the origin using 1/s.e. as weights. Cohort descriptions are in Supplementary Table 1. Variants are colored according to their primary associations in this study. The red dot outlier depicts the variant rs72854118-G near KCNK5, its effects on BRH exceeding its effects on all migraine. Pheno, phenotype; Migr, migraine.\n",
      "1\n",
      "Data are presented as OR for fracture per effect allele, with 95% CIs (for SOST, rs80107551-C; ESR1, rs2941741-G; WNT16, rs2908007-A; SALL1, rs62028332-G). OR for hip fractures are from Nethander et al.13 (11,516 cases and 723,838 controls), while OR for any fractures are from Morris et al.10 (53,184 cases and 373,611 controls). Statistically significant different associations for forearm fracture compared with the corresponding associations with hip fracture or any fracture are indicated with P values. Two-sided z test was used to test differences and the statistical significance limit was set to 0.0005 (Bonferroni adjustment considering 50 SNPs and two traits).\n",
      "12\n",
      "\n",
      "image 1/1 /Users/etowah/projects/PMC-figure-downloader/img/10632131_Fig2.jpg: 544x640 12 labels, 73.5ms\n",
      "Speed: 2.0ms preprocess, 73.5ms inference, 0.4ms postprocess per image at shape (1, 3, 544, 640)\n",
      "1\n",
      "Representative maximum intensity projection (×40 magnification) images of mouse tibiae labeled with fluorescence RNAscope for Tac4 (red) plus Runx2, Sost or Ctsk (green) mRNA and DAPI (blue). Tac4 and Runx2 or Sost double-positive osteocytes (solid arrows) can be detected both in the trabecular bone (TB) and cortical bone (CB). Tac4 mRNA also can be found in Runx2-expressing osteoblasts (open arrows) and Ctsk-expressing osteoclasts (solid arrowheads). Representative maximum intensity projection images are shown of mouse tibiae; n = 3 biologically independent mice.\n",
      "1\n",
      "Data are given as OR with 95% CIs estimated using IVW MR. OR for the risk of fracture are given per s.d. change in the risk factor for continuous trait or risk of fracture per doubling of odds (obtained by multiplying the causal estimate of log odds by ln2 ≈ 0.693) (ref. 41) of disease susceptibility for binary factors. For menopause and puberty, we used the estimated s.d. from the largest cohorts in the published GWAS (early menopause s.d. = 3.81 in Breast Cancer Association Consortium29; late puberty s.d. = 1.40 in Women’s Genome Health Study30) to translate the effect unit from year to s.d. For ever smoked regularly, ORs are expressed per unit increase in log odds of ever smoking regularly with a 1 s.d. increase in genetically predicted smoking initiation corresponding to a 10% increased risk of smoking38,42. Grip strength is given as grip strength per BW and s.d. for grip strength is given for kg grip strength per kg in BW and was estimated in the UK Biobank to be 0.127. P < 0.0029 (Bonferroni correction accounting for 17 tests) was considered statistically significant. Findings with a nonadjusted P value (based on a two-sided z test) below 0.0029 have their corresponding P value represented in the figure. For risk factors including UK Biobank in the GWAS and displaying statistically significant causal effects with forearm fractures, we performed sensitivity analyses excluding UK Biobank in the forearm fracture meta-analysis used for the MR, revealing essentially similar effect estimates (results excluding UK Biobank in the forearm fracture GWAS; decreased eBMD OR = 2.07; 95% CI, 1.96–2.20; P = 1.0 × 10−137; early menopause OR = 1.10; 95% CI, 1.02–1.18; P = 1.5 × 10−2).\n",
      "2\n",
      "\n",
      "image 1/1 /Users/etowah/projects/PMC-figure-downloader/img/10562256_Fig1.jpg: 576x640 2 labels, 77.3ms\n",
      "Speed: 2.4ms preprocess, 77.3ms inference, 0.4ms postprocess per image at shape (1, 3, 576, 640)\n",
      "2\n",
      "\n",
      "image 1/1 /Users/etowah/projects/PMC-figure-downloader/img/10562256_Fig2.jpg: 608x640 2 labels, 130.0ms\n",
      "Speed: 2.5ms preprocess, 130.0ms inference, 0.7ms postprocess per image at shape (1, 3, 608, 640)\n",
      "6\n",
      "\n",
      "image 1/1 /Users/etowah/projects/PMC-figure-downloader/img/10562256_Fig3.jpg: 576x640 6 labels, 96.6ms\n",
      "Speed: 2.5ms preprocess, 96.6ms inference, 1.6ms postprocess per image at shape (1, 3, 576, 640)\n",
      "6\n",
      "\n",
      "image 1/1 /Users/etowah/projects/PMC-figure-downloader/img/10562256_Fig4.jpg: 544x640 6 labels, 78.7ms\n",
      "Speed: 2.3ms preprocess, 78.7ms inference, 1.0ms postprocess per image at shape (1, 3, 544, 640)\n",
      "3\n",
      "\n",
      "image 1/1 /Users/etowah/projects/PMC-figure-downloader/img/10562256_Fig5.jpg: 640x576 3 labels, 82.3ms\n",
      "Speed: 2.4ms preprocess, 82.3ms inference, 0.4ms postprocess per image at shape (1, 3, 640, 576)\n",
      "4\n",
      "\n",
      "image 1/1 /Users/etowah/projects/PMC-figure-downloader/img/10562256_Fig6.jpg: 544x640 4 labels, 76.3ms\n",
      "Speed: 2.2ms preprocess, 76.3ms inference, 1.1ms postprocess per image at shape (1, 3, 544, 640)\n",
      "1\n",
      "Each column represents a sample in our study with tissue name on the top, and each row represents a sample from the reference epigenomes; for each sample in our study, the top five highly correlated reference samples are labeled with ‘*’; orange, red, green and blue boxes indicate tissue-matched pairs between our data and the reference data.\n",
      "3\n",
      "\n",
      "image 1/1 /Users/etowah/projects/PMC-figure-downloader/img/10562256_Fig8.jpg: 640x480 3 labels, 72.1ms\n",
      "Speed: 1.9ms preprocess, 72.1ms inference, 0.4ms postprocess per image at shape (1, 3, 640, 480)\n",
      "2\n",
      "\n",
      "image 1/1 /Users/etowah/projects/PMC-figure-downloader/img/10562256_Fig9.jpg: 640x480 2 labels, 78.5ms\n",
      "Speed: 2.0ms preprocess, 78.5ms inference, 0.8ms postprocess per image at shape (1, 3, 640, 480)\n",
      "2\n",
      "\n",
      "image 1/1 /Users/etowah/projects/PMC-figure-downloader/img/10562256_Fig10.jpg: 640x416 1 label, 78.3ms\n",
      "Speed: 1.9ms preprocess, 78.3ms inference, 0.8ms postprocess per image at shape (1, 3, 640, 416)\n",
      "Subfigure count mismatch: 1 detected, 2 expected\n",
      "3\n",
      "\n",
      "image 1/1 /Users/etowah/projects/PMC-figure-downloader/img/10562256_Fig11.jpg: 640x480 3 labels, 110.4ms\n",
      "Speed: 3.4ms preprocess, 110.4ms inference, 0.6ms postprocess per image at shape (1, 3, 640, 480)\n",
      "4\n",
      "\n",
      "image 1/1 /Users/etowah/projects/PMC-figure-downloader/img/10562256_Fig12.jpg: 640x576 4 labels, 114.5ms\n",
      "Speed: 2.8ms preprocess, 114.5ms inference, 1.2ms postprocess per image at shape (1, 3, 640, 576)\n",
      "5\n",
      "\n",
      "image 1/1 /Users/etowah/projects/PMC-figure-downloader/img/10562256_Fig13.jpg: 640x576 5 labels, 79.4ms\n",
      "Speed: 2.8ms preprocess, 79.4ms inference, 0.5ms postprocess per image at shape (1, 3, 640, 576)\n",
      "3\n",
      "\n",
      "image 1/1 /Users/etowah/projects/PMC-figure-downloader/img/10562256_Fig14.jpg: 640x544 3 labels, 73.1ms\n",
      "Speed: 1.8ms preprocess, 73.1ms inference, 0.5ms postprocess per image at shape (1, 3, 640, 544)\n",
      "5\n",
      "\n",
      "image 1/1 /Users/etowah/projects/PMC-figure-downloader/img/10562256_Fig15.jpg: 640x480 7 labels, 75.7ms\n",
      "Speed: 2.0ms preprocess, 75.7ms inference, 0.7ms postprocess per image at shape (1, 3, 640, 480)\n",
      "Subfigure count mismatch: 6 detected, 5 expected\n",
      "6\n",
      "\n",
      "image 1/1 /Users/etowah/projects/PMC-figure-downloader/img/10562256_Fig16.jpg: 640x512 6 labels, 79.6ms\n",
      "Speed: 2.0ms preprocess, 79.6ms inference, 0.5ms postprocess per image at shape (1, 3, 640, 512)\n",
      "5\n",
      "\n",
      "image 1/1 /Users/etowah/projects/PMC-figure-downloader/img/10562255_Fig1.jpg: 448x640 5 labels, 60.7ms\n",
      "Speed: 1.7ms preprocess, 60.7ms inference, 0.4ms postprocess per image at shape (1, 3, 448, 640)\n",
      "12\n",
      "\n",
      "image 1/1 /Users/etowah/projects/PMC-figure-downloader/img/10562255_Fig2.jpg: 576x640 12 labels, 102.7ms\n",
      "Speed: 2.0ms preprocess, 102.7ms inference, 0.6ms postprocess per image at shape (1, 3, 576, 640)\n",
      "15\n",
      "\n",
      "image 1/1 /Users/etowah/projects/PMC-figure-downloader/img/10562255_Fig3.jpg: 640x544 15 labels, 74.7ms\n",
      "Speed: 2.2ms preprocess, 74.7ms inference, 0.4ms postprocess per image at shape (1, 3, 640, 544)\n",
      "4\n",
      "\n",
      "image 1/1 /Users/etowah/projects/PMC-figure-downloader/img/10562255_Fig4.jpg: 416x640 4 labels, 74.4ms\n",
      "Speed: 2.0ms preprocess, 74.4ms inference, 0.4ms postprocess per image at shape (1, 3, 416, 640)\n",
      "11\n",
      "\n",
      "image 1/1 /Users/etowah/projects/PMC-figure-downloader/img/10562255_Fig5.jpg: 640x544 11 labels, 79.3ms\n",
      "Speed: 2.0ms preprocess, 79.3ms inference, 0.7ms postprocess per image at shape (1, 3, 640, 544)\n",
      "Subfigure count mismatch: 10 detected, 11 expected\n",
      "7\n",
      "\n",
      "image 1/1 /Users/etowah/projects/PMC-figure-downloader/img/10562255_Fig6.jpg: 640x576 7 labels, 117.6ms\n",
      "Speed: 2.1ms preprocess, 117.6ms inference, 1.0ms postprocess per image at shape (1, 3, 640, 576)\n",
      "9\n",
      "\n",
      "image 1/1 /Users/etowah/projects/PMC-figure-downloader/img/10562255_Fig7.jpg: 640x512 9 labels, 81.1ms\n",
      "Speed: 2.2ms preprocess, 81.1ms inference, 0.6ms postprocess per image at shape (1, 3, 640, 512)\n",
      "8\n",
      "\n",
      "image 1/1 /Users/etowah/projects/PMC-figure-downloader/img/10562255_Fig8.jpg: 640x544 8 labels, 71.1ms\n",
      "Speed: 2.2ms preprocess, 71.1ms inference, 0.4ms postprocess per image at shape (1, 3, 640, 544)\n",
      "6\n",
      "\n",
      "image 1/1 /Users/etowah/projects/PMC-figure-downloader/img/10562255_Fig9.jpg: 640x576 6 labels, 77.1ms\n",
      "Speed: 2.1ms preprocess, 77.1ms inference, 0.5ms postprocess per image at shape (1, 3, 640, 576)\n",
      "14\n",
      "\n",
      "image 1/1 /Users/etowah/projects/PMC-figure-downloader/img/10562255_Fig10.jpg: 608x640 14 labels, 75.4ms\n",
      "Speed: 2.1ms preprocess, 75.4ms inference, 0.4ms postprocess per image at shape (1, 3, 608, 640)\n",
      "7\n",
      "\n",
      "image 1/1 /Users/etowah/projects/PMC-figure-downloader/img/10562255_Fig11.jpg: 640x640 7 labels, 93.6ms\n",
      "Speed: 2.6ms preprocess, 93.6ms inference, 0.7ms postprocess per image at shape (1, 3, 640, 640)\n",
      "11\n",
      "\n",
      "image 1/1 /Users/etowah/projects/PMC-figure-downloader/img/10562255_Fig12.jpg: 640x544 11 labels, 72.8ms\n",
      "Speed: 1.9ms preprocess, 72.8ms inference, 0.4ms postprocess per image at shape (1, 3, 640, 544)\n",
      "7\n",
      "\n",
      "image 1/1 /Users/etowah/projects/PMC-figure-downloader/img/10562255_Fig13.jpg: 576x640 7 labels, 81.1ms\n",
      "Speed: 2.2ms preprocess, 81.1ms inference, 0.5ms postprocess per image at shape (1, 3, 576, 640)\n",
      "5\n",
      "\n",
      "image 1/1 /Users/etowah/projects/PMC-figure-downloader/img/10562255_Fig14.jpg: 640x576 5 labels, 91.7ms\n",
      "Speed: 2.1ms preprocess, 91.7ms inference, 0.6ms postprocess per image at shape (1, 3, 640, 576)\n",
      "3\n",
      "\n",
      "image 1/1 /Users/etowah/projects/PMC-figure-downloader/img/10562255_Fig15.jpg: 416x640 4 labels, 61.4ms\n",
      "Speed: 1.7ms preprocess, 61.4ms inference, 0.5ms postprocess per image at shape (1, 3, 416, 640)\n",
      "Subfigure count mismatch: 4 detected, 3 expected\n",
      "4\n",
      "\n",
      "image 1/1 /Users/etowah/projects/PMC-figure-downloader/img/10562253_Fig1.jpg: 576x640 4 labels, 79.4ms\n",
      "Speed: 2.1ms preprocess, 79.4ms inference, 1.1ms postprocess per image at shape (1, 3, 576, 640)\n",
      "3\n",
      "\n",
      "image 1/1 /Users/etowah/projects/PMC-figure-downloader/img/10562253_Fig2.jpg: 640x608 3 labels, 84.0ms\n",
      "Speed: 2.2ms preprocess, 84.0ms inference, 0.4ms postprocess per image at shape (1, 3, 640, 608)\n",
      "4\n",
      "\n",
      "image 1/1 /Users/etowah/projects/PMC-figure-downloader/img/10562253_Fig3.jpg: 480x640 4 labels, 70.9ms\n",
      "Speed: 1.7ms preprocess, 70.9ms inference, 0.4ms postprocess per image at shape (1, 3, 480, 640)\n",
      "1\n",
      "The relationship between inferred fitness effect and odds ratio (OR) of any blood cancer is shown for autosomal mCAs with a non-zero OR14 and that were observed in ≥30 individuals (n = 41 mCAs) (see Supplementary Table 4). Red, gains; blue, losses; yellow, CN-LOH events. The mCAs highlighted in bold are those that Loh et al.14 determined to have a statistically significant increased risk of any blood cancer. Pearson’s R correlation coefficient, two-tailed P value and 95% confidence intervals (gray shaded area) are shown. The blood cancers were diagnosed >1 year after DNA collection (within 4–9 years of follow-up) in individuals with no previous cancer and with no restriction on blood counts at assessment.\n",
      "4\n",
      "\n",
      "image 1/1 /Users/etowah/projects/PMC-figure-downloader/img/10562253_Fig5.jpg: 544x640 4 labels, 71.2ms\n",
      "Speed: 2.1ms preprocess, 71.2ms inference, 0.5ms postprocess per image at shape (1, 3, 544, 640)\n",
      "1\n",
      "The mutation rate DFE for all classes of autosomal mCAs (Fig. 2b combined) is shown in light purple, the mutation rate DFE for SNVs across a targeted ‘cancer panel’ of ∼1.1 MB (inferred in Poon et al.6) is shown in dark purple. The mutation rate to fitness effects of 10–25% per year is 1.5 × 10−7 for autosomal mCAs and 1.5 × 10−6 for SNVs.\n",
      "1\n",
      "The cell fraction probability density histogram is shown for each mCA (datapoints) with the theory distribution (solid line) fitted using maximum likelihood approaches (red = gains, blue = losses, yellow = CN-LOH). Datapoints are presented as mean values +/− SEM. Grey vertical dashed line shows the fitted ϕ parameter \\documentclass[12pt]{minimal}\n",
      "\t\t\t\t\\usepackage{amsmath}\n",
      "\t\t\t\t\\usepackage{wasysym} \n",
      "\t\t\t\t\\usepackage{amsfonts} \n",
      "\t\t\t\t\\usepackage{amssymb} \n",
      "\t\t\t\t\\usepackage{amsbsy}\n",
      "\t\t\t\t\\usepackage{mathrsfs}\n",
      "\t\t\t\t\\usepackage{upgreek}\n",
      "\t\t\t\t\\setlength{\\oddsidemargin}{-69pt}\n",
      "\t\t\t\t\\begin{document}$$\\left(\\frac{{e}^{{st}}-1}{{Ns}}\\right)$$\\end{document}est−1Ns, where the exponential fall-off in densities occurs. The white cross on the maximum likelihood heatmap marks the most likely mutation rate (µ) and fitness effect (s). n = number of individuals (with a single mCA) in whom the mCA was detected.\n",
      "1\n",
      "The cell fraction probability density histogram is shown for each mCA (datapoints) with the theory distribution (solid line) fitted using maximum likelihood approaches (red = gains, blue = losses, yellow = CN-LOH). Datapoints are presented as mean values +/− SEM. Grey vertical dashed line shows the fitted ϕ parameter \\documentclass[12pt]{minimal}\n",
      "\t\t\t\t\\usepackage{amsmath}\n",
      "\t\t\t\t\\usepackage{wasysym} \n",
      "\t\t\t\t\\usepackage{amsfonts} \n",
      "\t\t\t\t\\usepackage{amssymb} \n",
      "\t\t\t\t\\usepackage{amsbsy}\n",
      "\t\t\t\t\\usepackage{mathrsfs}\n",
      "\t\t\t\t\\usepackage{upgreek}\n",
      "\t\t\t\t\\setlength{\\oddsidemargin}{-69pt}\n",
      "\t\t\t\t\\begin{document}$$\\left(\\frac{{e}^{{st}}-1}{{Ns}}\\right)$$\\end{document}est−1Ns, where the exponential fall-off in densities occurs. The white cross on the maximum likelihood heatmap marks the most likely mutation rate (µ) and fitness effect (s). n = number of individuals (with a single mCA) in whom the mCA was detected. Gains (+) are shown in red, losses (−) are shown in blue.\n",
      "1\n",
      "The cell fraction probability density histogram is shown for each mCA (datapoints) with the theory distribution (solid line) fitted using maximum likelihood approaches (red = gains, blue = losses, yellow = CN-LOH). Datapoints are presented as mean values +/− SEM. Grey vertical dashed line shows the fitted ϕ parameter \\documentclass[12pt]{minimal}\n",
      "\t\t\t\t\\usepackage{amsmath}\n",
      "\t\t\t\t\\usepackage{wasysym} \n",
      "\t\t\t\t\\usepackage{amsfonts} \n",
      "\t\t\t\t\\usepackage{amssymb} \n",
      "\t\t\t\t\\usepackage{amsbsy}\n",
      "\t\t\t\t\\usepackage{mathrsfs}\n",
      "\t\t\t\t\\usepackage{upgreek}\n",
      "\t\t\t\t\\setlength{\\oddsidemargin}{-69pt}\n",
      "\t\t\t\t\\begin{document}$$\\left(\\frac{{e}^{{st}}-1}{{Ns}}\\right)$$\\end{document}est−1Ns, where the exponential fall-off in densities occurs. The white cross on the maximum likelihood heatmap marks the most likely mutation rate (µ) and fitness effect (s). n = number of individuals (with a single mCA) in whom the mCA was detected.\n",
      "1\n",
      "The cell fraction probability density histogram is shown for each mCA (datapoints) with the theory distribution (solid line) fitted using maximum likelihood approaches (red = gains, blue = losses, yellow = CN-LOH). Datapoints are presented as mean values +/− SEM. Grey vertical dashed line shows the fitted ϕ parameter \\documentclass[12pt]{minimal}\n",
      "\t\t\t\t\\usepackage{amsmath}\n",
      "\t\t\t\t\\usepackage{wasysym} \n",
      "\t\t\t\t\\usepackage{amsfonts} \n",
      "\t\t\t\t\\usepackage{amssymb} \n",
      "\t\t\t\t\\usepackage{amsbsy}\n",
      "\t\t\t\t\\usepackage{mathrsfs}\n",
      "\t\t\t\t\\usepackage{upgreek}\n",
      "\t\t\t\t\\setlength{\\oddsidemargin}{-69pt}\n",
      "\t\t\t\t\\begin{document}$$\\left(\\frac{{e}^{{st}}-1}{{Ns}}\\right)$$\\end{document}est−1Ns, where the exponential fall-off in densities occurs. The white cross on the maximum likelihood heatmap marks the most likely mutation rate (µ) and fitness effect (s). n = number of individuals (with a single mCA) in whom the mCA was detected. Losses (−) are shown in blue and CN-LOH (=) events are shown in yellow.\n",
      "1\n",
      "The cell fraction probability density histogram is shown for each mCA (datapoints) with the theory distribution (solid line) fitted using maximum likelihood approaches (red = gains, blue = losses, yellow = CN-LOH). Datapoints are presented as mean values +/− SEM. Grey vertical dashed line shows the fitted ϕ parameter \\documentclass[12pt]{minimal}\n",
      "\t\t\t\t\\usepackage{amsmath}\n",
      "\t\t\t\t\\usepackage{wasysym} \n",
      "\t\t\t\t\\usepackage{amsfonts} \n",
      "\t\t\t\t\\usepackage{amssymb} \n",
      "\t\t\t\t\\usepackage{amsbsy}\n",
      "\t\t\t\t\\usepackage{mathrsfs}\n",
      "\t\t\t\t\\usepackage{upgreek}\n",
      "\t\t\t\t\\setlength{\\oddsidemargin}{-69pt}\n",
      "\t\t\t\t\\begin{document}$$\\left(\\frac{{e}^{{st}}-1}{{Ns}}\\right)$$\\end{document}est−1Ns, where the exponential fall-off in densities occurs. The white cross on the maximum likelihood heatmap marks the most likely mutation rate (µ) and fitness effect (s). n = number of individuals (with a single mCA) in whom the mCA was detected.\n",
      "1\n",
      "The cell fraction probability density histogram is shown for each mCA (datapoints) with the theory distribution (solid line) fitted using maximum likelihood approaches (red = gains, blue = losses, yellow = CN-LOH). Datapoints are presented as mean values +/− SEM. Grey vertical dashed line shows the fitted ϕ parameter \\documentclass[12pt]{minimal}\n",
      "\t\t\t\t\\usepackage{amsmath}\n",
      "\t\t\t\t\\usepackage{wasysym} \n",
      "\t\t\t\t\\usepackage{amsfonts} \n",
      "\t\t\t\t\\usepackage{amssymb} \n",
      "\t\t\t\t\\usepackage{amsbsy}\n",
      "\t\t\t\t\\usepackage{mathrsfs}\n",
      "\t\t\t\t\\usepackage{upgreek}\n",
      "\t\t\t\t\\setlength{\\oddsidemargin}{-69pt}\n",
      "\t\t\t\t\\begin{document}$$\\left(\\frac{{e}^{{st}}-1}{{Ns}}\\right)$$\\end{document}est−1Ns, where the exponential fall-off in densities occurs. The white cross on the maximum likelihood heatmap marks the most likely mutation rate (µ) and fitness effect (s). n = number of individuals (with a single mCA) in whom the mCA was detected.\n",
      "1\n",
      "The cell fraction probability density histogram is shown for each mCA (datapoints) with the theory distribution (solid line) fitted using maximum likelihood approaches (red = gains, blue = losses, yellow = CN-LOH). Datapoints are presented as mean values +/− SEM. Grey vertical dashed line shows the fitted ϕ parameter \\documentclass[12pt]{minimal}\n",
      "\t\t\t\t\\usepackage{amsmath}\n",
      "\t\t\t\t\\usepackage{wasysym} \n",
      "\t\t\t\t\\usepackage{amsfonts} \n",
      "\t\t\t\t\\usepackage{amssymb} \n",
      "\t\t\t\t\\usepackage{amsbsy}\n",
      "\t\t\t\t\\usepackage{mathrsfs}\n",
      "\t\t\t\t\\usepackage{upgreek}\n",
      "\t\t\t\t\\setlength{\\oddsidemargin}{-69pt}\n",
      "\t\t\t\t\\begin{document}$$\\left(\\frac{{e}^{{st}}-1}{{Ns}}\\right)$$\\end{document}est−1Ns, where the exponential fall-off in densities occurs. The white cross on the maximum likelihood heatmap marks the most likely mutation rate (µ) and fitness effect (s). n = number of individuals (with a single mCA) in whom the mCA was detected.\n",
      "4\n",
      "\n",
      "image 1/1 /Users/etowah/projects/PMC-figure-downloader/img/10562253_Fig14.jpg: 384x640 4 labels, 61.2ms\n",
      "Speed: 1.7ms preprocess, 61.2ms inference, 1.3ms postprocess per image at shape (1, 3, 384, 640)\n",
      "2\n",
      "\n",
      "image 1/1 /Users/etowah/projects/PMC-figure-downloader/img/10562253_Fig15.jpg: 192x640 2 labels, 39.9ms\n",
      "Speed: 0.8ms preprocess, 39.9ms inference, 0.3ms postprocess per image at shape (1, 3, 192, 640)\n",
      "2\n",
      "\n",
      "image 1/1 /Users/etowah/projects/PMC-figure-downloader/img/10562253_Fig16.jpg: 160x640 2 labels, 38.4ms\n",
      "Speed: 0.6ms preprocess, 38.4ms inference, 0.4ms postprocess per image at shape (1, 3, 160, 640)\n",
      "8\n",
      "\n",
      "image 1/1 /Users/etowah/projects/PMC-figure-downloader/img/10562252_Fig1.jpg: 640x640 8 labels, 85.9ms\n",
      "Speed: 2.3ms preprocess, 85.9ms inference, 0.6ms postprocess per image at shape (1, 3, 640, 640)\n",
      "14\n",
      "\n",
      "image 1/1 /Users/etowah/projects/PMC-figure-downloader/img/10562252_Fig2.jpg: 640x544 14 labels, 78.2ms\n",
      "Speed: 2.0ms preprocess, 78.2ms inference, 0.5ms postprocess per image at shape (1, 3, 640, 544)\n",
      "11\n",
      "\n",
      "image 1/1 /Users/etowah/projects/PMC-figure-downloader/img/10562252_Fig3.jpg: 480x640 11 labels, 76.6ms\n",
      "Speed: 1.9ms preprocess, 76.6ms inference, 0.5ms postprocess per image at shape (1, 3, 480, 640)\n",
      "5\n",
      "\n",
      "image 1/1 /Users/etowah/projects/PMC-figure-downloader/img/10562252_Fig4.jpg: 288x640 5 labels, 105.5ms\n",
      "Speed: 1.0ms preprocess, 105.5ms inference, 0.6ms postprocess per image at shape (1, 3, 288, 640)\n",
      "10\n",
      "\n",
      "image 1/1 /Users/etowah/projects/PMC-figure-downloader/img/10562252_Fig5.jpg: 352x640 10 labels, 61.6ms\n",
      "Speed: 1.4ms preprocess, 61.6ms inference, 0.5ms postprocess per image at shape (1, 3, 352, 640)\n",
      "5\n",
      "\n",
      "image 1/1 /Users/etowah/projects/PMC-figure-downloader/img/10562252_Fig6.jpg: 160x640 5 labels, 32.2ms\n",
      "Speed: 0.6ms preprocess, 32.2ms inference, 0.4ms postprocess per image at shape (1, 3, 160, 640)\n",
      "20\n",
      "\n",
      "image 1/1 /Users/etowah/projects/PMC-figure-downloader/img/10562252_Fig7.jpg: 640x544 20 labels, 71.7ms\n",
      "Speed: 2.1ms preprocess, 71.7ms inference, 0.4ms postprocess per image at shape (1, 3, 640, 544)\n",
      "18\n",
      "\n",
      "image 1/1 /Users/etowah/projects/PMC-figure-downloader/img/10562252_Fig8.jpg: 640x544 19 labels, 78.3ms\n",
      "Speed: 2.3ms preprocess, 78.3ms inference, 0.4ms postprocess per image at shape (1, 3, 640, 544)\n",
      "14\n",
      "\n",
      "image 1/1 /Users/etowah/projects/PMC-figure-downloader/img/10562252_Fig9.jpg: 640x448 14 labels, 64.8ms\n",
      "Speed: 2.1ms preprocess, 64.8ms inference, 0.7ms postprocess per image at shape (1, 3, 640, 448)\n",
      "8\n",
      "\n",
      "image 1/1 /Users/etowah/projects/PMC-figure-downloader/img/10562252_Fig10.jpg: 512x640 8 labels, 72.4ms\n",
      "Speed: 2.0ms preprocess, 72.4ms inference, 0.4ms postprocess per image at shape (1, 3, 512, 640)\n",
      "16\n",
      "\n",
      "image 1/1 /Users/etowah/projects/PMC-figure-downloader/img/10562252_Fig11.jpg: 640x512 16 labels, 74.1ms\n",
      "Speed: 2.1ms preprocess, 74.1ms inference, 0.5ms postprocess per image at shape (1, 3, 640, 512)\n",
      "10\n",
      "\n",
      "image 1/1 /Users/etowah/projects/PMC-figure-downloader/img/10562252_Fig12.jpg: 544x640 10 labels, 98.3ms\n",
      "Speed: 2.9ms preprocess, 98.3ms inference, 0.5ms postprocess per image at shape (1, 3, 544, 640)\n",
      "5\n",
      "\n",
      "image 1/1 /Users/etowah/projects/PMC-figure-downloader/img/10484791_Fig1.jpg: 640x544 5 labels, 73.4ms\n",
      "Speed: 2.0ms preprocess, 73.4ms inference, 0.4ms postprocess per image at shape (1, 3, 640, 544)\n",
      "10\n",
      "\n",
      "image 1/1 /Users/etowah/projects/PMC-figure-downloader/img/10484791_Fig2.jpg: 640x640 11 labels, 114.6ms\n",
      "Speed: 2.5ms preprocess, 114.6ms inference, 0.5ms postprocess per image at shape (1, 3, 640, 640)\n",
      "Subfigure count mismatch: 11 detected, 10 expected\n",
      "4\n",
      "\n",
      "image 1/1 /Users/etowah/projects/PMC-figure-downloader/img/10484791_Fig3.jpg: 448x640 4 labels, 63.8ms\n",
      "Speed: 1.6ms preprocess, 63.8ms inference, 0.4ms postprocess per image at shape (1, 3, 448, 640)\n",
      "12\n",
      "\n",
      "image 1/1 /Users/etowah/projects/PMC-figure-downloader/img/10484791_Fig4.jpg: 640x640 12 labels, 88.1ms\n",
      "Speed: 2.3ms preprocess, 88.1ms inference, 1.0ms postprocess per image at shape (1, 3, 640, 640)\n",
      "8\n",
      "\n",
      "image 1/1 /Users/etowah/projects/PMC-figure-downloader/img/10484791_Fig5.jpg: 480x640 8 labels, 74.9ms\n",
      "Speed: 2.3ms preprocess, 74.9ms inference, 0.4ms postprocess per image at shape (1, 3, 480, 640)\n",
      "6\n",
      "\n",
      "image 1/1 /Users/etowah/projects/PMC-figure-downloader/img/10484791_Fig6.jpg: 608x640 6 labels, 104.9ms\n",
      "Speed: 3.2ms preprocess, 104.9ms inference, 0.7ms postprocess per image at shape (1, 3, 608, 640)\n",
      "4\n",
      "\n",
      "image 1/1 /Users/etowah/projects/PMC-figure-downloader/img/10484791_Fig7.jpg: 320x640 4 labels, 51.3ms\n",
      "Speed: 1.3ms preprocess, 51.3ms inference, 0.4ms postprocess per image at shape (1, 3, 320, 640)\n",
      "5\n",
      "\n",
      "image 1/1 /Users/etowah/projects/PMC-figure-downloader/img/10484791_Fig8.jpg: 640x416 6 labels, 64.7ms\n",
      "Speed: 1.9ms preprocess, 64.7ms inference, 0.4ms postprocess per image at shape (1, 3, 640, 416)\n",
      "4\n",
      "\n",
      "image 1/1 /Users/etowah/projects/PMC-figure-downloader/img/10484791_Fig9.jpg: 640x448 5 labels, 62.4ms\n",
      "Speed: 1.9ms preprocess, 62.4ms inference, 0.4ms postprocess per image at shape (1, 3, 640, 448)\n",
      "5\n",
      "\n",
      "image 1/1 /Users/etowah/projects/PMC-figure-downloader/img/10484791_Fig10.jpg: 640x448 6 labels, 61.3ms\n",
      "Speed: 2.0ms preprocess, 61.3ms inference, 0.7ms postprocess per image at shape (1, 3, 640, 448)\n",
      "5\n",
      "\n",
      "image 1/1 /Users/etowah/projects/PMC-figure-downloader/img/10484791_Fig11.jpg: 640x448 5 labels, 75.5ms\n",
      "Speed: 1.6ms preprocess, 75.5ms inference, 0.4ms postprocess per image at shape (1, 3, 640, 448)\n",
      "7\n",
      "\n",
      "image 1/1 /Users/etowah/projects/PMC-figure-downloader/img/10484791_Fig12.jpg: 640x512 7 labels, 75.0ms\n",
      "Speed: 1.9ms preprocess, 75.0ms inference, 0.4ms postprocess per image at shape (1, 3, 640, 512)\n",
      "2\n",
      "\n",
      "image 1/1 /Users/etowah/projects/PMC-figure-downloader/img/10484791_Fig13.jpg: 640x448 3 labels, 69.0ms\n",
      "Speed: 1.8ms preprocess, 69.0ms inference, 0.6ms postprocess per image at shape (1, 3, 640, 448)\n",
      "4\n",
      "\n",
      "image 1/1 /Users/etowah/projects/PMC-figure-downloader/img/10484791_Fig14.jpg: 640x448 5 labels, 61.6ms\n",
      "Speed: 1.9ms preprocess, 61.6ms inference, 0.4ms postprocess per image at shape (1, 3, 640, 448)\n",
      "5\n",
      "\n",
      "image 1/1 /Users/etowah/projects/PMC-figure-downloader/img/10484791_Fig15.jpg: 640x480 5 labels, 77.3ms\n",
      "Speed: 2.0ms preprocess, 77.3ms inference, 0.5ms postprocess per image at shape (1, 3, 640, 480)\n",
      "8\n",
      "\n",
      "image 1/1 /Users/etowah/projects/PMC-figure-downloader/img/10484791_Fig16.jpg: 640x480 8 labels, 68.1ms\n",
      "Speed: 1.8ms preprocess, 68.1ms inference, 0.8ms postprocess per image at shape (1, 3, 640, 480)\n",
      "6\n",
      "\n",
      "image 1/1 /Users/etowah/projects/PMC-figure-downloader/img/10484791_Fig17.jpg: 640x512 4 labels, 73.3ms\n",
      "Speed: 2.0ms preprocess, 73.3ms inference, 0.4ms postprocess per image at shape (1, 3, 640, 512)\n",
      "Subfigure count mismatch: 4 detected, 6 expected\n",
      "10\n",
      "\n",
      "image 1/1 /Users/etowah/projects/PMC-figure-downloader/img/10484789_Fig1.jpg: 352x640 10 labels, 60.4ms\n",
      "Speed: 1.4ms preprocess, 60.4ms inference, 0.4ms postprocess per image at shape (1, 3, 352, 640)\n",
      "12\n",
      "\n",
      "image 1/1 /Users/etowah/projects/PMC-figure-downloader/img/10484789_Fig2.jpg: 640x416 12 labels, 63.6ms\n",
      "Speed: 1.7ms preprocess, 63.6ms inference, 0.4ms postprocess per image at shape (1, 3, 640, 416)\n",
      "11\n",
      "\n",
      "image 1/1 /Users/etowah/projects/PMC-figure-downloader/img/10484789_Fig3.jpg: 544x640 11 labels, 77.5ms\n",
      "Speed: 1.9ms preprocess, 77.5ms inference, 0.4ms postprocess per image at shape (1, 3, 544, 640)\n",
      "5\n",
      "\n",
      "image 1/1 /Users/etowah/projects/PMC-figure-downloader/img/10484789_Fig4.jpg: 416x640 5 labels, 62.3ms\n",
      "Speed: 1.6ms preprocess, 62.3ms inference, 0.5ms postprocess per image at shape (1, 3, 416, 640)\n",
      "13\n",
      "\n",
      "image 1/1 /Users/etowah/projects/PMC-figure-downloader/img/10484789_Fig5.jpg: 640x512 13 labels, 67.4ms\n",
      "Speed: 1.8ms preprocess, 67.4ms inference, 0.8ms postprocess per image at shape (1, 3, 640, 512)\n",
      "5\n",
      "\n",
      "image 1/1 /Users/etowah/projects/PMC-figure-downloader/img/10484789_Fig6.jpg: 640x512 6 labels, 70.5ms\n",
      "Speed: 2.1ms preprocess, 70.5ms inference, 0.4ms postprocess per image at shape (1, 3, 640, 512)\n",
      "10\n",
      "\n",
      "image 1/1 /Users/etowah/projects/PMC-figure-downloader/img/10484789_Fig7.jpg: 640x480 11 labels, 78.1ms\n",
      "Speed: 2.0ms preprocess, 78.1ms inference, 0.4ms postprocess per image at shape (1, 3, 640, 480)\n",
      "17\n",
      "\n",
      "image 1/1 /Users/etowah/projects/PMC-figure-downloader/img/10484789_Fig8.jpg: 640x480 17 labels, 85.7ms\n",
      "Speed: 2.0ms preprocess, 85.7ms inference, 0.4ms postprocess per image at shape (1, 3, 640, 480)\n",
      "14\n",
      "\n",
      "image 1/1 /Users/etowah/projects/PMC-figure-downloader/img/10484789_Fig9.jpg: 640x544 14 labels, 116.5ms\n",
      "Speed: 2.2ms preprocess, 116.5ms inference, 0.5ms postprocess per image at shape (1, 3, 640, 544)\n",
      "4\n",
      "\n",
      "image 1/1 /Users/etowah/projects/PMC-figure-downloader/img/10484789_Fig10.jpg: 448x640 4 labels, 71.6ms\n",
      "Speed: 1.6ms preprocess, 71.6ms inference, 0.4ms postprocess per image at shape (1, 3, 448, 640)\n",
      "14\n",
      "\n",
      "image 1/1 /Users/etowah/projects/PMC-figure-downloader/img/10484789_Fig11.jpg: 640x576 14 labels, 90.6ms\n",
      "Speed: 2.3ms preprocess, 90.6ms inference, 0.5ms postprocess per image at shape (1, 3, 640, 576)\n",
      "1\n",
      "a-b, Distribution of p53-LSC scores in BeatAML (a) and TCGA (b) cohorts stratified by TP53 mutational status. c-e, Kaplan-Meier analysis of de novo AML patients from the full TCGA AML dataset (n = 132) (c), TP53-WT AML patients from BeatAML (n = 294) (d) and TP53-WT AML patients from TCGA (n = 124) (e) stratified according to high or low p53 LSC signature score. f-g, Hazard ratio of all AML patients (n = 322) (f) or secondary AML patients (n = 49) (g) from the BeatAML cohort using LSC17 score (Ng et al, 2016), p53-all-cells score (derived from all TP53-mutant sAML cells) and p53-LSC signature score (derived from transcriptionally-defined LSCs; related to Fig. 2a). Boxes represent hazard ratios and lower and upper bounds of error bars, 95% confidence intervals. Genes used for each score are listed in Supplementary Table 4. “p” indicates log-rank test p-value.\n",
      "9\n",
      "\n",
      "image 1/1 /Users/etowah/projects/PMC-figure-downloader/img/10484789_Fig13.jpg: 576x640 9 labels, 82.4ms\n",
      "Speed: 2.1ms preprocess, 82.4ms inference, 0.4ms postprocess per image at shape (1, 3, 576, 640)\n",
      "6\n",
      "\n",
      "image 1/1 /Users/etowah/projects/PMC-figure-downloader/img/10484789_Fig14.jpg: 576x640 6 labels, 73.4ms\n",
      "Speed: 2.0ms preprocess, 73.4ms inference, 0.4ms postprocess per image at shape (1, 3, 576, 640)\n",
      "15\n",
      "\n",
      "image 1/1 /Users/etowah/projects/PMC-figure-downloader/img/10484789_Fig15.jpg: 640x480 16 labels, 64.6ms\n",
      "Speed: 2.0ms preprocess, 64.6ms inference, 1.2ms postprocess per image at shape (1, 3, 640, 480)\n",
      "Subfigure count mismatch: 16 detected, 15 expected\n",
      "14\n",
      "\n",
      "image 1/1 /Users/etowah/projects/PMC-figure-downloader/img/10484789_Fig16.jpg: 640x480 15 labels, 99.1ms\n",
      "Speed: 2.3ms preprocess, 99.1ms inference, 0.9ms postprocess per image at shape (1, 3, 640, 480)\n",
      "Subfigure count mismatch: 15 detected, 14 expected\n",
      "1\n",
      "The red line shows the genome-wide significance threshold (5 × 10−8). Chromosome and position are displayed on the x axis, and two-sided −log10\n",
      "P value is on the y axis. New genome-wide significant loci are highlighted in red, and loci previously associated with epilepsy in orange. New loci were those previously unreported as GWAS significant in previous epilepsy GWASs. Annotated genes are those implicated by our gene prioritization analyses. See Supplementary Fig. 7 for QQ plots. QQ plots, quantile–quantile plot.\n",
      "1\n",
      "The genetic correlation coefficient was calculated with LDSC and is denoted by color scale from −1 (red; negatively (anti-)correlated) to +1 (blue; positively correlated). The square size relates to the absolute value of the corresponding correlation coefficient. Single asterisk indicates two-sided P < 0.05 and double asterisk indicates two-sided P < 0.0009 (Bonferroni corrected).\n",
      "1\n",
      "The x axis is the chromosomal position, and the y axis is the meta-analyzed z score from testing H0: β = ln(OR) = 0 in the UK Biobank and BCAC datasets (two-tailed). The blue lines correspond to z = ±3.29, P = 0.001, the red lines correspond to z = ±4.71, P = 2.5 × 10−6. All labeled genes are those with P < 0.001. All P values are unadjusted for multiple testing.\n",
      "1\n",
      "P values are from the meta-analyzed z score from testing H0: β = ln(OR) = 0 in the UKB and BCAC datasets (two-tailed). The x axis is the expected log10\n",
      "P values from the null hypothesis, the y axis is the observed log10\n",
      "P values. All highlighted genes have P < 0.0005 and are associated with an increased risk of breast cancer. All P values are unadjusted for multiple testing.\n",
      "1\n",
      "The x-axis gives chromosomal position, and the y values are the meta-analyzed Z-scores from testing H0: β = ln (OR)=0 β =ln =0 in the UKB and BCAC datasets (2-tailed). The blue lines correspond to Z=±3.29 ± 3.29, P = 0.001, the red lines correspond to Z=±4.71 ± 4.71, P = 2.5 × 10−6. The labeled genes are those with P < 0.001. All P-values are unadjusted for multiple testing.\n",
      "1\n",
      "P-values are from the meta-analyzed Z-score from testing H0: β=ln(OR)=0β=ln =0 in the UKB and BCAC datasets (2-tailed). The x-axis gives the expected log10 P-values under the null hypothesis and the y-axis the observed log10 P-values Highlighted genes are those with P < 0.0005. Blue corresponds to an increased risk of breast cancer, and cream corresponds to a decreased risk of breast cancer. All P-values are unadjusted for multiple testing.\n",
      "1\n",
      "The x-axis gives the chromosomal position, and the y values are meta-analyzed Z-scores from testing H0: β = ln (OR) = 0 β = ln =0 in the UKB and BCAC datasets (2-tailed). The blue lines correspond to Z=±3.29 ± 3.29, P = 0.001, the red lines correspond to Z= ± 4.71 ± 4.71, P = 2.5 × 10−6. All labeled genes are those with P < 0.001. All P-values are unadjusted for multiple testing.\n",
      "1\n",
      "P-values are from the meta-analyzed Z-score from testing H0: β = ln(OR)=0 β = ln =0 in the UKB and BCAC datasets (2-tailed). The x-axis gives the expected log10\n",
      "P-values under the null hypothesis and the y-axis the observed log10\n",
      "P-values Highlighted genes are those with P < 0.0005. Blue corresponds to an increased risk of breast cancer, and cream corresponds to a decreased risk of breast cancer. All P-values are unadjusted for multiple testing.\n",
      "1\n",
      "DNMT3A1 has a disordered N-terminal domain, missing from the other isoform DNMT3A2, that binds ubiquitinated lysine 119 on histone H2A (H2AK119ub). This binding directs DNMT3A1 to bivalent domains (H3K4me3+/H3K27me3+) in some contexts such as in neurons or where the PWWP domain is mutated.\n",
      "      \n",
      "4\n",
      "\n",
      "image 1/1 /Users/etowah/projects/PMC-figure-downloader/img/10412458_Fig1.jpg: 384x640 4 labels, 70.8ms\n",
      "Speed: 1.6ms preprocess, 70.8ms inference, 0.5ms postprocess per image at shape (1, 3, 384, 640)\n",
      "1\n",
      "Core steps are shown as yellow rectangles. The procedure involves dividing the data into different data groups, shown here as blue rectangles. Gray rectangles show additional quality control steps, implemented to reduce or adjust for genotyping and data-processing errors. Pink rectangles show within-study validation steps.\n",
      "1\n",
      "Centers of error bars correspond to effect estimates in Table 1 for the pPGS based on combined weights; here, the association analyses were performed for male (blue error bars) and female (red error bars) unrelateds separately. Error bars correspond to 95% CI (estimate ±1.96 s.e.). Quantitative phenotypes were standardized to have a variance of 1 in males and females separately (Supplementary Note section 5). For secondary participation traits, the sample size shown equals number of cases plus controls.\n",
      "1\n",
      "For \\documentclass[12pt]{minimal}\n",
      "\t\t\t\t\\usepackage{amsmath}\n",
      "\t\t\t\t\\usepackage{wasysym} \n",
      "\t\t\t\t\\usepackage{amsfonts} \n",
      "\t\t\t\t\\usepackage{amssymb} \n",
      "\t\t\t\t\\usepackage{amsbsy}\n",
      "\t\t\t\t\\usepackage{mathrsfs}\n",
      "\t\t\t\t\\usepackage{upgreek}\n",
      "\t\t\t\t\\setlength{\\oddsidemargin}{-69pt}\n",
      "\t\t\t\t\\begin{document}$$\\alpha =0.055$$\\end{document}α=0.055, the participation rate of UKBB, three frequency difference ratios of an SNP (indicated in the figure) with participation effects are displayed as functions of the sibling recurrence participation rate ratio, \\documentclass[12pt]{minimal}\n",
      "\t\t\t\t\\usepackage{amsmath}\n",
      "\t\t\t\t\\usepackage{wasysym} \n",
      "\t\t\t\t\\usepackage{amsfonts} \n",
      "\t\t\t\t\\usepackage{amssymb} \n",
      "\t\t\t\t\\usepackage{amsbsy}\n",
      "\t\t\t\t\\usepackage{mathrsfs}\n",
      "\t\t\t\t\\usepackage{upgreek}\n",
      "\t\t\t\t\\setlength{\\oddsidemargin}{-69pt}\n",
      "\t\t\t\t\\begin{document}$${\\lambda }_{S}$$\\end{document}λS. The results are from simulations under a liability-threshold model where the participation of an individual is determined by its liability score and the participation rate \\documentclass[12pt]{minimal}\n",
      "\t\t\t\t\\usepackage{amsmath}\n",
      "\t\t\t\t\\usepackage{wasysym} \n",
      "\t\t\t\t\\usepackage{amsfonts} \n",
      "\t\t\t\t\\usepackage{amssymb} \n",
      "\t\t\t\t\\usepackage{amsbsy}\n",
      "\t\t\t\t\\usepackage{mathrsfs}\n",
      "\t\t\t\t\\usepackage{upgreek}\n",
      "\t\t\t\t\\setlength{\\oddsidemargin}{-69pt}\n",
      "\t\t\t\t\\begin{document}$$\\alpha$$\\end{document}α. Given \\documentclass[12pt]{minimal}\n",
      "\t\t\t\t\\usepackage{amsmath}\n",
      "\t\t\t\t\\usepackage{wasysym} \n",
      "\t\t\t\t\\usepackage{amsfonts} \n",
      "\t\t\t\t\\usepackage{amssymb} \n",
      "\t\t\t\t\\usepackage{amsbsy}\n",
      "\t\t\t\t\\usepackage{mathrsfs}\n",
      "\t\t\t\t\\usepackage{upgreek}\n",
      "\t\t\t\t\\setlength{\\oddsidemargin}{-69pt}\n",
      "\t\t\t\t\\begin{document}$$\\alpha$$\\end{document}α, \\documentclass[12pt]{minimal}\n",
      "\t\t\t\t\\usepackage{amsmath}\n",
      "\t\t\t\t\\usepackage{wasysym} \n",
      "\t\t\t\t\\usepackage{amsfonts} \n",
      "\t\t\t\t\\usepackage{amssymb} \n",
      "\t\t\t\t\\usepackage{amsbsy}\n",
      "\t\t\t\t\\usepackage{mathrsfs}\n",
      "\t\t\t\t\\usepackage{upgreek}\n",
      "\t\t\t\t\\setlength{\\oddsidemargin}{-69pt}\n",
      "\t\t\t\t\\begin{document}$${\\lambda }_{S}$$\\end{document}λS is a function of the correlation between the liability scores of two siblings (Methods). In particular, for \\documentclass[12pt]{minimal}\n",
      "\t\t\t\t\\usepackage{amsmath}\n",
      "\t\t\t\t\\usepackage{wasysym} \n",
      "\t\t\t\t\\usepackage{amsfonts} \n",
      "\t\t\t\t\\usepackage{amssymb} \n",
      "\t\t\t\t\\usepackage{amsbsy}\n",
      "\t\t\t\t\\usepackage{mathrsfs}\n",
      "\t\t\t\t\\usepackage{upgreek}\n",
      "\t\t\t\t\\setlength{\\oddsidemargin}{-69pt}\n",
      "\t\t\t\t\\begin{document}$$\\alpha =0.055$$\\end{document}α=0.055, a correlation of 0.193 between the siblings’ liability scores leads to \\documentclass[12pt]{minimal}\n",
      "\t\t\t\t\\usepackage{amsmath}\n",
      "\t\t\t\t\\usepackage{wasysym} \n",
      "\t\t\t\t\\usepackage{amsfonts} \n",
      "\t\t\t\t\\usepackage{amssymb} \n",
      "\t\t\t\t\\usepackage{amsbsy}\n",
      "\t\t\t\t\\usepackage{mathrsfs}\n",
      "\t\t\t\t\\usepackage{upgreek}\n",
      "\t\t\t\t\\setlength{\\oddsidemargin}{-69pt}\n",
      "\t\t\t\t\\begin{document}$${\\lambda }_{S}=2$$\\end{document}λS=2, the reported enrichment of sib-pairs in the UKBB data. We simulated 500 replications from a population of \\documentclass[12pt]{minimal}\n",
      "\t\t\t\t\\usepackage{amsmath}\n",
      "\t\t\t\t\\usepackage{wasysym} \n",
      "\t\t\t\t\\usepackage{amsfonts} \n",
      "\t\t\t\t\\usepackage{amssymb} \n",
      "\t\t\t\t\\usepackage{amsbsy}\n",
      "\t\t\t\t\\usepackage{mathrsfs}\n",
      "\t\t\t\t\\usepackage{upgreek}\n",
      "\t\t\t\t\\setlength{\\oddsidemargin}{-69pt}\n",
      "\t\t\t\t\\begin{document}$$\\,5\\times {10}^{7}$$\\end{document}5×107 sib-pairs. Allele 1 of the SNP is assumed to have a population frequency of 0.5 and the effect of the SNP is assumed to account for 0.1% of the variance of the liability score. The simulated averages of the two ratios, \\documentclass[12pt]{minimal}\n",
      "\t\t\t\t\\usepackage{amsmath}\n",
      "\t\t\t\t\\usepackage{wasysym} \n",
      "\t\t\t\t\\usepackage{amsfonts} \n",
      "\t\t\t\t\\usepackage{amssymb} \n",
      "\t\t\t\t\\usepackage{amsbsy}\n",
      "\t\t\t\t\\usepackage{mathrsfs}\n",
      "\t\t\t\t\\usepackage{upgreek}\n",
      "\t\t\t\t\\setlength{\\oddsidemargin}{-69pt}\n",
      "\t\t\t\t\\begin{document}$$({F}_{{\\mathrm{IBD}}1\\mathrm{S}}-{F}_{{\\mathrm{IBD}}1{\\mathrm{NS}}})/({F}_{{\\mathrm{samp}}}-{f}_{{\\mathrm{pop}}})$$\\end{document}(FIBD1S−FIBD1NS)/(Fsamp−fpop) and \\documentclass[12pt]{minimal}\n",
      "\t\t\t\t\\usepackage{amsmath}\n",
      "\t\t\t\t\\usepackage{wasysym} \n",
      "\t\t\t\t\\usepackage{amsfonts} \n",
      "\t\t\t\t\\usepackage{amssymb} \n",
      "\t\t\t\t\\usepackage{amsbsy}\n",
      "\t\t\t\t\\usepackage{mathrsfs}\n",
      "\t\t\t\t\\usepackage{upgreek}\n",
      "\t\t\t\t\\setlength{\\oddsidemargin}{-69pt}\n",
      "\t\t\t\t\\begin{document}$$\\left({F}_{{\\mathrm{IBD}}2}-{F}_{{\\mathrm{IBD}}0}\\right)/\\left({F}_{{\\mathrm{samp}}}-{f}_{{\\mathrm{pop}}}\\right),$$\\end{document}FIBD2−FIBD0/Fsamp−fpop, shown with red hollow squares and blue solid triangles respectively, are virtually indistinguishable from each other; they are roughly 1 when \\documentclass[12pt]{minimal}\n",
      "\t\t\t\t\\usepackage{amsmath}\n",
      "\t\t\t\t\\usepackage{wasysym} \n",
      "\t\t\t\t\\usepackage{amsfonts} \n",
      "\t\t\t\t\\usepackage{amssymb} \n",
      "\t\t\t\t\\usepackage{amsbsy}\n",
      "\t\t\t\t\\usepackage{mathrsfs}\n",
      "\t\t\t\t\\usepackage{upgreek}\n",
      "\t\t\t\t\\setlength{\\oddsidemargin}{-69pt}\n",
      "\t\t\t\t\\begin{document}$${\\lambda }_{S}$$\\end{document}λS is close to 1 and decrease gradually as \\documentclass[12pt]{minimal}\n",
      "\t\t\t\t\\usepackage{amsmath}\n",
      "\t\t\t\t\\usepackage{wasysym} \n",
      "\t\t\t\t\\usepackage{amsfonts} \n",
      "\t\t\t\t\\usepackage{amssymb} \n",
      "\t\t\t\t\\usepackage{amsbsy}\n",
      "\t\t\t\t\\usepackage{mathrsfs}\n",
      "\t\t\t\t\\usepackage{upgreek}\n",
      "\t\t\t\t\\setlength{\\oddsidemargin}{-69pt}\n",
      "\t\t\t\t\\begin{document}$${\\lambda }_{S}$$\\end{document}λS increases, but are always positive. The simulated average of the third ratio \\documentclass[12pt]{minimal}\n",
      "\t\t\t\t\\usepackage{amsmath}\n",
      "\t\t\t\t\\usepackage{wasysym} \n",
      "\t\t\t\t\\usepackage{amsfonts} \n",
      "\t\t\t\t\\usepackage{amssymb} \n",
      "\t\t\t\t\\usepackage{amsbsy}\n",
      "\t\t\t\t\\usepackage{mathrsfs}\n",
      "\t\t\t\t\\usepackage{upgreek}\n",
      "\t\t\t\t\\setlength{\\oddsidemargin}{-69pt}\n",
      "\t\t\t\t\\begin{document}$$({F}_{{\\mathrm{SIBS}}}-{F}_{{\\mathrm{SING}}})/({F}_{{\\mathrm{samp}}}-{f}_{{\\mathrm{pop}}})$$\\end{document}(FSIBS−FSING)/(Fsamp−fpop), where \\documentclass[12pt]{minimal}\n",
      "\t\t\t\t\\usepackage{amsmath}\n",
      "\t\t\t\t\\usepackage{wasysym} \n",
      "\t\t\t\t\\usepackage{amsfonts} \n",
      "\t\t\t\t\\usepackage{amssymb} \n",
      "\t\t\t\t\\usepackage{amsbsy}\n",
      "\t\t\t\t\\usepackage{mathrsfs}\n",
      "\t\t\t\t\\usepackage{upgreek}\n",
      "\t\t\t\t\\setlength{\\oddsidemargin}{-69pt}\n",
      "\t\t\t\t\\begin{document}$${F}_{{\\mathrm{SIBS}}}$$\\end{document}FSIBS is the allele frequency in the participating sibling pairs and \\documentclass[12pt]{minimal}\n",
      "\t\t\t\t\\usepackage{amsmath}\n",
      "\t\t\t\t\\usepackage{wasysym} \n",
      "\t\t\t\t\\usepackage{amsfonts} \n",
      "\t\t\t\t\\usepackage{amssymb} \n",
      "\t\t\t\t\\usepackage{amsbsy}\n",
      "\t\t\t\t\\usepackage{mathrsfs}\n",
      "\t\t\t\t\\usepackage{upgreek}\n",
      "\t\t\t\t\\setlength{\\oddsidemargin}{-69pt}\n",
      "\t\t\t\t\\begin{document}$${F}_{{\\mathrm{SING}}}$$\\end{document}FSING is the allele frequency in the participating individuals whose sibling does not participate, is shown with gray solid circles. For \\documentclass[12pt]{minimal}\n",
      "\t\t\t\t\\usepackage{amsmath}\n",
      "\t\t\t\t\\usepackage{wasysym} \n",
      "\t\t\t\t\\usepackage{amsfonts} \n",
      "\t\t\t\t\\usepackage{amssymb} \n",
      "\t\t\t\t\\usepackage{amsbsy}\n",
      "\t\t\t\t\\usepackage{mathrsfs}\n",
      "\t\t\t\t\\usepackage{upgreek}\n",
      "\t\t\t\t\\setlength{\\oddsidemargin}{-69pt}\n",
      "\t\t\t\t\\begin{document}$${\\lambda }_{S}=2$$\\end{document}λS=2, the first two ratios are around 0.86 and the third ratio is around 0.32.\n",
      "1\n",
      "Displayed are the 16 possible equally likely combinations of transmissions of parental genetic segments to a sibling pair at a given locus. The father has two blue genetic segments, one solid and one striped, and the mother has two orange genetic segments, solid and striped. The different colors and fills indicate distinct origins of inheritance. That is, the four parental genetic segments could be identical by state but they are all distinct with regard to grandparental origin. The sibling pairs are shown as diamond shapes, each carrying one blue genetic segment (solid or striped) inherited from the father and one orange genetic segment (solid or striped) inherited from the mother. In the case the siblings share one segment IBD (IBD1), 8 out of the 16 combinations, the shared segment is paternal for 4 combinations and maternal for the other 4 combinations.\n",
      "1\n",
      "The figure shows, for each SNP, chromosomal position (x-axis, build 37) and the estimated IBD fractions among the 16,668 white British sibling pairs in UKBB (y-axis). The black solid lines denote the theoretical expected fraction for each IBD state, equals 0.25, 0.5, and 0.25 for IBD0, IBD1, and IBD2 respectively. The two black dashed lines indicate the theoretical 95% probability interval, that is expectation \\documentclass[12pt]{minimal}\n",
      "\t\t\t\t\\usepackage{amsmath}\n",
      "\t\t\t\t\\usepackage{wasysym} \n",
      "\t\t\t\t\\usepackage{amsfonts} \n",
      "\t\t\t\t\\usepackage{amssymb} \n",
      "\t\t\t\t\\usepackage{amsbsy}\n",
      "\t\t\t\t\\usepackage{mathrsfs}\n",
      "\t\t\t\t\\usepackage{upgreek}\n",
      "\t\t\t\t\\setlength{\\oddsidemargin}{-69pt}\n",
      "\t\t\t\t\\begin{document}$$\\pm$$\\end{document}± 1.96 SD. Figure a) shows the empirical sibling fractions for each of the three IBD states computed based on results from the program KING18, and figure b) shows the sibling fractions computed with the results from the program snipar9.\n",
      "1\n",
      "Noting that the error-rate of inferring IBD state is higher in the beginning and end of IBD regions, we trimmed away 250 SNPs from the beginning and end of each called IBD segment.\n",
      "1\n",
      "Within the IBD1 region, we search for a neighboring SNP for which one individual is heterozygous while the other is homozygous. If such a neighboring SNP exists, and is phased with the target SNP, the shared allele of the target SNP can be inferred through the shared haplotype. This method was also used by Young et al.9 to infer the IBD1 shared allele.\n",
      "1\n",
      "For each biallelic sequence variant, we estimate the phasing error rate (y-axis) from trios where the offspring and one parent are heterozygous while the other parent is homozygous. The latter allowed us to determine the shared allele without using phasing and is taken as the truth. Error is when the shared allele deduced through phasing for the double-heterozygotes parent–offspring pair differs from the ‘truth’ supported by the genotype of the homozygous parent. The error rate here is, for instances where the true shared allele is 1, the fraction of times that allele 0 is deduced as the shared allele through phasing. The solid line shows the fit from regressing the estimated error rate on allele frequency up to the third power in the set of 500,632 SNPs.\n",
      "1\n",
      "The solid line shows the fit from regressing the unadjusted WSPC t-statistics through the origin on centered allele frequency (cf = f - 0.5) and cf3 in the set of 500,632 SNPs. The dashed line shows the fit for the estimated bias induced by miscalling the shared allele for the double-heterozygotes as a function of allele frequency. As described in Supplementary Note section 3, the estimated phasing induced bias was computed as 2f (1-f) [fϵ 1-f - (1-f) ϵf]/SEf with f being the frequency for the allele coded as 1, ϵ1-f and ϵf being the estimated phasing error rate for a given f (see Extended Data Fig. 5) and SEf being the standard error of the shared-not-shared allele frequency difference for a given f. We note that, mainly due to the variation in sample sizes, SEf has modest variation among SNPs with very similar f. For the figure here, a fitted value of SEf is used.\n",
      "1\n",
      "The two solid lines show the fit from regressing the χ2 statistics, computed from the allele-frequency adjusted TNTC and WSPC t-statistics, on MAF up to the third power. The t-statistics are for the 500,632 SNPs. The fitted value for a particular MAF can be interpreted as the average χ2 values for SNPs with MAFs close to that. The broken line is the corresponding fit for the BSPC χ2 statistics. Given that BSPC and WSPC capture similar true effects with comparable power, the difference between the MAF-specific fitted/average χ2 values is a measure of the average inflation of the WSPC χ2 values. Notably, for WSPC, the fitted χ2 value is much higher for SNPs with low MAFs. By contrast, the fitted χ2 value for BSPC has an increasing trend as MAF gets bigger. When MAF is low, the WSPC fitted χ2 value is substantially higher than that of BSPC, indicating that data errors are inducing a higher inflation there. As MAF increases, the difference between the WSPC and BSPC fitted χ2 values decreases. The fitted χ2 value of BSPC actually becomes slightly bigger than that of WSPC for MAF > 0.46, although that difference is not statistically significant. This is consistent with the WSPC results being close to unbiased when MAF is close to 0.5, which makes sense as the difference between major and minor alleles is small, and so is the major allele effect, when MAF is close to 0.5. The TNTC fitted χ2 value is in general smaller than that of WSPC. That is mainly due to the smaller effective sample size of TNTC, which affects the contributions of both the true effect and the bias to the χ2 statistics.\n",
      "1\n",
      "Displayed are relative allele frequency differences for different groups and segments as functions of the sibling recurrence participation ratio, λS. These differences are estimated from the same simulations underlying Fig. 4 and are described in the main text and Methods. FIBD2 and FIBD0 denote the allele frequency among sibling pairs sharing the SNP IBD2 and IBD0 respectively, while FIBD1S and FIBD1NS denote the allele frequency among the shared and not-shared alleles among sibling pairs sharing the SNP IBD1. FSIBS is the allele frequency in the participating sibling pairs, FSING is the allele frequency in the participating individuals whose sibling does not participate and fpop is the population allele frequency.\n",
      "4\n",
      "\n",
      "image 1/1 /Users/etowah/projects/PMC-figure-downloader/img/10412456_Fig1.jpg: 544x640 4 labels, 87.3ms\n",
      "Speed: 2.0ms preprocess, 87.3ms inference, 0.6ms postprocess per image at shape (1, 3, 544, 640)\n",
      "6\n",
      "\n",
      "image 1/1 /Users/etowah/projects/PMC-figure-downloader/img/10412456_Fig2.jpg: 640x480 7 labels, 68.7ms\n",
      "Speed: 1.9ms preprocess, 68.7ms inference, 0.4ms postprocess per image at shape (1, 3, 640, 480)\n",
      "1\n",
      "NDD-associated variants, including 14 novel variants, mapped on to the 3D structure of the human cBAF complex (PDB:6LTJ). Residues shown in red spheres represent NDD-associated variants in the subunit indicated, residues in blue represent those mapped from the paralog subunit, and residues in purple represent NDD-variants mapped in both the primary subunit present on the cBAF structure and paralog mapped subunit. Recurrent variants (n ≥ 3) are emphasized in red text. Caution is needed when evaluating these variants in a clinical context since not all variants are confirmed as causal.\n",
      "4\n",
      "\n",
      "image 1/1 /Users/etowah/projects/PMC-figure-downloader/img/10412456_Fig4.jpg: 544x640 4 labels, 83.7ms\n",
      "Speed: 2.2ms preprocess, 83.7ms inference, 0.5ms postprocess per image at shape (1, 3, 544, 640)\n",
      "5\n",
      "\n",
      "image 1/1 /Users/etowah/projects/PMC-figure-downloader/img/10412456_Fig5.jpg: 640x608 6 labels, 90.1ms\n",
      "Speed: 2.3ms preprocess, 90.1ms inference, 0.8ms postprocess per image at shape (1, 3, 640, 608)\n",
      "1\n",
      "NDD-associated mSWI/SNF mutations occur across several subunits of the mSWI/SNF family of chromatin remodeling complexes and cluster in key structural hubs. Missense and in-frame deletions accumulate within the catalytic ATPase, nucleosome interacting, histone-binding or DNA-binding domains, as well as the ARP module, underscoring their convergence in producing neurodevelopmental aberrations. Interpretation of NDD-associated variants in the context of this framework enables mechanistic dissection of mSWI/SNF activities and provides functional links relevant to clinical phenotypes.\n",
      "16\n",
      "\n",
      "image 1/1 /Users/etowah/projects/PMC-figure-downloader/img/10412456_Fig7.jpg: 640x480 16 labels, 66.4ms\n",
      "Speed: 2.0ms preprocess, 66.4ms inference, 0.4ms postprocess per image at shape (1, 3, 640, 480)\n",
      "6\n",
      "\n",
      "image 1/1 /Users/etowah/projects/PMC-figure-downloader/img/10412456_Fig8.jpg: 640x640 6 labels, 128.8ms\n",
      "Speed: 2.3ms preprocess, 128.8ms inference, 0.5ms postprocess per image at shape (1, 3, 640, 640)\n",
      "3\n",
      "\n",
      "image 1/1 /Users/etowah/projects/PMC-figure-downloader/img/10412456_Fig9.jpg: 640x608 3 labels, 86.2ms\n",
      "Speed: 2.3ms preprocess, 86.2ms inference, 0.7ms postprocess per image at shape (1, 3, 640, 608)\n",
      "9\n",
      "\n",
      "image 1/1 /Users/etowah/projects/PMC-figure-downloader/img/10412456_Fig10.jpg: 640x512 10 labels, 81.1ms\n",
      "Speed: 1.9ms preprocess, 81.1ms inference, 0.4ms postprocess per image at shape (1, 3, 640, 512)\n",
      "10\n",
      "\n",
      "image 1/1 /Users/etowah/projects/PMC-figure-downloader/img/10412456_Fig11.jpg: 640x544 10 labels, 72.8ms\n",
      "Speed: 2.0ms preprocess, 72.8ms inference, 0.4ms postprocess per image at shape (1, 3, 640, 544)\n",
      "4\n",
      "\n",
      "image 1/1 /Users/etowah/projects/PMC-figure-downloader/img/10412453_Fig1.jpg: 640x512 5 labels, 74.8ms\n",
      "Speed: 2.2ms preprocess, 74.8ms inference, 0.4ms postprocess per image at shape (1, 3, 640, 512)\n",
      "9\n",
      "\n",
      "image 1/1 /Users/etowah/projects/PMC-figure-downloader/img/10412453_Fig2.jpg: 640x480 9 labels, 77.0ms\n",
      "Speed: 2.1ms preprocess, 77.0ms inference, 0.6ms postprocess per image at shape (1, 3, 640, 480)\n",
      "4\n",
      "\n",
      "image 1/1 /Users/etowah/projects/PMC-figure-downloader/img/10412453_Fig3.jpg: 384x640 4 labels, 63.3ms\n",
      "Speed: 1.4ms preprocess, 63.3ms inference, 0.4ms postprocess per image at shape (1, 3, 384, 640)\n",
      "19\n",
      "\n",
      "image 1/1 /Users/etowah/projects/PMC-figure-downloader/img/10412453_Fig4.jpg: 640x512 7 labels, 70.3ms\n",
      "Speed: 2.1ms preprocess, 70.3ms inference, 1.3ms postprocess per image at shape (1, 3, 640, 512)\n",
      "Subfigure count mismatch: 7 detected, 19 expected\n",
      "3\n",
      "\n",
      "image 1/1 /Users/etowah/projects/PMC-figure-downloader/img/10412453_Fig5.jpg: 640x480 3 labels, 66.1ms\n",
      "Speed: 1.9ms preprocess, 66.1ms inference, 0.4ms postprocess per image at shape (1, 3, 640, 480)\n",
      "1\n",
      "The breakpoints of all SVs within every ‘best’ genome were matched to the corresponding gff annotation file. These were then compared to the genome wide proportion of events to calculate a per genome enrichment. This was done by getting the breakpoint-associated features (left: n = 1776 CDS, n = 5 centromere, n = 1743 LTR, n = 274 tRNA, n = 760 TY, n = 354 X_element and n = 498 Y_prime_element) and the closest element for intergenic breakpoints (right: n = 2041 CDS, n = 145 centromere, n = 1784 LTR, n = 1291 tRNA, n = 748 TY, n = 427 X_element and n = 278 Y_prime_element). The horizontal lines in the boxplots correspond to the median, the lower and upper hinges correspond to the first and third quartiles and the whiskers extend up to 1.5 times the interquartile range.\n",
      "2\n",
      "\n",
      "image 1/1 /Users/etowah/projects/PMC-figure-downloader/img/10412453_Fig7.jpg: 640x544 2 labels, 77.0ms\n",
      "Speed: 2.3ms preprocess, 77.0ms inference, 0.7ms postprocess per image at shape (1, 3, 640, 544)\n",
      "1\n",
      "The vertical lines in the boxplots correspond to the median, the lower and upper hinges correspond to the first and third quartiles and the whiskers extend up to 1.5 times the interquartile range. For each of the 142 strains, the number of telomeres used to derive the boxplot is indicated in Supplementary Table 1. The inset plot shows the distribution of the average telomere length per strain for each dataset. Median values are indicated by the gray crossbars in each violin plot (for de novo assemblies n = 100 strains, for public assemblies n = 24 strains and for re-assemblies n = 18). Two-sided Wilcoxon mean comparison p-values are indicated.\n",
      "3\n",
      "\n",
      "image 1/1 /Users/etowah/projects/PMC-figure-downloader/img/10412453_Fig9.jpg: 544x640 4 labels, 87.2ms\n",
      "Speed: 2.3ms preprocess, 87.2ms inference, 0.4ms postprocess per image at shape (1, 3, 544, 640)\n",
      "Subfigure count mismatch: 4 detected, 3 expected\n",
      "2\n",
      "\n",
      "image 1/1 /Users/etowah/projects/PMC-figure-downloader/img/10412453_Fig10.jpg: 640x480 9 labels, 76.0ms\n",
      "Speed: 2.0ms preprocess, 76.0ms inference, 0.4ms postprocess per image at shape (1, 3, 640, 480)\n",
      "Subfigure count mismatch: 9 detected, 2 expected\n",
      "1\n",
      "Two-sided Wilcoxon mean comparison p-values are indicated. The number of telomeres that is used to derive each violin plot is indicated on the top of each individual plot.\n",
      "1\n",
      "The phylogenetic tree on the left is identical to Fig. 1 and corresponds to a tree based on the concatenated protein sequence alignment of 1,612 1:1 ortholog. The green, red, blue and yellow symbols indicate the ecological origins. Ploidy levels and zygosity are symbolized by the shapes of the symbols as in Fig. 1.\n",
      "2\n",
      "\n",
      "image 1/1 /Users/etowah/projects/PMC-figure-downloader/img/10412453_Fig13.jpg: 640x352 2 labels, 92.9ms\n",
      "Speed: 1.9ms preprocess, 92.9ms inference, 0.8ms postprocess per image at shape (1, 3, 640, 352)\n",
      "6\n",
      "\n",
      "image 1/1 /Users/etowah/projects/PMC-figure-downloader/img/10412453_Fig14.jpg: 640x480 7 labels, 68.6ms\n",
      "Speed: 2.0ms preprocess, 68.6ms inference, 1.0ms postprocess per image at shape (1, 3, 640, 480)\n",
      "Subfigure count mismatch: 7 detected, 6 expected\n",
      "1\n",
      "Created with BioRender.com.\n",
      "        \n",
      "1\n",
      "Pairwise Spearman’s Rank correlation. Red shades denote positive correlation, blue shared denote negative correlation between trait pairs. X denotes no significant correlation (P > 0.05).\n",
      "        \n",
      "1\n",
      "Pairwise Spearman’s Rank correlation. Red shades denote positive correlation, blue shared denote negative correlation between trait pairs, legend along bottom of heatmap shows colour scale relative to rho value. X denotes no significant correlation (P > 0.05). Abbreviations denote: IFC_OGTT – IFC calculated using OGTT measures, ISI_OGTT – Modified Stumvoll ISI calculated using OGTT measures. M_I – M/I index of insulin sensitivity measured by clamp. InsC0c – insulin measured at 0min during clamp. InsO0c - insulin measured at 0min during OGTT. InsC120c – insulin measured at 120min during clamp. InsO120c - insulin measured at 120min during OGTT. InsC_20c – insulin measured at 20min before clamp. EGP_B -basal glucose production, EGP_SS - glucose production during clamp, GCR_B - basal glucose clearance, ml/min/kg lean body mass, GCR_SS - steady state glucose clearance, ml/min/kg lean body mass, icl_clamp - peripheral insulin clearance (1/min/m2), icl_OGTT - endogenous \"pre-hepatic\" clearance during the OGTT, hie_0 - hepatic insulin extraction during clamp, OGIS - oral glucose insulin sensitivity index (ml min-1 m-2). ISR5dr - insulin secretion 5 mM glucose, beta cell dose response (pmol min-1m-2). ISR0 basal insulin secretion (pmol min-1m-2). ISRtot - total insulin secretion (nmol m-2).\n",
      "        \n",
      "1\n",
      "Analysis workflow for the meta-analysis of study-level GWAS results for Insulin fold change and Modified Stumvoll ISI. Created with BioRender.com\n",
      "        \n",
      "1\n",
      "Conditional analyses identify a second independent signal at PPP1R3B for insulin fold change adjusted for BMI. The regional association plot shows the primary signal in red and the secondary signal in blue for marginal summary statistics for insulin fold change adjusted for BMI. Shade of point indicates pairwise LD (R2) with indicated lead variant.\n",
      "        \n",
      "1\n",
      "Labels on the right-hand side indicate the ancestry of the study and study name. EUR-European ancestry, HISAMR – Hispanic American ancestry, EAS – East Asian ancestry. Left-hand side values are beta estimate and 95% confidence interval. Error bars denote a 95% confidence interval. X-axis denotes the beta estimate of associations with insulin fold change in BMI adjusted analyses.\n",
      "        \n",
      "1\n",
      "Unadjusted -log10 p-values are indicated on the y axis. Lead variant indicated by purple diamond.\n",
      "        \n",
      "1\n",
      "An EMSA using 6μg per lane of nuclear extract from undifferentiated LHCN-M2 cells shows protein-DNA interactions for probes centered around each both alleles of rs117643180. The probe containing rs117643180-A shows allele-specific protein binding (Arrow A, lane 6), relative to the probe containing rs117643180-C (lane 2). A 25-fold excess unlabeled probe containing the A allele competed away A-specific bands more effectively (lane 7) than 25-fold excess unlabeled probe containing the C allele (lane 8). Arrow B shows a biotinylated free probe (200 fmol per lane). Uncropped image is available in Source Data.\n",
      "        \n",
      "1\n",
      "Representative blot from N=2. Marker indicates protein size in kDa is outlined on the right-hand side of the blot. siGenome and OT+ represent siRNA pools with their corresponding targets indicated below (see methods) and NT denotes non-targeting control. Antibodies are outlined on the left-hand side of the blot with Tubulin and B-actin used as loading controls. Uncropped blots are provided in Source Data.\n",
      "        \n",
      "1\n",
      "Manhattan plot of results for IFC, ISI and relative association with key glycaemic traits of interest. Y-axis denotes the -log10(p-value) of association for each trait. X-axis outlines the chromosomal position, with alternate chromosomes represented in light and dark grey. The dashed line defines genome-wide significance threshold (P = 5x10-8), and the dotted line denotes P = 1x10-5. Points highlighted in blue on each plot represent those that meet genome-wide significance for either IFC and ISI (P < 5x10-8). Labels for indicate the nearest gene at a locus defined at genome-wide significance for either IFC and ISI, and their relative association with other traits. Published GWAS were used for fasting insulin and 2 h glucose3,, and T2D26. For T2D y axis is cut at 1x10-80 for clarity, with the minimum p value truncated to P = 1x10-300. Uncorrected p-values are shown. All GWAS are in cohorts of European ancestry and adjusted for BMI, except T2D which is unadjusted for BMI.\n",
      "      \n",
      "1\n",
      "A) Regional association plot showing ± 500kb around rs117643180 (SLC2A4) : insulin fold change (IFC) - blue, Modified Stumvoll (ISI) – red, and 2 h glucose3 – green, fasting insulin3 – purple, type 2 diabetes26 – pink. Y-axis denotes the unadjusted log10(p-value) of association. The dashed lines and dotted line indicate genome-wide (P = 5x10-8) and suggestive (P = 1x10-5) significance threshold, respectively. All association statistics are from BMI adjusted analyses in studies of European ancestry, except T2D which is unadjusted for BMI. Labels indicate the lead variant for each trait, with location indicating position. B) Association of rs117643180 with key metabolic traits of interest. Plot shows beta estimate or odds ratio of association for rs117643180 with traits outlined in (A) and fasting glucose3 and waist-to-hip ratio53 in grey. Point represents effect size and error bars indicate 95% CI. C) rs117643180 overlaps a transcriptionally active region in muscle tissues. rs117643180 is located ~730 bp downstream of the main SLC2A4 transcription start site (TSS) and ~330 bp downstream of an alternate SLC2A4 TSS. A 229-bp region flanking the variant, as shown by the orange bar, overlaps with accessible chromatin in LHCN-M2 (DNAase)54, chromatin marks of regulatory activity in fetal muscle leg tissue 55, a region conserved across species (vertebrate conservation), and a skeletal muscle ChromHMM56 (red =ActiveTSS). Created using http://genome.ucsc.edu57\n",
      "D-E) rs117643180 exhibits allelic differences in transcriptional activity. Values represent fold-change of firefly luciferase/Renilla activity normalized to empty pGL4.23 vector in C2C12 cells, for the A (effect) and C (non-effect) alleles clones in forward and reverse orientation (See Methods) (D) and LHCN-M2 myotubes (E). Error bars represent the SEM of 4 to 5 independent clones tested in duplicate wells; points represent individual biological replicates. P-values are calculated from two-sided t-tests.\n",
      "      \n",
      "1\n",
      "a) Effect of gene knockdown on GLUT4 trafficking and glucose transport at 0.5 nM insulin stimulation in WT 3T3-L1 adipocytes (N = 3 biologically independent replicates, 2 technical replicates per N). Genes are clustered using hierarchical clustering. All results are normalised to non-targeting (NT) control. Grey represents genes for which glucose transport data was not collected as their role is established in the literature. Scale represents relative effect of gene knockdown compared to NT control for given measures – red positive, and blue negative, with the maximal scale representing the maximum absolute response relative to NT (0) across all measures, expressed as a percentage, to allow direct comparison across measures.-. B-D) Representative fluorescence microscopy images of example siRNA knockdown in WT 3T3-L1 adipocytes (N = 3 biologically independent replicates, 2 technical replicates per N) at 0.5 and 100nM insulin stimulation. Green represents the surface GLUT4 signal, orange the total GLUT4 signal and blue indicates individual nuclei. Scale bars indicate 100 μm. Quantification of Slc36a4 and Pdzk1ip1 compared to NT control and images at 0 nM are shown in Supplementary Figure 25. b) shows non-targeting control, c) Slc36a4 and d) Pdzk1ip1.\n",
      "      \n",
      "3\n",
      "\n",
      "image 1/1 /Users/etowah/projects/PMC-figure-downloader/img/10335940_Fig1.jpg: 640x512 3 labels, 72.0ms\n",
      "Speed: 2.1ms preprocess, 72.0ms inference, 0.9ms postprocess per image at shape (1, 3, 640, 512)\n",
      "18\n",
      "\n",
      "image 1/1 /Users/etowah/projects/PMC-figure-downloader/img/10335940_Fig2.jpg: 640x288 19 labels, 58.1ms\n",
      "Speed: 1.3ms preprocess, 58.1ms inference, 0.4ms postprocess per image at shape (1, 3, 640, 288)\n",
      "11\n",
      "\n",
      "image 1/1 /Users/etowah/projects/PMC-figure-downloader/img/10335940_Fig3.jpg: 640x448 12 labels, 94.0ms\n",
      "Speed: 2.0ms preprocess, 94.0ms inference, 0.6ms postprocess per image at shape (1, 3, 640, 448)\n",
      "Subfigure count mismatch: 12 detected, 11 expected\n",
      "11\n",
      "\n",
      "image 1/1 /Users/etowah/projects/PMC-figure-downloader/img/10335940_Fig4.jpg: 512x640 6 labels, 77.1ms\n",
      "Speed: 1.9ms preprocess, 77.1ms inference, 0.4ms postprocess per image at shape (1, 3, 512, 640)\n",
      "Subfigure count mismatch: 6 detected, 11 expected\n",
      "3\n",
      "\n",
      "image 1/1 /Users/etowah/projects/PMC-figure-downloader/img/10335940_Fig5.jpg: 480x640 4 labels, 69.7ms\n",
      "Speed: 1.8ms preprocess, 69.7ms inference, 0.4ms postprocess per image at shape (1, 3, 480, 640)\n",
      "Subfigure count mismatch: 4 detected, 3 expected\n",
      "7\n",
      "\n",
      "image 1/1 /Users/etowah/projects/PMC-figure-downloader/img/10335940_Fig6.jpg: 640x512 7 labels, 80.1ms\n",
      "Speed: 2.3ms preprocess, 80.1ms inference, 0.4ms postprocess per image at shape (1, 3, 640, 512)\n",
      "16\n",
      "\n",
      "image 1/1 /Users/etowah/projects/PMC-figure-downloader/img/10335940_Fig7.jpg: 640x576 18 labels, 77.6ms\n",
      "Speed: 2.4ms preprocess, 77.6ms inference, 0.6ms postprocess per image at shape (1, 3, 640, 576)\n",
      "Subfigure count mismatch: 18 detected, 16 expected\n",
      "11\n",
      "\n",
      "image 1/1 /Users/etowah/projects/PMC-figure-downloader/img/10335940_Fig8.jpg: 608x640 10 labels, 101.2ms\n",
      "Speed: 2.5ms preprocess, 101.2ms inference, 0.6ms postprocess per image at shape (1, 3, 608, 640)\n",
      "Subfigure count mismatch: 10 detected, 11 expected\n",
      "6\n",
      "\n",
      "image 1/1 /Users/etowah/projects/PMC-figure-downloader/img/10335940_Fig9.jpg: 640x448 7 labels, 74.2ms\n",
      "Speed: 1.8ms preprocess, 74.2ms inference, 0.4ms postprocess per image at shape (1, 3, 640, 448)\n",
      "3\n",
      "\n",
      "image 1/1 /Users/etowah/projects/PMC-figure-downloader/img/10335940_Fig10.jpg: 640x512 2 labels, 97.6ms\n",
      "Speed: 2.2ms preprocess, 97.6ms inference, 1.0ms postprocess per image at shape (1, 3, 640, 512)\n",
      "Subfigure count mismatch: 2 detected, 3 expected\n",
      "9\n",
      "\n",
      "image 1/1 /Users/etowah/projects/PMC-figure-downloader/img/10335940_Fig11.jpg: 384x640 8 labels, 86.9ms\n",
      "Speed: 1.5ms preprocess, 86.9ms inference, 0.4ms postprocess per image at shape (1, 3, 384, 640)\n",
      "Subfigure count mismatch: 8 detected, 9 expected\n",
      "6\n",
      "\n",
      "image 1/1 /Users/etowah/projects/PMC-figure-downloader/img/10335940_Fig12.jpg: 352x640 7 labels, 65.3ms\n",
      "Speed: 1.5ms preprocess, 65.3ms inference, 0.4ms postprocess per image at shape (1, 3, 352, 640)\n",
      "6\n",
      "\n",
      "image 1/1 /Users/etowah/projects/PMC-figure-downloader/img/10335940_Fig13.jpg: 320x640 4 labels, 91.6ms\n",
      "Speed: 1.4ms preprocess, 91.6ms inference, 0.8ms postprocess per image at shape (1, 3, 320, 640)\n",
      "Subfigure count mismatch: 4 detected, 6 expected\n",
      "4\n",
      "\n",
      "image 1/1 /Users/etowah/projects/PMC-figure-downloader/img/10335940_Fig14.jpg: 640x480 5 labels, 105.4ms\n",
      "Speed: 2.3ms preprocess, 105.4ms inference, 0.5ms postprocess per image at shape (1, 3, 640, 480)\n",
      "Subfigure count mismatch: 5 detected, 4 expected\n",
      "15\n",
      "\n",
      "image 1/1 /Users/etowah/projects/PMC-figure-downloader/img/10335940_Fig15.jpg: 544x640 16 labels, 103.8ms\n",
      "Speed: 2.5ms preprocess, 103.8ms inference, 0.5ms postprocess per image at shape (1, 3, 544, 640)\n",
      "21\n",
      "\n",
      "image 1/1 /Users/etowah/projects/PMC-figure-downloader/img/10335940_Fig16.jpg: 640x480 2 labels, 81.1ms\n",
      "Speed: 2.3ms preprocess, 81.1ms inference, 0.8ms postprocess per image at shape (1, 3, 640, 480)\n",
      "Subfigure count mismatch: 2 detected, 21 expected\n",
      "18\n",
      "\n",
      "image 1/1 /Users/etowah/projects/PMC-figure-downloader/img/10335940_Fig17.jpg: 640x480 (no detections), 72.8ms\n",
      "Speed: 2.0ms preprocess, 72.8ms inference, 0.2ms postprocess per image at shape (1, 3, 640, 480)\n",
      "Subfigure count mismatch: 0 detected, 18 expected\n",
      "3\n",
      "\n",
      "image 1/1 /Users/etowah/projects/PMC-figure-downloader/img/10335940_Fig18.jpg: 448x640 2 labels, 56.9ms\n",
      "Speed: 1.6ms preprocess, 56.9ms inference, 0.8ms postprocess per image at shape (1, 3, 448, 640)\n",
      "Subfigure count mismatch: 2 detected, 3 expected\n",
      "3\n",
      "\n",
      "image 1/1 /Users/etowah/projects/PMC-figure-downloader/img/10335939_Fig1.jpg: 416x640 3 labels, 57.1ms\n",
      "Speed: 2.0ms preprocess, 57.1ms inference, 0.6ms postprocess per image at shape (1, 3, 416, 640)\n",
      "7\n",
      "\n",
      "image 1/1 /Users/etowah/projects/PMC-figure-downloader/img/10335939_Fig2.jpg: 480x640 8 labels, 68.3ms\n",
      "Speed: 1.8ms preprocess, 68.3ms inference, 0.4ms postprocess per image at shape (1, 3, 480, 640)\n",
      "Subfigure count mismatch: 8 detected, 7 expected\n",
      "7\n",
      "\n",
      "image 1/1 /Users/etowah/projects/PMC-figure-downloader/img/10335939_Fig3.jpg: 576x640 7 labels, 97.5ms\n",
      "Speed: 2.4ms preprocess, 97.5ms inference, 0.5ms postprocess per image at shape (1, 3, 576, 640)\n",
      "7\n",
      "\n",
      "image 1/1 /Users/etowah/projects/PMC-figure-downloader/img/10335939_Fig4.jpg: 576x640 8 labels, 81.1ms\n",
      "Speed: 2.0ms preprocess, 81.1ms inference, 0.4ms postprocess per image at shape (1, 3, 576, 640)\n",
      "Subfigure count mismatch: 8 detected, 7 expected\n",
      "5\n",
      "\n",
      "image 1/1 /Users/etowah/projects/PMC-figure-downloader/img/10335939_Fig5.jpg: 544x640 5 labels, 88.1ms\n",
      "Speed: 1.9ms preprocess, 88.1ms inference, 0.6ms postprocess per image at shape (1, 3, 544, 640)\n",
      "6\n",
      "\n",
      "image 1/1 /Users/etowah/projects/PMC-figure-downloader/img/10335939_Fig6.jpg: 576x640 8 labels, 86.4ms\n",
      "Speed: 2.1ms preprocess, 86.4ms inference, 0.7ms postprocess per image at shape (1, 3, 576, 640)\n",
      "Subfigure count mismatch: 7 detected, 6 expected\n",
      "1\n",
      "The disease originates from the rearrangement of BCR-ABL1 in a multipotent stem cell (HSC or MPP), which is the cell-of-origin. BCR-ABL1 increases cell proliferation and impairs DNA damage response. It then transforms by acquiring cooperating alterations in a downstream B-cell progenitor, which is the cell-of-transformation. Transformation-related events are mostly generated by RAG-mediated recombination. Inactivation of B-cell transcription factors and tumor suppressors causes a block in differentiation and evasion of programmed cell death, respectively. The molecular subtypes of BCR-ABL1 lymphoblastic leukemia arise from different target cells of transformation.\n",
      "3\n",
      "\n",
      "image 1/1 /Users/etowah/projects/PMC-figure-downloader/img/10335939_Fig8.jpg: 512x640 3 labels, 85.2ms\n",
      "Speed: 2.0ms preprocess, 85.2ms inference, 0.5ms postprocess per image at shape (1, 3, 512, 640)\n",
      "1\n",
      "Consensus hierarchical clustering using 163 NMF component genes produced three molecular subtypes of BCR-ABL1 lymphoblastic leukemia. Rows represent the genes grouped into NMF components, and columns represent the 57 samples. Molecular subtypes, consensus clustering confidence scores, BCR-ABL1 isoforms, diagnoses (B-ALL, MPAL, CML-LBC), and sample types (primary/diagnostic, relapse) are shown in tracks.\n",
      "1\n",
      "Comparison of clinical and sample characteristics of three molecular subtypes. Plots are separated into main cohort (left; 23 C1, 8 C2, 22 C3), 3’-seq cohort (centre; 13 C1, 13 C2, 14 C3), and combined cohort (right; 36 C1, 21 C2, 36 C3). Counts represent numbers of patients/primary leukemias. Kruskal-Wallis test was used for comparing age at diagnosis and WBC, and Fisher’s exact test was used for the rest. PB, peripheral blood; BM, bone marrow; WBC, white blood cell.\n",
      "3\n",
      "\n",
      "image 1/1 /Users/etowah/projects/PMC-figure-downloader/img/10335939_Fig11.jpg: 640x608 3 labels, 96.0ms\n",
      "Speed: 2.2ms preprocess, 96.0ms inference, 1.4ms postprocess per image at shape (1, 3, 640, 608)\n",
      "7\n",
      "\n",
      "image 1/1 /Users/etowah/projects/PMC-figure-downloader/img/10335939_Fig12.jpg: 640x608 7 labels, 106.0ms\n",
      "Speed: 2.9ms preprocess, 106.0ms inference, 1.2ms postprocess per image at shape (1, 3, 640, 608)\n",
      "4\n",
      "\n",
      "image 1/1 /Users/etowah/projects/PMC-figure-downloader/img/10335939_Fig13.jpg: 448x640 4 labels, 65.9ms\n",
      "Speed: 1.9ms preprocess, 65.9ms inference, 0.4ms postprocess per image at shape (1, 3, 448, 640)\n",
      "3\n",
      "\n",
      "image 1/1 /Users/etowah/projects/PMC-figure-downloader/img/10335939_Fig14.jpg: 608x640 3 labels, 84.9ms\n",
      "Speed: 2.3ms preprocess, 84.9ms inference, 0.8ms postprocess per image at shape (1, 3, 608, 640)\n",
      "16\n",
      "\n",
      "image 1/1 /Users/etowah/projects/PMC-figure-downloader/img/10335939_Fig15.jpg: 640x640 5 labels, 85.2ms\n",
      "Speed: 2.5ms preprocess, 85.2ms inference, 0.5ms postprocess per image at shape (1, 3, 640, 640)\n",
      "Subfigure count mismatch: 5 detected, 16 expected\n",
      "3\n",
      "\n",
      "image 1/1 /Users/etowah/projects/PMC-figure-downloader/img/10335939_Fig16.jpg: 640x640 3 labels, 89.3ms\n",
      "Speed: 2.4ms preprocess, 89.3ms inference, 0.9ms postprocess per image at shape (1, 3, 640, 640)\n",
      "1\n",
      "The degree of innate resistance to chemotherapy and TKI (that is imatinib) influences the probability of acquiring BCR-ABL1 KD mutations and relapsing. Late-Pro leukemias respond extremely well to induction and most patients achieve remission/MMR. Inter-Pro leukemias do not respond as well and most patients relapse without KD mutations. Early-Pro leukemias show minimal response to induction and most patients relapse with or without KD mutations. Cell quiescence, STAT5 signaling, and UPR signaling potentially contribute to treatment resistance in the Early-Pro and Inter-Pro subtypes.\n",
      "6\n",
      "\n",
      "image 1/1 /Users/etowah/projects/PMC-figure-downloader/img/10335937_Fig1.jpg: 640x640 7 labels, 88.6ms\n",
      "Speed: 2.4ms preprocess, 88.6ms inference, 0.4ms postprocess per image at shape (1, 3, 640, 640)\n",
      "Subfigure count mismatch: 7 detected, 6 expected\n",
      "3\n",
      "\n",
      "image 1/1 /Users/etowah/projects/PMC-figure-downloader/img/10335937_Fig2.jpg: 640x640 5 labels, 91.8ms\n",
      "Speed: 2.6ms preprocess, 91.8ms inference, 2.1ms postprocess per image at shape (1, 3, 640, 640)\n",
      "Subfigure count mismatch: 5 detected, 3 expected\n",
      "6\n",
      "\n",
      "image 1/1 /Users/etowah/projects/PMC-figure-downloader/img/10335937_Fig3.jpg: 640x480 6 labels, 64.8ms\n",
      "Speed: 2.0ms preprocess, 64.8ms inference, 0.6ms postprocess per image at shape (1, 3, 640, 480)\n",
      "6\n",
      "\n",
      "image 1/1 /Users/etowah/projects/PMC-figure-downloader/img/10335937_Fig4.jpg: 640x608 7 labels, 85.1ms\n",
      "Speed: 2.2ms preprocess, 85.1ms inference, 2.6ms postprocess per image at shape (1, 3, 640, 608)\n",
      "Subfigure count mismatch: 7 detected, 6 expected\n",
      "6\n",
      "\n",
      "image 1/1 /Users/etowah/projects/PMC-figure-downloader/img/10335937_Fig5.jpg: 256x640 7 labels, 47.7ms\n",
      "Speed: 1.2ms preprocess, 47.7ms inference, 0.3ms postprocess per image at shape (1, 3, 256, 640)\n",
      "6\n",
      "\n",
      "image 1/1 /Users/etowah/projects/PMC-figure-downloader/img/10335937_Fig6.jpg: 640x480 7 labels, 81.6ms\n",
      "Speed: 2.0ms preprocess, 81.6ms inference, 0.4ms postprocess per image at shape (1, 3, 640, 480)\n",
      "Subfigure count mismatch: 5 detected, 6 expected\n",
      "6\n",
      "\n",
      "image 1/1 /Users/etowah/projects/PMC-figure-downloader/img/10335937_Fig7.jpg: 640x480 7 labels, 85.5ms\n",
      "Speed: 2.0ms preprocess, 85.5ms inference, 0.4ms postprocess per image at shape (1, 3, 640, 480)\n",
      "Subfigure count mismatch: 7 detected, 6 expected\n",
      "6\n",
      "\n",
      "image 1/1 /Users/etowah/projects/PMC-figure-downloader/img/10335937_Fig8.jpg: 640x480 6 labels, 84.0ms\n",
      "Speed: 2.2ms preprocess, 84.0ms inference, 0.5ms postprocess per image at shape (1, 3, 640, 480)\n",
      "5\n",
      "\n",
      "image 1/1 /Users/etowah/projects/PMC-figure-downloader/img/10335937_Fig9.jpg: 640x544 8 labels, 117.1ms\n",
      "Speed: 2.2ms preprocess, 117.1ms inference, 1.0ms postprocess per image at shape (1, 3, 640, 544)\n",
      "Subfigure count mismatch: 7 detected, 5 expected\n",
      "5\n",
      "\n",
      "image 1/1 /Users/etowah/projects/PMC-figure-downloader/img/10335937_Fig10.jpg: 608x640 8 labels, 89.0ms\n",
      "Speed: 2.4ms preprocess, 89.0ms inference, 0.5ms postprocess per image at shape (1, 3, 608, 640)\n",
      "Subfigure count mismatch: 7 detected, 5 expected\n",
      "5\n",
      "\n",
      "image 1/1 /Users/etowah/projects/PMC-figure-downloader/img/10335937_Fig11.jpg: 640x608 5 labels, 86.3ms\n",
      "Speed: 2.3ms preprocess, 86.3ms inference, 0.5ms postprocess per image at shape (1, 3, 640, 608)\n",
      "18\n",
      "\n",
      "image 1/1 /Users/etowah/projects/PMC-figure-downloader/img/10335937_Fig12.jpg: 480x640 5 labels, 91.5ms\n",
      "Speed: 1.7ms preprocess, 91.5ms inference, 2.0ms postprocess per image at shape (1, 3, 480, 640)\n",
      "Subfigure count mismatch: 5 detected, 18 expected\n",
      "7\n",
      "\n",
      "image 1/1 /Users/etowah/projects/PMC-figure-downloader/img/10335937_Fig13.jpg: 640x512 8 labels, 74.1ms\n",
      "Speed: 3.0ms preprocess, 74.1ms inference, 0.4ms postprocess per image at shape (1, 3, 640, 512)\n",
      "6\n",
      "\n",
      "image 1/1 /Users/etowah/projects/PMC-figure-downloader/img/10335937_Fig14.jpg: 640x384 5 labels, 63.6ms\n",
      "Speed: 2.0ms preprocess, 63.6ms inference, 0.4ms postprocess per image at shape (1, 3, 640, 384)\n",
      "Subfigure count mismatch: 5 detected, 6 expected\n",
      "7\n",
      "\n",
      "image 1/1 /Users/etowah/projects/PMC-figure-downloader/img/10335937_Fig15.jpg: 512x640 6 labels, 70.2ms\n",
      "Speed: 1.9ms preprocess, 70.2ms inference, 0.4ms postprocess per image at shape (1, 3, 512, 640)\n",
      "Subfigure count mismatch: 6 detected, 7 expected\n",
      "7\n",
      "\n",
      "image 1/1 /Users/etowah/projects/PMC-figure-downloader/img/10335936_Fig1.jpg: 640x512 7 labels, 68.5ms\n",
      "Speed: 1.9ms preprocess, 68.5ms inference, 0.6ms postprocess per image at shape (1, 3, 640, 512)\n",
      "3\n",
      "\n",
      "image 1/1 /Users/etowah/projects/PMC-figure-downloader/img/10335936_Fig2.jpg: 224x640 3 labels, 46.4ms\n",
      "Speed: 1.9ms preprocess, 46.4ms inference, 0.7ms postprocess per image at shape (1, 3, 224, 640)\n",
      "4\n",
      "\n",
      "image 1/1 /Users/etowah/projects/PMC-figure-downloader/img/10335936_Fig3.jpg: 480x640 4 labels, 72.9ms\n",
      "Speed: 1.9ms preprocess, 72.9ms inference, 0.4ms postprocess per image at shape (1, 3, 480, 640)\n",
      "4\n",
      "\n",
      "image 1/1 /Users/etowah/projects/PMC-figure-downloader/img/10335936_Fig4.jpg: 512x640 4 labels, 70.9ms\n",
      "Speed: 1.8ms preprocess, 70.9ms inference, 0.8ms postprocess per image at shape (1, 3, 512, 640)\n",
      "3\n",
      "\n",
      "image 1/1 /Users/etowah/projects/PMC-figure-downloader/img/10335936_Fig5.jpg: 640x448 5 labels, 74.2ms\n",
      "Speed: 1.9ms preprocess, 74.2ms inference, 0.5ms postprocess per image at shape (1, 3, 640, 448)\n",
      "4\n",
      "\n",
      "image 1/1 /Users/etowah/projects/PMC-figure-downloader/img/10335936_Fig6.jpg: 512x640 3 labels, 77.7ms\n",
      "Speed: 2.0ms preprocess, 77.7ms inference, 0.8ms postprocess per image at shape (1, 3, 512, 640)\n",
      "Subfigure count mismatch: 3 detected, 4 expected\n",
      "1\n",
      "There was a total of 20 contigs which were anchored and oriented onto ten pseudomolecules of the Mo17 genome. The contig 3 and contig 6 were generated by split of a raw ONT contig with assembly error (see Supplementary Fig. 1). The red lines on the blue blocks refer the gaps on the Mo17ref_V1.\n",
      "1\n",
      "According to the alignment with the PacBio assembly, 5 LCRs on the basal Mo17 assembly could be corrected by the contigs of PacBio Hifiasm assembly. Corrected assembly was confirmed by the uniform ONT and/or PacBio reads coverage, and tiling ONT reads. Black shades refer local coverage-anomalous regions.\n",
      "1\n",
      "The validity of four gaps closed by the contigs of PacBio Hifiasm assembly were confirmed by the uniform ONT and/or PacBio reads coverage and tiling ONT reads. Black shades refer local coverage-anomalous regions. We noted that in gaps 7 and 10, a part of the ONT contigs not aligned to HiFi contigs were removed in the final assembly. Compared to the normal gap ends (which were remained in the final assembly) with gradually decreased coverage, almost no aligned ONT reads were observed for these regions removed in the final assembly, which suggested that these removed parts of contigs were in fact redundant or misassembled fragments, and further confirmed the assemblies of these gap-closed regions.\n",
      "1\n",
      "The assembly of the terminal 1 Mb regions of all 10 chromosomes were confirmed by tiling ONT reads. ONT reads coverage analysis showed that expect for the 20 telomeres of chromosomes, as well as subtelomeric regions and long TAG repeat region on chromosome 2, uniform ONT reads coverage were observed in general. Black shades of ONT coverage verification refer the regions with reads depth lower than 100. Red pentagrams refer the reads harbored with the longest telomeric repeats for corresponding chromosomal ends, which were used to correct corresponding telomeric regions.\n",
      "1\n",
      "The green dotted box indicated the TAG repeat region in gap2 which length was estimated (see Method).\n",
      "1\n",
      "The blue and red arrows indicated the transcription directions of corresponding 45S rDNAs were toward to the centromere and telomere, respectively. The black arrows indicated the direction of extension during gap closure.\n",
      "1\n",
      "Ultra-long ONT reads longer than 10 kb and PacBio HiFi reads were used for analysis. Local coverage-anomalous regions were shown in black shades.\n",
      "2\n",
      "\n",
      "image 1/1 /Users/etowah/projects/PMC-figure-downloader/img/10335936_Fig14.jpg: 224x640 3 labels, 47.9ms\n",
      "Speed: 0.8ms preprocess, 47.9ms inference, 1.2ms postprocess per image at shape (1, 3, 224, 640)\n",
      "Subfigure count mismatch: 3 detected, 2 expected\n",
      "3\n",
      "\n",
      "image 1/1 /Users/etowah/projects/PMC-figure-downloader/img/10335936_Fig15.jpg: 480x640 3 labels, 77.9ms\n",
      "Speed: 1.8ms preprocess, 77.9ms inference, 0.4ms postprocess per image at shape (1, 3, 480, 640)\n",
      "3\n",
      "\n",
      "image 1/1 /Users/etowah/projects/PMC-figure-downloader/img/10335936_Fig16.jpg: 288x640 4 labels, 56.8ms\n",
      "Speed: 1.1ms preprocess, 56.8ms inference, 0.4ms postprocess per image at shape (1, 3, 288, 640)\n",
      "1\n",
      "The flow chart summarizes the overall study design in terms of cohorts, phenotypes and types of genetic analyses performed. ICD, International Classification of Diseases.\n",
      "1\n",
      "Quantile–quantile (QQ) plot of the rare variant associations (both variant and burden associations) with six smoking phenotypes (ever smoker, heavy smoker, former smoker, nicotine dependence, cig per day and age started smoking). The dashed line corresponds to the exome-wide significant threshold, 4.5 × 10−8, determined based on a 1% FDR correction applied across all the associations (n tests = 8,417,987).\n",
      "1\n",
      "Cohort-level and meta-analysis summary statistics of the most significant burden–trait associations for each of the three exome-wide significant genes are summarized using forest plots. The ORs and 95% CIs are plotted. The columns ‘case counts’ and ‘control counts’ show the case and control sample sizes, respectively, broken down to the number of carriers of the homozygous reference, heterozygous and homozygous alternative genotypes. For burden definitions, refer to Supplementary Table 2. ALL, all ancestries; AAF, alternative allele frequency (combined frequency of all the variants aggregated in the burden mask).\n",
      "4\n",
      "\n",
      "image 1/1 /Users/etowah/projects/PMC-figure-downloader/img/10335934_Fig4.jpg: 512x640 5 labels, 71.6ms\n",
      "Speed: 2.0ms preprocess, 71.6ms inference, 0.4ms postprocess per image at shape (1, 3, 512, 640)\n",
      "Subfigure count mismatch: 5 detected, 4 expected\n",
      "2\n",
      "\n",
      "image 1/1 /Users/etowah/projects/PMC-figure-downloader/img/10335934_Fig5.jpg: 640x544 1 label, 75.0ms\n",
      "Speed: 2.0ms preprocess, 75.0ms inference, 0.5ms postprocess per image at shape (1, 3, 640, 544)\n",
      "Subfigure count mismatch: 1 detected, 2 expected\n",
      "1\n",
      "We constructed pLOF-only and pLOF-plus-missense burden masks at five allele frequency thresholds using all variants (wCHIP) and excluding CHIP variants (woCHIP) in the UKB and the GHS cohorts and tested their associations with the six major smoking phenotypes using REGENIE (Methods). The burden association P values are plotted, and the summary statistics including sample sizes are provided in Supplementary Table 8. The dashed line corresponds to the significance threshold after adjusting for multiple testing (1% FDR correction).\n",
      "1\n",
      "Prevalence estimates of heavy smokers among CHRNB2 rare variant carriers and non-carriers within each of the five PGS quintiles in the UKB are plotted. Standard errors of the prevalence estimates, displayed as error bars, were calculated using the formula √(pq/n), where n is the number of individuals in each group, p is the prevalence of heavy smoker in the group and q is 1 − p. The PGS was based on a GWAS meta-analysis of the ever smoker phenotype (Methods). CHRNB2 rare variants are those that were aggregated into the CHRNB2 pLOF-plus-missense (AAF < 0.001) burden mask. Statistical differences in the prevalence between carriers and non-carriers were tested using a logistic regression analysis within each quintile; ORs, 95% CIs and P values are shown.\n",
      "1\n",
      "The forest plot displays the cohort-level and meta-analysis associations of the CHRNB2 pLOF-only (AAF<0.001) and pLOF plus missense (AAF<0.001) burden masks with heavy-smoker and ever-smoker tested using REGENIE (Methods). The odds ratios and 95% confidence intervals are plotted. The columns ‘case counts’ and ‘control counts’ show the case and control sample sizes, respectively, broken down to the number of carriers of the homozygous reference, heterozygous and homozygous alternative genotypes.\n",
      "1\n",
      "The forest plots display the cohort-level or meta-analysis associations of CHRNB2 pLOF plus missense (AAF<0.001) burden mask with binary (P<0.1) and quantitative smoking phenotypes (major smoking phenotypes and phenotypes derived based on UKB lifestyle questionnaire) and smoking-related diseases tested using REGENIE (Methods). The odds ratios (or Beta estimates) and 95% confidence intervals are plotted. The columns ‘case counts’ and ‘control counts’ show the case and control sample sizes, respectively, broken down to the number of carriers of the homozygous reference, heterozygous and homozygous alternative genotypes. FEV1 – Forced expiratory volume in 1 sec; FVC – Forced vital capacity; FEV1_FVC – FEV1:FVC ratio; COPD – Chronic obstructive pulmonary disease.\n",
      "1\n",
      "The forest plots display the cohort-level or meta-analysis associations of ASXL1 pLOF only burden mask (AAF<0.01) with binary and quantitative smoking phenotypes (major smoking phenotypes and phenotypes derived based on UKB lifestyle questionnaire with P<0.1) and smoking-related diseases tested using REGENIE (Methods). The odds ratios (or beta estimates) and 95% confidence intervals are plotted. The columns ‘case counts’ and ‘control counts’ show the case and control sample sizes, respectively, broken down to the number of carriers of the homozygous reference, heterozygous and homozygous alternative genotypes.\n",
      "1\n",
      "The forest plots display the cohort-level or meta-analysis associations of DNMT3A pLOF plus missense burden mask (AAF<0.01) with binary and quantitative smoking phenotypes (major smoking phenotypes and phenotypes derived based on the UKB lifestyle questionnaire) and smoking-related diseases tested using REGENIE (Methods). The odds ratios (or beta estimates) and 95% confidence intervals are plotted. The columns ‘case counts’ and ‘control counts’ show the case and control sample sizes, respectively, broken down to the number of carriers of the homozygous reference, heterozygous and homozygous alternative genotypes.\n",
      "3\n",
      "\n",
      "image 1/1 /Users/etowah/projects/PMC-figure-downloader/img/10335934_Fig12.jpg: 640x416 3 labels, 71.1ms\n",
      "Speed: 1.8ms preprocess, 71.1ms inference, 0.4ms postprocess per image at shape (1, 3, 640, 416)\n",
      "3\n",
      "\n",
      "image 1/1 /Users/etowah/projects/PMC-figure-downloader/img/10335934_Fig13.jpg: 640x480 3 labels, 73.9ms\n",
      "Speed: 2.0ms preprocess, 73.9ms inference, 0.5ms postprocess per image at shape (1, 3, 640, 480)\n",
      "1\n",
      "Assuming an 80% power and P value of 5e-8, detectable effect sizes at various minor allele frequency values were calculated for the current sample size of cig-per-day (the smoking trait most associated with CHRNA5 locus) as well for a series of sample sizes up to 1 million. The observed effect sizes for pLOF only burden and pLOF and missense burden associations of CHRNA5, CHRNA3 and CHRNB4 are plotted; all the points lay below the red line, which marks the detection limit of our current sample size, suggesting that we are underpowered. Based on the intersections of the grey lines with the points marking the observed effect sizes, we can approximately guess what sample size will be required to detect these burden signals at P value 5e-8.\n",
      "1\n",
      "Rare pLOF only and pLOF and missense burden associations were tested focusing only on the genes located at the known GWAS loci identified by the recent largest GWAS of smoking to date. We studied two gene lists prioritized by Saunders et al.19: a list of genes mapped to all the identified GWAS loci and a list of ‘high-priority genes’ mapped to GWAS loci with less than five fine-mapped variants. QQ plots of the meta-analysis P values of burden associations are shown. The dashed line corresponds to FDR 1% P value threshold.\n",
      "1\n",
      "Assuming an 80% power, P value threshold of 4e-8 (exome-wide significant threshold of the current study based on FDR 1%), effect sizes (that is, beta values) were computed for a range of minor allele frequencies (combined allele frequency in case of burden masks) for a given sample size (varies across phenotypes). The computed effect sizes (absolute values of beta estimates) are plotted against minor allele frequencies (carrier frequency) for six smoking phenotypes. The carrier frequency corresponding to 100 carriers, calculated for each of the phenotype based on the corresponding sample size, in the X axis and the corresponding effect size in the Y axis are marked with straight lines. The top association of the three genes identified as exome-wide significant are plotted with the color corresponding to the associated phenotype. Based on these power curves, we had 80% power to detect any variant or burden associations with ever-smoker, heavy-smoker and /former-smoker/ with odds ratio ~2.5 or higher (0.4 or lower) when there are at least 100 carriers. And we had 80% power to detect any variant or burden associations with cig-per-day and age started smoking with beta 0.45 (equivalent to 4.7 extra cigarettes for cig-per-day and 1.9 yr earlier age for age started smoking) when there are at least 100 carriers. These calculations assume that there is no heterogeneity in the effect sizes across the cohorts, which is never the case for complex traits such as smoking. Hence, these estimates should be considered arbitrary. Importantly, the effect sizes for protective associations with binary phenotypes are likely overestimated due to imbalances in the case-control ratios.\n",
      "3\n",
      "\n",
      "image 1/1 /Users/etowah/projects/PMC-figure-downloader/img/10335933_Fig1.jpg: 416x640 5 labels, 71.6ms\n",
      "Speed: 1.6ms preprocess, 71.6ms inference, 0.4ms postprocess per image at shape (1, 3, 416, 640)\n",
      "Subfigure count mismatch: 5 detected, 3 expected\n",
      "4\n",
      "\n",
      "image 1/1 /Users/etowah/projects/PMC-figure-downloader/img/10335933_Fig2.jpg: 640x608 4 labels, 78.4ms\n",
      "Speed: 2.4ms preprocess, 78.4ms inference, 0.4ms postprocess per image at shape (1, 3, 640, 608)\n",
      "5\n",
      "\n",
      "image 1/1 /Users/etowah/projects/PMC-figure-downloader/img/10335933_Fig3.jpg: 544x640 5 labels, 78.9ms\n",
      "Speed: 2.0ms preprocess, 78.9ms inference, 0.4ms postprocess per image at shape (1, 3, 544, 640)\n",
      "8\n",
      "\n",
      "image 1/1 /Users/etowah/projects/PMC-figure-downloader/img/10335933_Fig4.jpg: 640x576 8 labels, 74.3ms\n",
      "Speed: 2.3ms preprocess, 74.3ms inference, 0.4ms postprocess per image at shape (1, 3, 640, 576)\n",
      "14\n",
      "\n",
      "image 1/1 /Users/etowah/projects/PMC-figure-downloader/img/10335933_Fig5.jpg: 640x544 13 labels, 78.5ms\n",
      "Speed: 2.2ms preprocess, 78.5ms inference, 0.4ms postprocess per image at shape (1, 3, 640, 544)\n",
      "Subfigure count mismatch: 13 detected, 14 expected\n",
      "7\n",
      "\n",
      "image 1/1 /Users/etowah/projects/PMC-figure-downloader/img/10335933_Fig6.jpg: 608x640 7 labels, 78.3ms\n",
      "Speed: 2.3ms preprocess, 78.3ms inference, 0.4ms postprocess per image at shape (1, 3, 608, 640)\n",
      "4\n",
      "\n",
      "image 1/1 /Users/etowah/projects/PMC-figure-downloader/img/10335929_Fig1.jpg: 256x640 4 labels, 49.0ms\n",
      "Speed: 1.1ms preprocess, 49.0ms inference, 0.4ms postprocess per image at shape (1, 3, 256, 640)\n",
      "4\n",
      "\n",
      "image 1/1 /Users/etowah/projects/PMC-figure-downloader/img/10335929_Fig2.jpg: 448x640 5 labels, 63.2ms\n",
      "Speed: 1.7ms preprocess, 63.2ms inference, 0.6ms postprocess per image at shape (1, 3, 448, 640)\n",
      "4\n",
      "\n",
      "image 1/1 /Users/etowah/projects/PMC-figure-downloader/img/10335929_Fig3.jpg: 480x640 4 labels, 75.1ms\n",
      "Speed: 1.9ms preprocess, 75.1ms inference, 0.4ms postprocess per image at shape (1, 3, 480, 640)\n",
      "2\n",
      "\n",
      "image 1/1 /Users/etowah/projects/PMC-figure-downloader/img/10335929_Fig4.jpg: 320x640 (no detections), 66.7ms\n",
      "Speed: 1.2ms preprocess, 66.7ms inference, 0.2ms postprocess per image at shape (1, 3, 320, 640)\n",
      "Subfigure count mismatch: 0 detected, 2 expected\n",
      "1\n",
      "Switch error locations for 50 validation samples when phased as part of 480 k samples across the entire chromosome 20 (x-axis) with SHAPEIT5. A switch between dark and light blue represents a switch error. At the top is shown the Axiom array phasing when using Axiom array genotypes as validation and at the bottom when using the WGS data as validation genotypes. Many of the small segments in the top panel are due to genotyping errors in the duo/trio parents, which involve incorrect duo/trio phasing as validation. When using the WGS as validation, many of these inconsistencies disappear.\n",
      "4\n",
      "\n",
      "image 1/1 /Users/etowah/projects/PMC-figure-downloader/img/10335929_Fig6.jpg: 640x640 4 labels, 89.5ms\n",
      "Speed: 2.4ms preprocess, 89.5ms inference, 0.4ms postprocess per image at shape (1, 3, 640, 640)\n",
      "1\n",
      "Non-reference discordance rate (NRD, y-axis) stratified by minor allele count (x-axis) for 1,000 white British samples genotyped with the Axiom array when using reference panels phased with either SHAPEIT5 (blue) or Beagle5.4 (black) whole genome sequencing (a) or whole exome sequencing (b). In (a) the NRD is also reported for the imputation experiment using the HRC reference panel (gray).\n",
      "1\n",
      "Switch Error Rate (SER, y-axis) for WGS (a) and WES (c) datasets stratified by minor allele count (x-axis) at different phasing confidence score thresholds. Number of heterozygous sites filtered out at different phasing confidence score thresholds in the WGS (b) and WES (d) datasets. Filtering for a confidence score of 0.5 is equivalent to no filtering.\n",
      "3\n",
      "\n",
      "image 1/1 /Users/etowah/projects/PMC-figure-downloader/img/10335929_Fig9.jpg: 224x640 3 labels, 41.3ms\n",
      "Speed: 1.0ms preprocess, 41.3ms inference, 0.4ms postprocess per image at shape (1, 3, 224, 640)\n",
      "1\n",
      "‘Full data’: all LoF variants in the study, except singletons, ‘High confidence’: LoF variants excluding calls with phasing confidence score < 0.99, ‘Random phasing’: shuffling phasing of all LoF variants.\n",
      "4\n",
      "\n",
      "image 1/1 /Users/etowah/projects/PMC-figure-downloader/img/10335929_Fig11.jpg: 448x640 4 labels, 77.5ms\n",
      "Speed: 1.8ms preprocess, 77.5ms inference, 0.4ms postprocess per image at shape (1, 3, 448, 640)\n",
      "4\n",
      "\n",
      "image 1/1 /Users/etowah/projects/PMC-figure-downloader/img/10335929_Fig12.jpg: 448x640 5 labels, 60.1ms\n",
      "Speed: 1.7ms preprocess, 60.1ms inference, 0.4ms postprocess per image at shape (1, 3, 448, 640)\n",
      "5\n",
      "\n",
      "image 1/1 /Users/etowah/projects/PMC-figure-downloader/img/10632151_Fig1.jpg: 320x640 5 labels, 78.6ms\n",
      "Speed: 1.4ms preprocess, 78.6ms inference, 0.8ms postprocess per image at shape (1, 3, 320, 640)\n",
      "9\n",
      "\n",
      "image 1/1 /Users/etowah/projects/PMC-figure-downloader/img/10632151_Fig2.jpg: 640x512 9 labels, 76.2ms\n",
      "Speed: 2.0ms preprocess, 76.2ms inference, 0.5ms postprocess per image at shape (1, 3, 640, 512)\n",
      "7\n",
      "\n",
      "image 1/1 /Users/etowah/projects/PMC-figure-downloader/img/10632151_Fig3.jpg: 544x640 7 labels, 88.1ms\n",
      "Speed: 2.0ms preprocess, 88.1ms inference, 0.5ms postprocess per image at shape (1, 3, 544, 640)\n",
      "5\n",
      "\n",
      "image 1/1 /Users/etowah/projects/PMC-figure-downloader/img/10632151_Fig4.jpg: 352x640 5 labels, 94.7ms\n",
      "Speed: 4.1ms preprocess, 94.7ms inference, 1.0ms postprocess per image at shape (1, 3, 352, 640)\n",
      "7\n",
      "\n",
      "image 1/1 /Users/etowah/projects/PMC-figure-downloader/img/10632151_Fig5.jpg: 640x640 7 labels, 128.0ms\n",
      "Speed: 3.4ms preprocess, 128.0ms inference, 0.9ms postprocess per image at shape (1, 3, 640, 640)\n",
      "13\n",
      "\n",
      "image 1/1 /Users/etowah/projects/PMC-figure-downloader/img/10632151_Fig6.jpg: 640x640 13 labels, 87.8ms\n",
      "Speed: 2.6ms preprocess, 87.8ms inference, 0.6ms postprocess per image at shape (1, 3, 640, 640)\n",
      "6\n",
      "\n",
      "image 1/1 /Users/etowah/projects/PMC-figure-downloader/img/10632151_Fig7.jpg: 448x640 6 labels, 58.4ms\n",
      "Speed: 1.7ms preprocess, 58.4ms inference, 0.5ms postprocess per image at shape (1, 3, 448, 640)\n",
      "4\n",
      "\n",
      "image 1/1 /Users/etowah/projects/PMC-figure-downloader/img/10632151_Fig8.jpg: 512x640 4 labels, 80.0ms\n",
      "Speed: 1.8ms preprocess, 80.0ms inference, 0.5ms postprocess per image at shape (1, 3, 512, 640)\n",
      "3\n",
      "\n",
      "image 1/1 /Users/etowah/projects/PMC-figure-downloader/img/10632151_Fig9.jpg: 640x640 4 labels, 81.8ms\n",
      "Speed: 2.3ms preprocess, 81.8ms inference, 0.4ms postprocess per image at shape (1, 3, 640, 640)\n",
      "Subfigure count mismatch: 4 detected, 3 expected\n",
      "10\n",
      "\n",
      "image 1/1 /Users/etowah/projects/PMC-figure-downloader/img/10632151_Fig10.jpg: 640x448 12 labels, 68.1ms\n",
      "Speed: 1.9ms preprocess, 68.1ms inference, 0.4ms postprocess per image at shape (1, 3, 640, 448)\n",
      "Subfigure count mismatch: 11 detected, 10 expected\n",
      "9\n",
      "\n",
      "image 1/1 /Users/etowah/projects/PMC-figure-downloader/img/10632151_Fig11.jpg: 640x480 9 labels, 66.2ms\n",
      "Speed: 1.8ms preprocess, 66.2ms inference, 0.4ms postprocess per image at shape (1, 3, 640, 480)\n",
      "7\n",
      "\n",
      "image 1/1 /Users/etowah/projects/PMC-figure-downloader/img/10632151_Fig12.jpg: 640x480 7 labels, 88.5ms\n",
      "Speed: 2.4ms preprocess, 88.5ms inference, 0.4ms postprocess per image at shape (1, 3, 640, 480)\n",
      "3\n",
      "\n",
      "image 1/1 /Users/etowah/projects/PMC-figure-downloader/img/10632151_Fig13.jpg: 640x576 3 labels, 99.0ms\n",
      "Speed: 2.2ms preprocess, 99.0ms inference, 1.1ms postprocess per image at shape (1, 3, 640, 576)\n",
      "5\n",
      "\n",
      "image 1/1 /Users/etowah/projects/PMC-figure-downloader/img/10632151_Fig14.jpg: 640x480 6 labels, 69.4ms\n",
      "Speed: 1.8ms preprocess, 69.4ms inference, 0.4ms postprocess per image at shape (1, 3, 640, 480)\n",
      "Subfigure count mismatch: 6 detected, 5 expected\n",
      "4\n",
      "\n",
      "image 1/1 /Users/etowah/projects/PMC-figure-downloader/img/10632151_Fig15.jpg: 480x640 6 labels, 62.7ms\n",
      "Speed: 1.6ms preprocess, 62.7ms inference, 0.4ms postprocess per image at shape (1, 3, 480, 640)\n",
      "Subfigure count mismatch: 6 detected, 4 expected\n",
      "5\n",
      "\n",
      "image 1/1 /Users/etowah/projects/PMC-figure-downloader/img/10632143_Fig1.jpg: 640x608 8 labels, 81.3ms\n",
      "Speed: 2.3ms preprocess, 81.3ms inference, 0.5ms postprocess per image at shape (1, 3, 640, 608)\n",
      "Subfigure count mismatch: 8 detected, 5 expected\n",
      "4\n",
      "\n",
      "image 1/1 /Users/etowah/projects/PMC-figure-downloader/img/10632143_Fig2.jpg: 416x640 4 labels, 81.3ms\n",
      "Speed: 1.9ms preprocess, 81.3ms inference, 0.5ms postprocess per image at shape (1, 3, 416, 640)\n",
      "6\n",
      "\n",
      "image 1/1 /Users/etowah/projects/PMC-figure-downloader/img/10632143_Fig3.jpg: 640x640 6 labels, 85.6ms\n",
      "Speed: 2.4ms preprocess, 85.6ms inference, 0.6ms postprocess per image at shape (1, 3, 640, 640)\n",
      "4\n",
      "\n",
      "image 1/1 /Users/etowah/projects/PMC-figure-downloader/img/10632143_Fig4.jpg: 512x640 4 labels, 76.3ms\n",
      "Speed: 1.7ms preprocess, 76.3ms inference, 0.5ms postprocess per image at shape (1, 3, 512, 640)\n",
      "1\n",
      "a) Time since psoriasis diagnosis as a function of age broken down by sex of the donor. b) Fraction of patients who have a history of any of four common treatments for psoriasis. c) The number of treatment cycles for two common phototreatments: Psoralens + UV-A (PUVA) and UV-B phototreatment. Unk = Number of treatments unknown. d) Histograms showing the volume of the microbiopsies by type. e) Histograms showing the estimated surface area of the skin each microbiopsy covers. f) Median on-target coverage for each microbiopsy. The medians for each type are 58X and 46X for the lesional and non-lesional microbiopsies, respectively. g) Sensitivity estimates derived by sequencing 18 near-replicate samples of epidermis where the same histological features were visible on adjacent sections.\n",
      "1\n",
      "Apart from the Unassigned, components are ordered by the fraction of mutations assigned to each component in the dataset. Three unknown components, N1-N3, were extracted that together account for 2.6% of the mutations. These have been merged with the Unassigned component in Main Fig. 2.\n",
      "1\n",
      "The outlier, patient 34, is the leftmost patient that donated a sample from the leg.\n",
      "1\n",
      "a) A comparison of the indel mutation spectra between samples with high and low burden of the psoralen substitution signature. The spectra are identical (cosine similarity >0.99), indicating that psoralens don’t cause indels. n is the number of mutation events. b) A heatmap showing the effect of sequence context on psoralen-mutagenesis extends beyond the trinucleotide model, with bases 2 bp either side of the mutated base affecting the mutation probability. c) The effect of replication direction of psoralen mutagenesis in whole-genome sequenced samples. More mutations are observed on the leading versus lagging strand. P-values are from a two-sided Poisson test and are not corrected for multiple testing. d) The effect of replication timing on psoralen mutagenesis. Across all samples that underwent whole-genome sequencing the mutation rate is greater in late-replicating regions as compared with early replicating regions.\n",
      "1\n",
      "a) Histogram showing the fraction of cells carrying one or more mutations in each gene for each patient. b) Lollipop plots showing the location and annotation of the identified mutations in each gene in lesional (upper) and non-lesional (lower) skin.\n",
      "2\n",
      "\n",
      "image 1/1 /Users/etowah/projects/PMC-figure-downloader/img/10632132_Fig1.jpg: 480x640 2 labels, 77.9ms\n",
      "Speed: 1.9ms preprocess, 77.9ms inference, 0.7ms postprocess per image at shape (1, 3, 480, 640)\n",
      "4\n",
      "\n",
      "image 1/1 /Users/etowah/projects/PMC-figure-downloader/img/10632132_Fig2.jpg: 608x640 4 labels, 79.9ms\n",
      "Speed: 2.1ms preprocess, 79.9ms inference, 0.5ms postprocess per image at shape (1, 3, 608, 640)\n",
      "2\n",
      "\n",
      "image 1/1 /Users/etowah/projects/PMC-figure-downloader/img/10632132_Fig3.jpg: 512x640 3 labels, 71.4ms\n",
      "Speed: 1.9ms preprocess, 71.4ms inference, 0.7ms postprocess per image at shape (1, 3, 512, 640)\n",
      "Subfigure count mismatch: 3 detected, 2 expected\n",
      "3\n",
      "\n",
      "image 1/1 /Users/etowah/projects/PMC-figure-downloader/img/10632132_Fig4.jpg: 576x640 3 labels, 80.6ms\n",
      "Speed: 2.2ms preprocess, 80.6ms inference, 1.7ms postprocess per image at shape (1, 3, 576, 640)\n",
      "1\n",
      "The percent of mCAs (number of mCA calls spanning genomic location / all mCA calls) is shown on the vertical axis and genomic location is shown on the horizontal axis. Bins span 1megabase (Mb) and mCAs with CF>20% are in dark red, CF\\documentclass[12pt]{minimal}\n",
      "\t\t\t\t\\usepackage{amsmath}\n",
      "\t\t\t\t\\usepackage{wasysym} \n",
      "\t\t\t\t\\usepackage{amsfonts} \n",
      "\t\t\t\t\\usepackage{amssymb} \n",
      "\t\t\t\t\\usepackage{amsbsy}\n",
      "\t\t\t\t\\usepackage{mathrsfs}\n",
      "\t\t\t\t\\usepackage{upgreek}\n",
      "\t\t\t\t\\setlength{\\oddsidemargin}{-69pt}\n",
      "\t\t\t\t\\begin{document}$$\\ge$$\\end{document}≥10% in light red, CF\\documentclass[12pt]{minimal}\n",
      "\t\t\t\t\\usepackage{amsmath}\n",
      "\t\t\t\t\\usepackage{wasysym} \n",
      "\t\t\t\t\\usepackage{amsfonts} \n",
      "\t\t\t\t\\usepackage{amssymb} \n",
      "\t\t\t\t\\usepackage{amsbsy}\n",
      "\t\t\t\t\\usepackage{mathrsfs}\n",
      "\t\t\t\t\\usepackage{upgreek}\n",
      "\t\t\t\t\\setlength{\\oddsidemargin}{-69pt}\n",
      "\t\t\t\t\\begin{document}$$\\ge$$\\end{document}≥3% in grey, CF\\documentclass[12pt]{minimal}\n",
      "\t\t\t\t\\usepackage{amsmath}\n",
      "\t\t\t\t\\usepackage{wasysym} \n",
      "\t\t\t\t\\usepackage{amsfonts} \n",
      "\t\t\t\t\\usepackage{amssymb} \n",
      "\t\t\t\t\\usepackage{amsbsy}\n",
      "\t\t\t\t\\usepackage{mathrsfs}\n",
      "\t\t\t\t\\usepackage{upgreek}\n",
      "\t\t\t\t\\setlength{\\oddsidemargin}{-69pt}\n",
      "\t\t\t\t\\begin{document}$$\\ge$$\\end{document}≥1% in light blue, and CF<1% in dark blue. CHR; chromosome.\n",
      "1\n",
      "Age bins in years are plotted on the horizontal axis and the fraction of females carrying an mCA is plotted on the vertical axis. Autosomal mCAs are shown in blue, and chromosome X (chrX) mCAs are shown in red. Error bars represent 95% confidence intervals. There were n = 41,895 biologically independent individuals included in this analysis.\n",
      "1\n",
      "The fraction of chromosome X (chrX) mCAs is plotted by ancestry and type of mCA with copy-neutral loss of heterozygosity (CN-LOH) in green, Gain in orange, Loss in blue, and Undetermined in grey. AA; African American ancestry, HA; Hispanic ancestry, EA; European ancestry, EAS; East Asian ancestry.\n",
      "1\n",
      "African American ancestry (AA) and European ancestry (EA) groups. The percent of mCAs (number of mCA calls spanning genomic location / all mCA calls) is shown on the vertical axis and genomic location is shown on the horizontal axis. Bins span 1 megabase (Mb) and mCAs with CF>20% are in dark red, CF ≥ 10% in light red, CF ≥ 3% in grey, CF ≥ 1% in light blue, and CF<1% in dark blue. The left panel displays mCAs detected in AAs and the right panel displays mCAs detected in EAs.\n",
      "2\n",
      "\n",
      "image 1/1 /Users/etowah/projects/PMC-figure-downloader/img/10632132_Fig9.jpg: 640x640 2 labels, 109.7ms\n",
      "Speed: 2.3ms preprocess, 109.7ms inference, 0.4ms postprocess per image at shape (1, 3, 640, 640)\n",
      "8\n",
      "\n",
      "image 1/1 /Users/etowah/projects/PMC-figure-downloader/img/10260402_Fig1.jpg: 640x544 8 labels, 76.0ms\n",
      "Speed: 1.9ms preprocess, 76.0ms inference, 0.4ms postprocess per image at shape (1, 3, 640, 544)\n",
      "6\n",
      "\n",
      "image 1/1 /Users/etowah/projects/PMC-figure-downloader/img/10260402_Fig2.jpg: 640x480 6 labels, 58.9ms\n",
      "Speed: 2.0ms preprocess, 58.9ms inference, 0.4ms postprocess per image at shape (1, 3, 640, 480)\n",
      "8\n",
      "\n",
      "image 1/1 /Users/etowah/projects/PMC-figure-downloader/img/10260402_Fig3.jpg: 640x576 10 labels, 75.0ms\n",
      "Speed: 2.2ms preprocess, 75.0ms inference, 0.4ms postprocess per image at shape (1, 3, 640, 576)\n",
      "18\n",
      "\n",
      "image 1/1 /Users/etowah/projects/PMC-figure-downloader/img/10260402_Fig4.jpg: 640x608 8 labels, 73.6ms\n",
      "Speed: 2.4ms preprocess, 73.6ms inference, 0.4ms postprocess per image at shape (1, 3, 640, 608)\n",
      "Subfigure count mismatch: 7 detected, 18 expected\n",
      "13\n",
      "\n",
      "image 1/1 /Users/etowah/projects/PMC-figure-downloader/img/10260402_Fig5.jpg: 640x544 7 labels, 66.9ms\n",
      "Speed: 2.0ms preprocess, 66.9ms inference, 0.4ms postprocess per image at shape (1, 3, 640, 544)\n",
      "Subfigure count mismatch: 7 detected, 13 expected\n",
      "10\n",
      "\n",
      "image 1/1 /Users/etowah/projects/PMC-figure-downloader/img/10260402_Fig6.jpg: 640x512 10 labels, 78.9ms\n",
      "Speed: 2.3ms preprocess, 78.9ms inference, 0.4ms postprocess per image at shape (1, 3, 640, 512)\n",
      "8\n",
      "\n",
      "image 1/1 /Users/etowah/projects/PMC-figure-downloader/img/10260402_Fig7.jpg: 640x512 7 labels, 71.6ms\n",
      "Speed: 2.2ms preprocess, 71.6ms inference, 0.5ms postprocess per image at shape (1, 3, 640, 512)\n",
      "Subfigure count mismatch: 6 detected, 8 expected\n",
      "6\n",
      "\n",
      "image 1/1 /Users/etowah/projects/PMC-figure-downloader/img/10260402_Fig8.jpg: 640x448 7 labels, 58.9ms\n",
      "Speed: 1.7ms preprocess, 58.9ms inference, 0.6ms postprocess per image at shape (1, 3, 640, 448)\n",
      "7\n",
      "\n",
      "image 1/1 /Users/etowah/projects/PMC-figure-downloader/img/10260402_Fig9.jpg: 640x608 7 labels, 97.2ms\n",
      "Speed: 2.4ms preprocess, 97.2ms inference, 0.4ms postprocess per image at shape (1, 3, 640, 608)\n",
      "5\n",
      "\n",
      "image 1/1 /Users/etowah/projects/PMC-figure-downloader/img/10260402_Fig10.jpg: 640x512 6 labels, 72.0ms\n",
      "Speed: 1.8ms preprocess, 72.0ms inference, 0.4ms postprocess per image at shape (1, 3, 640, 512)\n",
      "Subfigure count mismatch: 6 detected, 5 expected\n",
      "9\n",
      "\n",
      "image 1/1 /Users/etowah/projects/PMC-figure-downloader/img/10260402_Fig11.jpg: 640x640 10 labels, 85.4ms\n",
      "Speed: 2.4ms preprocess, 85.4ms inference, 0.4ms postprocess per image at shape (1, 3, 640, 640)\n",
      "Subfigure count mismatch: 10 detected, 9 expected\n",
      "5\n",
      "\n",
      "image 1/1 /Users/etowah/projects/PMC-figure-downloader/img/10260402_Fig12.jpg: 640x512 5 labels, 88.3ms\n",
      "Speed: 2.2ms preprocess, 88.3ms inference, 4.4ms postprocess per image at shape (1, 3, 640, 512)\n",
      "1\n",
      "Diagram depicts genetic variants on human chromosome 21 (HSA21) that may contribute to risk of congenital heart defects (CHDs) in humans with trisomy 21 (T21, yellow), genes with functional evidence whose triplication is necessary or sufficient to increase incidence of CHDs in mouse models of Down syndrome (DS, blue), and genes with supporting evidence in both humans and mouse models of DS (green)3,16,23,24,57–59. Relative cytogenetic locations and number of bolded protein-coding genes are indicated along ideogram of the q arm of HSA21 colored according to Giemsa banding. Sources supporting this summary overview is provided in Source Data Extended Data Fig. 7.\n",
      "1\n",
      "This study included TOPMed WGS and whole transcriptome data generated from whole-blood samples of SAGE AA and GALA II Latino individuals (n = 2,733). We compared elements of the genetic architecture gene expression, such as cis-heritability and genetic variance, across participant groups defined based on self-identified race/ethnicity and genetic ancestry. We performed eQTL mapping and identified eQTLs that were specific to AFR or IAM ancestry. Finally, we developed genetic prediction models of whole-blood transcriptomes and performed comparative TWASs using GWAS summary statistics generated from the PAGE study and the UKB. Figure created with BioRender.com.\n",
      "6\n",
      "\n",
      "image 1/1 /Users/etowah/projects/PMC-figure-downloader/img/10260401_Fig2.jpg: 640x512 6 labels, 82.8ms\n",
      "Speed: 2.1ms preprocess, 82.8ms inference, 0.5ms postprocess per image at shape (1, 3, 640, 512)\n",
      "3\n",
      "\n",
      "image 1/1 /Users/etowah/projects/PMC-figure-downloader/img/10260401_Fig3.jpg: 640x640 3 labels, 92.8ms\n",
      "Speed: 2.6ms preprocess, 92.8ms inference, 0.7ms postprocess per image at shape (1, 3, 640, 640)\n",
      "4\n",
      "\n",
      "image 1/1 /Users/etowah/projects/PMC-figure-downloader/img/10260401_Fig4.jpg: 640x416 4 labels, 63.4ms\n",
      "Speed: 2.0ms preprocess, 63.4ms inference, 0.4ms postprocess per image at shape (1, 3, 640, 416)\n",
      "5\n",
      "\n",
      "image 1/1 /Users/etowah/projects/PMC-figure-downloader/img/10260401_Fig5.jpg: 640x448 5 labels, 61.8ms\n",
      "Speed: 1.9ms preprocess, 61.8ms inference, 0.7ms postprocess per image at shape (1, 3, 640, 448)\n",
      "2\n",
      "\n",
      "image 1/1 /Users/etowah/projects/PMC-figure-downloader/img/10260401_Fig6.jpg: 640x512 2 labels, 94.6ms\n",
      "Speed: 2.2ms preprocess, 94.6ms inference, 0.4ms postprocess per image at shape (1, 3, 640, 512)\n",
      "1\n",
      "Analyses of a, h2 and b, VG were stratified by self-identified race/ethnicity and compared African American (AA), Puerto Rican (PR), Mexican American (MX) participants. Within each self-identified race/ethnicity group, individuals were down-sampled to n = 600. Violin plots show the full distribution of h2 or VG values across all genes within each population. Box plots extend from the 25th to the 75th percentile. Each group-specific median value of h2 or VG is annotated and indicated by the shaded circle. All Wilcoxon p-values are two-sided.\n",
      "1\n",
      "Analyses were performed separately for low-frequency variants with 0.01 ≤MAF ≤ 0.1 and for common variants with 0.01 ≤MAF < 0.5. Split violin plots show the distribution of a, b, h2 and c, d, VG in individuals with >50% global genetic ancestry (High) to participants with <10% of the same ancestry (Low). Analyses were conducted separately for African (AFR) and Indigenous American (IAM) ancestry within each MAF bin. Box plots extend from the 25th to the 75th percentile and the median value for h2 and VG value is annotated. All Wilcoxon p-values are two-sided.\n",
      "1\n",
      "Sample size was fixed to n = 600 for eQTL mapping analyses in each ancestry group. Independent cis-eQTLs were identified by performing LD-based clumping (r2 < 0.10) of statistically significant eQTLs identified in each population. Individuals with >50% global genetic ancestry (High) were compared to participants with <10% of the same ancestry (Low). Analyses were conducted separately for African (AFR) and Indigenous American (IAM) ancestry. The distribution of independent cis-eQTLs per gene between was compared between a, AFRhigh and AFRlow populations and b, IAMhigh and IAMlow groups using a two-sided Wilcoxon test. Pie charts visualize the proportion of genes with a greater number of cis-eQTLs in c, AFRhigh compared to AFRlow and in d, IAMhigh compared to IAMlow.\n",
      "1\n",
      "Estimates of a, h2 and c, VG for each gene in individuals with ≥50% global African ancestry (AFRHigh) are plotted on the y-axis and in participants with <10% AFR ancestry (AFRLow) on the x-axis. Genes containing ancestry-specific eQTLs are highlighted. The proportion of genes falling off the diagonal, with higher b, h2 or d, VG in AFRHigh than AFRLow, is show using stacked bar plots and compared using a two-sided binomial test.\n",
      "2\n",
      "\n",
      "image 1/1 /Users/etowah/projects/PMC-figure-downloader/img/10260401_Fig11.jpg: 320x640 2 labels, 69.8ms\n",
      "Speed: 1.4ms preprocess, 69.8ms inference, 0.6ms postprocess per image at shape (1, 3, 320, 640)\n",
      "1\n",
      "TWAS of 22 blood-based biomarkers and quantitative traits was performed using GWAS summary statistics from African and European ancestry subjects in the UK Biobank. We applied GTEx v8 whole blood models and GALA/SAGE models trained in African Americans (AA) to each set of summary statistics. The number of genes with false discovery rate (FDR) < 0.05 detected in ancestry matched and ancestry discordant analyses is summarized for TWAS of a, African (AFR) and b, European (EUR) ancestry subjects. Correlation between TWAS z-scores for statistically significant associations detected in c, UKB EUR and d, UKB AFR are shown for genes that were present in GTEx and GALA/SAGE models. Association between TWAS z-scores from each model is visualized by a linear regression line with shaded 95% confidence intervals. Genes highlighted in orange had FDR < 0.05 using ancestry-matched models but did not reach nominal significance (TWAS p-value>0.05) using ancestry discordant models.\n",
      "3\n",
      "\n",
      "image 1/1 /Users/etowah/projects/PMC-figure-downloader/img/10260400_Fig1.jpg: 640x416 6 labels, 65.4ms\n",
      "Speed: 1.9ms preprocess, 65.4ms inference, 0.5ms postprocess per image at shape (1, 3, 640, 416)\n",
      "Subfigure count mismatch: 5 detected, 3 expected\n",
      "5\n",
      "\n",
      "image 1/1 /Users/etowah/projects/PMC-figure-downloader/img/10260400_Fig2.jpg: 640x448 5 labels, 85.6ms\n",
      "Speed: 4.3ms preprocess, 85.6ms inference, 0.4ms postprocess per image at shape (1, 3, 640, 448)\n",
      "5\n",
      "\n",
      "image 1/1 /Users/etowah/projects/PMC-figure-downloader/img/10260400_Fig3.jpg: 640x480 6 labels, 92.3ms\n",
      "Speed: 1.8ms preprocess, 92.3ms inference, 0.5ms postprocess per image at shape (1, 3, 640, 480)\n",
      "4\n",
      "\n",
      "image 1/1 /Users/etowah/projects/PMC-figure-downloader/img/10260400_Fig4.jpg: 640x512 4 labels, 82.3ms\n",
      "Speed: 2.4ms preprocess, 82.3ms inference, 1.6ms postprocess per image at shape (1, 3, 640, 512)\n",
      "4\n",
      "\n",
      "image 1/1 /Users/etowah/projects/PMC-figure-downloader/img/10260400_Fig5.jpg: 640x512 4 labels, 66.2ms\n",
      "Speed: 2.5ms preprocess, 66.2ms inference, 0.4ms postprocess per image at shape (1, 3, 640, 512)\n",
      "3\n",
      "\n",
      "image 1/1 /Users/etowah/projects/PMC-figure-downloader/img/10260400_Fig6.jpg: 640x480 4 labels, 91.1ms\n",
      "Speed: 2.2ms preprocess, 91.1ms inference, 0.6ms postprocess per image at shape (1, 3, 640, 480)\n",
      "3\n",
      "\n",
      "image 1/1 /Users/etowah/projects/PMC-figure-downloader/img/10260400_Fig7.jpg: 640x640 3 labels, 90.9ms\n",
      "Speed: 2.9ms preprocess, 90.9ms inference, 0.6ms postprocess per image at shape (1, 3, 640, 640)\n",
      "3\n",
      "\n",
      "image 1/1 /Users/etowah/projects/PMC-figure-downloader/img/10260400_Fig8.jpg: 640x544 4 labels, 92.8ms\n",
      "Speed: 2.2ms preprocess, 92.8ms inference, 1.8ms postprocess per image at shape (1, 3, 640, 544)\n",
      "2\n",
      "\n",
      "image 1/1 /Users/etowah/projects/PMC-figure-downloader/img/10260400_Fig9.jpg: 512x640 2 labels, 68.8ms\n",
      "Speed: 1.8ms preprocess, 68.8ms inference, 0.4ms postprocess per image at shape (1, 3, 512, 640)\n",
      "2\n",
      "\n",
      "image 1/1 /Users/etowah/projects/PMC-figure-downloader/img/10260400_Fig10.jpg: 480x640 2 labels, 62.7ms\n",
      "Speed: 1.7ms preprocess, 62.7ms inference, 0.4ms postprocess per image at shape (1, 3, 480, 640)\n",
      "1\n",
      "IFC staining of CYP17A1, VSNL1 (a ZG marker), and GJA1 was performed on serial sections from an adrenal harboring an APA. IFC staining color-coded as indicated in image. A no primary (1°) antibody control was performed to take into account autofluorescence that occurs in the region. GJA1 punctate expression, annular GJ (white arrow), and GJ plaque (yellow arrow) co-localizes with WGA in VSNL1 positive (+ve) cells. Left, scans of the serial sections. Right, zoomed image within boxed regions in scans. Bottom, image of the CYP17A1 positive cells shown in Extended Data Fig. 4.\n",
      "1\n",
      "IFC staining of CYP11B2 and GJA1 was performed on serial sections of the same adrenal shown in Extended Data Fig. 5. IFC staining is color-coded as indicated in image. GJA1 expression in APM regions was lower compared to adjacent adrenal cortex. Annular GJs (red arrows) were present in APM. Top, scan of the serial adrenal section; Bottom, zoomed images of ZF region and APM regions highlighted in scan.\n",
      "2\n",
      "\n",
      "image 1/1 /Users/etowah/projects/PMC-figure-downloader/img/10260400_Fig13.jpg: 640x576 2 labels, 83.6ms\n",
      "Speed: 2.2ms preprocess, 83.6ms inference, 0.5ms postprocess per image at shape (1, 3, 640, 576)\n",
      "1\n",
      "IHC for APQ2 was performed on a positive control tissue section (from a human kidney) and on adrenal tissue sections containing APA of P1 (top right), APA of P4 (bottom left), and APA of P5 (bottom right). Selective membranous staining (black arrows) was seen in the collecting ducts of the human kidney, whereas using the same parameters, APA of P1 had solitary cytoplasmic bodies staining with AQP2 (red arrows), while APAs of P4 and P5 had apparent cytoplasmic staining (blue arrows).\n",
      "2\n",
      "\n",
      "image 1/1 /Users/etowah/projects/PMC-figure-downloader/img/10260400_Fig15.jpg: 480x640 3 labels, 64.5ms\n",
      "Speed: 1.7ms preprocess, 64.5ms inference, 0.4ms postprocess per image at shape (1, 3, 480, 640)\n",
      "Subfigure count mismatch: 3 detected, 2 expected\n",
      "6\n",
      "\n",
      "image 1/1 /Users/etowah/projects/PMC-figure-downloader/img/10260397_Fig1.jpg: 640x640 5 labels, 83.6ms\n",
      "Speed: 2.4ms preprocess, 83.6ms inference, 0.4ms postprocess per image at shape (1, 3, 640, 640)\n",
      "Subfigure count mismatch: 5 detected, 6 expected\n",
      "3\n",
      "\n",
      "image 1/1 /Users/etowah/projects/PMC-figure-downloader/img/10260397_Fig2.jpg: 640x608 3 labels, 85.4ms\n",
      "Speed: 2.3ms preprocess, 85.4ms inference, 0.5ms postprocess per image at shape (1, 3, 640, 608)\n",
      "20\n",
      "\n",
      "image 1/1 /Users/etowah/projects/PMC-figure-downloader/img/10260397_Fig3.jpg: 480x640 (no detections), 81.9ms\n",
      "Speed: 2.1ms preprocess, 81.9ms inference, 0.2ms postprocess per image at shape (1, 3, 480, 640)\n",
      "Subfigure count mismatch: 0 detected, 20 expected\n",
      "1\n",
      "DAPI (x-axis) vs. FITC (y-axis) dot plot was obtained after the analysis of DAPI-stained chromosome suspensions labeled by FISHIS with FITC-conjugated probes for GAA and ACG microsatellites. The 7D/7el2 translocation chromosomes were sorted from the sorting window shown as red rectangle at purities of 60–65%. Inset: 7D/7el2 translocation chromosome after FISH with probes for pSc119.2 repeat (green), Afa family repeat (red) and 45S rDNA (yellow) that was used to identify chromosomes in the sorted fraction. Chromosomes were counterstained by DAPI (blue).\n",
      "2\n",
      "\n",
      "image 1/1 /Users/etowah/projects/PMC-figure-downloader/img/10260397_Fig5.jpg: 416x640 2 labels, 57.4ms\n",
      "Speed: 1.4ms preprocess, 57.4ms inference, 0.4ms postprocess per image at shape (1, 3, 416, 640)\n",
      "3\n",
      "\n",
      "image 1/1 /Users/etowah/projects/PMC-figure-downloader/img/10260397_Fig6.jpg: 640x480 4 labels, 67.4ms\n",
      "Speed: 1.9ms preprocess, 67.4ms inference, 1.4ms postprocess per image at shape (1, 3, 640, 480)\n",
      "Subfigure count mismatch: 4 detected, 3 expected\n",
      "2\n",
      "\n",
      "image 1/1 /Users/etowah/projects/PMC-figure-downloader/img/10260397_Fig7.jpg: 640x640 2 labels, 77.3ms\n",
      "Speed: 2.4ms preprocess, 77.3ms inference, 0.4ms postprocess per image at shape (1, 3, 640, 640)\n",
      "1\n",
      "A minimum threshold of 94% coverage and an identity of ≥60% were applied. ATSS, Aegilops tauschii subsp. strangulata; HVSV, Hordeum vulgare subsp. vulgare; LR, Lolium rigidum; OS, Oryza sativa; PV, Panicum virgatum; SI, Setaria italica; TA, Triticum aestivum; TD, Triticum dicoccoides; TTSD, Triticum turgidum subsp. durum; ZM, Zea mays.\n",
      "1\n",
      "A minimum threshold of ≥99% coverage and an identity of ≥70% were applied. ATSS, Aegilops tauschii subsp. strangulata; BD, Brachypodium distachyon; DE, Digitaria exilis; EC, Eragrostis curvula; HV, Hordeum vulgare; HVSV, Hordeum vulgare subsp. vulgare; LR, Lolium rigidum; ML, Miscanthus lutarioriparius; OB, Oryza brachyantha; OMVG, Oryza meyeriana var. granulata; OS, Oryza sativa; OSIG, Oryza sativa Indica Group; OSJG, Oryza sativa Japonica Group; PH, Panicum hallii; PHVH, Panicum hallii var. hallii; PM, Panicum miliaceum; PV, Panicum virgatum; SI, Setaria italica; SV, Setaria viridis; SB, Sorghum bicolor; TA, Triticum aestivum; TD, Triticum dicoccoides; TTSD, Triticum turgidum subsp. durum; TU, Triticum urartu; ZM, Zea mays; ZP, Zizania palustris.\n",
      "1\n",
      "The ATP-binding and active sites of the kinases are indicated by boxes and circles in the active and pseudo kinases, respectively, as determined by Interpro scanning, homology searching in AlphaFold-derived 3D models and/or BLAST. White bars indicate loss-of-function missense mutations obtained from EMS mutant screens. Eight mutations in the linker domains are not shown. The proteins are organized from top to bottom by date of publication: Rpg1, Yr36, Tsn1, Rpg5, Yr15, Sr60, Pm24, Sm1, Snn3, Pm4 (encoded by splice version 2), WTK4, Rwt4, Sr62 (refs. 9–21), Sr43 (this study) and Lr9 (ref. 22). START, Steroidogenic Acute Regulatory Protein-related Lipid Transfer; NLR, nucleotide binding and leucine-rich repeat; MSP, major sperm protein; C2, C2 domain; PRT C, phosphoribosyl transferase C-terminal domain; DUF, domain of unknown function; vWA, von Willebrand factor type A domain. White bars indicate the position of EMS-derived loss-of-function mutations (excluding early stop codon mutations) where they are available. The number 9 underneath the active site in the active kinase of Lr9 indicates the presence of nine mutations.\n",
      "1\n",
      "The protein kinase (PK) fusion protein contains an integrated decoy (ID) that traps the avirulence effector protein (red square). This triggers an autophosphorylation of either the protein kinase or decoy, the effector, or the nucleotide binding leucine-rich repeat (NLR) guard. This results in a conformational change which in turn triggers a signal cascade leading to downstream defense responses and immunity.\n",
      "11\n",
      "\n",
      "image 1/1 /Users/etowah/projects/PMC-figure-downloader/img/10562254_Fig1.jpg: 640x608 12 labels, 83.3ms\n",
      "Speed: 2.4ms preprocess, 83.3ms inference, 0.5ms postprocess per image at shape (1, 3, 640, 608)\n",
      "8\n",
      "\n",
      "image 1/1 /Users/etowah/projects/PMC-figure-downloader/img/10562254_Fig2.jpg: 640x512 10 labels, 81.4ms\n",
      "Speed: 2.0ms preprocess, 81.4ms inference, 0.6ms postprocess per image at shape (1, 3, 640, 512)\n",
      "4\n",
      "\n",
      "image 1/1 /Users/etowah/projects/PMC-figure-downloader/img/10562254_Fig3.jpg: 512x640 4 labels, 96.8ms\n",
      "Speed: 3.2ms preprocess, 96.8ms inference, 0.7ms postprocess per image at shape (1, 3, 512, 640)\n",
      "6\n",
      "\n",
      "image 1/1 /Users/etowah/projects/PMC-figure-downloader/img/10562254_Fig4.jpg: 640x576 6 labels, 105.5ms\n",
      "Speed: 2.9ms preprocess, 105.5ms inference, 0.5ms postprocess per image at shape (1, 3, 640, 576)\n",
      "7\n",
      "\n",
      "image 1/1 /Users/etowah/projects/PMC-figure-downloader/img/10562254_Fig5.jpg: 640x544 8 labels, 73.9ms\n",
      "Speed: 2.1ms preprocess, 73.9ms inference, 0.5ms postprocess per image at shape (1, 3, 640, 544)\n",
      "7\n",
      "\n",
      "image 1/1 /Users/etowah/projects/PMC-figure-downloader/img/10562254_Fig6.jpg: 512x640 7 labels, 76.7ms\n",
      "Speed: 2.4ms preprocess, 76.7ms inference, 0.8ms postprocess per image at shape (1, 3, 512, 640)\n",
      "3\n",
      "\n",
      "image 1/1 /Users/etowah/projects/PMC-figure-downloader/img/10562254_Fig7.jpg: 640x512 3 labels, 74.4ms\n",
      "Speed: 1.9ms preprocess, 74.4ms inference, 0.4ms postprocess per image at shape (1, 3, 640, 512)\n",
      "2\n",
      "\n",
      "image 1/1 /Users/etowah/projects/PMC-figure-downloader/img/10562254_Fig8.jpg: 640x576 2 labels, 75.6ms\n",
      "Speed: 2.1ms preprocess, 75.6ms inference, 0.4ms postprocess per image at shape (1, 3, 640, 576)\n",
      "3\n",
      "\n",
      "image 1/1 /Users/etowah/projects/PMC-figure-downloader/img/10562254_Fig9.jpg: 640x448 2 labels, 64.8ms\n",
      "Speed: 1.9ms preprocess, 64.8ms inference, 0.4ms postprocess per image at shape (1, 3, 640, 448)\n",
      "Subfigure count mismatch: 2 detected, 3 expected\n",
      "2\n",
      "\n",
      "image 1/1 /Users/etowah/projects/PMC-figure-downloader/img/10562254_Fig10.jpg: 640x448 4 labels, 60.4ms\n",
      "Speed: 1.9ms preprocess, 60.4ms inference, 0.4ms postprocess per image at shape (1, 3, 640, 448)\n",
      "Subfigure count mismatch: 3 detected, 2 expected\n",
      "11\n",
      "\n",
      "image 1/1 /Users/etowah/projects/PMC-figure-downloader/img/10562254_Fig11.jpg: 640x480 12 labels, 66.7ms\n",
      "Speed: 2.2ms preprocess, 66.7ms inference, 1.4ms postprocess per image at shape (1, 3, 640, 480)\n",
      "3\n",
      "\n",
      "image 1/1 /Users/etowah/projects/PMC-figure-downloader/img/10562254_Fig12.jpg: 640x608 3 labels, 106.2ms\n",
      "Speed: 2.6ms preprocess, 106.2ms inference, 0.6ms postprocess per image at shape (1, 3, 640, 608)\n",
      "6\n",
      "\n",
      "image 1/1 /Users/etowah/projects/PMC-figure-downloader/img/10562254_Fig13.jpg: 640x480 21 labels, 101.0ms\n",
      "Speed: 2.5ms preprocess, 101.0ms inference, 0.9ms postprocess per image at shape (1, 3, 640, 480)\n",
      "Subfigure count mismatch: 20 detected, 6 expected\n",
      "1\n",
      "In rice hybrids, male parents generally have much better yield performance than female parents, so better parent heterosis mostly refers to the advantage over male parents. Therefore, the over-parent heterosis observed in hybrids is predominantly attributable to the influence of positive partial dominance effect loci contributed by female parents, as well as over-dominance effect loci from both parental lines. Thus, female-contributed QTLs that displayed positive dominance effects and the QTLs that demonstrated over-dominance effects were included for analysis. The d/a index is applied to measure the magnitude of dominance effect. d/a ≤ 1 and >1 refer to positive partial dominance (blue) and over dominance effect (orange), respectively.\n",
      "4\n",
      "\n",
      "image 1/1 /Users/etowah/projects/PMC-figure-downloader/img/10562254_Fig15.jpg: 576x640 4 labels, 112.7ms\n",
      "Speed: 2.3ms preprocess, 112.7ms inference, 0.6ms postprocess per image at shape (1, 3, 576, 640)\n",
      "4\n",
      "\n",
      "image 1/1 /Users/etowah/projects/PMC-figure-downloader/img/10484790_Fig1.jpg: 512x640 4 labels, 85.8ms\n",
      "Speed: 2.1ms preprocess, 85.8ms inference, 0.5ms postprocess per image at shape (1, 3, 512, 640)\n",
      "8\n",
      "\n",
      "image 1/1 /Users/etowah/projects/PMC-figure-downloader/img/10484790_Fig2.jpg: 640x544 8 labels, 56.2ms\n",
      "Speed: 2.1ms preprocess, 56.2ms inference, 0.4ms postprocess per image at shape (1, 3, 640, 544)\n",
      "5\n",
      "\n",
      "image 1/1 /Users/etowah/projects/PMC-figure-downloader/img/10484790_Fig3.jpg: 640x640 5 labels, 91.4ms\n",
      "Speed: 2.5ms preprocess, 91.4ms inference, 0.6ms postprocess per image at shape (1, 3, 640, 640)\n",
      "4\n",
      "\n",
      "image 1/1 /Users/etowah/projects/PMC-figure-downloader/img/10484790_Fig4.jpg: 480x640 4 labels, 65.4ms\n",
      "Speed: 1.8ms preprocess, 65.4ms inference, 0.7ms postprocess per image at shape (1, 3, 480, 640)\n",
      "4\n",
      "\n",
      "image 1/1 /Users/etowah/projects/PMC-figure-downloader/img/10484790_Fig5.jpg: 640x640 4 labels, 85.4ms\n",
      "Speed: 2.4ms preprocess, 85.4ms inference, 0.5ms postprocess per image at shape (1, 3, 640, 640)\n",
      "4\n",
      "\n",
      "image 1/1 /Users/etowah/projects/PMC-figure-downloader/img/10484790_Fig6.jpg: 416x640 4 labels, 64.4ms\n",
      "Speed: 1.6ms preprocess, 64.4ms inference, 0.7ms postprocess per image at shape (1, 3, 416, 640)\n",
      "3\n",
      "\n",
      "image 1/1 /Users/etowah/projects/PMC-figure-downloader/img/10484790_Fig7.jpg: 640x512 3 labels, 75.4ms\n",
      "Speed: 2.0ms preprocess, 75.4ms inference, 0.4ms postprocess per image at shape (1, 3, 640, 512)\n",
      "1\n",
      "Per-gene DMS results for the 9 VEP methods that are closest to ESM1b in performance according to the head-to-head comparison (Fig. 3c). The numbers of unique variants scored by each VEP method, out of the total 76,133 variants in the full DMS dataset, are shown in square brackets next to the method names. The numbers of variants per gene are shown in parentheses next to the gene names.\n",
      "1\n",
      "In gray: the distribution of ESM1b PLLR effect scores across indels in ClinVar annotated as variants of uncertain significance (VUS). We estimated the fraction of pathogenic and benign variants among these VUS indels by decomposing the VUS distribution of effect scores as a mixture of the distributions over pathogenic and benign variants (Fig. 6a) approximated by kernel density estimation. Red and blue curves: the mixture components of pathogenic and benign effect scores, respectively. Black dashed curve: the sum of the pathogenic (red) and benign (blue) components as an estimate of the empirical distribution of VUS (gray).\n",
      "2\n",
      "\n",
      "image 1/1 /Users/etowah/projects/PMC-figure-downloader/img/10484790_Fig10.jpg: 256x640 2 labels, 51.9ms\n",
      "Speed: 1.2ms preprocess, 51.9ms inference, 0.5ms postprocess per image at shape (1, 3, 256, 640)\n",
      "5\n",
      "\n",
      "image 1/1 /Users/etowah/projects/PMC-figure-downloader/img/10484790_Fig11.jpg: 640x608 5 labels, 85.4ms\n",
      "Speed: 2.4ms preprocess, 85.4ms inference, 0.4ms postprocess per image at shape (1, 3, 640, 608)\n",
      "2\n",
      "\n",
      "image 1/1 /Users/etowah/projects/PMC-figure-downloader/img/10484790_Fig12.jpg: 640x576 2 labels, 76.1ms\n",
      "Speed: 2.2ms preprocess, 76.1ms inference, 0.4ms postprocess per image at shape (1, 3, 640, 576)\n",
      "2\n",
      "\n",
      "image 1/1 /Users/etowah/projects/PMC-figure-downloader/img/10484788_Fig1.jpg: 640x480 2 labels, 103.8ms\n",
      "Speed: 2.3ms preprocess, 103.8ms inference, 0.6ms postprocess per image at shape (1, 3, 640, 480)\n",
      "11\n",
      "\n",
      "image 1/1 /Users/etowah/projects/PMC-figure-downloader/img/10484788_Fig2.jpg: 640x544 11 labels, 155.0ms\n",
      "Speed: 3.0ms preprocess, 155.0ms inference, 2.7ms postprocess per image at shape (1, 3, 640, 544)\n",
      "2\n",
      "\n",
      "image 1/1 /Users/etowah/projects/PMC-figure-downloader/img/10484788_Fig3.jpg: 640x544 3 labels, 78.1ms\n",
      "Speed: 2.2ms preprocess, 78.1ms inference, 0.9ms postprocess per image at shape (1, 3, 640, 544)\n",
      "Subfigure count mismatch: 3 detected, 2 expected\n",
      "1\n",
      "CELLECT prioritization of 115 cell types from Tabula Muris highlights pancreatic polypeptide (PP) cells (shown in black, one-sided Wilcoxon rank-sum test with significance threshold depicted by a dotted line indicating cell types with a nominal PS-LDSC < 4.3 × 10−4).\n",
      "1\n",
      "The x axis provides the estimated rg genetic correlation values for traits or diseases (y axis) reaching at least nominal significance (P < 0.05). Correlations reaching P < 0.010 are labeled with the prime symbol, and those P < 2.1 × 10−4 are labeled with the asterisk symbol. P values were calculated from the two-sided z statistics computed by dividing the estimated rg by the estimated standard error, without adjustment. Each error bar represents the standard error of the estimate.\n",
      "7\n",
      "\n",
      "image 1/1 /Users/etowah/projects/PMC-figure-downloader/img/10484788_Fig6.jpg: 640x640 7 labels, 100.5ms\n",
      "Speed: 2.4ms preprocess, 100.5ms inference, 0.6ms postprocess per image at shape (1, 3, 640, 640)\n",
      "1\n",
      "a-i, Look-up of effects was done in previously published genome-wide association studies for HbA1c (a), fasting glucose (b), fasting insulin (c), type 2 diabetes (d), forced expiratory volume in one second (FEV1) (e), forced vital capacity (FVC) (f), FEV1/FVC (g), lung cancer (h) and squamous cell lung cancer (i). RG and other phenotype effect sizes are plotted along the y- and x-axes, respectively. Point size and color are proportional to the significance of the variant in each phenotype, with red indicating higher and blue lower significance, respectively. The dashed line represents the line of best fit. P < 5.0 × 10−8 was considered statistically significant after adjusting for multiple testing. Two-tailed P-values are reported. Variants with P-values in the lowest decile are labeled.\n",
      "1\n",
      "a-h, Plots are shown for GCKR (a), TET2 (b), RREB1 (c), NMT1 (d) and WIPI1 (e) loci and low-frequency coding variants at EDEM3 (f), NEUROD1 (g) and GLP1R (h) loci. The x-axis shows the chromosomal position, and the y-axis shows the uncorrected two-sided −log10\n",
      "P values from the UKBB GWAS conducted using linear mixed-modeling in BOLT-LMM. Horizontal line corresponds to P = 5 × 10−8 and blue peaks show the recombination rate.\n",
      "1\n",
      "Minor allele frequency-weighted linear regression was used to test if mini-Gs response to GLP-1 stimulation significantly predicted point estimates of GLP1R variant effect on RG levels (AST20 βRG as estimated in whole-exome sequencing data from the UKBB study). Mini-Gs response to GLP-1 stimulation was corrected for variant surface expression (nmax = 22, exact n for each variant is provided in Supplementary Table 11). Error bars extend one standard error above and below the point estimate. Size of the dots is proportional to the weight applied in the regression model (Methods). The regression results (coefficient of determination R2 = 0.56, F(1, 14) = 20.1, P = 5.2 × 10−4) suggest that mini-Gs coupling in response to GLP-1 stimulation predicts the effect of these coding variants on RG levels (AST20 βRG = − 0.028; 95% CI = −0.042 to −0.015; P = 5.2 × 10−4). The gray shaded area around the regression line corresponds to the 95% confidence interval of predictions from the model. Variants in red showed no detectable surface expression (NDE) and are not included in regression analysis.\n",
      "1\n",
      "The analyses were performed using generalized linear modeling in GARFIELD software. We considered enrichment to be statistically significant if the RG GWAS P-value reached P = 1 × 10−8 and the enrichment analysis P-value was < 2.5 × 10−5 (Bonferonni corrected for 2,040 annotations).\n",
      "1\n",
      "All variant effects were aligned to the RG risk allele. HapMap2 based summary statistics were imputed using SS-Imp v0.5.565 to minimize missingness. Missing summary statistics values were imputed via mean imputation. The heatmap was produced using the Pheatmap package. For visualization, the Z-scores were truncated to the value corresponding to genome-wide significance (Z = 5.45), and 11 phenotypes with the lowest median absolute Z-scores were excluded.\n",
      "3\n",
      "\n",
      "image 1/1 /Users/etowah/projects/PMC-figure-downloader/img/10484788_Fig12.jpg: 640x640 3 labels, 86.3ms\n",
      "Speed: 2.4ms preprocess, 86.3ms inference, 0.5ms postprocess per image at shape (1, 3, 640, 640)\n",
      "5\n",
      "\n",
      "image 1/1 /Users/etowah/projects/PMC-figure-downloader/img/10484787_Fig1.jpg: 544x640 5 labels, 72.8ms\n",
      "Speed: 2.5ms preprocess, 72.8ms inference, 0.4ms postprocess per image at shape (1, 3, 544, 640)\n",
      "10\n",
      "\n",
      "image 1/1 /Users/etowah/projects/PMC-figure-downloader/img/10484787_Fig2.jpg: 640x608 10 labels, 86.9ms\n",
      "Speed: 2.3ms preprocess, 86.9ms inference, 0.5ms postprocess per image at shape (1, 3, 640, 608)\n",
      "5\n",
      "\n",
      "image 1/1 /Users/etowah/projects/PMC-figure-downloader/img/10484787_Fig3.jpg: 416x640 5 labels, 61.2ms\n",
      "Speed: 1.7ms preprocess, 61.2ms inference, 0.4ms postprocess per image at shape (1, 3, 416, 640)\n",
      "8\n",
      "\n",
      "image 1/1 /Users/etowah/projects/PMC-figure-downloader/img/10484787_Fig4.jpg: 640x640 8 labels, 83.9ms\n",
      "Speed: 2.3ms preprocess, 83.9ms inference, 0.5ms postprocess per image at shape (1, 3, 640, 640)\n",
      "5\n",
      "\n",
      "image 1/1 /Users/etowah/projects/PMC-figure-downloader/img/10484787_Fig5.jpg: 512x640 5 labels, 74.7ms\n",
      "Speed: 2.1ms preprocess, 74.7ms inference, 0.4ms postprocess per image at shape (1, 3, 512, 640)\n",
      "4\n",
      "\n",
      "image 1/1 /Users/etowah/projects/PMC-figure-downloader/img/10484787_Fig6.jpg: 640x640 6 labels, 92.5ms\n",
      "Speed: 2.8ms preprocess, 92.5ms inference, 0.5ms postprocess per image at shape (1, 3, 640, 640)\n",
      "Subfigure count mismatch: 5 detected, 4 expected\n",
      "1\n",
      "H3–H4 asymmetry in MCM2-2A cells creates a lagging strand largely devoid of parental histone PTMs, which affects the accuracy of chromatin restoration and creates a permissive environment for unscheduled expression in every cell cycle. This broadly alters the histone PTM epigenome with both local and global changes in repressive modifications. MCM2-2A cells show loss of H3K9me3-based repeat repression, misregulation of H3K27me3 and bivalent genes, and reduced ESC plasticity and embryonic differentiation. Naïve, naïve pluripotency; primed, lineage-primed states; new histones, green; parental histones, purple.\n",
      "6\n",
      "\n",
      "image 1/1 /Users/etowah/projects/PMC-figure-downloader/img/10484787_Fig8.jpg: 512x640 6 labels, 98.5ms\n",
      "Speed: 1.9ms preprocess, 98.5ms inference, 0.5ms postprocess per image at shape (1, 3, 512, 640)\n",
      "9\n",
      "\n",
      "image 1/1 /Users/etowah/projects/PMC-figure-downloader/img/10484787_Fig9.jpg: 640x448 9 labels, 85.0ms\n",
      "Speed: 2.0ms preprocess, 85.0ms inference, 0.9ms postprocess per image at shape (1, 3, 640, 448)\n",
      "9\n",
      "\n",
      "image 1/1 /Users/etowah/projects/PMC-figure-downloader/img/10484787_Fig10.jpg: 640x480 11 labels, 73.6ms\n",
      "Speed: 2.0ms preprocess, 73.6ms inference, 0.5ms postprocess per image at shape (1, 3, 640, 480)\n",
      "1\n",
      "a-f, Box plots showing relative levels of H3K4 methylation and acetylation (a), H3K18 and H3K23 acetylation (b), H3K36 methylation (c), H3K79 methylation and acetylation (d), H4K5/8/12/16 acetylation (e), and H4K20 methylation (f). Histone modifications were quantified by mass spectrometry in WT and MCM2-2A cells growing asynchronously. WT#1 (n = 4), WT#2 (n = 4), WT#3 (n = 4), WT#4 (n = 4), MCM2-2A#1 (n = 4), MCM2-2A#3 (n = 4), MCM2-2A#4 (n = 4), MCM2-2A#1 (n = 4); n = biological replicates. Lines represent median, boxes represent the 1st and 3rd quartiles, whiskers extend 1.5x interquartile range (IQR), dots depict individual data points. Two-sided Welch’s t-test FDR.\n",
      "9\n",
      "\n",
      "image 1/1 /Users/etowah/projects/PMC-figure-downloader/img/10484787_Fig12.jpg: 544x640 9 labels, 85.9ms\n",
      "Speed: 2.0ms preprocess, 85.9ms inference, 0.5ms postprocess per image at shape (1, 3, 544, 640)\n",
      "10\n",
      "\n",
      "image 1/1 /Users/etowah/projects/PMC-figure-downloader/img/10484787_Fig13.jpg: 640x480 12 labels, 79.4ms\n",
      "Speed: 1.9ms preprocess, 79.4ms inference, 0.4ms postprocess per image at shape (1, 3, 640, 480)\n",
      "12\n",
      "\n",
      "image 1/1 /Users/etowah/projects/PMC-figure-downloader/img/10484787_Fig14.jpg: 640x480 13 labels, 89.4ms\n",
      "Speed: 1.9ms preprocess, 89.4ms inference, 0.6ms postprocess per image at shape (1, 3, 640, 480)\n",
      "Subfigure count mismatch: 11 detected, 12 expected\n",
      "9\n",
      "\n",
      "image 1/1 /Users/etowah/projects/PMC-figure-downloader/img/10484787_Fig15.jpg: 640x480 10 labels, 82.0ms\n",
      "Speed: 2.2ms preprocess, 82.0ms inference, 0.5ms postprocess per image at shape (1, 3, 640, 480)\n",
      "11\n",
      "\n",
      "image 1/1 /Users/etowah/projects/PMC-figure-downloader/img/10484787_Fig16.jpg: 640x448 14 labels, 79.0ms\n",
      "Speed: 4.3ms preprocess, 79.0ms inference, 0.8ms postprocess per image at shape (1, 3, 640, 448)\n",
      "Subfigure count mismatch: 12 detected, 11 expected\n",
      "5\n",
      "\n",
      "image 1/1 /Users/etowah/projects/PMC-figure-downloader/img/10484787_Fig17.jpg: 288x640 9 labels, 55.1ms\n",
      "Speed: 1.1ms preprocess, 55.1ms inference, 1.0ms postprocess per image at shape (1, 3, 288, 640)\n",
      "Subfigure count mismatch: 9 detected, 5 expected\n",
      "2\n",
      "\n",
      "image 1/1 /Users/etowah/projects/PMC-figure-downloader/img/10484786_Fig1.jpg: 576x640 2 labels, 84.5ms\n",
      "Speed: 2.2ms preprocess, 84.5ms inference, 0.4ms postprocess per image at shape (1, 3, 576, 640)\n",
      "6\n",
      "\n",
      "image 1/1 /Users/etowah/projects/PMC-figure-downloader/img/10484786_Fig2.jpg: 384x640 6 labels, 72.7ms\n",
      "Speed: 1.4ms preprocess, 72.7ms inference, 1.1ms postprocess per image at shape (1, 3, 384, 640)\n",
      "1\n",
      "A total of 315 RXLR effectors were transiently expressed in 52 S. americanum accessions. The HR index (2, strong HR; 1, weak HR; 0, no HR, NA, not available) was used for the heatmap. These effectors were screened on N. benthamiana30, and their recognitions are included in this heatmap. Empty PVX vector was used as negative control, and co-expression of Rpi-amr3–HisFlag and AVRamr3–GFP was used as positive control. The S. americanum accessions were ordered based on the phylogenetic tree; SP3400 is not included in this tree. The scale bar represents the number of amino acid substitutions per site. The S. americanum accessions were classified into four groups (gray or yellow shading). The four reference accessions are marked by red arrows. The effectors were ordered based on the total HR index. For each effector, the numbers of responsive S. americanum accessions is visualized by a bar chart on the top of the heatmap. For each S. americanum accession, the numbers of recognized effectors is visualized by a bar chart on the right of the heatmap. Some RXLR effectors previously characterized or mentioned in this study are indicated by gray arrows.\n",
      "6\n",
      "\n",
      "image 1/1 /Users/etowah/projects/PMC-figure-downloader/img/10484786_Fig4.jpg: 384x640 6 labels, 53.4ms\n",
      "Speed: 1.5ms preprocess, 53.4ms inference, 0.4ms postprocess per image at shape (1, 3, 384, 640)\n",
      "8\n",
      "\n",
      "image 1/1 /Users/etowah/projects/PMC-figure-downloader/img/10484786_Fig5.jpg: 608x640 9 labels, 102.3ms\n",
      "Speed: 2.5ms preprocess, 102.3ms inference, 0.6ms postprocess per image at shape (1, 3, 608, 640)\n",
      "Subfigure count mismatch: 9 detected, 8 expected\n",
      "6\n",
      "\n",
      "image 1/1 /Users/etowah/projects/PMC-figure-downloader/img/10484784_Fig1.jpg: 384x640 7 labels, 60.5ms\n",
      "Speed: 1.6ms preprocess, 60.5ms inference, 1.4ms postprocess per image at shape (1, 3, 384, 640)\n",
      "4\n",
      "\n",
      "image 1/1 /Users/etowah/projects/PMC-figure-downloader/img/10484784_Fig2.jpg: 512x640 4 labels, 66.6ms\n",
      "Speed: 1.8ms preprocess, 66.6ms inference, 0.4ms postprocess per image at shape (1, 3, 512, 640)\n",
      "7\n",
      "\n",
      "image 1/1 /Users/etowah/projects/PMC-figure-downloader/img/10484784_Fig3.jpg: 448x640 7 labels, 70.4ms\n",
      "Speed: 1.8ms preprocess, 70.4ms inference, 0.5ms postprocess per image at shape (1, 3, 448, 640)\n",
      "6\n",
      "\n",
      "image 1/1 /Users/etowah/projects/PMC-figure-downloader/img/10484784_Fig4.jpg: 576x640 6 labels, 87.4ms\n",
      "Speed: 2.4ms preprocess, 87.4ms inference, 1.0ms postprocess per image at shape (1, 3, 576, 640)\n",
      "3\n",
      "\n",
      "image 1/1 /Users/etowah/projects/PMC-figure-downloader/img/10484784_Fig5.jpg: 640x608 4 labels, 82.1ms\n",
      "Speed: 2.3ms preprocess, 82.1ms inference, 0.7ms postprocess per image at shape (1, 3, 640, 608)\n",
      "Subfigure count mismatch: 4 detected, 3 expected\n",
      "2\n",
      "\n",
      "image 1/1 /Users/etowah/projects/PMC-figure-downloader/img/10484784_Fig6.jpg: 192x640 3 labels, 42.2ms\n",
      "Speed: 0.7ms preprocess, 42.2ms inference, 0.4ms postprocess per image at shape (1, 3, 192, 640)\n",
      "Subfigure count mismatch: 3 detected, 2 expected\n",
      "4\n",
      "\n",
      "image 1/1 /Users/etowah/projects/PMC-figure-downloader/img/10484784_Fig7.jpg: 416x640 5 labels, 61.0ms\n",
      "Speed: 1.6ms preprocess, 61.0ms inference, 1.1ms postprocess per image at shape (1, 3, 416, 640)\n",
      "Subfigure count mismatch: 5 detected, 4 expected\n",
      "2\n",
      "\n",
      "image 1/1 /Users/etowah/projects/PMC-figure-downloader/img/10484784_Fig8.jpg: 288x640 3 labels, 49.0ms\n",
      "Speed: 1.1ms preprocess, 49.0ms inference, 0.5ms postprocess per image at shape (1, 3, 288, 640)\n",
      "Subfigure count mismatch: 3 detected, 2 expected\n",
      "2\n",
      "\n",
      "image 1/1 /Users/etowah/projects/PMC-figure-downloader/img/10484784_Fig9.jpg: 320x640 2 labels, 55.0ms\n",
      "Speed: 1.3ms preprocess, 55.0ms inference, 0.4ms postprocess per image at shape (1, 3, 320, 640)\n",
      "1\n",
      "Comparison of the predicted probability of developing any MN with the observed MN incidence in the UKB validation cohort of 207,039 individuals at any time during the follow-up/observation period (dots showing the mean and error bars showing 1.96 standard deviations that is 5–95% CI). Samples were binned according to predicted probability ranges as follows: 0–0.05, 0.05–0.1, 0.1–0.3, 0.3–0.5 and 0.5–1. Individuals who died during the observation period without having developed MN were not included in the calculations. The plot shows close agreement (along the dotted line y = x) between prediction and observed incidence.\n",
      "4\n",
      "\n",
      "image 1/1 /Users/etowah/projects/PMC-figure-downloader/img/10484784_Fig11.jpg: 640x640 4 labels, 91.4ms\n",
      "Speed: 2.2ms preprocess, 91.4ms inference, 0.4ms postprocess per image at shape (1, 3, 640, 640)\n",
      "2\n",
      "\n",
      "image 1/1 /Users/etowah/projects/PMC-figure-downloader/img/10484784_Fig12.jpg: 320x640 2 labels, 72.7ms\n",
      "Speed: 1.3ms preprocess, 72.7ms inference, 0.6ms postprocess per image at shape (1, 3, 320, 640)\n",
      "3\n",
      "\n",
      "image 1/1 /Users/etowah/projects/PMC-figure-downloader/img/10484783_Fig1.jpg: 640x576 3 labels, 85.8ms\n",
      "Speed: 2.3ms preprocess, 85.8ms inference, 0.5ms postprocess per image at shape (1, 3, 640, 576)\n",
      "4\n",
      "\n",
      "image 1/1 /Users/etowah/projects/PMC-figure-downloader/img/10484783_Fig2.jpg: 448x640 4 labels, 72.9ms\n",
      "Speed: 1.9ms preprocess, 72.9ms inference, 0.5ms postprocess per image at shape (1, 3, 448, 640)\n",
      "6\n",
      "\n",
      "image 1/1 /Users/etowah/projects/PMC-figure-downloader/img/10484783_Fig3.jpg: 640x640 6 labels, 103.5ms\n",
      "Speed: 2.3ms preprocess, 103.5ms inference, 5.3ms postprocess per image at shape (1, 3, 640, 640)\n",
      "1\n",
      "Loci are either associated with keratinocyte cancer or tan response (Tan) or previously found to be under selection in Singaporean genomes; – for genotype indicates that no call could be made (Methods). Associated genes and mechanisms are suggested using phenome-wide association studies (PheWAS) and expression quantitative trait locus (eQTL) data from the Open Targets Genetics database. ECM, extra-cellular matrix; – indicates that the mechanism is unknown. Population allele frequencies are reported for East Asia (EA freq) and Great Britain (GBR freq) using data from the 1000 Genomes Project Phase 3 (Methods). Purple shading indicates the frequency of the alternative (ALT) allele in each population. Note that all loci in the HYAL region form part of the same linkage disequilibrium block on chromosome 3p.21. For each donor, green indicates homozygous reference (REF) alleles, yellow heterozygous and red homozygous ALT alleles.\n",
      "4\n",
      "\n",
      "image 1/1 /Users/etowah/projects/PMC-figure-downloader/img/10484783_Fig5.jpg: 512x640 4 labels, 70.6ms\n",
      "Speed: 1.8ms preprocess, 70.6ms inference, 0.4ms postprocess per image at shape (1, 3, 512, 640)\n",
      "1\n",
      "a Estimates of genome-wide mutation burden per donor. b Proportion of SBS1 mutations/mm2 per donor (n = 11) by country (two-sided t-test: p = 5.8 × 10−3). Tukey boxplot where lower and upper hinges = 1st and 3rd quartile, centre = median, outliers > 1.5 x inter-quartile range. c Proportion of SBS5 mutations/mm2 per donor (n = 11) by country (two-sided t-test: p = 2.1 × 10−4). Tukey boxplot where lower and upper hinges = 1st and 3rd quartile, centre = median, outliers > 1.5 × inter-quartile range. d Proportion of SBS assigned to each signature, split by mutations above and below median variant allele frequency (VAF) for each country (Pearson’s chi-square UK: p < 2.2 × 10−16; SG: p = 1.3 × 10−14). e Counts of double-base substitutions (DBS) per mm2 of skin of donors from each country (UK mean = 1.08 DBS/mm2, SG mean = 0.126 DBS/mm2, two-sided Welch’s t-test: p = 2.3 × 10−3). f Count of insertions and deletions per mm2 in each donor (n = 11) by country (UK mean = 0.72 indels/mm2, SG mean = 0.37 indels/mm2, two-sided Welch’s t-test: p = 0.07). Tukey boxplot where lower and upper hinges = 1st and 3rd quartile, centre = median, outliers > 1.5 x inter-quartile range.\n",
      "1\n",
      "a Ratio of observed/expected non-synonymous mutations for positively selected genes by country (q < 0.01). No synonymous mutations were detected in Singaporean skin for TP53 and AJUBA, leading to high dN/dS ratios. Line drawn at y = 1. b Sizes of all clones with protein-altering mutations in the top four positively selected genes, by country (samples with known CNA removed), two-sided t-test adjusted by Bonferroni multiple test correction: p = 6 × 10−4 (NOTCH1, n = 1,020), 1.5 × 10−2 (NOTCH2, n = 390), 0.68 (FAT1, n = 432) and 1.0 (TP53, n = 366). Tukey boxplot where lower and upper hinges = 1st and 3rd quartile, centre = median, outliers > 1.5 x inter-quartile range. c Distributions of mutations across codons in TP53. The most frequently mutated codon in cancer, R248 (shown red), is the most common codon change in UK skin but is absent in Singaporean skin.\n",
      "1\n",
      "Correlation between mean quality sequencing coverage per sample and the number of mutations detected for a Singapore and b UK (error bands = 95% confidence interval). Mean depth of coverage was calculated after removing off-target reads, duplicates and those with mapping quality of 25 or less and base quality of 30 or less. Samples of neither country show a correlation between depth of coverage and mutation counts (linear regression: (SG) R = 0.13, p = 0.01; (UK) R = 0.26, p = 5.8 × 10−9).\n",
      "9\n",
      "\n",
      "image 1/1 /Users/etowah/projects/PMC-figure-downloader/img/10412459_Fig1.jpg: 640x480 9 labels, 76.6ms\n",
      "Speed: 2.0ms preprocess, 76.6ms inference, 0.4ms postprocess per image at shape (1, 3, 640, 480)\n",
      "8\n",
      "\n",
      "image 1/1 /Users/etowah/projects/PMC-figure-downloader/img/10412459_Fig2.jpg: 640x512 8 labels, 69.7ms\n",
      "Speed: 2.1ms preprocess, 69.7ms inference, 0.4ms postprocess per image at shape (1, 3, 640, 512)\n",
      "6\n",
      "\n",
      "image 1/1 /Users/etowah/projects/PMC-figure-downloader/img/10412459_Fig3.jpg: 416x640 6 labels, 79.6ms\n",
      "Speed: 1.7ms preprocess, 79.6ms inference, 0.4ms postprocess per image at shape (1, 3, 416, 640)\n",
      "8\n",
      "\n",
      "image 1/1 /Users/etowah/projects/PMC-figure-downloader/img/10412459_Fig4.jpg: 576x640 8 labels, 82.2ms\n",
      "Speed: 2.3ms preprocess, 82.2ms inference, 0.4ms postprocess per image at shape (1, 3, 576, 640)\n",
      "9\n",
      "\n",
      "image 1/1 /Users/etowah/projects/PMC-figure-downloader/img/10412459_Fig5.jpg: 576x640 10 labels, 94.4ms\n",
      "Speed: 2.2ms preprocess, 94.4ms inference, 0.5ms postprocess per image at shape (1, 3, 576, 640)\n",
      "5\n",
      "\n",
      "image 1/1 /Users/etowah/projects/PMC-figure-downloader/img/10412459_Fig6.jpg: 384x640 5 labels, 58.8ms\n",
      "Speed: 1.5ms preprocess, 58.8ms inference, 0.5ms postprocess per image at shape (1, 3, 384, 640)\n",
      "9\n",
      "\n",
      "image 1/1 /Users/etowah/projects/PMC-figure-downloader/img/10412459_Fig7.jpg: 640x640 9 labels, 84.6ms\n",
      "Speed: 2.4ms preprocess, 84.6ms inference, 0.6ms postprocess per image at shape (1, 3, 640, 640)\n",
      "1\n",
      "When exposed to PARPi, SF3B1MUT cells show an impaired replication stress response (lack of pCHK1 (S317), pATR and pRPA32) due to reduced CINP protein expression. This leads to increased origin firing, unchecked fork progression and unresolved replication intermediates via the lack of MUS81-positive FANCD2 foci. This results in ATM activation and the induction of pCHK2 (T68), stalling SF3B1MUT cells at the G2/M checkpoint.\n",
      "13\n",
      "\n",
      "image 1/1 /Users/etowah/projects/PMC-figure-downloader/img/10412459_Fig9.jpg: 640x480 13 labels, 71.3ms\n",
      "Speed: 2.1ms preprocess, 71.3ms inference, 0.5ms postprocess per image at shape (1, 3, 640, 480)\n",
      "6\n",
      "\n",
      "image 1/1 /Users/etowah/projects/PMC-figure-downloader/img/10412459_Fig10.jpg: 640x480 7 labels, 90.9ms\n",
      "Speed: 2.3ms preprocess, 90.9ms inference, 0.7ms postprocess per image at shape (1, 3, 640, 480)\n",
      "5\n",
      "\n",
      "image 1/1 /Users/etowah/projects/PMC-figure-downloader/img/10412459_Fig11.jpg: 640x576 5 labels, 133.4ms\n",
      "Speed: 2.5ms preprocess, 133.4ms inference, 0.5ms postprocess per image at shape (1, 3, 640, 576)\n",
      "4\n",
      "\n",
      "image 1/1 /Users/etowah/projects/PMC-figure-downloader/img/10412459_Fig12.jpg: 640x544 4 labels, 77.3ms\n",
      "Speed: 2.2ms preprocess, 77.3ms inference, 0.6ms postprocess per image at shape (1, 3, 640, 544)\n",
      "6\n",
      "\n",
      "image 1/1 /Users/etowah/projects/PMC-figure-downloader/img/10412459_Fig13.jpg: 576x640 8 labels, 77.7ms\n",
      "Speed: 2.3ms preprocess, 77.7ms inference, 0.4ms postprocess per image at shape (1, 3, 576, 640)\n",
      "Subfigure count mismatch: 8 detected, 6 expected\n",
      "3\n",
      "\n",
      "image 1/1 /Users/etowah/projects/PMC-figure-downloader/img/10412459_Fig14.jpg: 544x640 3 labels, 73.3ms\n",
      "Speed: 2.0ms preprocess, 73.3ms inference, 0.4ms postprocess per image at shape (1, 3, 544, 640)\n",
      "2\n",
      "\n",
      "image 1/1 /Users/etowah/projects/PMC-figure-downloader/img/10412459_Fig15.jpg: 640x640 6 labels, 100.9ms\n",
      "Speed: 2.7ms preprocess, 100.9ms inference, 0.4ms postprocess per image at shape (1, 3, 640, 640)\n",
      "Subfigure count mismatch: 6 detected, 2 expected\n",
      "9\n",
      "\n",
      "image 1/1 /Users/etowah/projects/PMC-figure-downloader/img/10412459_Fig16.jpg: 640x640 9 labels, 84.8ms\n",
      "Speed: 2.4ms preprocess, 84.8ms inference, 0.4ms postprocess per image at shape (1, 3, 640, 640)\n",
      "6\n",
      "\n",
      "image 1/1 /Users/etowah/projects/PMC-figure-downloader/img/10412459_Fig17.jpg: 640x512 9 labels, 93.8ms\n",
      "Speed: 2.0ms preprocess, 93.8ms inference, 0.9ms postprocess per image at shape (1, 3, 640, 512)\n",
      "Subfigure count mismatch: 9 detected, 6 expected\n",
      "10\n",
      "\n",
      "image 1/1 /Users/etowah/projects/PMC-figure-downloader/img/10412459_Fig18.jpg: 640x544 10 labels, 79.3ms\n",
      "Speed: 2.1ms preprocess, 79.3ms inference, 0.7ms postprocess per image at shape (1, 3, 640, 544)\n",
      "1\n",
      "Blood is collected from a population of individuals, some of whom have cancer. Then, cfDNA is extracted from plasma and subjected to single-molecule sequencing using massively parallel sequencing approaches. Sequence alterations are used to obtain genome‐wide mutation profiles, and regional differences in cancer and non-cancer mutation frequencies are identified using machine learning to distinguish individuals with and without cancer.\n",
      "10\n",
      "\n",
      "image 1/1 /Users/etowah/projects/PMC-figure-downloader/img/10412448_Fig2.jpg: 576x640 10 labels, 94.1ms\n",
      "Speed: 2.2ms preprocess, 94.1ms inference, 0.7ms postprocess per image at shape (1, 3, 576, 640)\n",
      "2\n",
      "\n",
      "image 1/1 /Users/etowah/projects/PMC-figure-downloader/img/10412448_Fig3.jpg: 640x640 2 labels, 90.7ms\n",
      "Speed: 2.4ms preprocess, 90.7ms inference, 0.5ms postprocess per image at shape (1, 3, 640, 640)\n",
      "6\n",
      "\n",
      "image 1/1 /Users/etowah/projects/PMC-figure-downloader/img/10412448_Fig4.jpg: 640x640 6 labels, 91.3ms\n",
      "Speed: 2.4ms preprocess, 91.3ms inference, 0.4ms postprocess per image at shape (1, 3, 640, 640)\n",
      "6\n",
      "\n",
      "image 1/1 /Users/etowah/projects/PMC-figure-downloader/img/10412448_Fig5.jpg: 640x512 6 labels, 67.3ms\n",
      "Speed: 2.1ms preprocess, 67.3ms inference, 0.4ms postprocess per image at shape (1, 3, 640, 512)\n",
      "1\n",
      "Average somatic mutation frequencies computed in sliding 2.5 megabase (Mb) windows with a step size of 100 kb across chromosome 1 obtained from an analysis of 2,511 PCAWG samples across 25 common cancer types. Adeno, adenocarcinoma; TCC, transitional cell carcinoma; Osteo, osteosarcoma; CNS, central nervous system; GBM, glioblastoma multiforme; Medullo, medulloblastoma; SCC, squamous cell carcinoma; ChRCC, chromophobe renal cell carcinoma; RCC, renal cell carcinoma; HCC, hepatocellular carcinoma; BNHL, B cell non-Hodgkin lymphoma; CLL, chronic lymphoid leukemia; MPN, myeloproliferative neoplasm; Endo, endocrine.\n",
      "2\n",
      "\n",
      "image 1/1 /Users/etowah/projects/PMC-figure-downloader/img/10412448_Fig7.jpg: 256x640 2 labels, 46.7ms\n",
      "Speed: 1.1ms preprocess, 46.7ms inference, 0.4ms postprocess per image at shape (1, 3, 256, 640)\n",
      "1\n",
      "Single molecule C > A somatic mutation frequencies computed in sliding 2.5 megabase (Mb) windows with a step size of 100 kb across the autosomes obtained from an aggregated analysis of the 31 PCAWG lung cancer samples showed widespread differences in mutation frequencies depending on genomic location. Chr, Chromosome.\n",
      "1\n",
      "Single molecule C > A somatic mutation frequencies computed in a sliding 2.5 megabase (Mb) window with a step size of 100 kb across chromosome 4 from PCAWG lung cancer samples (n = 31) revealed similar mutation profiles among different lung cancers. Patient IDs (for example DO23744) are indicated for each sample.\n",
      "1\n",
      "The genome is divided into 1,144 non-overlapping 2.5 Mb bins (20 bins are depicted here) and the single molecule mutation frequency (SMMF) is computed in each bin as the number of sequence changes per million evaluable bases, defined as the number of positions in fragments in which each sequence change could be detected after quality and germline filtering. Samples in the training set are used to identify the bins that are most differentially mutated between cancer and non-cancer samples. In the training set, sequence data from all cancer samples and all non-cancer samples are combined, and the cancer and non-cancer single molecule mutation frequencies are computed in each bin. Next, the difference in single molecule mutation frequency is computed between cancer and non-cancer samples in each bin, and the 10% of bins most mutated in cancer samples relative to non-cancer samples, as well as the 10% of bins most mutated in non-cancer samples relative to cancer samples, are identified (indicated by triangles and circles respectively). In the testing set, the difference in single molecule mutation frequency is computed between these two sets of bins in a new sample not included in the training set, generating a regional difference in mutation frequency that can be used to classify the sample into being derived from a healthy individual or an individual with cancer. By taking the difference in single molecule mutation frequency between two sets of regions in the genome within an individual sample, this approach controls for the overall number of sequence changes in that sample that may result from technical variability in sequencing runs.\n",
      "1\n",
      "Cross-validated regional differences in single molecule mutation frequencies in cell-free DNA (cfDNA) in individuals with lung cancer (n = 89) and individuals without cancer (n = 74), median-centered within each mutation type. Regional C > A mutation frequencies were preferentially altered between lung cancer and non-cancer samples, but not when randomly permuting class labels (p = 0.36, Wilcoxon rank sum test, two-sided). Adjusted p-values (padj) were generated using the two-sided Wilcoxon rank sum test and were corrected for multiple comparisons using the Benjamini-Hochberg method. The horizontal dashed line indicates a p-value of 0.05. All boxplots represent the interquartile range with whiskers drawn to the highest value within the upper and lower fences (upper fence = 0.75 quantile + 1.5 × interquartile range; lower fence = 0.25 quantile – 1.5 × interquartile range). The solid middle line in the boxplot corresponds to the median value.\n",
      "2\n",
      "\n",
      "image 1/1 /Users/etowah/projects/PMC-figure-downloader/img/10412448_Fig12.jpg: 448x640 2 labels, 63.1ms\n",
      "Speed: 1.7ms preprocess, 63.1ms inference, 0.5ms postprocess per image at shape (1, 3, 448, 640)\n",
      "1\n",
      "Patients with lung cancer in the high-risk LUCAS cohort (n = 89) were stratified in two groups based on the median GEMINI / DELFI score among lung cancer patients of 0.84. Patients with a GEMINI / DELFI score ≥0.84 (yellow) had a significantly worse overall survival compared to patients with a GEMINI / DELFI score < 0.84 (blue) (p = 0.004, Log-rank test).\n",
      "3\n",
      "\n",
      "image 1/1 /Users/etowah/projects/PMC-figure-downloader/img/10412448_Fig14.jpg: 576x640 3 labels, 88.9ms\n",
      "Speed: 2.6ms preprocess, 88.9ms inference, 0.5ms postprocess per image at shape (1, 3, 576, 640)\n",
      "1\n",
      "Individuals with a smoking history as well as availability of targeted deep sequencing11 and low coverage WGS data13 were analyzed before and during treatment with tyrosine kinase inhibitors (arrows indicate initiation of treatment). GEMINI scores were associated with the median mutant allele fraction (MAF) of detectable mutations at each timepoint with values of zero used in CGPLLU269 samples where no mutations were detected (Spearman’s correlation coefficient = 0.53, p = 0.02, two-sided). The range of median MAFs for all GEMINI positive patients was 0.17% to 50.91% at 80% specificity.\n",
      "5\n",
      "\n",
      "image 1/1 /Users/etowah/projects/PMC-figure-downloader/img/10335938_Fig1.jpg: 640x384 5 labels, 62.5ms\n",
      "Speed: 1.8ms preprocess, 62.5ms inference, 0.4ms postprocess per image at shape (1, 3, 640, 384)\n",
      "3\n",
      "\n",
      "image 1/1 /Users/etowah/projects/PMC-figure-downloader/img/10335938_Fig2.jpg: 640x416 2 labels, 67.5ms\n",
      "Speed: 1.7ms preprocess, 67.5ms inference, 0.7ms postprocess per image at shape (1, 3, 640, 416)\n",
      "Subfigure count mismatch: 2 detected, 3 expected\n",
      "4\n",
      "\n",
      "image 1/1 /Users/etowah/projects/PMC-figure-downloader/img/10335938_Fig3.jpg: 640x320 4 labels, 54.7ms\n",
      "Speed: 1.4ms preprocess, 54.7ms inference, 0.4ms postprocess per image at shape (1, 3, 640, 320)\n",
      "2\n",
      "\n",
      "image 1/1 /Users/etowah/projects/PMC-figure-downloader/img/10335938_Fig4.jpg: 640x320 3 labels, 86.0ms\n",
      "Speed: 1.7ms preprocess, 86.0ms inference, 0.4ms postprocess per image at shape (1, 3, 640, 320)\n",
      "Subfigure count mismatch: 3 detected, 2 expected\n",
      "8\n",
      "\n",
      "image 1/1 /Users/etowah/projects/PMC-figure-downloader/img/10335938_Fig5.jpg: 640x416 10 labels, 74.2ms\n",
      "Speed: 1.6ms preprocess, 74.2ms inference, 0.4ms postprocess per image at shape (1, 3, 640, 416)\n",
      "Subfigure count mismatch: 9 detected, 8 expected\n",
      "4\n",
      "\n",
      "image 1/1 /Users/etowah/projects/PMC-figure-downloader/img/10335938_Fig6.jpg: 640x480 5 labels, 79.6ms\n",
      "Speed: 2.1ms preprocess, 79.6ms inference, 0.4ms postprocess per image at shape (1, 3, 640, 480)\n",
      "1\n",
      "Each square represents the average coverage value for a given time-point. The increase in H3K27ac coverage over the HoxD cluster and T-DOM (with a strong preference for sub-TAD1) contrasts with C-DOM (bottom). Graphs showing quantifications over time of the colored vertical segments illustrated in Fig. 1b (red for C-DOM and green for T-DOM). Each line represents a specific H3K27 acetylated element within the HoxD regulatory landscape. The time sequence in the activation of these elements in T-DOM does not correlate with their distance from the genes cluster (I and II).\n",
      "2\n",
      "\n",
      "image 1/1 /Users/etowah/projects/PMC-figure-downloader/img/10335938_Fig8.jpg: 544x640 2 labels, 70.1ms\n",
      "Speed: 2.0ms preprocess, 70.1ms inference, 0.7ms postprocess per image at shape (1, 3, 544, 640)\n",
      "2\n",
      "\n",
      "image 1/1 /Users/etowah/projects/PMC-figure-downloader/img/10335938_Fig9.jpg: 640x576 3 labels, 102.9ms\n",
      "Speed: 2.3ms preprocess, 102.9ms inference, 0.6ms postprocess per image at shape (1, 3, 640, 576)\n",
      "Subfigure count mismatch: 3 detected, 2 expected\n",
      "5\n",
      "\n",
      "image 1/1 /Users/etowah/projects/PMC-figure-downloader/img/10335938_Fig10.jpg: 608x640 5 labels, 94.0ms\n",
      "Speed: 2.4ms preprocess, 94.0ms inference, 0.6ms postprocess per image at shape (1, 3, 608, 640)\n",
      "3\n",
      "\n",
      "image 1/1 /Users/etowah/projects/PMC-figure-downloader/img/10335938_Fig11.jpg: 640x512 3 labels, 67.4ms\n",
      "Speed: 2.6ms preprocess, 67.4ms inference, 1.5ms postprocess per image at shape (1, 3, 640, 512)\n",
      "1\n",
      "The contacts between the various viewpoints (Hoxd genes shown on top) and the Evx2-Hoxd12 region showed a progressive decrease upon activation of the cluster, particularly visible for both Hoxd8 and Hoxd9. Contacts values were normalized to 48 h for each viewpoint. Asterisks represent significant changes against as determined by the Kruskal-Wallis followed by Dunn’s multiple comparisons test (* is p-value < 0.05, ** < 0.01, *** < 0.001, **** < 0.0001). Box plots with median value and 25–75% percentiles, whiskers represent minimum and maximum (sum of 3 independent replicates).\n",
      "4\n",
      "\n",
      "image 1/1 /Users/etowah/projects/PMC-figure-downloader/img/10335938_Fig13.jpg: 640x512 4 labels, 96.0ms\n",
      "Speed: 2.1ms preprocess, 96.0ms inference, 0.4ms postprocess per image at shape (1, 3, 640, 512)\n",
      "5\n",
      "\n",
      "image 1/1 /Users/etowah/projects/PMC-figure-downloader/img/10335938_Fig14.jpg: 640x512 6 labels, 65.0ms\n",
      "Speed: 2.1ms preprocess, 65.0ms inference, 0.5ms postprocess per image at shape (1, 3, 640, 512)\n",
      "2\n",
      "\n",
      "image 1/1 /Users/etowah/projects/PMC-figure-downloader/img/10335938_Fig15.jpg: 640x384 2 labels, 55.4ms\n",
      "Speed: 1.4ms preprocess, 55.4ms inference, 0.5ms postprocess per image at shape (1, 3, 640, 384)\n",
      "4\n",
      "\n",
      "image 1/1 /Users/etowah/projects/PMC-figure-downloader/img/10335938_Fig16.jpg: 640x416 5 labels, 57.8ms\n",
      "Speed: 1.8ms preprocess, 57.8ms inference, 0.4ms postprocess per image at shape (1, 3, 640, 416)\n",
      "Subfigure count mismatch: 5 detected, 4 expected\n",
      "1\n",
      "MGB Biobank, Mass General Brigham Biobank; DD, disc diameter; LDSC, linkage disequilibrium score regression.\n",
      "2\n",
      "\n",
      "image 1/1 /Users/etowah/projects/PMC-figure-downloader/img/10335935_Fig2.jpg: 512x640 2 labels, 69.6ms\n",
      "Speed: 1.9ms preprocess, 69.6ms inference, 1.0ms postprocess per image at shape (1, 3, 512, 640)\n",
      "4\n",
      "\n",
      "image 1/1 /Users/etowah/projects/PMC-figure-downloader/img/10335935_Fig3.jpg: 608x640 4 labels, 128.3ms\n",
      "Speed: 2.5ms preprocess, 128.3ms inference, 0.9ms postprocess per image at shape (1, 3, 608, 640)\n",
      "3\n",
      "\n",
      "image 1/1 /Users/etowah/projects/PMC-figure-downloader/img/10335935_Fig4.jpg: 288x640 3 labels, 57.4ms\n",
      "Speed: 1.2ms preprocess, 57.4ms inference, 1.0ms postprocess per image at shape (1, 3, 288, 640)\n",
      "1\n",
      "Plots show 14 traits that were associated with POAG from MR (FDR P < 0.05, direction: complex traits → POAG). Different outcome traits are shown in different colors. Different MR methods are displayed in different line types. The dots show the effect sizes of MR estimations, and the error bars show the 95% confidence interval of the estimations. The number of SNPs, effect sizes and P values are presented in Supplementary Table 14. All tests were two-sided. BMI, body mass index; 95% CI, 95% confidence interval.\n",
      "4\n",
      "\n",
      "image 1/1 /Users/etowah/projects/PMC-figure-downloader/img/10335935_Fig6.jpg: 544x640 4 labels, 79.5ms\n",
      "Speed: 1.9ms preprocess, 79.5ms inference, 0.4ms postprocess per image at shape (1, 3, 544, 640)\n",
      "1\n",
      "‘1’ corresponds to novel SNPs; ‘0’ corresponds to known SNPs.\n",
      "2\n",
      "\n",
      "image 1/1 /Users/etowah/projects/PMC-figure-downloader/img/10335935_Fig8.jpg: 640x480 2 labels, 84.6ms\n",
      "Speed: 2.0ms preprocess, 84.6ms inference, 0.4ms postprocess per image at shape (1, 3, 640, 480)\n",
      "2\n",
      "\n",
      "image 1/1 /Users/etowah/projects/PMC-figure-downloader/img/10335935_Fig9.jpg: 384x640 2 labels, 76.7ms\n",
      "Speed: 1.6ms preprocess, 76.7ms inference, 0.4ms postprocess per image at shape (1, 3, 384, 640)\n",
      "2\n",
      "\n",
      "image 1/1 /Users/etowah/projects/PMC-figure-downloader/img/10335935_Fig10.jpg: 320x640 2 labels, 56.8ms\n",
      "Speed: 1.3ms preprocess, 56.8ms inference, 0.4ms postprocess per image at shape (1, 3, 320, 640)\n",
      "1\n",
      "This scatter plot shows the effect sizes of 263 MTAG POAG loci on VCDR (x-axis) and IOP (y-axis). The assigned SNPs (IOP and VCDR in different point shapes) were based on a hierarchical clustering approach. The different point colors for trait combination show the results from multi-trait colocalization. This figure shows a consistent classification of POAG loci into VCDR- and IOP-specific SNPs based on the hierarchical clustering method and the multi-trait colocalization method.\n",
      "1\n",
      "The x-axis shows the genetic correlations and their 95% confidence intervals. The y-axis shows the genetically correlated traits. The dots show the effect size of Mendelian randomization estimations, and the error bars show the 95% confidence interval of the estimations. All tests were two-sided.\n",
      "2\n",
      "\n",
      "image 1/1 /Users/etowah/projects/PMC-figure-downloader/img/10335935_Fig13.jpg: 640x544 2 labels, 141.6ms\n",
      "Speed: 2.1ms preprocess, 141.6ms inference, 0.5ms postprocess per image at shape (1, 3, 640, 544)\n",
      "1\n",
      "Plots show the reverse-directional Mendelian randomization for traits that were associated with POAG (celiac disease was not shown because of the limited number of overlapping SNPs; direction: POAG -> complex traits). Different exposure traits are shown in different colors. Different MR methods are displayed in different line types. In this reverse-directional MR analysis, only the association between VCDR and optic disc area passed multiple testing.\n",
      "1\n",
      "In the current study, we first performed a TAAD discovery GWAS in the MVP, with replication from six external datasets. Secondary analyses included a PheWAS of lead TAAD risk variants, MR analyses with known epidemiologic risk factors for disease, a series of analyses to identify causal genes, variants and cell types for TAAD leveraging colocalization techniques and scRNA-seq or snRNA-seq data, and the generation and testing of a TAAD PRS. This figure was created with the assistance of BioRender. Abbreviations: LDL-C, LDL cholesterol; HDL-C, HDL cholesterol; TG, triglycerides; SBP, systolic blood pressure; DBP, diastolic blood pressure.\n",
      "2\n",
      "\n",
      "image 1/1 /Users/etowah/projects/PMC-figure-downloader/img/10335930_Fig2.jpg: 640x320 2 labels, 128.8ms\n",
      "Speed: 1.6ms preprocess, 128.8ms inference, 0.9ms postprocess per image at shape (1, 3, 640, 320)\n",
      "1\n",
      "Logistic regression (inverse-variance-weighted; IVW) association results for multiple epidemiologic risk factor exposures with the TAAD outcome in two-sample MR analyses. The lifetime smoking TAAD OR reflects a per-genetic increase in smoking ~20 cigarettes a day for 15 years and stopping 17 years ago. The OR for height reflects a 1-s.d. genetic increase in standing height (~7.6 cm). The SBP, DBP, MAP and PP ORs correspond to the change in TAAD risk per 10-mmHg increase in the blood pressure trait. The lipid ORs reflect the change in TAAD risk per s.d. genetic increase in lipid fraction. Two-sided P values are displayed, and we set a two-sided P < 0.0055 (0.05 ÷ 9 traits) for statistical significance. Error bars represent 95% CIs of the displayed ORs.\n",
      "6\n",
      "\n",
      "image 1/1 /Users/etowah/projects/PMC-figure-downloader/img/10335930_Fig4.jpg: 640x512 6 labels, 78.6ms\n",
      "Speed: 2.1ms preprocess, 78.6ms inference, 0.6ms postprocess per image at shape (1, 3, 640, 512)\n",
      "2\n",
      "\n",
      "image 1/1 /Users/etowah/projects/PMC-figure-downloader/img/10335930_Fig5.jpg: 192x640 2 labels, 37.8ms\n",
      "Speed: 0.8ms preprocess, 37.8ms inference, 0.4ms postprocess per image at shape (1, 3, 192, 640)\n",
      "4\n",
      "\n",
      "image 1/1 /Users/etowah/projects/PMC-figure-downloader/img/10335927_Fig1.jpg: 640x640 4 labels, 87.8ms\n",
      "Speed: 2.4ms preprocess, 87.8ms inference, 0.5ms postprocess per image at shape (1, 3, 640, 640)\n",
      "3\n",
      "\n",
      "image 1/1 /Users/etowah/projects/PMC-figure-downloader/img/10335927_Fig2.jpg: 640x576 3 labels, 74.5ms\n",
      "Speed: 2.3ms preprocess, 74.5ms inference, 0.5ms postprocess per image at shape (1, 3, 640, 576)\n",
      "2\n",
      "\n",
      "image 1/1 /Users/etowah/projects/PMC-figure-downloader/img/10335927_Fig3.jpg: 640x640 2 labels, 91.8ms\n",
      "Speed: 2.4ms preprocess, 91.8ms inference, 0.9ms postprocess per image at shape (1, 3, 640, 640)\n",
      "1\n",
      "Chromosome 1 imputation accuracy (Aggregate r2, y-axis) for 10,000 British samples of three simulated SNP arrays (Omni 2.5 Array, yellow; GSA array, orange; Axiom Array, red), and sequencing coverages (0.1–4.0x, shades of blue) using the UKB reference panel. The lifted-over (non-simulated) Axiom array data from the UK Biobank is shown in black. We imputed low-coverage data using GLIMPSE2 and SNP array data using BEAGLE v5.4.\n",
      "2\n",
      "\n",
      "image 1/1 /Users/etowah/projects/PMC-figure-downloader/img/10335927_Fig5.jpg: 320x640 2 labels, 59.1ms\n",
      "Speed: 1.3ms preprocess, 59.1ms inference, 0.4ms postprocess per image at shape (1, 3, 320, 640)\n",
      "2\n",
      "\n",
      "image 1/1 /Users/etowah/projects/PMC-figure-downloader/img/10335927_Fig6.jpg: 320x640 2 labels, 84.0ms\n",
      "Speed: 1.4ms preprocess, 84.0ms inference, 0.6ms postprocess per image at shape (1, 3, 320, 640)\n",
      "1\n",
      "Schematic representation of the genome-wide screens for plasma and urine metabolite levels and their follow-up analyses. Analyses based on data from plasma are presented in red, analyses based on data from urine in blue, comparative analyses of results based on plasma and urine are shown in a red–blue gradient, and matrix-independent analyses are in white. Icon credit, Servier Medical Art by Servier (licensed under a Creative Commons Attribution 3.0 Unported License). HMGD, Human Gene Mutation Database; HPA, Human Protein Atlas; HRC, Haplotype Reference Consortium; v8, version 8.\n",
      "1\n",
      "The light red band shows the −log10 (P values) for genetic associations with metabolite levels in plasma by chromosomal position, and the light blue band shows the associations with metabolite levels in urine. Results from all 1,296 plasma and 1,401 urine GWAS traits are based on linear regressions and are overlaid in the respective bands, with P values truncated at 1 × 10−60. The horizontal lines (blue and red) indicate genome-wide significance (Pplasma = 3.9 × 10−11 and Purine = 3.6 × 10−11). Supplementary Table 3 contains details about index SNP associations (mQTLs). Gene labels for significant loci were assigned based on mQTL annotations, colocalization analysis with gene expression and protein levels, and literature research (Methods). Black gene labels indicate genetic regions identified in both plasma and urine with intermatrix colocalization (PP H4 > 0.8), gray labels indicate genetic regions identified in both plasma and urine without intermatrix colocalization, and red or blue labels indicate genetic regions exclusively identified in plasma or urine, respectively. The number of plasma and urine mQTLs annotated to a gene is given in parentheses (plasma, urine). The pie chart reflects the proportions of the 282 unique genes that were annotated as enzymes and transporters. Official gene symbols for PYCRL and ERO1L are PYCR3 and ERO1A, respectively.\n",
      "3\n",
      "\n",
      "image 1/1 /Users/etowah/projects/PMC-figure-downloader/img/10260405_Fig3.jpg: 416x640 3 labels, 72.5ms\n",
      "Speed: 1.8ms preprocess, 72.5ms inference, 1.4ms postprocess per image at shape (1, 3, 416, 640)\n",
      "3\n",
      "\n",
      "image 1/1 /Users/etowah/projects/PMC-figure-downloader/img/10260405_Fig4.jpg: 480x640 3 labels, 75.4ms\n",
      "Speed: 1.8ms preprocess, 75.4ms inference, 0.4ms postprocess per image at shape (1, 3, 480, 640)\n",
      "7\n",
      "\n",
      "image 1/1 /Users/etowah/projects/PMC-figure-downloader/img/10260405_Fig5.jpg: 640x512 7 labels, 74.6ms\n",
      "Speed: 2.1ms preprocess, 74.6ms inference, 1.0ms postprocess per image at shape (1, 3, 640, 512)\n",
      "4\n",
      "\n",
      "image 1/1 /Users/etowah/projects/PMC-figure-downloader/img/10260405_Fig6.jpg: 640x480 4 labels, 70.7ms\n",
      "Speed: 2.1ms preprocess, 70.7ms inference, 0.5ms postprocess per image at shape (1, 3, 640, 480)\n",
      "4\n",
      "\n",
      "image 1/1 /Users/etowah/projects/PMC-figure-downloader/img/10260405_Fig7.jpg: 640x608 5 labels, 87.4ms\n",
      "Speed: 2.3ms preprocess, 87.4ms inference, 1.2ms postprocess per image at shape (1, 3, 640, 608)\n",
      "Subfigure count mismatch: 5 detected, 4 expected\n",
      "4\n",
      "\n",
      "image 1/1 /Users/etowah/projects/PMC-figure-downloader/img/10260405_Fig8.jpg: 608x640 4 labels, 79.5ms\n",
      "Speed: 2.1ms preprocess, 79.5ms inference, 0.4ms postprocess per image at shape (1, 3, 608, 640)\n",
      "1\n",
      "Each point represents the index SNP of one of 459 (EA) and 430 (AA) associations that could be matched between the Metabolon platforms of the GCKD and ARIC studies (see Supplementary Table 6). Data are presented as effect size estimate +/- 1.96x standard errors in each study and the dot size is proportional to the two-sided -log10(P-value) in GCKD (NGCKD = 4960, NARIC EA = 3603, NARIC AA = 818).\n",
      "1\n",
      "The positive correlation between the estimated heritabilities for a given metabolite’s plasma and urine levels is consistent with the metabolites’ filtration from plasma to urine, without substantial additional genetic influences on their tubular handling. The blue line is the linear regression line and the gray shaded area represents the 95%-confidence interval. Differences in estimated heritability for plasma and urine (instances with >25% are labeled with the associated metabolite and most likely gene; error bars represent h2 variance) can contain interesting biological information: for example, three metabolites with larger estimated heritabilities in urine than in plasma are N-acetylated amino acids, all of which have an mQTL at NAT8. NAT8 is highly and selectively expressed in the kidney, where the encoded enzyme N-acetylates molecules to make them water soluble for subsequent excretion.\n",
      "1\n",
      "Power analyses are based on a sample size of 5,000, the genome-wide statistical significance thresholds used in our study, and are conducted across a range of minor allele frequencies. For each matrix-super-pathway subgroup, the median observed effect size across mQTLs as well as the median standard deviation of the metabolites with an mQTL within the group were used.\n",
      "1\n",
      "Comparison of effect sizes and explained variance for colocalization signals for mQTLs detected for the same metabolite in both plasma and urine (N = 204; only the 99 mQTLs for which the explained variance in metabolite levels in at least one of both matrices is >3% are shown). The two inner bands represent the effect size of the mQTL in plasma (framed in red) and urine (framed in blue). Shades of orange indicate positive effect sizes, shades of aquamarine negative ones. The two outer bands represent the variance in metabolite levels in plasma and urine explained by the index SNP of the corresponding mQTL, where a darker shade of green corresponds to a greater explained variance.\n",
      "1\n",
      "The mQTLs are represented by the implicated genes on the rows and the colocalized clinical markers are on the columns. Liver function markers include alanine aminotransferase (ALT), aspartate aminotransferase (AST), gamma glutamyltransferase (GGT), albumin and bilirubin. Kidney function markers include eGFRcrea, eGFRcys and urea. The size of pie represents the total number of colocalizations grouped into four categories. The slices in each pie colored in red and blue represent the proportion of colocalizations of plasma and urine mQTLs with the respective clinical markers.\n",
      "1\n",
      "The proportion of a metabolite’s variance explained by eGFR is represented on the x-axis. All metabolites quantified from plasma and urine are shown along the y-axis, ordered by the maximum variance explained across plasma (red color) and urine (blue color). The metabolite with the largest amount of variance explained by eGFR was plasma creatinine.\n",
      "1\n",
      "(a) Similarities and differences between terms and pathways enriched for genes identified by all plasma vs. all urine mQTLs; (b) mQTLs exclusively identified in plasma and urine; (c) between tissues enriched for genes identified by all plasma vs. all urine mQTLs, and (d) between cell types enriched for genes identified by all plasma vs. all urine mQTLs. Terms significantly (adjusted P-value < 0.05) enriched for genes identified by mQTLs from only one matrix are colored in red and blue respectively and terms significantly enriched for genes from both matrices are colored in purple. OR: odds ratio.\n",
      "1\n",
      "The upper part of the figure shows the same RNA-seq, ATAC-seq, chromatin state and histone ChIP-seq tracks as Fig. 6. The RNA-seq and ATAC-seq tracks show the overlayed signal from tissue of three different donors. The index SNP rs16961281, that is associated with urine glycocholate, is located at the vertical dashed line. The bottom part shows publicly available single nucleus (sn)ATAC-seq data for different kidney cell types, which was derived from primary human kidney samples38. The position of rs16961281 is nearly exclusively accessible in cells of all proximal tubule segments (PT-S1, PT-S2, PT-S3). PTs are the predominant cell type in kidney cortex, underscoring the consistency of the snATAC-seq data and the bulk ATAC-seq data. Other cell types shown include: Endothelial cells (Endo), podocytes (Podo), loop of Henle cells (LOH), distal convoluted tubule cells (DCT), collecting duct principal cells (PC), collecting duct intercalated cells (IC), stroma cells (Stroma), immune cells (Immune), lymph cells (Lymph).\n",
      "1\n",
      "The upper part of the figure shows the same RNA-seq, ATAC-seq, chromatin state and histone ChIP-seq tracks as Fig. 7. The index SNP rs6124828, that is associated with malate, fumarate, and methylsuccinoylcarnitine in plasma as well as with methylsuccinoylcarnitine in urine is located at the second vertical dashed line from the left. The bottom part shows single nucleus (sn)ATAC-seq data for different kidney cell types, which was derived from primary human kidney samples39. The position of rs6124828 is nearly exclusively accessible in proximal tubule cells (PT). PTs are the predominant cell type in the kidney cortex, underscoring the consistency of the snATAC-seq data and the bulk ATAC-seq data. Other cell types shown include: Endothelial cells (Endo), podocytes (Podo), loop of Henle cells (LOH), distal convoluted tubule cells (DCT), collecting duct principal cells (CDPC), collecting duct intercalated cells (CDIC), immune cells (Immune).\n",
      "2\n",
      "\n",
      "image 1/1 /Users/etowah/projects/PMC-figure-downloader/img/10260404_Fig1.jpg: 320x640 3 labels, 77.6ms\n",
      "Speed: 1.3ms preprocess, 77.6ms inference, 0.4ms postprocess per image at shape (1, 3, 320, 640)\n",
      "11\n",
      "\n",
      "image 1/1 /Users/etowah/projects/PMC-figure-downloader/img/10260404_Fig2.jpg: 288x640 6 labels, 68.4ms\n",
      "Speed: 1.5ms preprocess, 68.4ms inference, 0.4ms postprocess per image at shape (1, 3, 288, 640)\n",
      "Subfigure count mismatch: 6 detected, 11 expected\n",
      "5\n",
      "\n",
      "image 1/1 /Users/etowah/projects/PMC-figure-downloader/img/10260404_Fig3.jpg: 320x640 5 labels, 52.8ms\n",
      "Speed: 1.3ms preprocess, 52.8ms inference, 0.4ms postprocess per image at shape (1, 3, 320, 640)\n",
      "6\n",
      "\n",
      "image 1/1 /Users/etowah/projects/PMC-figure-downloader/img/10260404_Fig4.jpg: 320x640 6 labels, 60.2ms\n",
      "Speed: 1.2ms preprocess, 60.2ms inference, 0.4ms postprocess per image at shape (1, 3, 320, 640)\n",
      "4\n",
      "\n",
      "image 1/1 /Users/etowah/projects/PMC-figure-downloader/img/10260404_Fig5.jpg: 640x640 4 labels, 82.4ms\n",
      "Speed: 2.5ms preprocess, 82.4ms inference, 0.5ms postprocess per image at shape (1, 3, 640, 640)\n",
      "8\n",
      "\n",
      "image 1/1 /Users/etowah/projects/PMC-figure-downloader/img/10260404_Fig6.jpg: 480x640 9 labels, 65.6ms\n",
      "Speed: 2.0ms preprocess, 65.6ms inference, 0.5ms postprocess per image at shape (1, 3, 480, 640)\n",
      "2\n",
      "\n",
      "image 1/1 /Users/etowah/projects/PMC-figure-downloader/img/10260404_Fig7.jpg: 480x640 2 labels, 72.9ms\n",
      "Speed: 1.8ms preprocess, 72.9ms inference, 0.4ms postprocess per image at shape (1, 3, 480, 640)\n",
      "3\n",
      "\n",
      "image 1/1 /Users/etowah/projects/PMC-figure-downloader/img/10260404_Fig8.jpg: 384x640 3 labels, 56.0ms\n",
      "Speed: 1.4ms preprocess, 56.0ms inference, 0.3ms postprocess per image at shape (1, 3, 384, 640)\n",
      "6\n",
      "\n",
      "image 1/1 /Users/etowah/projects/PMC-figure-downloader/img/10260404_Fig9.jpg: 544x640 3 labels, 76.2ms\n",
      "Speed: 2.0ms preprocess, 76.2ms inference, 1.0ms postprocess per image at shape (1, 3, 544, 640)\n",
      "Subfigure count mismatch: 3 detected, 6 expected\n",
      "11\n",
      "\n",
      "image 1/1 /Users/etowah/projects/PMC-figure-downloader/img/10260404_Fig10.jpg: 576x640 11 labels, 91.9ms\n",
      "Speed: 2.1ms preprocess, 91.9ms inference, 0.4ms postprocess per image at shape (1, 3, 576, 640)\n",
      "1\n",
      "Prior probability inferred by the pairwise hierarchical model (Online Methods) fitted between our eQTLs and those for the 48 GTEx tissues as well as immune cells (monocytes and iPSC-derived macrophages) under different stimulation conditions.\n",
      "4\n",
      "\n",
      "image 1/1 /Users/etowah/projects/PMC-figure-downloader/img/10260404_Fig12.jpg: 320x640 4 labels, 76.0ms\n",
      "Speed: 1.5ms preprocess, 76.0ms inference, 0.7ms postprocess per image at shape (1, 3, 320, 640)\n",
      "Subfigure count mismatch: 3 detected, 4 expected\n",
      "6\n",
      "\n",
      "image 1/1 /Users/etowah/projects/PMC-figure-downloader/img/10260404_Fig13.jpg: 640x608 6 labels, 94.0ms\n",
      "Speed: 2.4ms preprocess, 94.0ms inference, 0.5ms postprocess per image at shape (1, 3, 640, 608)\n",
      "7\n",
      "\n",
      "image 1/1 /Users/etowah/projects/PMC-figure-downloader/img/10260403_Fig1.jpg: 640x544 7 labels, 64.0ms\n",
      "Speed: 2.2ms preprocess, 64.0ms inference, 0.4ms postprocess per image at shape (1, 3, 640, 544)\n",
      "2\n",
      "\n",
      "image 1/1 /Users/etowah/projects/PMC-figure-downloader/img/10260403_Fig2.jpg: 640x544 4 labels, 63.3ms\n",
      "Speed: 2.2ms preprocess, 63.3ms inference, 0.4ms postprocess per image at shape (1, 3, 640, 544)\n",
      "Subfigure count mismatch: 4 detected, 2 expected\n",
      "1\n",
      "Distribution of EDU and VNR scores for KDM5B PTV carriers in the UKB. ClinVar probably pathogenic variants for IDD (MIM 618109) were annotated. Samples with inpatient ICD-10 records of psychiatric (SCZ, BD, depression, substance use disorder or anxiety and stress disorders), neurodegenerative and neurodevelopmental disorders were annotated. Phenotypes were residualized by sex, age, age2, sex by age, sex by age2, top 20 principal components and recruitment centers and were rank-based inverse-normal transformed. The blue (for EDU) and red (for VNR) lines represent fitted locally estimated scatterplot smoothing (LOESS) regression on standardized, residualized phenotypes. The gray bands represent 95% CIs for the fitted LOESS regression.\n",
      "3\n",
      "\n",
      "image 1/1 /Users/etowah/projects/PMC-figure-downloader/img/10260403_Fig4.jpg: 640x640 4 labels, 74.0ms\n",
      "Speed: 2.4ms preprocess, 74.0ms inference, 1.0ms postprocess per image at shape (1, 3, 640, 640)\n",
      "2\n",
      "\n",
      "image 1/1 /Users/etowah/projects/PMC-figure-downloader/img/10260403_Fig5.jpg: 640x544 2 labels, 79.2ms\n",
      "Speed: 2.4ms preprocess, 79.2ms inference, 1.2ms postprocess per image at shape (1, 3, 640, 544)\n",
      "1\n",
      "FEV1 Z-score is Inverted GLI 2012 z-score for FEV1. Phenotypes were grouped and color-coded from left to right in the following categories: biomarker; composite phenotypes; family history; ICD-10 cause of death, ICD-10 congenital malformations; deformations and chromosomal abnormalities; ICD-10 diseases of the circulatory system; ICD-10 diseases of the digestive system; ICD-10 diseases of the eye and adnexa; ICD-10 diseases of the genitourinary system; ICD-10 diseases of the musculoskeletal system and connective tissue; ICD-10 diseases of the nervous system; ICD-10 diseases of the respiratory system; ICD-10 diseases of the skin and subcutaneous tissue; ICD-10 endocrine, nutritional and metabolic diseases; ICD-10 mental, behavioral and neurodevelopmental disorders; ICD-10 neoplasms; ICD-10 pregnancy, childbirth and the puerperium; ICD-10 symptoms, signs and abnormal clinical and laboratory findings, not elsewhere classified; operation code; self-reported illness: cancer; self-reported illness: non−cancer; self-reported medication. Bonferroni corrected p-value (two-sided t-test) significance threshold was 0.05/3150 = 1.59×10−5.\n",
      "2\n",
      "\n",
      "image 1/1 /Users/etowah/projects/PMC-figure-downloader/img/10260403_Fig7.jpg: 640x576 2 labels, 87.8ms\n",
      "Speed: 2.2ms preprocess, 87.8ms inference, 1.0ms postprocess per image at shape (1, 3, 640, 576)\n",
      "1\n",
      "ClinVar pathogenic/likely pathogenic variants for epileptic encephalopathy (MIM 617106) and/or type 2 episodic ataxia (MIM 108500) was annotated. Samples with inpatient ICD-10 (International Classification of Diseases version-10) records of psychiatric (schizophrenia, bipolar disorder, depression, substance use disorder and/or anxiety and stress disorders), neurodegenerative and neurodevelopmental disorders were annotated. Phenotypes were residualized by sex, age, age2, sex by age interaction, sex by age2 interaction, top 20 PCs, and recruitment center and inverse rank-based normal transformed. The blue line (for EDU) and red line (for VNR) represent fitted loess regression on standardized, residualized phenotypes. The gray bands represent 95% confidence intervals for the fitted loess regression.\n",
      "1\n",
      "An intermediate effect on cranial length (additive genotype effect two-sided P = 0.0268) and height (additive genotype effect two-sided P = 0.0056) is detected in Kdm5b+/− mice, but not in cranial width (additive genotype effect two-sided P = 0.3090). For the boxplot, the center line represents the median, the box limits represent the IQR, and the whiskers indicate the minimum and maximum values. A fully penetrant transitional vertebrae phenotype seen in Kdm5b−/− mice (N = 21) is observed at a lower frequency in Kdm5b+/− mice (N = 40, Fisher’s exact test vs Kdm5b+/+ two-sided P = 0.0189).\n",
      "1\n",
      "Log2-fold change of differentially expressed genes plotted for Kdm5b+/− (y-axis) and Kdm5b−/− (x-axis) mice across embryonic and adult brain tissues as indicated. There is a strong correlation between direction of change in expression in both mutant genotypes (robust linear regression line and slope shown in red).\n",
      "1\n",
      "Regional plot of educational attainment GWAS association test results were generated around top independent SNP rs10798888. Additional associations from GWAS catalog were annotated with the associated phenotypes in the regional plot. EDU and VNR score for ADGRB2 PTV carriers in UKB were plotted (both phenotypes were residualized by sex, age, age2, sex by age, sex by age2, top 20 PCs and recruitment centers and were inverse rank-based normal transformed). Samples with inpatient ICD-10 (International Classification of Diseases version-10) records of psychiatric, neurodegenerative, and neurodevelopmental disorders were annotated. The blue line (for EDU) and red line (for VNR) represent fitted loess regression on standardized, residualized phenotypes. The gray bands represent 95% confidence intervals for the fitted loess regression.\n",
      "1\n",
      "Regional plot of cognitive function GWAS association test results were generated for top independent SNP rs5751191 and the extended LD region. Additional associations from GWAS catalog were annotated with the associated phenotypes in the regional plot. Number of PTV carriers and gene-based PTV burden association p-value were extracted for genes in the region.\n",
      "1\n",
      "Unrelated UKB EUR samples were included for this analysis (N = 318,844 for EDU, 319,536 for RT, and 128,812 for VNR). Genes were stratified by elevated expression in brain tissue (2,587 genes), elevated expression in other tissues but also expressed in brain (5,298 genes) and no tissue specific expression (8,342 genes). Number of genes included in the burden is annotated for each set. Data are presented in effect size estimates (β) with 95% confidence intervals.\n",
      "1\n",
      "The impact of cognitive function polygenic score (PRS) and carrier status of PTV and/or damaging missense variants (MPC > 2) in LoF intolerant genes (pLI > 0.9) on a) EDU and b) VNR. EDU and VNR were residualized by sex, age, age2, sex by age, sex by age2, top 20 PCs and recruitment centers and inverse rank-based normal transformed. Median of educational attainment was calculated for individuals stratified by PRS quantiles (in 2% groups) and PTV and/or damaging missense variant carrier status. The blue and red lines represent fitted linear regressions for standardized, residualized phenotypes by PRS percentile groups, stratified by rare coding variant carrier status. The gray bands represent 95% confidence intervals for each of the fitted linear regression.\n",
      "5\n",
      "\n",
      "image 1/1 /Users/etowah/projects/PMC-figure-downloader/img/10260399_Fig1.jpg: 544x640 5 labels, 75.7ms\n",
      "Speed: 2.1ms preprocess, 75.7ms inference, 0.4ms postprocess per image at shape (1, 3, 544, 640)\n",
      "2\n",
      "\n",
      "image 1/1 /Users/etowah/projects/PMC-figure-downloader/img/10260399_Fig2.jpg: 544x640 2 labels, 83.2ms\n",
      "Speed: 2.0ms preprocess, 83.2ms inference, 2.2ms postprocess per image at shape (1, 3, 544, 640)\n",
      "6\n",
      "\n",
      "image 1/1 /Users/etowah/projects/PMC-figure-downloader/img/10260399_Fig3.jpg: 480x640 6 labels, 85.7ms\n",
      "Speed: 2.1ms preprocess, 85.7ms inference, 0.4ms postprocess per image at shape (1, 3, 480, 640)\n",
      "1\n",
      "Seedling inoculations on the Lr58-carrying wheat line TA5605 (near isogenic line in the genetic background of bread wheat line WL711) and the Lr9-containing bread wheat line ThatcherLr9. Images were taken 12 days after inoculation with seven different Pt isolates. Isolates B9414, 93012, 96237, 95012, 94015, and 96007 are avirulent on Lr9. Pt isolate MNPSD is virulent on Lr9. Scale bar = 1 cm.\n",
      "1\n",
      "TA5605 M3 mutants (a) and M2 mutants (b) were inoculated with P. triticina isolate B9414. Shown are leaves 12 days after inoculation. Spt = Suceptible to Puccinia triticina. Scale bar = 1 cm.\n",
      "1\n",
      "Shown are representatives images of an F2 population derived from a cross between ThatcherLr9 and Avocet S inoculated with the avirulent P. triticina isolate B9414. M-VWA represents a PCR marker developed from the WTK6-vWA coding sequence. SCS5550 represents a PCR marker reported to be linked to Lr9. Markers were genotyped as present (+) or absent (-). R = resistant, S = susceptible. Images were taken 12 days after inoculation. Scale bar = 1 cm.\n",
      "1\n",
      "Shown are representative images of an F2 population derived from a cross between TA5605 and Avocet S inoculated with the avirulent P. triticina isolate 96007. M-VWA represents a PCR marker developed from the WTK6-vWA coding sequence. Xncw-Lr58-1 represents a PCR marker reported to be linked with Lr58. Markers were genotyped as present (+) or absent (-). R = resistant, S = susceptible. Images were taken 12 days after inoculation. Scale bar = 1 cm.\n",
      "1\n",
      "The kinase 1 domain is surrounded by a red box, kinase 2 by a brown box, the vWA domain by a purple box, and the Vwaint domain by a blue box. Roman numerals represent conserved kinase subdomains. Letters above the Lr9 amino acid sequence represent EMS-induced amino acid substitutions. Letters in gray were found in partially susceptible mutants, indicating a knock-down of protein function. ‘*’ indicates mutations causing a premature stop codon. Black triangles = ATP binding site; red triangles = key conserved residues, black asterisks = putative substrate binding site, blue squares = residue determining RD and non-RD kinases, orange triangle = glutamic acid to aspartic acid substitution in subdomain III of kinase 2.\n",
      "1\n",
      "Rectangular boxes in red and brown colors represent kinase domains belonging to the LRR_8B (cysteine rich kinases) and LRR_3 (leucine-rich-repeat receptor kinase subfamily 3) families. Bent ovals in blue, purple and orange represent vWA domains. Blue dots indicate kinase vWA gene fusion events. Numbers in brackets represent the maximum number of genes found in a given accession of each species.\n",
      "2\n",
      "\n",
      "image 1/1 /Users/etowah/projects/PMC-figure-downloader/img/10260399_Fig10.jpg: 608x640 2 labels, 118.0ms\n",
      "Speed: 2.3ms preprocess, 118.0ms inference, 0.5ms postprocess per image at shape (1, 3, 608, 640)\n",
      "2\n",
      "\n",
      "image 1/1 /Users/etowah/projects/PMC-figure-downloader/img/10260399_Fig11.jpg: 352x640 4 labels, 135.6ms\n",
      "Speed: 1.6ms preprocess, 135.6ms inference, 1.6ms postprocess per image at shape (1, 3, 352, 640)\n",
      "Subfigure count mismatch: 3 detected, 2 expected\n",
      "2\n",
      "\n",
      "image 1/1 /Users/etowah/projects/PMC-figure-downloader/img/10260398_Fig1.jpg: 384x640 2 labels, 98.5ms\n",
      "Speed: 1.7ms preprocess, 98.5ms inference, 0.7ms postprocess per image at shape (1, 3, 384, 640)\n",
      "2\n",
      "\n",
      "image 1/1 /Users/etowah/projects/PMC-figure-downloader/img/10260398_Fig2.jpg: 448x640 2 labels, 76.1ms\n",
      "Speed: 1.8ms preprocess, 76.1ms inference, 0.6ms postprocess per image at shape (1, 3, 448, 640)\n",
      "2\n",
      "\n",
      "image 1/1 /Users/etowah/projects/PMC-figure-downloader/img/10260398_Fig3.jpg: 320x640 2 labels, 72.9ms\n",
      "Speed: 1.4ms preprocess, 72.9ms inference, 0.5ms postprocess per image at shape (1, 3, 320, 640)\n",
      "2\n",
      "\n",
      "image 1/1 /Users/etowah/projects/PMC-figure-downloader/img/10260398_Fig4.jpg: 288x640 2 labels, 54.3ms\n",
      "Speed: 1.3ms preprocess, 54.3ms inference, 0.3ms postprocess per image at shape (1, 3, 288, 640)\n",
      "[['10937393', 'Fig3'], ['10937393', 'Fig12'], ['10937382', 'Fig10'], ['10937379', 'Fig9'], ['10937375', 'Fig3'], ['10937375', 'Fig10'], ['10937375', 'Fig12'], ['10937375', 'Fig14'], ['10864188', 'Fig12'], ['10864188', 'Fig13'], ['10864188', 'Fig14'], ['10864183', 'Fig8'], ['10864183', 'Fig10'], ['10864183', 'Fig11'], ['10864183', 'Fig13'], ['10864174', 'Fig3'], ['10937399', 'Fig4'], ['10937392', 'Fig4'], ['10937392', 'Fig13'], ['10937392', 'Fig16'], ['10937392', 'Fig18'], ['10937387', 'Fig2'], ['10937387', 'Fig7'], ['10937387', 'Fig9'], ['10937381', 'Fig14'], ['10786726', 'Fig3'], ['10786726', 'Fig7'], ['10786715', 'Fig11'], ['10786714', 'Fig1'], ['10786714', 'Fig3'], ['10786714', 'Fig6'], ['10703699', 'Fig13'], ['10703699', 'Fig15'], ['10703699', 'Fig16'], ['10703693', 'Fig7'], ['10703693', 'Fig9'], ['10703692', 'Fig8'], ['10703692', 'Fig13'], ['10703692', 'Fig16'], ['10703688', 'Fig6'], ['10703688', 'Fig15'], ['10703679', 'Fig2'], ['10703679', 'Fig7'], ['10703679', 'Fig9'], ['10703679', 'Fig10'], ['10703679', 'Fig13'], ['10703676', 'Fig4'], ['10786727', 'Fig6'], ['10786727', 'Fig10'], ['10786727', 'Fig11'], ['10786727', 'Fig12'], ['10786727', 'Fig13'], ['10786727', 'Fig15'], ['10786724', 'Fig1'], ['10786724', 'Fig9'], ['10786720', 'Fig7'], ['10786720', 'Fig11'], ['10786716', 'Fig4'], ['10703697', 'Fig3'], ['10703696', 'Fig10'], ['10703691', 'Fig4'], ['10703691', 'Fig7'], ['10703691', 'Fig14'], ['10703686', 'Fig2'], ['10703686', 'Fig3'], ['10703686', 'Fig5'], ['10703686', 'Fig7'], ['10703684', 'Fig3'], ['10703684', 'Fig4'], ['10703684', 'Fig5'], ['10703684', 'Fig6'], ['10703684', 'Fig7'], ['10703684', 'Fig8'], ['10703684', 'Fig9'], ['10703684', 'Fig10'], ['10703684', 'Fig11'], ['10703684', 'Fig12'], ['10632150', 'Fig4'], ['10632150', 'Fig12'], ['10562256', 'Fig10'], ['10562256', 'Fig15'], ['10562255', 'Fig5'], ['10562255', 'Fig15'], ['10484791', 'Fig2'], ['10484791', 'Fig17'], ['10484789', 'Fig15'], ['10484789', 'Fig16'], ['10412453', 'Fig4'], ['10412453', 'Fig9'], ['10412453', 'Fig10'], ['10412453', 'Fig14'], ['10335940', 'Fig3'], ['10335940', 'Fig4'], ['10335940', 'Fig5'], ['10335940', 'Fig7'], ['10335940', 'Fig8'], ['10335940', 'Fig10'], ['10335940', 'Fig11'], ['10335940', 'Fig13'], ['10335940', 'Fig14'], ['10335940', 'Fig16'], ['10335940', 'Fig17'], ['10335940', 'Fig18'], ['10335939', 'Fig2'], ['10335939', 'Fig4'], ['10335939', 'Fig6'], ['10335939', 'Fig15'], ['10335937', 'Fig1'], ['10335937', 'Fig2'], ['10335937', 'Fig4'], ['10335937', 'Fig6'], ['10335937', 'Fig7'], ['10335937', 'Fig9'], ['10335937', 'Fig10'], ['10335937', 'Fig12'], ['10335937', 'Fig14'], ['10335937', 'Fig15'], ['10335936', 'Fig6'], ['10335936', 'Fig14'], ['10335934', 'Fig4'], ['10335934', 'Fig5'], ['10335933', 'Fig1'], ['10335933', 'Fig5'], ['10335929', 'Fig4'], ['10632151', 'Fig9'], ['10632151', 'Fig10'], ['10632151', 'Fig14'], ['10632151', 'Fig15'], ['10632143', 'Fig1'], ['10632132', 'Fig3'], ['10260402', 'Fig4'], ['10260402', 'Fig5'], ['10260402', 'Fig7'], ['10260402', 'Fig10'], ['10260402', 'Fig11'], ['10260400', 'Fig1'], ['10260400', 'Fig15'], ['10260397', 'Fig1'], ['10260397', 'Fig3'], ['10260397', 'Fig6'], ['10562254', 'Fig9'], ['10562254', 'Fig10'], ['10562254', 'Fig13'], ['10484788', 'Fig3'], ['10484787', 'Fig6'], ['10484787', 'Fig14'], ['10484787', 'Fig16'], ['10484787', 'Fig17'], ['10484786', 'Fig5'], ['10484784', 'Fig5'], ['10484784', 'Fig6'], ['10484784', 'Fig7'], ['10484784', 'Fig8'], ['10412459', 'Fig13'], ['10412459', 'Fig15'], ['10412459', 'Fig17'], ['10335938', 'Fig2'], ['10335938', 'Fig4'], ['10335938', 'Fig5'], ['10335938', 'Fig9'], ['10335938', 'Fig16'], ['10260405', 'Fig7'], ['10260404', 'Fig2'], ['10260404', 'Fig9'], ['10260404', 'Fig12'], ['10260403', 'Fig2'], ['10260399', 'Fig11']]\n"
     ]
    }
   ],
   "source": [
    "figure_data = pl.read_parquet(\"figure_data.parquet\")\n",
    "out_dir = \"test\"\n",
    "failed = []\n",
    "\n",
    "for row in figure_data.rows():\n",
    "    base_dir = \"/Users/etowah/projects/PMC-figure-downloader/img\"\n",
    "    # Access the values of each column in the current row\n",
    "    figure_name = f\"{row[0]}_{row[1]}\"\n",
    "    img_path = f\"{base_dir}/{figure_name}.jpg\"\n",
    "    caption = row[4]\n",
    "    n_subfigures = count_subfigures(caption)\n",
    "    print(n_subfigures)\n",
    "    if n_subfigures == 1:\n",
    "        print(caption)\n",
    "        image = cv2.imread(img_path)\n",
    "        cv2.imwrite(f\"{out_dir}/{figure_name}.jpg\", image)\n",
    "    else:\n",
    "        bboxes = detect_subfigure_labels(img_path)\n",
    "        scores = [box[4] for box in bboxes]\n",
    "        keep = non_max_suppression(bboxes, scores, 0.5)\n",
    "        bounding_boxes = [bboxes[i] for i in keep]\n",
    "        # show_image_with_bboxes(image_file, bounding_boxes)\n",
    "        if (len(bounding_boxes) != n_subfigures):\n",
    "            print(f\"Subfigure count mismatch: {len(bounding_boxes)} detected, {n_subfigures} expected\")\n",
    "            # show_image_with_bboxes(img_path, bounding_boxes)\n",
    "            failed.append([row[0], row[1]])\n",
    "            continue\n",
    "        subfigure_info, concate_img = dummy_detect_label_letter(img_path, bounding_boxes)\n",
    "        # show_labels_on_image(image_file, subfigure_info, save=False, show=True)\n",
    "        figure_json = classify_subfigures(img_path, subfigure_info, concate_img)\n",
    "        save_subfigures(img_path, figure_json, out_dir=out_dir)\n",
    "\n",
    "    \n",
    "print(failed)\n"
   ]
  }
 ],
 "metadata": {
  "kernelspec": {
   "display_name": "venv",
   "language": "python",
   "name": "python3"
  },
  "language_info": {
   "codemirror_mode": {
    "name": "ipython",
    "version": 3
   },
   "file_extension": ".py",
   "mimetype": "text/x-python",
   "name": "python",
   "nbconvert_exporter": "python",
   "pygments_lexer": "ipython3",
   "version": "3.9.18"
  }
 },
 "nbformat": 4,
 "nbformat_minor": 2
}
